# Shared genes between Alzheimer’s disease and ischemic stroke  

Chang‐Juan Wei   |    Pan Cui   |    He Li   |    Wen‐Jing Lang   |    Gui‐You Liu 3   | Xiao‐Feng Ma  

1 Department of Neurology, Tianjin Medical  University General Hospital, Tianjin, China 2 Tianjin Neurological Institute, Key  Laboratory of Post‐neuro trauma Neuro‐ repair and Regeneration in Central Nervous  System, Ministry of Education, Tianjin, China 3 School of Life Science and  Technology, Harbin Institute of Technology,  Harbin, China  

# Correspondence  

Gui‐You Liu, School of Life Science and  Technology, Harbin Institute of Technology,  Harbin, China. Email: li ugui you 1981@163.com  and  Xiao‐Feng Ma, Department of Neurology,  Tianjin Medical University General Hospital,  Tianjin, China. Email: xfengma@tmu.edu.cn  

Funding information National Natural Science Foundation of  China, Grant/Award Number: 81300945,  81401361 and 81870954  

# Summary  

Aims:  Although converging evidence from experimental and epidemiological studies  indicates Alzheimer's disease (AD) and ischemic stroke (IS) are related, the genetic  basis underlying their links is less well characterized. Traditional SNP‐based genome‐ wide association studies (GWAS) have failed to uncover shared susceptibility variants  of AD and IS. Therefore, this study was designed to investigate whether pleiotropic  genes existed between AD and IS to account for their phenotypic association, al‐ though this was not reported in previous studies.  

Methods:  Taking advantage of large‐scale GWAS summary statistics of AD (17,008  AD cases and 37,154 controls) and IS (10,307 IS cases and 19,326 controls), we per‐ formed gene‐based analysis implemented in VEGAS2 and Fisher's meta‐analysis of  the set of overlapped genes of nominal significance in both diseases. Subsequently,  gene expression analysis in AD‐ or IS‐associated expression datasets was conducted  to explore the transcriptional alterations of pleiotropic genes identified.  

Results:  16 AD‐IS pleiotropic genes surpassed the cutoff for Bonferroni‐corrected  significance. Notably,  MS4A4A  and  TREM2 , two established AD‐susceptibility genes  showed remarkable alterations in the spleens and brains afflicted by IS, respectively.  Among the prioritized genes identified by virtue of literature‐based knowledge, most  are immune‐relevant genes ( EPHA1 ,  MS4A4A ,  UBE2L3  and  TREM2 ), implicating cru‐ cial roles of the immune system in the pathogen es is of AD and IS.  

Conclusions:  The observation that AD and IS had shared disease‐associated genes  offered mechanistic insights into their common pathogen es is, predominantly involv‐ ing the immune system. More importantly, our findings have important implications  for future research directions, which are encouraged to verify the involvement of  these candidates in AD and IS and interpret the exact molecular mechanisms of  action.  

# K E Y W O R D S  

Alzheimer’s disease, gene expression analysis, gene‐based analysis, genome‐wide association  studies, ischemic stroke, pleiotropic genes, Versatile Gene‐based Association Study‐2 version 2  

# 1  |  INTRODUCTION  

Alzheimer's disease (AD) is the world's leading cause of dementia. The  hallmarks of AD are extracellular amyloid‐  $\cdot\upbeta$   peptide (A β ) accumulation  and intracellular neuro fi brill ary tangles (NFTs), the latter of which is  formed by hyper phosphor y late d tau protein. Along with the notori‐ ous reputation as the second leading cause of mortality and disability  worldwide,  stroke is another major cause of age‐related cognitive  decline and dementia.  As the most prevalent form, ischemic stroke (IS)  accounts for   ${\sim}85\%$   of stroke incidents.  Collectively, AD and IS both  exert a large burden on global public healthcare and clinical practice.  

Growing evidence indicates that there are links between AD  and IS. Firstly, emerging epidemiologic research shows that AD  is associated with considerable increased risk of IS,  and vice  versa.  Secondly, neuro pathological studies show that cerebro‐ vascular lesions frequently coexist with AD pathology.  The two  mixed path o logie s act synergistic ally in increasing the odds of clini‐ cal dementia.  Indeed, a handful of studies have reported that brain  ischemia is a non‐neglect able factor driving the development of AD  through dys regulated expression of AD‐associated genes, such as   $\mathsf{A}\upbeta$   precursor processing genes and tau protein gene.  Lastly, tau  protein, a core hallmark of AD, can exacerbate brain injury in exper‐ imental IS through tau‐mediated iron export and ex ci to toxicity.   Taken together, we hypothesized that there might be a shared ge‐ netic basis underlying these connections between AD and IS.  

Genome‐wide association studies (GWAS) have yielded new in‐ sights into the genetics of  $\mathsf{A}\mathsf{D}^{13,14}$   and IS.  Shared genetic variants  between AD and IS or its subtypes, have been first determined by  Traylor et al.  They tested whether established genome‐wide single  nucleotide polymorphisms (SNPs) for AD or IS were significantly asso‐ ciated with the other disease. Yet no such variants have been found.  Conventional GWAS methods just focus on significant SNPs judging by  overly stringent criterion   $(P<5.00\mathsf{E}–08)$  ) when exploring the genome.  There is an emerging consensus that, however, complex diseases are  mostly driven by the joint action of a large proportion of SNPs having  modest effects well below genome‐wide significance.  Alternatively,  gene‐based analysis can obtain more validated associations by com‐ bining the effects of all SNPs within corresponding genes, thus expand  knowledge about genetic architectures of complex diseases.  

Hence, in our present study, we performed gene‐based associa‐ tion tests to identify potential candidate genes shared between AD  and IS. Next, gene expression analyses were conducted to evaluate  shared genes’ expression alterations in AD and IS brains or periph‐ eral blood versus matched controls. Furthermore, to further inter‐ pret the molecular mechanisms that underpin AD and IS, systematic  dissection of individual genes’ functionalities was conducted.  

# 2  |  MATERIALS AND METHODS  

# 2.1  |  Samples  

The GWAS statistics data for AD and IS analyses were from the  International Genomics of Alzheimer's Project   $({\mathsf{I G A P}})^{13}$   and the  METASTROKE consortium of the International Stroke Genetics  Consortium,  respectively. Both GWAS datasets are based on pop‐ ulations of European descent imputed to the 1000 Genomes Project  (1000G) reference panel. Poorly genotyped or imputed SNPs and  SNPs with a minor allele frequency (MAF) of less than 0.01 were  filtered out in both datasets.  

IGAP is a large‐scale, two‐stage GWAS (for more details, see  the research by Lambert et  ${\mathsf{a}}|^{13}.$  ). In stage 1, a meta‐analysis of four  published GWAS samples comprising 17,008 AD cases and 37,154  controls was conducted. After quality control, 7,055,881 SNPs were  available for analysis. In stage 2, 11,632 SNPs showing moderate ev‐ idence of association   $(P<1.00\mathsf{E}–03)$   in stage 1 were genotyped and  tested for replication in an independent sample totaling 8,572 cases  and 11,312 controls. Lastly, a meta‐analysis combining results from  stages 1 and 2 was performed. For the present study, we used only  summary data from stage 1.  

The METASTROKE collaboration genotyped and imputed ap‐ proximate ly 9 million SNPs from a meta‐analysis of 12 independent  GWAS comprising 10,307 IS cases and 19,326 controls. Differing  from prior IS GWAS data imputed to the HapMap panel, which  comprised up to 2.5 million SNPs with MAF more than 0.05, this  expanded set of SNPs informed by 1000G project also included low‐ frequency variants (MAF 0.01‐0.05) and totaled 8.3 million quality‐ controlled SNPs for analysis. For more detailed information, refer to  the original study.  

# 2.2  |  Statistical analyses  

# 2.2.1  |  VEGAS2 method  

Using the GWAS summary data for AD and IS, we performed a  gene‐based association test implemented in an updated version of  Versatile Gene‐based Association Study‐2 version 2 (VEGAS2).   Among various methods of gene‐based analysis, VEGAS2 is par‐ ticularly feasible for analyzing GWAS summary statistics where  individual‐level genotypic and phenotypic data are unavailable. By  uploading the individual SNPs’ IDs and their association   $P$  ‐values,  VEGAS2 sums the effects of all the SNPs within a gene and corrects  for linkage d is equilibrium (LD) referring to 1000G reference set and  thus, generates a gene‐based test statistic by doing simulations from  the multivariate normal distribution. The simulation approach is com‐ put ation ally more efficient than other methods that rely on permuta‐ tions, such as PLINK, minSNP.  The default “symmetric boundaries   $\pm0\,\mathsf{k b}$   outside gene and SNPs in LD above   $r^{2}=0.8"$   was chosen to  define gene boundaries, which meant that the effects of SNPs within  a gene, also outside of the gene with   $r^{2}>0.8$   with the ones in the  gene, were considered to calculate gene‐based  $P$  ‐values. This option  both took account into the effects of nearby regulatory SNPs and  reduced the non‐specificity caused by large boundaries like  $\pm50\,\mathsf{k b}$  .  

To reduce the possibility of a single disease driving the cross‐dis‐ ease associations and uncover truly pleiotropic genes shared by AD  and IS, we focused on shared genes that were nominally significant  in each disease   $(P_{\mathsf{A D}}<0.05$   and  $P_{1\mathsf{S}}<0.05,$  .  

Gene‐based association results from gene‐based meta‐analyses of Alzheimer's disease (AD) and ischemic stroke (IS) TA B LE 1  
![](images/f2d58a3aaf1c653e2d26d94fcb144ced9e220772eb90234d3f7e70ea4a15e145.jpg)  
‐values in both phenotypes were below 0.001 were shown in bold. P Top‐SNP, most significant SNP from each gene. Significantly associated genes whose  

# 2.2.2  |  Meta‐analysis using Fisher's method  

We used Fisher's method to combine the  $P$  ‐values calculated by the  VEGAS2 for each gene shared by AD and IS. For a given gene, the  Fisher formula for meta‐analysis is:  

$$
\!\!\!\!\!\!\!x^{2}\!=\!-2\sum_{i=1}^{k}\mathsf{I n}\left(P_{i}\right)
$$  

where  $P_{j}$   are the   $P$  ‐values of the genes in the  $\dot{I}_{\mathrm{th}}$   study and  $k$   is the  total number of studies.  $x^{2}$   follows a chi‐square distribution with  $_{2\sf K}$    degrees of freedom.  The gene‐based meta‐analysis was carried  out using the R software package.  

To avoid false positive signals, we applied the stringent Bonferroni  correction accounting for the number of genes and phenotypes  tested, that is, the significance threshold was set at 0.05/2N, where  N represented the number of shared genes with nominal signifi‐ cance in both AD and IS.  

# 2.2.3  |  Gene expression analysis method  

To explore the expression alterations of shared genes in each dis‐ ease, we surveyed the expression datasets of AD and IS from the  Gene Expression Omnibus (GEO) repository. Gene expression analy‐ sis was mainly restricted to brains and peripheral blood, as they are  the most affected by AD‐ or IS‐associated pathology.  

Because AD‐associated neuro pathology shows regional spec‐ ificity, expression profiles from discrete brain regions are more  informative for discerning AD molecular signatures than anal‐ yses based on whole‐brain expression data. Thus, we examined  expression data from separate regions from postmortem human  AD and control brains, including the dor so lateral prefrontal cor‐ tex (PFC) and hippocampus regions. The former (GSE44770) sam‐ pled 549 brains of 376 late‐onset AD patients and 173 controls.   The latter (GSE48350) comprised 19 AD cases versus 43 matched  controls.  

For IS, samples of brains and other tissues (eg spleen) from  patients were generally not available. Considering that the core  features of IS path o physiology in rodents and humans are anal‐ ogous, we included the expression data from peri‐infarct brain  areas of rats (GSE55260).  We also analyzed transcriptional data  from peripheral blood collected within 24 hours of stroke onset  in 39 IS patients and 24 controls (GSE16561).  Additionally, we  explored transcriptional profiles from mouse spleens (GSE70841)  as spleen is the major lymphoid organ involved in the inflammatory  milieu secondary to brain ischemia. All gene aliases in rat or mouse  were transformed to the official symbols corresponding to human  genes.  

If there were multiple transcripts within the same gene, the one  with the smallest  P ‐value was selected. Due to between‐study het‐ erogeneity, not all transcripts of AD‐IS pleiotropic genes appeared  in each dataset. The differential expression was determined by  Bonferroni correction accounting for the number of shared genes  present in each dataset   $\d_{1}P=0.05/\upeta$  ,  $\mathsf{n}\leq16\mathsf{I}$  ).  

# 3  |  RESULTS  

# 3.1  |  Gene‐based testing for risk genes of AD and IS  

Firstly, the VEGAS2 method was applied to individual GWAS sum‐ mary data from AD and IS. For AD, 34 genes exceeded gene‐wide  significance   $(P<2.35\mathsf{E}–6$   for 21,244 gene tests). Apart from 20  genes at   $19\mathsf{q}13.31\mathsf{-q}13.32$   harboring the well‐known  APOE  locus,  we confirmed 12 genes ( CD33 ,  ABCA7 ,  HLA‐DRB6 ,  MS4A2/MS4A6A ,  EPHA1 ,  PICALM ,  CR1 ,  CLU ,  MTCH2 / SLC39A13 , and  BIN1 ) within 10  established risk loci ( CD33 ,  ABCA7 ,  HLA‐DRB5‐DRB1  region,  MS4A  locus,  EPHA1 ,  PICALM ,  CR1 ,  CLU ,  CELF1 , and  BIN1 ).  Considering  the other 39 genes outside the  APOE  locus passing a loose signifi‐ cance threshold   $\mathsf{\dot{P}}<1.00\mathsf{E}\mathrm{-}4\mathsf{\dot{e}}$  ), 23 genes belonged to the established  risk loci, with 4 more loci validated ( CD2AP ,  PTK2B ,  SORL1 , and  SLC24A4 ).  The well‐replicated results showed the good perfor‐ mance of VEGAS2 for gene‐based analysis. For the full results, see  Supporting Information Table S1.  

For IS, no genes of gene‐wide significance were recognized   $(P<2.28\mathsf{E}\!-\!06\$   for 21,913 gene tests), while two genes surpassed a  loose significance threshold   $(P<1.00\mathsf{E}–04\$  ):  ZYX  (  $|P=2.70\mathsf{E}\!-\!05$  ) and  NCR3LG1    $(P=7.30\mathsf{E}–05)$  ).  

# 3.2  |  Gene‐based testing for shared genes between  AD and IS  

After overlapping 1915 AD genes and 1288 IS genes with nominal  significance, 130 shared genes remained for further Fisher's meta‐ analysis. We identified 17 genes reaching the significance threshold  at   $P_{\mathrm{compound}}<1.92\mathsf{E}–4\ [0.05/(2^{*}130)]$  located in adjacent positions, that is,  ZYX/EPHA1  at   $7\mathsf{q}34\!\!-\!\!7\mathsf{q}35$  ,  EFTUD1/FAM154B at 15q25.2, YDJC/UBE2L3 and SLC2A11 at 22q11, HECTD4/OAS2  at   $12\mathsf{q}24.13$  , and  ANKHD1‐EIF4EBP3/ANKHD1  at   $5\upalpha31.3$  . The latter was regarded as one gene as they were likely to  arise from the same SNPs signals as  ANKHD1‐EIF4EBP3  is a fusion  transcript of  ANKHD1  and its downstream  EIF4EBP3 . It's worth not‐ ing that  ZYX  and  EFTUD1  even reached  $P<0.001$   in both AD and IS  datasets. For the full results, refer to Supporting Information Table  S2.  

# 3.3  |  Gene expression analyses of shared genes  

To validate the relevance of the 16 AD‐IS genes ,  we evaluated  their expressions in brains or peripheral blood of AD or IS cases  versus controls. Importantly,  MS4A4A  and  TREM2 , two established  AD‐susceptibility genes, showed remarkable alterations in the  spleens and brains afflicted by IS, respectively. A comparison of  differential ly expressed genes in each dataset was presented in  Table 2.  

AD‐associated expression profiles from the PFC region  (Supporting Information Table S3A) revealed that 8 genes were  differential ly expressed, with  MS4A4A, UBE2L3,   TREM2,  and  HECTD4  being the most significant, while just 3 genes’ expression  

TA B LE 2  Expression changes of shared genes in distinct expression datasets for Alzheimer's disease (AD) and ischemic stroke (IS) 
![](images/e4238afcde882338f10b842a4b8566f5319ce06021965605196b704155024f34.jpg)  
GEO accession, GSE44772, GSE48350, GSE55260, GSE16561 and GSE70841; PFC, dor so lateral prefrontal cortex; “—”, (no data) meant that the tran‐ scripts of this gene did not exist in corresponding datasets. Bolded  P ‐values of genes achieved Bonferroni‐corrected significance, adjusted for the  number of shared genes present in each expression dataset   $(0.05/\mathsf{n}$  ,  $\mathsf{n}\leq16\mathsf{I}$  . \*Gene expression level showing more than 2 folds up regulation or down regulation compared to controls.  \*\*Genes near the significance threshold.  

levels ( HECTD4, YDJC,  and  PABPC1 ) were altered in the hippocam‐ pal regions compared with control brains (Supporting Information  Table S3B). In peri‐infarcted rat brains (Supporting Information  Table S4A), we observed expression changes of 4 genes ( TREM2,   PABPC1,   ZYX , and  YDJC ). Notably,  TREM2  showed nearly 4‐ fold up regulation versus controls (log2‐fold change  $=1.99.$  ).  Transcriptional changes of four genes ( MS4A4A ,  ZYX ,  EPHA1  and  SLC2A11) were present in the peripheral blood of IS patients  (Supporting Information Table S4B). Particularly, differential ex‐ pression of  MS4A4A  was even more evident in mouse spleens,  with over 5‐fold increase (log2‐fold change  $=2.37$  ; Supporting  Information Table S4C). It's necessary to point out that, however,  the transcriptional data from rat brains and mouse spleens were  not compelling enough due to the small sample size. More robust  transcriptional datasets are needed to further validate the candi‐ date genes.  

# 4  |  DISCUSSION  

Different from conventional SNP‐based GWAS studies, our re‐ search used VEGAS2 gene‐based association test to detect  pleiotropic genes jointly associated with AD and IS. To avoid the  problem of joint effects arising from a dominant association with  one single disease, we focused on shared genes with nominal sig‐ nificance in both AD and IS   $(P<0.05)$  . By this criterion, 16 genes  survived the stringent Bonferroni correction. Next, they were  screened for differential expression in AD and IS cases versus  controls. To search for supportive evidence for their relevance to  AD and IS, we paid extra attention to the molecular functions of  these genes.  

# 4.1  |   ZYX  and  EPHA1  gene at 7q34‐7q35  

Zyxin  ( ZYX ) encodes a zinc‐binding adaptor protein that trans locates  from focal adhesions to the nucleus to conduct signal transduction  and modulate gene expression. Recently, zyxin has been identified  as a novel target of  $\mathsf{A}\upbeta$   metabolism in AD.  Besides, Zyxin is a novel  interacting partner of  ${\mathsf{S}}|{\mathsf{R}}{\mathsf{T}}1^{27}$   that is protective against aging‐asso‐ ciated path o logie s like   $\mathsf{A}\mathsf{D}^{28}$   and IS.  Here,  ZYX  was jointly associ‐ ated with AD and IS   $(P_{\mathtt{c o m b i n e d}}=2.63\mathtt{E}{-07})$   with   $P<0.001$   in each  disease. Of note was that  ZYX  showed the most significant associa‐ tion   $(P=2.70\mathsf{E}–05)$  ) in the gene‐based analysis of IS.  

EPHA1  is an established risk locus of AD, and our gene‐ based analysis confirmed its gene‐wide association with AD   $(P=1.00\mathsf{E}\!-\!06)$  . Ephrin type‐A receptor 1 (EPHA1) belongs to the  eph receptor subfamily that is the largest family of receptor tyro‐ sine kinases, mediating axonal guidance, synaptic plasticity, and  cell‐to‐cell communication in the central nervous system (CNS).   Moreover, EPHA1 can modulate leukocyte extra vasa tion, chemo‐ taxis, and inflammatory cell migration.  Indeed, ample evidence  has implicated the involvement of  EPHA 1, as well as  MS4A4A  and  TREM2  listed below, in the immune module of AD.  As previ‐ ously described,  we observed no aberrant  EPHA1  expression in the  PFC region of AD patients, neither in the hippocampus.  

# 4.2 | MS4A4A gene at 11q12.2  

MS4A4A  is a strong AD candidate gene  38  within the membrane‐span‐ ning 4‐domains subfamily A ( MS4A ) gene cluster, an established risk  locus for AD.  It is predominantly expressed in immune cells,  including resident microglia in the CNS. Following microglial activa‐ tion,  MS4A4A  shows increased expression.  Similarly,  MS4A4A  is  up regulated in activated dendritic cells (DCs) and M1 macrophages,  while not detected in immature DCs and M2 macrophages.  Here,  MS4A4A  showed significantly altered expression in the PFC region  of AD brains   $(P=1.08\mathsf{E}{-}24)$  ), yet displayed an expression alteration  of suggestive evidence in the hippo camp al region   $(P=3.87\mathsf{E}–03)$  .  

Notably,  MS4A4A  was also differential ly expressed in peripheral  blood of IS patients   $(P=2.73\mathsf{E}–05)$  ) and in mouse spleens after IS with  over 5‐fold up regulation   $(P=1.57\mathsf{E}–05$  ; log2‐fold change  $=2.37$  ).  Following IS, increased incidence of infections occurs, mainly in the  form of pneumonia and urinary tract infections.  The underlying  mechanism is insufficient antigen‐presentation of monocytes/macro‐ phages and DCs in peripheral immune organs, resulting from downreg‐ ulation of MHC class II and co‐sti mula tory molecules and remarkable  reduction of pro inflammatory cytokines.  Herein, we speculated that  the pronounced up regulation of  MS4A4A  in the spleen following IS  might reflect a phenotypic switch of monocytes from the proinflam‐ matory M1 phenotype to the anti‐inflammatory M2 phenotype.  

# 4.3 | UBE2L3, YDJC and SLC2A11 genes at 22q11  

Ubiquitin‐conjugating enzyme E2 L3  (UBE2L3)  and  YDJC  are located  at   $22\mathsf{q}11.21$  . The genetic relationship between the   $22\mathsf{q}11.21$   re‐ gion and multiple autoimmune diseases has been extensively eluci‐ dated.  Besides, SNPs near  YDJC  are suggested to be a pleiotropic  locus between AD and Crohn disease.  Lately, UBE2L3 has been  identified as a hub gene in the gene regulatory networks of AD.  

UBE2L3  encodes an E2 ubiquitin‐conjugating enzyme. Through  its action on ubiquitin ation in  $\mathsf{N F-K B}$   signaling, UBE2L3 promotes  NF‐  $\cdot\kappa\mathsf{B}$   activation, thus mediates its link with numerous autoim‐ mune diseases.  Moreover, UBE2L3 modulates pro‐IL‐1  $.\upbeta$   pro‐ cessing and mature   $^{1|\mathsf{L}-1\upbeta}$   secretion,  the deregulation of which  pronounced ly intensifies neuronal damage in both AD and IS.   In addition, UBE2L3 directly interacts with the parkin protein, a  ubiquitin‐protein ligase that is protective against not only neuro‐ degenerative diseases,  but also cerebral ischemia‐reperfusion  injury.  Nonetheless, there is no conclusive evidence to date  demonstrating a causative link between  UBE2L3  and AD or IS.  The function of  YDJC  remains largely obscure. Solute‐carrier 2A  family member 11 ( SLC2A11 ) encodes GLUT11, a fructose‐trans‐ porting protein that might participate in fructose consumption in  the CNS.  

# 4.4 | PABPC1 gene at 8q22.3  

Poly(A) binding protein cytoplasmic 1 (PABPC1), one type of  RNA‐binding proteins, is a central component of cytoplasmic  stress granules comprising proteins and mRNAs stalled at the  translation initiation step.  Pathological stress granules play  crucial roles in neuro degenerative disorders,  also in brain is‐ chemia.  Moreover, abnormal cytoplasmic inclusions of PABPC1  have been observed in human ALS spinal cord neurons.  In  the present work,  PABPC1  was differential ly expressed in both  the PFC   $(P=1.90\mathsf{E}{-}11$  ) and hippo camp al regions   $(P=2.99\mathsf{E}–04)$  )  of AD. Evidently in IS brains, its expression was significantly en‐ hanced (log2‐fold change  $=1.19.$  ) compared to control subjects  (Supporting Information Table S4A).  

# 4.5  |   HECTD4  and  OAS2  gene at 12q24  

Mounting GWAS studies have demonstrated the pleiotropic effects  of   $12\mathfrak{q}24$   locus on type 1 diabetes,  celiac disease,  coronary ar‐ tery disease   and a number of cardiovascular risk factors, includ‐ ing hypertension,  cholesterol levels,  whist‐hip ration,  and  glycemia.  Importantly, the 12q24 region has been suggested to be  a risk locus of IS.  Recently, SNPs near  HECTD4  are shown to be as‐ sociated with memory performance.  HECT domain E3 ubiquitin‐ protein ligase 4 (HECTD4) is a E3 ubiquitin‐protein ligase. HECT‐type  E3s can function with UBE2L3 discussed above in the ubiquitin sys‐ tem.  Here,  HECTD4  was differential ly expressed in both AD's PFC   $(P=8.74\mathsf{E}–16)$   and hippo camp al   $(P=1.73\mathsf{E}\!-\!03)$  ) regions compared to  controls. Regrettably, the transcripts of  HECTD4  were not detected in  any expression datasets of IS. 2'‐5'‐oli go a deny late synthetase 2 ( OAS2 )  gene, together with neighboring  OAS1  and  OAS3  gene, encodes en‐ zymes participating in innate immunity response to viral infection.  

# 4.6  |   PINX1  gene at 8p23.1  

PIN2 (TERF1) interacting telomerase inhibitor 1 (PINX1) protein is a  potent telomerase inhibitor,  and a micro tubule‐binding protein es‐ sential for chromosome segregation in mitosis.  In addition to its bio‐ logical significance in various cancers,   PINX1  gene is associated with  sub clinical cardiovascular events like carotid intima media thickness,   blood lipids,  and involved in AD as a potential interactor of A β .  

# 4.7  |   TREM2  gene at 6p21.1  

Triggering receptor expressed on myeloid cells‐2 ( TREM2 ) is highly  expressed on microglia as an innate immune receptor involved in  phagocytosis, clearance of damaged neurons, and inhibition of the  microglial pro inflammatory response.  Mutation of rare variants in  TREM2  confers a substantial increase in AD risk,  which has been  experimentally proved.   TREM2  has been indicated to be upregu‐ lated and participate in ischemic brain damage by modulating micro‐ glial phenotypes despite conflicting findings.  Here,  TREM2  was  shared by AD and IS with individual  $P=0.0015$   and  $P=0.0095$  , respec‐ tively. Since rare variants   $({\mathsf{M A F}}<0.01$  ) were excluded from the GWAS  panels of both AD and IS ,  we assumed that the true joint association  signal of  TREM2  with AD and IS might be stronger than we observed.  TREM2  transcriptional changes were observed in the PFC region   $(P=6.80\mathsf{E}–20)$  ) of AD, and in rat peri‐infarcted brains   $(P=9.29\mathsf{E}–06)$  .  Expectedly, the latter showed nearly 4‐fold increased expression for  TREM2  (log2‐fold change  $=1.99$  ; Supporting Information Table S4A).  

# 4.8  |  Others  

Though we found that the following genes (ie  EFTUD1 / FAM154B  at  15q25.2,  RRN3P1 ,  SLC16A5 , and  ANKHD1‐EIF4EBP3 ) were common  association signals for AD and IS in terms of bioinformatics, less is  known about their biological roles due to lack of overwhelming evi‐ dence related to AD or IS.  

Based on the concise discussion about their biological signif‐ icance, partial pleotropic genes underlying AD and IS were priori‐ tized, including ZYX ,  EPHA1 ,  MS4A4A ,  UBE2L3 ,  PABPC1 ,  HECTD4 ,  PINX1 , and  TREM2 . Consistent with our previous findings from the  pathway‐oriented perspective,  we once again highlighted the  critical roles of neuro inflammation in the development and progres‐ sion of AD and IS, since half of them ( EPHA1 ,  MS4A4A ,  UBE2L3  and  TREM2 ) were engaged in immune signaling.  

Although gene‐based tests increase the power to detect dis‐ ease‐associated genes harboring multiple associated variants, they  do have limitations. Firstly, the VEGAS test is prone to underesti‐ mating effects of low‐frequency SNPs correlated with few SNPs in  LD blocks,  but may unable to distinguish the truly casual genes  from several adjacent ones co localizing in one significant locus.  Secondly, genes revealed by positional proximity to significant  variants are not necessarily the casual ones for disease pathogen‐ esis. In complex diseases, significant variants are mostly located in  intronic/intergenic areas, presumably regulating gene expression,  including acting on distant genes. Next, we leveraged the GEO  dataset to estimate the expression alteration of candidate genes  in disease‐related tissues, the reliability of which largely relied on  the raw data, for instance, the size of tissue samples in the original  studies. Further powerful approaches and sophisticated functional  interpretation analyses are warranted to prioritize causal genes.  Moreover, being pathologically and genetically heterogeneous,   IS has different e tio logical subtypes (ie large vessel disease, cardio‐ embolic stroke, and small vessel disease, undetermined and other).  Here, we just surveyed the genetic link between AD and overall IS.  

# 5  |  CONCLUSIONS  

We presented a gene‐based strategy that corroborates shared candi‐ date genes between AD and IS, with gene expression analysis ensued,  which provided a typical example of how genetic studies could add  to biological understanding of cross‐trait etiology. Literature mining  supported the potential association of partial novel candidate genes  with both AD and IS. Our findings yielded mechanistic insights into  the common pathogen es is underlying AD and IS, predominantly in‐ volving the immune system, and might suggest common intervention  targets. More importantly, our findings should encourage more stud‐ ies to verify the involvement of these candidates in AD and IS and  interpret the exact molecular mechanisms of action.  

# ACKNOWLEDGMENTS  

This work was supported by the National Natural Science Foundation  of China (81300945 to G‐Y.L.; and 81401361 and 81870954 to X‐F.  M.). We're greatly appreciative of the kind offer of GWAS summary  statistics from the IGAP and METASTROKE collaboration.  

# CONFLICT OF INTEREST  

The authors declare no conflict of interest.  

# REFERENCES  

1. Kassebaum NJ, Arora M, Barber RM, et al. Global, regional, and  national disability‐adjusted life‐years (DALYs) for 315 diseases and  injuries and healthy life expectancy (HALE), 1990‐2015: a system‐ atic analysis for the Global Burden of Disease Study 2015.  The  Lancet . 2016;388(10053):1603‐1658. 2. Wang H, Naghavi M, Allen C, et al. Global, regional, and national  life expectancy, all‐cause mortality, and cause‐specific mor‐ tality for 249 causes of death, 1980‐2015: a systematic anal‐ ysis for the Global Burden of Disease Study 2015.  The Lancet .  2016;388(10053):1459‐1544. 3. Langhorne P, Bernhardt J, Kwakkel G. Stroke rehabilitation.  Lancet .  2011;377(9778):1693‐1702. 4. Tolppanen AM, Lavikainen P, Solomon A, Kivipelto M, Soininen  H, Hart ika inen S. Incidence of stroke in people with Alzheimer  disease:  a  national  register‐based  approach.  Neurology .  2013;80(4):353‐358. 5. Chi NF, Chien LN, Ku HL, Hu CJ, Chiou HY. Alzheimer disease  and risk of stroke: a population‐based cohort study.  Neurology .  2013;80(8):705‐711. 6. Gamaldo A, Moghekar A, Kilada S, Resnick SM, Zonderman AB,  O'Brien R. Effect of a clinical stroke on the risk of dementia in a  prospective cohort.  Neurology . 2006;67(8):1363‐1369. 7. Dong S, Maniar S, Manole MD, Sun D. Cerebral hypo perfusion and  other shared brain path o logie s in ischemic stroke and Alzheimer's  disease.  Transl Stroke Res . 2018;9(3):238‐250. 8. Gorelick PB, Scuteri A, Black SE, et al. Vascular contributions to  cognitive impairment and dementia: a statement for healthcare  professionals from the american heart association/american  stroke association.  Stroke . 2011;42(9):2672‐2713. 9. Pluta R, Jabłoński M, Ułamek‐Kozioł M, et al. Sporadic Alzheimer's  disease begins as episodes of brain ischemia and ischemi‐ cally dys regulated Alzheimer's disease genes.  Mol Neurobiol .  2013;48(3):500‐515.  10. Salminen A, Kauppinen A, Ka arni rant a K. Hypoxia/ischemia ac‐ tivate processing of Amyloid Precursor Protein: impact of vas‐ cular dysfunction in the pathogen es is of Alzheimer's disease.  J  Neurochem . 2017;140(4):536‐549.  

11. Bi M, Gladbach A, van Eersel J, et al. Tau exacerbates excito‐ toxic brain damage in an animal model of stroke.  Nat Commun .  2017;8(1):473.

  12. Tuo Q‐z, Lei P, Jackman Ka, et al. Tau‐mediated iron export pre‐ vents ferro p to tic damage after ischemic stroke.  Mol Psychiatry .  2017;22(11):1520‐1530.

  13. Lambert J‐C, Ibrahim‐Verbaas CA, Harold D, et al. Meta‐analy‐ sis of 74,046 individuals identifies 11 new susceptibility loci for  Alzheimer's disease.  Nat Genet . 2013;45(12):1452‐1458.

  14. Jansen IE, Savage JE, Watanabe K, et al. Genome‐wide meta‐ analysis identifies new loci and functional pathways influencing  Alzheimer's disease risk.  Nat Genet . 2019;51(3):404‐413.

  15. Malik R, Traylor M, Pulit SL, et al. Low‐frequency and common  genetic variation in ischemic stroke: The METASTROKE collabo‐ ration.  Neurology . 2016;86(13):1217‐1226.

  16. Malik R, Chauhan G, Traylor M, et al. Multi ancestry genome‐wide  association study of 520,000 subjects identifies 32 loci associated  with stroke and stroke subtypes.  Nat Genet . 2018;50(4):524‐537.

  17. Traylor M, Adib‐Samii P, Harold D, et al. Shared genetic contri‐ bution to Ischaemic Stroke and Alzheimer's Disease.  Ann Neurol .  2016;79(5):739‐747.

 18. Boyle EA, Li YI, Pritchard JK. An expanded view of complex traits:  from polygenic to omnigenic.  Cell . 2017;169(7): 1177‐1186.

  19. Mishra A, Macgregor S. VEGAS2: Software for more flexible gene‐ based testing.  Twin Res Hum Genet . 2015;18(1):86‐91.

  20. Liu JZ, Mcrae AF, Nyholt DR, et al. A versatile gene‐based  test for genome‐wide association studies.  Am J Hum Genet .  2010;87(1):139‐145.

  21. Begum F, Ghosh D, Tseng GC, Feingold E. Comprehensive litera‐ ture review and statistical considerations for GWAS meta‐analy‐ sis.  Nucleic Acids Res . 2012;40(9):3777‐3784.

  22. Zhang B, Gaiteri C, Bodea L‐G, et al. Integrated systems approach  identifies genetic nodes and networks in late‐onszheimer's dis‐ ease. Cell. 2013;153(3):707‐720.

 23. Berchtold NC, Coleman PD, Cribbs DH, Rogers J, Gillen DL,  Cotman CW. Synaptic genes are extensively down regulated across  multiple brain regions in normal human aging and Alzheimer's dis‐ ease.  Neurobiol Aging . 2013;34(6):1653‐1661.

  24. Buga AM, Marga rites cu C, Scholz CJ, Radu E, Zelenak C, Popa‐ Wagner A. Transcript om ics of post‐stroke ang io genesis in the  aged brain.  Front Aging Neurosci . 2014;6:44.

  25. Barr Tl, Conley Y, Ding J, et al. Genomic biomarkers and cellular  pathways of ischemic stroke by RNA gene expression profiling.  Neurology . 2010;75(11):1009‐1014.

  26. Lanni C, Necchi D, Pinto A, et al. Zyxin is a novel target for beta‐ amyloid peptide: characterization of its role in Alzheimer's patho‐ genesis.  J Neurochem . 2013;125(5):790‐799.

  27. Fujita Y, Yamaguchi A, Hata K, Endo M, Yamaguchi N, Yamashita  T. Zyxin is a novel interacting partner for SIRT1.  BMC Cell Biol .  2009;10:6.

  28. Herskovits AZ, Guarente L. SIRT1 in neuro development and brain  senescence.  Neuron . 2014;81(3):471‐483.

  29. Yang Y, Duan W, Li Y, et al. New role of silent information regulator  1 in cerebral ischemia.  Neurobiol Aging . 2013;34(12):2879‐2888.

  30. Martinez A, Otal R, Sieber BA, Ibanez C, Soriano E. Disruption of  ephrin‐A/EphA binding alters syn ap to genesis and neural connec‐ tivity in the hippocampus.  Neuroscience . 2005;135(2):451‐461.

  31. Lai KO, Ip NY. Synapse development and plasticity: roles of ephrin/ Eph receptor signaling.  Curr Opin Neurobiol . 2009;19(3):275‐283.

  32. Ivanov AI, Romanovsky AA. Putative dual role of ephrin‐Eph recep‐ tor interactions in inflammation.  IUBMB Life . 2006;58(7):389‐394.

  33. Aasheim HC, Delabie J, Finne EF. Ephrin‐A1 binding to   $\mathsf{C D4+}$   T  lymphocytes stimulates migration and induces tyrosine phosphor‐ ylation of PYK2.  Blood . 2005;105(7):2869‐2876.  

34. Ahmad S, Bannister C, van der Lee SJ, et al. Disentangling the bio‐ logical pathways involved in early features of Alzheimer's disease  in the Rotterdam Study.  Alzheimers Dement . 2018;14(7):848‐857.

  35. Heneka MT, Golenbock DT, Latz E. Innate immunity in Alzheimer's  disease.  Nat Immunol . 2015;16(3):229‐236.

  36. Karch CM, Cruchaga C, Goate A. Alzheimer’s Disease Genetics:  From the bench to the clinic.  Neuron . 2014;83(1):11‐26.

  37. Karch CM, Jeng AT, Nowotny P, Cady J, Cruchaga C, Goate AM.  Expression of Novel Alzheimer’s disease risk genes in control and  Alzheimer’s disease brains. PLoS ONE. 2012;7(11):e50976.

 38. Allen M, Zou F, Chai Hs, et al. Novel late‐onszheimer disease  loci variants associate with brain gene expression.  Neurology .  2012;79(3):221‐228.

  39. Holl ing worth P, Harold D, Sims R, et al. Common variants at  ABCA7, MS4A6A/MS4A4E, EPHA1, CD33 and CD2AP are asso‐ ciated with Alzheimer's disease.  Nat Genet . 2011;43(5):429‐435.

  40. Naj AC, Jun G, Beecham GW, et al. Common variants at MS4A4/ MS4A6E, CD2AP, CD33 and EPHA1 are associated with late‐ onszheimer's disease.  Nat Genet . 2011;43(5):436‐441.

  41. Kofler J, Bissel S, Wiley C, Stauffer M, Murdoch G. Differential  microglial expression of new Alzheimer's disease associated genes  MS4A4A and MS4A6A.  Alzheimers Dement . 2012;8(4):P253‐P253.

  42. Sanyal R, Polyak MJ, Zuccolo J, et al. MS4A4A: a novel cell surface  marker for M2 macrophages and plasma cells.  Immunol Cell Biol .  2017;95(7):611‐619.

 43. Rohweder G, Ellekjaer H, Salvesen O, Naalsund E, Indredavik B.  Functional outcome after common poststroke complications oc‐ curring in the first 90 days.  Stroke . 2015;46(1):65‐70.

  44. Meisel C, Schwab JM, Prass K, Meisel A, Dirnagl U. Central ner‐ vous system injury‐induced immune deficiency syndrome.  Nat Rev  Neurosci . 2005;6(10):775‐786.

  45. Chamorro A, Meisel A, Planas AM, Urra X, van de Beek D,  Veltkamp R. The immunology of acute stroke.  Nat Rev Neurol .  2012;8(7):401‐410.

  46. Han J‐W, Zheng H‐F, Cui Y, et al. Genome‐wide association study in  a Chinese Han population identifies nine new susceptibility loci for  systemic lupus erythematosus.  Nat Genet . 2009;41(11):1234‐1237.

  47. Zhernakova A, Stahl EA, Trynka G, et al. Meta‐analysis of ge‐ nome‐wide association studies in celiac disease and rheumatoid  arthritis identifies fourteen non‐HLA shared loci.  PLoS Genet .  2011;7(2):e1002004.

  48. Ellinghaus D, Ellinghaus E, Nair R, et al. Combined analysis of  genome‐wide association studies for crohn disease and psoria‐ sis identifies seven shared susceptibility loci.  Am J Hum Genet .  2012;90(4):636‐647.

  49. Yokoyama JS, Wang Y, Schork AJ, et al. Association between ge‐ netic traits for immune‐mediated diseases and alzheimer disease.  JAMA Neurol . 2016;73(6):691‐697.

  50. Kawalia SB, Raschka T, Naz M, de Matos SR, Senger P, Hofmann‐ Apitius M. Analytical strategy to prioritize Alzheimer’s disease  candidate genes in gene regulatory networks using public expres‐ sion data.  J Alzheimers Dis . 2017;59(4):1237‐1254.

  51. Fu B, Li S, Wang L, Berman MA, Dorf ME. The ubiquitin conjugat‐ ing enzyme UBE2L3 regulates TNF α ‐induced linear ubiquitin ation.  Cell Res . 2014;24(3):376‐379.

  52. Lewis MJ, Vyse S, Shields AM, et al. UBE2L3 polymorphism ampli‐ fies NF‐kappaB activation and promotes plasma cell development,  linking linear ubiquitin ation to multiple autoimmune diseases.  Am J  Hum Genet . 2015;96(2):221‐234.

  53. Lewis M, Vyse S, Shields A, et al. Effect of UBE2L3 genotype on  regulation of the linear ubiquitin chain assembly complex in sys‐ temic lupus erythematosus.  Lancet . 2015;385(Suppl 1):S9.

  54. Eldridge M, Sanchez‐Garrido J, Hoben GF, Goddard PJ, Shenoy  AR. The Atypical Ubiquitin E2 Conjugase UBE2L3 Is an Indirect  Caspase‐1 Target and Controls IL‐1 β  Secretion by In fl amma some s.  Cell Rep . 2017;18(5):1285‐1297.  

55. Gelderblom M, Sobey CG, Klein sch nitz C, Magnus T. Danger sig‐ nals in stroke.  Ageing Res Rev . 2015;24(Pt A):77‐82.

  56. Heneka MT, Kummer MP, Stutz A, et al. NLRP3 is activated in  Alzheimer's disease and contributes to pathology in APP/PS1  mice.  Nature . 2013;493(7434):674‐678.

  57. Shimura H, Hattori N, Kubo S‐I, et al. Familial Parkinson disease  gene product, parkin, is a ubiquitin‐protein ligase.  Nat Genet .  2000;25(3):302‐305.

  58. Burns MP, Zhang L, Rebeck GW, Querfurth HW, Moussa CE.  Parkin promotes intracellular Abeta1‐42 clearance.  Hum Mol  Genet . 2009;18(17):3206‐3216.

  59. Zheng Q, Huang T, Zhang L, et al. Dys regulation of ubiquitin‐ proteasome system in neuro degenerative diseases.  Front Aging  Neurosci . 2016;8:303.

  60. Tang J, Hu Z, Tan J, Yang S, Zeng L. Parkin protects against ox‐ ygen‐glucose deprivation/reperfusion insult by promoting Drp1  degradation.  Oxid Med Cell Longev . 2016;2016:8474303.

  61. Izumi Y, Zorumski CF. Glial‐neuronal interactions underly‐ ing fructose utilization in rat hippo camp al slices.  Neuroscience .  2009;161(3):847‐854.

  62. Bag J, Bhatt a char j ee RB. Multiple levels of post‐transcriptional  control of expression of the poly (A)‐binding protein.  RNA Biol .  2010;7(1):5‐12.

  63. Wolozin B. Physiological protein aggregation run amuck: stress  granules and the genesis of neuro degenerative disease.  Discov  Med . 2014;17(91):47‐52.

  64. Dewey CM, Cenik B, Sephton CF, Johnson BA, Herz J, Yu G.  TDP‐43 aggregation in neuro degeneration: are stress granules the  key?  Brain Res . 2012;1462:16‐25.

  65. Ayuso MI, Martinez‐Alonso E, Regidor I, Alcazar A. Stress granule  induction after brain ischemia is independent of eukaryotic trans‐ lation initiation factor (eIF) 2alpha phosphor yl ation and is cor‐ related with a decrease in eIF4B and eIF4E proteins.  J Biol Chem .  2016;291(53):27252‐27264.

  66. Kim H‐J, Raphael AR, LaDow ES, et al. Therapeutic modulation of  eIF2 α ‐phosphor yl ation rescues TDP‐43 toxicity in amyotrophic  lateral sclerosis disease models.  Nat Genet . 2014;46(2):152‐160.

  67. Fittschen M, Lastres‐Becker I, Halbach Mv, et al. Genetic ablation  of ataxin‐2 increases several global translation factors in their tran‐ script abundance but decreases translation rate.  Neuro genetics .  2015;16(3):181‐192.

  68. Burton PR, Clayton DG, Cardon LR, et al. Genome‐wide associa‐ tion study of 14,000 cases of seven common diseases and 3,000  shared controls.  Nature . 2007;447(7145):661‐678.

  69. Todd JA, Walker NM, Cooper JD, et al. Robust associations of four  new chromosome regions from genome‐wide analyses of type 1  diabetes.  Nat Genet . 2007;39(7):857‐864.

  70. Hunt KA, Zhernakova A, Turner G, et al. Newly identified genetic  risk variants for celiac disease related to the immune response.  Nat  Genet . 2008;40(4):395‐402.

  71. Lu X, Wang L, Chen S, et al. Genome‐wide association study in Han  Chinese identifies four new susceptibility loci for coronary artery  disease.  Nat Genet . 2012;44(8):890‐894.

  72. Kato N, Takeuchi F, Tabara Y, et al. Meta‐analysis of genome‐ wide association studies identifies common variants associ‐ ated with blood pressure variation in east Asians.  Nat Genet .  2011;43(6):531‐538.

  73. Soranzo N, Spector TD, Mangino M, et al. A genome‐wide  meta‐analysis identifies 22 loci associated with eight hema‐ tological parameters in the HaemGen consortium.  Nat Genet .  2009;41(11):1182‐1190.  

74. Newton‐Cheh C, Johnson T, Gateva V, et al. Eight blood pressure  loci identified by genome‐wide association study of 34,433 people  of European ancestry.  Nat Genet . 2009;41(6):666‐676.

  75. Levy D, Ehret GB, Rice K, et al. Genome‐wide association study of  blood pressure and hypertension.  Nat Genet . 2009;41(6):677‐687.

  76. Kim YJ, Go MJ, Hu C, et al. Large‐scale genome‐wide association  studies in East Asians identify new genetic loci influencing meta‐ bolic traits.  Nat Genet . 2011;43(10):990‐995.

  77. Cho YS, Go MJ, Kim YJ, et al. A large‐scale genome‐wide associa‐ tion study of Asian populations uncovers genetic factors influenc‐ ing eight quantitative traits.  Nat Genet . 2009;41(5):527‐534.

  78. Go MJ, Hwang J‐Y, Kim YJ, et al. New susceptibility loci in MYL2,  C12orf51 and OAS1 associated with 1‐h plasma glucose as predis‐ posing risk factors for type 2 diabetes in the Korean population.  J  Hum Genet . 2013;58(6):362‐365.

  79. Kilarski LL, Achterberg S, Devan WJ, et al. Meta‐analysis in more  than 17,900 cases of ischemic stroke reveals a novel association at  12q24.12.  Neurology . 2014;83(8):678‐685.

  80. Ninomiya‐Baba M, Matsuo J, Sasayama D, et al. Association of  body mass index‐related single nucleotide polymorphisms with  psychiatric disease and memory performance in a Japanese popu‐ lation.  Acta Neuro psych i a tr . 2017;29(5):299‐308.

  81. Wenzel DM, Lissounov A, Brzovic PS, Klevit RE. UbcH7 reactiv‐ ity profile reveals Parkin and HHARI to be RING/HECT hybrids.  Nature . 2011;474(7349):105‐108.

  82. Field LL, Bonnevie‐Nielsen V, Pociot F, Lu S, Nielsen TB, Beck‐ Nielsen H. OAS1 splice site polymorphism controlling antiviral en‐ zyme activity influences susceptibility to type 1 diabetes.  Diabetes .  2005;54(5):1588‐1591.

  83. Zhou XZ, Lu KP. The Pin2/TRF1‐interacting protein PinX1 is a po‐ tent telomerase inhibitor.  Cell . 2001;107(3):347‐359.

  84. Yuan K, Li Na, Jiang K, et al. PinX1 is a novel micro tubule‐bind‐ ing protein essential for accurate chromosome segregation.  J Biol  Chem . 2009;284(34):23072‐23082.

  85. Zhang B, Bai Yx, Ma Hh, et al. Silencing PinX1 compromises telo‐ mere length maintenance as well as tumor i geni city in telomerase‐ positive human cancer cells. Cancer Res. 2009;69(1):75‐83.

 86. Bis JC, Kavousi M, France sch in i N, et al. Meta‐analysis of genome‐ wide association studies from the CHARGE consortium identifies  common variants associated with carotid intima media thickness  and plaque.  Nat Genet . 2011;43(10):940‐947.

  87. Willer CJ, Schmidt EM, Sengupta S, et al. Discovery and re‐ finement of loci associated with lipid levels.  Nat Genet .  2013;45(11):1274‐1283.

  88. Teslovich TM, Musunuru K, Smith AV, et al. Biological, clin‐ ical and population relevance of 95 loci for blood lipids.  Nature .  2010;466(7307):707‐713.

  89. Virok DP, Simon Dóra, Bozsó Z, et al. Protein array based interac‐ tome analysis of Amyloid‐  $\cdot\upbeta$   indicates an inhibition of protein trans‐ lation.  J Proteome Res . 2011;10(4):1538‐1547.

  90. Colonna M. TREMs in the immune system and beyond.  Nat Rev  Immunol . 2003;3(6):445‐453.

  91. Guerreiro R, Wojtas A, Bras J, et al. TREM2 variants in Alzheimer's  disease.  N Engl J Med . 2013;368(2):117‐127.

  92. Jonsson T, Stefansson H, Steinberg S, et al. Variant of TREM2  associated with the risk of Alzheimer's disease.  N Engl J Med .  2013;368(2):107‐116.

  93. Sims R, van der Lee SJ, Naj AC, et al. Rare coding variants in PLCG2,  ABI3, and TREM2 implicate microglial‐mediated innate immunity  in Alzheimer's disease.  Nat Genet . 2017;49(9):1373‐1384.

  94. Wang Y, Cella M, Mallinson K, et al. TREM2 lipid sensing sustains  the microglial response in an Alzheimer's disease model.  Cell .  2015;160(6):1061‐1071.  

95. Ulland TK, Song WM, Huang SC, et al. TREM2 maintains microglial  metabolic fitness in Alzheimer's disease.  Cell . 2017;170(4):649‐663. e613.

  96. Sieber MW, Jaenisch N, Brehm M, et al. Attenuated inflam‐ matory response in triggering receptor expressed on myeloid  cells 2 (TREM2) knock‐out mice following stroke.  PLoS ONE .  2013;8(1):e52982.

  97. Kawabori M, Kacimi R, Kauppinen T, et al. Triggering receptor  expressed on myeloid cells 2 (TREM2) deficiency attenu‐ ates phagocytic activities of microglia and exacerbates isch‐ emic damage in experimental stroke.  J Neurosci . 2015;35(8):  3384‐3396.

  98. Zhai Q, Li F, Chen X, et al. Triggering receptor expressed on my‐ eloid cells 2, a novel regulator of immunocyte phenotypes,  Confers  Neuro protection by Relieving Neuro inflammation. Anesthesiology .  2017;127(1):98‐110.

 99. Kawabori M, Hokari M, Zheng Z, et al. Triggering receptor ex‐ pressed on myeloid cells‐2 correlates to hypo the r mic neuro‐ protection in ischemic stroke.  Ther Hypothermia Temp Manag .  2013;3(4):189‐198.

  100. Wu R, Li X, Xu P, et al. TREM2 protects against cerebral ischemia/ reperfusion injury.  Mol Brain . 2017;10(1):20.  

101. Cui P, Ma X, Li H, Lang W, Hao J. Shared biological pathways be‐ tween Alzheimer’s disease and ischemic stroke.  Front Neurosci .  2018;12(605).

  102. Huang H, Chanda P, Alonso A, Bader JS, Arking DE. Gene‐based  tests of association.  PLoS Genet . 2011;7(7):e1002177.

  103. Markus HS, Bevan S. Mechanisms and treatment of ischaemic  stroke–insights from genetic associations.  Nat Rev Neurol .  2014;10(12):723‐730.  

# SUPPORTING INFORMATION  

Additional supporting information may be found online in the  Supporting Information section at the end of the article.  

How to cite this article:  Wei C‐J, Cui P, Li H, Lang W‐J, Liu G‐Y,  Ma X‐F. Shared genes between Alzheimer’s disease and  ischemic stroke.  CNS Neurosci Ther . 2019;25:855–864.  https:// doi.org/10.1111/cns.13117  