# Exploring the interaction between SNP genotype and post menopausal hormone therapy effects on stroke risk  

1 2 3 2 4 5 Ying Huang , Dennis G Ballinger , Renee Stokowski , Erica Beilharz , Jennifer G Robinson , Simin Liu , 6 7 8   Randal D Robinson , Victor W Henderson , Jacques E Rossouw and Ross L Prentice  

# Abstract  

Background:  Genome-wide association studies have identified several genomic regions that are associated with stroke risk, but these provide an explanation for only a small fraction of familial stroke aggregation. Genotype by environment interactions may contribute further to such an explanation. The Women ’ s Health Initiative (WHI) clinical trial found increased stroke risk with post menopausal hormone therapy (HT) and provides an efficient setting for evaluating genotype-HT interaction on stroke risk.  

Methods:  We examined HT by genotype interactions for 392 SNPs selected from candidate gene studies, and 2,371 SNPs associated with changes in blood protein concentrations after hormone therapy, in analyses that included 2,045 post menopausal women who developed stroke during WHI clinical trial and observational study follow-up and one-to-one matched controls. A two-stage procedure was implemented where SNPs passing the first stage screening based on marginal association with stroke risk were tested in the second stage for interaction with HT using case-only analysis.  

Results:  The two-stage procedure identified two SNPs, rs2154299 and rs12194855, in the coagulation factor XIII subunit A ( F13A1 ) region and two SNPs, rs630431 and rs560892, in the proprotein convertase subtilisin kexin 9  $(P C S K9)$   region, with an estimated false discovery rate   ${<}0.05$   based on interaction tests. Further analyses showed significant stroke risk interaction between these  F13A1  SNPs and estrogen plus progestin   $(\mathsf E+\mathsf P)$   treatment for ischemic stroke and for ischemic and hemorrhagic stroke combined, and suggested interactions between  $P C S K9$  SNPs with either  $\mathsf{E}\!+\!\mathsf{P}$   or estrogen-alone treatment.  

Conclusions:  Genotype by environment interaction information may help to define genomic regions relevant to stroke risk. Two-stage analysis among post menopausal women generates novel hypotheses concerning the  F13A1 and    $P C S K9$   genomic regions and the effects of hormonal exposures on post menopausal stroke risk for subsequent independent validation.  

# Background  

In the past few years, genome-wide association studies have identified many common genetic variants associated with complex diseases. Most of the associations identified so far, however, explain only a small fraction of familial disease aggregation, which suggests that studies exploring additional human genome-related associations, such as those related to rare genetic variants, gene-gene interactions, and gene-environment interactions, could play an important role in accounting for the  ‘ missing heritability ’ [1]. In the case of stroke, several genome-wide association studies have been performed to identify SNP associations, yet no single locus identified has been successfully replicated in a second study [2-9]. The complex nature of stroke risk, which may involve interactions between biological pathways, and the largely unexplained heritability suggest that gene-gene and gene-environment interactions  

could have a significant impact on the search for strokerelated SNPs [10,11].  

A major challenge in assessing gene-environment interaction is the difficulty in characterizing environmental exposures precisely in an observational study setting, which has been the setting for most of the gene-environment interaction studies to date. Randomized controlled intervention trials, on the other hand, provide natural settings for incorporating gene-environment interaction assessment into the search for disease-susceptible SNPs since the treatment/intervention assignment is known exactly. An additional advantage of random iz ation is the resulting independence between intervention assignment and genotype, which justifies highly efficient case-only testing of interaction between SNPs and intervention in relation to disease risk [12-17].  

The Women ’ s Health Initiative (WHI) studies include two major components: a randomized controlled trial (CT) and an observational study (OS). The WHI trial included four randomized and controlled comparisons among post menopausal women, in a partial factorial design [18,19]. Specifically, it comprised a postmen o pausal hormone therapy component that involved two nonoverlapping trials: an estrogen versus placebo (E-alone) trial among women who were post-hysterectomy, and an estrogen plus progestin versus placebo   $\mathrm{(E+P)}$   trial among women with a uterus; a low-fat dietary modification (DM) versus usual diet component, and a calcium and vitamin D (CaD) supplement ation versus placebo component.  

The  $_{\mathrm{E+P}}$   trial was stopped early in 2002, triggered by an elevation of breast cancer risk and an overall unfavorable health benefit versus risk profile [20,21]. An important health risk was an approximate  $40\%$   elevation of stroke risk [22]. The E-alone trial was also stopped early, in 2004, primarily due to an elevation of stroke risk of similar magnitude to that seen for   $_{\mathrm{E+P}}$   [23,24]. The elevated stroke risks associated with the use of estrogen or estrogen plus progestin were not observed in the OS, where there was evidence of residual confounding [25]. Neither the DM nor the CaD trial yielded evidence of an intervention effect on stroke risk [26,27].  

Here we study 2,763 candidate SNPs in relation to stroke incidence and hormone therapy (HT) intervention (E-alone and  $\mathrm{E+P}$  ) effects. These SNPs were evaluated in stroke cases and one-to-one matched controls in WHI cohorts as a  ‘ core ’  WHI project that sought to identify SNPs that interact with hormone therapy effects on stroke risk.  

Our analyses apply a novel two-stage procedure by first screening out unpromising SNPs based on marginal association tests with stroke using all available case-control samples from the WHI CT and OS, and then in the second stage investigating SNP-HT interaction only for SNPs meeting first-stage filtering criteria among stroke cases in the HT trials [28-30]. The independence between the test statistics for marginal association of SNPs and the caseonly test statistics for interaction has been demonstrated in [29]. As a result, with a pre-determined threshold for marginal significance in the first-stage screening (chosen as 0.05), we only need to correct for the number of SNPs passing the first-stage filtering in evaluating gene-intervention interaction. This analytic approach preserves statisti- cal power for interaction identification for the most promising SNPs.  

# Materials and methods  

# Study design and population  

Enrollees in WHI trials were post menopausal women aged 50 to 79 years who met component-specific eligibility criteria [31]. Women were randomized to a HT component, or a DM component, or both. At the oneyear anniversary from enrollment participating women could be further randomized into the CaD supplement ation component. A total of 68,132 women were enrolled into the trials between 1993 and 1998, among which there were 10,739 in E-alone, 16,608 in   $_{\mathrm{E+P}}$  , 48,835 in DM, and 36,282 in CaD components. The WHI OS enrolled 93,676 post menopausal women and participants were followed for 6 to 10 years during the intervention phase of the WHI program. Details about distributions of demographic variables and stroke risk factors in the study cohort were published previously [31].  

# Case and control selection  

Overall, 2,096 stroke cases were considered for inclusion, including all stroke cases in the WHI clinical trials that developed between random iz ation and August 2007 and a subset of stroke cases in the OS that occurred following the time of case selection in a WHI pooled data genomewide association study but prior to August 2007. Also considered were corresponding controls one-to-one matched to cases on baseline age, self-reported ethnicity, cohort (CT versus OS) and participation in each component if in the clinical trial, years since random iz ation/ enrollment, baseline hysterectomy status, and prevalent stroke at enrollment. Among these, a total of 2,045  $(97.6\%)$   cases had adequate quantity and quality of DNA for this project. Of these, 149   $(7.3\%)$   cases had a prior history of stroke at baseline. Each case was matched to a qualifying control also having a suitable DNA specimen available, leading to an equal number of cases and controls in each CT or OS component. The number of included cases (or controls) was 351 in the E-alone trial, 438 in the  $\mathrm{E}\!+\!\mathrm{P}$   trial, 1,110 in the DM trial, 838 in the CaD trial (cases arising after CaD random iz ation only), and 373 in the OS (Table 1). The comparatively small number of cases from the OS occurs because of the  

Table 1 Distribution of stroke cases and controls by type and CT/OS component 
![](images/99a4ab05e1dd7b8cd3dc908774efc6a38089c5fefe64350de36d96fe640e3986.jpg)  
CaD, calcium and vitamin D versus placebo supplement ation; CT, randomized controlled trial; DM, low-fat dietary modification versus usual diet; E-alone, estrogen versus placebo;  $\mathsf{E}\!+\!\mathsf{P}$  , estrogen plus progestin versus placebo; OS, observational study.  

exclusion of stroke cases included in the earlier WHI pooled DNA study mentioned above. Table 1 also presents the number of cases stratified further by stroke subcategory.  

Informed consent was obtained from each study participant and the research conformed to the Helsinki Declaration and to all pertinent local legislation.  

# Laboratory methods and SNP selection  

SNPs were genotyped and quality control criteria were applied at Perlegen Sciences (Mountain View, CA, USA). Genotyping and data cleaning methods have been described [32], with an average call rate of   $99.8\%$   and an average concordance rate of   $99.8\%$   for 157 blind duplicate samples.  

Principal component analysis was used to characterize population structure and to identify genotyping artifacts. The top 20 principal components did not associate with common sources of experimental variability such as date of sample processing or hybridization performance for either chip design.  

The 2,763 candidate SNPs evaluated in this paper cover 400 chromosomal regions with pairwise correlation   $\mathrm{r^{2}}$    between regions   ${<}0.2$  . Among these, 392 were selected from candidate genes previously reported to be related to stroke risk [4,33-35], and 2,371 were selected based on WHI studies of potential changes in blood protein concentrations following the initiation of hormone therapy [36,37].  

# Statistical methods  

A two-stage procedure as proposed in [29,30] was implemented for identification of SNP-HT interaction. First stage filtering  

In the first stage, the marginal effect for each SNP individually was estimated from a standard logistic regression of case (1) versus control (0) status on number of minor SNP alleles and potential confounding factors based on all case-control samples in CT and OS. The logistic regression model included smoking status, physical functioning score, history of treated diabetes, prevalent hypertension, current aspirin use and current statin use. Also included were the variables used for matching controls to cases in control selection and clinical trial random iz ation assignments. In addition, eigenvectors from the first ten principal components from correlation analysis of the genotype data were included to adjust for population stratification [38]. All SNPs with two-sided  $P$  -value   $<\,0.05$   were entered into the second stage for evaluation of interaction with HT.  

# Second stage  

At the second stage, a two-component test statistic was used for each SNP to test for interaction with HT on stroke risk. The independence between HT treatment assignment and genotype in E-alone and   $_{\mathrm{E+P}}$   ensured by random iz ation allows the use of the case-only analysis for interaction testing, a more efficient method compared to standard case-control analysis. The two test statistic components are case-only tests for dependence of intervention odds ratios on SNP genotype for each of E-alone and   $\mathrm{E}\!+\!\mathrm{P}$  , respectively. These statistics arise as likelihood ratio test statistics in logistic regression of active (1) versus placebo random iz ation assignment on the number of minor SNP alleles with logistic regression location parameter offset by log  ${\bf q}/(1\textrm{-}{\bf q}),$  , where q is the fraction of women assigned to active intervention for the pertinent clinical trial component. That is, for cases within the E-alone or   $\mathrm{E}\!+\!\mathrm{P}$   trial separately, we fit a model with:  

$$
\mathrm{Pr}(Z=1|\mathrm{G}\ )=\log(\mathrm{q}/(1-\mathrm{q}))+\upbeta_{0}+\upbeta_{2}\mathrm{G}
$$
 logit Pr(  

where   $Z\,=\,0$  , 1 indicates assignment to placebo or treatment arm, respectively, and   $G$   is the number of minor alleles for the SNP considered.  

The interaction tests for E-alone and   $_{\mathrm{E+P}}$   are independent of each other since they are based on non-overlapping sets of women. The two test statistics were added to yield a chi-square test with two degrees of freedom to test SNP interaction with HT on stroke risk for each of the SNPs. Control for multiple testing was carried out by requiring the estimated false discovery rate (FDR) [39] to be  $<0.05$  .  

SNPs of interest in the interaction test were subsequently examined for evidence of interaction effects with E-alone and   $\mathrm{E}\!+\!\mathrm{P}$   separately, both for all stroke cases and by major stroke subcategory (ischemic, hemorrhagic). Again, case-only analyses were employed, and for descriptive purposes, intervention odds ratios were estimated separately at zero, one, and two minor SNP alleles. Likelihood ratio tests with both one and two degrees of freedom (reflecting whether SNP genotype was modeled as a linear term in the number of minor SNP alleles, or a separate indicator variables for one and for two minor alleles) were examined to assess SNP by intervention interaction in these analyses. All significance levels (  $\boldsymbol{\mathrm{i}}P$  -values) were two-sided. Note that the odds ratio estimates in these case-only analyses are (asymptotically) independent of those from the stage 1 analyses [29,30], so that only the number of SNPs included in stage 2 need to be considered in examining these odds ratios estimates for multiple testing-related biases.  

The potential of SNP by HT interactions to contribute to the ability to discriminate between stroke cases and controls was evaluated by estimating areas under the receiver operating characteristic curves (AUC), and associated confidence intervals.  

# Results Tests of interaction with hormonal therapy  

Among the 2,763 SNPs studied, 112 SNPs passed the first stage screening threshold with a marginal effect  $P$  - value  $<\ 0.05$  , based on the additive allele log-odds model. Among these, 22 were from selected earlier literature reports and 90 were selected based on WHI proteomics studies. These 112 SNPs were then entered into the second stage analysis for testing of interaction with HT based on the simultaneous case-only test with two degrees of freedom. Information about the 112 SNPs is presented in Additional file 1.  

Table 2 presents the top 10 SNPs ranked by    $P$  -value of the simultaneous test of interaction with HT, among the   $112~\mathrm{SNPS}$   entering the second stage analysis. Four SNPs had an interaction test FDR   ${<}0.05$  . Specifically, the top two SNPs, rs2154299 and rs12194855, ranked by interaction test    $P_{*}$  -value are located in an intronic region of the coagulation factor XIII subunit A gene ( F13A1 ) on chromosome region 6p25. These SNPs were found to be associated with risk of ischemic stroke in an earlier study [35]. The two SNPs are in high linkage d is equilibrium with each other, with pairwise correlation  $\mathrm{r}^{2}>\mathrm{\bar{0}}.98$  , and have significant interaction with HT at the 0.05 level even after Bonferroni correction for the 112 interaction tests conducted in the second stage analysis. The other two SNPs identified are from an intronic region of the proprotein convertase subtilisin kexin 9 gene ( PCSK9 ) in the genomic region of chromosome 1p32, the protein product of which was found to be increased by E-alone intervention in the WHI E-alone trial [36]. The two SNPs in the    $P C S K9$   region are also in high linkage d is equilibrium   $(\mathrm{r}^{2}>0.98)$   with each other.  

We further investigated these four SNPs for their interaction with the E-alone and  $\mathrm{E}{+}\mathrm{P}$   interventions separately and by stroke subcategory. Table 3 shows estimated HT intervention odds ratios and   $95\%$   confidence intervals as a function of the number of minor alleles for the two SNPs in the  F13A1  region, for all stroke types together and for ischemic and hemorrhagic stroke separately. For both SNPs, a larger number of minor alleles (A for rs2154299 and G for rs12194855) appears to be associated with a lower   $_{\mathrm{E+P}}$   odds ratio. The pattern is consistent within ischemic or hemorrhagic stroke cases. The trends are in the same direction for E-alone, but not significant. Corresponding results for the two SNPs in the  PCSK9  region are shown in Table 4. For both SNPs, there are suggestions that a larger number of minor alleles (G for rs630431 and G for rs568052) are associated with a lower   $_{\mathrm{E+P}}$   odds ratio, but a higher E-alone odds ratio, for all stroke types altogether and for ischemic stroke. The interactions of the two    $P C S K9$  SNPs with either E-alone or   $_{\mathrm{E+P}}$   alone are only nominally significant.  

The majority   $(81\%)$   of the case-control samples are from European-ancestry women. In Additional files 2 and 3 we provide    $P$  -values for interaction between HT  

Table 2 Top ten SNPs identified by two-component test of interaction with E-alone or  $\tt E+{\tt P}$  
![](images/ae8cd3fa52123b383151636a416d662dec428cb93c9df9465c7f376af62beedd.jpg)  
Rank, rank of SNPs based on HT interaction test with two degrees of freedom. Rs#, SNP identification (rs) number in dbSNP database. Chr, chromosome. Allele, minor/major allele. MAF, minor allele frequency in the study population. OR, odds ratio for increase of per minor allele. Marginal association test  P -value,  P -value based on test of SNP main effect assuming additive effect. HT interaction  P -value:  P -value based on two degrees of freedom joint test for interaction with HT. HT interaction FDR, FDR based on two degrees of freedom joint test for interaction with HT. E-alone interaction  $P_{\l}$  -value,  P -value based on test for interaction with Ealone.  $\mathsf{E}\!+\!\mathsf{P}$   interaction  $P$  -value,    $P$  -value based on test for interaction with  $\mathsf{E}\!+\!\mathsf{P}.$  .  

Table 3 Stroke odds ratio for E-alone and   $\mathsf{E}\!+\!\mathsf{P},$   by genotype of SNPs in  F13A1  region 
![](images/bd9b0cfb2b937994e4b40427039e82dbbd04bdc8873061340ed9ac845e85426f.jpg)  
OR, estimated intervention odds ratio. p-2df,  P -value regressing random iz ation assignment on indicator for one or two minor alleles. p-1df,  P -value regressing random iz ation assignment on number of minor alleles. NA indicates information (data) not available. CI, confidence interval.  

component and the four SNPs in the  F13A1 and PCSK9 regions, and the estimated intervention odds ratios and  $95\%$   confidence intervals as a function of the number of minor alleles, among women of European ancestry specifically. The patterns that we observe are quite similar to the overall patterns.  

Table 4 Stroke odds ratio for E-alone and   $\mathsf{E}\!+\!\mathsf{P},$   by genotype of SNPs in  PCSK9  region 
![](images/daa0690d6370b8ce43b8eeee49adaf54bbaf306757ae3f5036518df3da90b097.jpg)  
OR, estimated intervention odds ratio. p-2df,  P -value regressing random iz ation assignment on indicator for one or two minor alleles. p-1df,  P -value regressing random iz ation assignment on number of minor alleles. CI, confidence interval.  

Additionally, we examined the joint effect of the F13A1  SNPs and  PCSK9  SNPs in interacting with   $\mathrm{E}\mathrm{+P}$  intervention, using case-only analysis. Based on logistic regression applied to cases in the  $_{\mathrm{E+P}}$   trial, where the indicator for active treatment is regressed on genotypes of rs2154299 and rs630431 together, both SNPs showed nominally significant interactions. Specifically, when the indicators for one or two minor alleles are included in the regression, the nominal  $P.$  -values based on likelihood ratio test with two degrees of freedom for rs2154299 and rs630431 were 0.0009 and 0.013 when all stroke types are considered and 0.004 and 0.025 for ischemic stroke. When numbers of minor alleles are included in the regression assuming  ‘ monotone ’  interaction, the nominal    $P_{*}$  -values based on a likelihood ratio test with one degree of freedom for rs2154299 and rs630431 were 0.0002 and 0.005 when all stroke types were considered and 0.001 and 0.008 for ischemic stroke.  

To examine the added benefit of including SNPs in the two regions for discriminating stroke cases from controls, we calculated the AUC from logistic regression analyses that included matching covariates, clinical trial randomization assignments for each of the four interventions, and the potential confounding factors listed above. This gave an AUC (  $\cdot95\%$   confidence interval) of 0.645 (0.634, 0.666) for overall stroke. When rs2154299 and rs630431 were further incorporated into the risk model including main effect indicator variables for one and two minor alleles and corresponding HT intervention interaction indicator variables, the AUC increased to only 0.651 (0.642, 0.675). A bootstrap test of significance for the increase in AUC gave a nominal    $P.$  -value of 0.045.  

# Discussion  

In this report we evaluated the interaction between hormone therapy and 2,763 SNPs on stroke incidence in the WHI clinical trial, through the use of a two-stage testing procedure that conducts interaction tests among only the SNPs passing a first stage filtering based on a marginal association test with stroke incidence. With gene-environment interaction being the major focus here, this type of two-stage procedure allows multiple-testing adjustment to be restricted to a smaller set of SNPs passing the first stage screening. This separate adjustment of interaction testing from main SNP effect testing is justified by the independence between the two test statistics [29], and provides a powerful alternative to the procedure of applying interaction testing to all SNPs. Note, however, that this testing procedure would not be sensitive for interaction detection for SNPs for which there is little evidence for a marginal association with stroke risk.  

Our study is novel in being nested within the randomized controlled WHI clinical trial, implying randomization assignments that are known and statistically independent of genotype. This independence allows caseonly interaction tests with their substantial efficiency gains compared to case-control interaction tests.  

Our analyses identified interesting SNPs on two genomic regions: rs2154299 and rs12194855 from the  F13A1 region of chromosome 6 with a strong interaction with E  $\mathrm{+P;}$   and rs630431 and rs568052 from the  PCSK9  region of chromosome 1 with suggestive evidence for an interaction with HT. The  F13A1  gene encodes subunit A of coagulation factor XIII, which in its active form is involved in the blood coagulation process. Upon activation by thrombin, factor XIIIa acts on fibrin to form cross-links between fibrin molecules to form an insoluble clot. Previous analyses of the WHI trial data have shown that ischemic stroke risk is directly associated with baseline D-dimer levels and that HT increases D-dimer levels [40]. D-dimer is a fibrin degradation product containing two cross-linked D fragments of the fibrinogen protein, which is present in the blood after a blood clot is degraded by fi bri no lysis. Previously, SNPs in the coding region of    $F I3A I$   (rs5985 for Val34Leu and rs3024477 for Tyr204Phe) were reported to be associated with risk of ischemic stroke [4]. However, neither of these SNPs passed the first stage screening in our study, and both are in very low linkage d is equilibrium with rs2154299 and rs12194855   $(\mathrm{r}^{2}<0.01)$  .  

Variations in the  PCSK9  gene have been shown to be related to the risk of large-vessel atherosclerosis stroke [41].  PCSK9  encodes the proprotein convertase subtilisin/ kexin type 9 protein, an important regulator of plasma low-density lipoprotein cholesterol. PCSK9 protein binds to and degrades the low density lipoprotein cholesterol receptor.  PCSK9  loss-of-function mutations result in low levels of low density lipoprotein cholesterol and protect against coronary heart disease while gain-of-function mutations have the opposite effect [42]. The possibility of using PCSK9 in the treatment of hypercholesterolemia has fueled considerable research into related molecular mechanisms [43]. In the WHI hormone trials, higher levels of baseline low-density lipoprotein cholesterol were related to the risk of ischemic stroke, and interacted with E-alone to increase risk [40]. Moreover, in the WHI E-alone trial, blood PCSK9 protein levels were observed to increase with E-alone intervention [36]. These findings, together with the significant interactions discovered in this study between  PCSK9  and both E-alone and   $\mathrm{E}{+}\mathrm{P}$  , makes  $P C S K9$  another interesting candidate for future study of a relationship with hormone therapy and stroke risk and suggests plausible actions through blood low density lipoprotein cholesterol levels.  

In our analysis, interactions of SNPs from the two regions with hormone therapy led to only a very small increase in the estimated ability to distinguish stroke cases from controls. Note that AUCs reported here represent classification accuracy of the models in the matched case-control sample and are of exploratory nature only. On the one hand, matching by factors such as age and prevalent stroke in the case-control sample makes the estimated AUC a somewhat distorted, and likely reduced, estimate of the population AUC; on the other hand, even the small increase in estimated AUC when interacting SNPs are added may be optimistic since these SNPs were identified in the same dataset used to estimate AUC.  

When the simultaneous interaction test with E-alone and  $_{\mathrm{E+P}}$   is further separated into its constituents, we observe highly significant evidence of interaction between the two  F13A1  SNPs with   $_{\mathrm{E+P}}$   and nominally significant evidence of interactions between the two  PCSK9  SNPs with   $\mathrm{E}\mathrm{+P}$   and E-alone for total stroke and for ischemic stroke specifically. The number of hemorrhagic stroke events is too small for any clear conclusions. The results suggest that among women with identified SNPs in the F13A1  region (GG homozygotes for rs2154299 and AA homozygotes for rs12104885), E-alone or  $_{\mathrm{E+P}}$   increased risk of ischemic stroke by about half, with little evidence of an HT effect on stroke risk if one or more minor SNP alleles are present. Within the  $P C S K9$   region, among women with the AA genotype for SNPs rs630431 or rs568052 the risk of ischemic stroke was approximately doubled by E-alone; in contrast, risk elevations of a similar magnitude by   $_{\mathrm{E+P}}$   were noted among women with the GG genotype. Considering our procedure of selection based on most significant interactions, the magnitude of HR odds ratio variations between genotypes observed in this project is subject to  ‘ winner ’ s curse ’  and could be exaggerated due to the fact that 112 SNPs were considered in the second stage analyses. These analyses represent an early step in assessing the role of gene-environment interactions to help explain familial stroke patterns. The analyses also generate interesting hypotheses concerning genotype interactions with hormone therapy in relation to stroke risk that require further confirmatory study.  

# Conclusions  

Two-stage analysis among post menopausal women generates novel hypotheses about the interaction between F13A1  and  PCSK9  genomic regions and the effects of hormonal exposures on post menopausal stroke risk for subsequent independent validation. These analyses represent an early step in assessing the role of genotype by environment interaction to help explain familial stroke aggregation.  

# Additional material  

Additional file 1: The 112 SNPs tested in stage 2, ranked by significance of two-component test of interaction with E-alone or E  $+\mathsf{P}.$  . This file extends the information provided in Table 2 for the 10 SNPs  

that were top-ranked according to hormone therapy interaction with stroke risk to the entire set of 112 SNPs having a marginal association nominal significance level  ${<}0.05$  . Additional file 2: Stroke odds ratio for E-alone and  $\mathsf{E}\!+\!\mathsf{P},$  , by genotype of SNPs in the  F13A1 region  in women of European ancestry . This file presents analyses corresponding to Table 3, with cases and controls restricted to be of European ancestry. Additional file 3: Stroke odds ratio for E-alone and  $\mathsf{E}\!+\!\mathsf{P},$  , by genotype of SNPs in  PCSK9 region  for women with European ancestry . This file presents analyses corresponding to Table 4, with cases and controls restricted to be of European ancestry.  

# Abbreviations  

AUC: area under the receiver operating characteristic curve; CaD trial: calcium and vitamin D versus placebo supplement ation component; CT: randomized controlled trial; DM trial: low-fat dietary modification versus usual diet component; E-alone trial: estrogen versus placebo;   $\mathsf{E}\!+\!\mathsf{P}$   trial: estrogen plus progestin versus placebo;  F13A1 : coagulation factor XIII subunit A; FDR: false discovery rate; HT: hormone therapy; OS: observational study;  PCSK9 : proprotein convertase subtilisin kexin 9; SNP: single nucleotide polymorphism; WHI: Women ’ s Health Initiative.  

# Acknowledgements  

Decisions concerning study design, data collection and analysis, interpretation of the results, the preparation of the manuscript, or the decision to submit the manuscript for publication resided with committees composed of WHI investigators that included NHLBI representatives. This work was supported by the National Heart, Lung, and Blood Institute, National Institutes of Health, US Department of Health and Human Services (contracts HHSN268200764314C, N01WH22110, 24152, 32100-2, 32105-6, 32108-9, 32111-13, 32115, 32118-19, 32122, 42107-26, 42129-32, and 44221). Clinical Trials Registration: Clinical Trials.gov identifier: NC T 00000611. The work of Drs Huang and Prentice was partially supported by NIH grants CA53996 and R 21 HL 109527. Program Office: (National Heart, Lung, and Blood Institute, Bethesda, Maryland) Jacques Rossouw, Shari Ludlam, Dale Burwen, Joan McGowan, Leslie Ford, and Nancy Geller. Clinical Coordinating Center: (Fred Hutchinson Cancer Research Center, Seattle, WA) Garnet Anderson, Ross Prentice, Andrea LaCroix, and Charles Kooperberg. Investigators and Academic Centers: (Brigham and Women ’ s Hospital, Harvard Medical School, Boston, MA) JoAnn E Manson; (MedStar Health Research Institute/Howard University, Washington, DC) Barbara V Howard; (Stanford Prevention Research Center, Stanford, CA) Marcia L Stefanick; (The Ohio State University, Columbus, OH) Rebecca Jackson; (University of Arizona, Tucson/Phoenix, AZ) Cynthia A Thomson; (University at Buffalo, Buffalo, NY) Jean WactawskiWende; (University of Florida, Gainesville/Jacksonville, FL) Marian Limacher; (University of Iowa, Iowa City/Davenport, IA) Robert Wallace; (University of Pittsburgh, Pittsburgh, PA) Lewis Kuller; (Wake Forest University School of Medicine, Winston-Salem, NC) Sally Shumaker. Women ’ s Health Initiative Memory Study: (Wake Forest University School of Medicine, Winston-Salem, NC) Sally Shumaker.  

# Author details  

1 Public Health Sciences Division, Fred Hutchinson Cancer Research Center,   2 1100 Fairview Ave N, Seattle, WA 98109, USA. Complete Genomics, Inc,   3 2071 Stierlin Court, Mountain View, CA 94043, USA. Aria Diagnostics, 5945   4 Optical Court, San Jose, CA 95138, USA. Departments of Epidemiology and Medicine, University of Iowa, 200 Medicine Admin. Bldg, Iowa City, IA 52242,   5 USA. School of Public Health, University of California at Los Angeles, 650   6 Charles E Young Dr. South, Los Angeles, CA 90095, USA. School of Medicine, UT Heath Science Center, 7703 Floyd Curl Drive, San Antonio, TX   7 78229, USA. Health Research & Policy (Epidemiology) and Neurology & Neurological Sciences, Stanford School of Medicine, 450 Serra Mall, Stanford,   8 CA 94305, USA. Women ’ s Health Initiative Project Office, National Heart, Lung, and Blood Institute, 6701 Rockledge Drive, Bethesda, MD 20892, USA.  

# Authors ’  contributions  

All authors were involved in development and/or critical review and revision of the manuscript. Additionally, DB, RS, and EB had primary responsibility for project genotyping; YH and RP had primary responsibility for data analysis; JR and RP had responsibility for clinical data; and DB and RP had primary administrative responsibility for this research project. All authors have read and approved the manuscript for publication.  

# Competing interests  

The authors declare that they have no competing interests.  

Received: 29 February 2012 Revised: 27 June 2012 Accepted: 13 July 2012 Published: 13 July 2012  

# References  

1. Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI, Ramos EM, Cardon LR, Chakra var ti A, Cho JH, Guttmacher AE, Kong A, Kruglyak L, Mardis E, Rotimi CN, Slatkin M, Valle D, Whittemore AS, Boehnke M, Clark AG, Eichler EE, Gibson G, Haines JL, Mackay TFC, McCarroll SA, Visscher PM:  Finding the missing heritability of complex diseases.  Nature  2009,  461 :747-753.

 2. Hata J, Matsuda K, Ninomiya T, Yonemoto K, Matsushita T, Ohnishi Y, Saito S, Kitazono T, Ibayashi S, Iida M, Kiyohara Y, Nakamura Y, Kubo M: Functional snp in an sp1-binding site of agtrl1 gene is associated with susceptibility to brain infarction.  Hum Mol Genet  2007,  16 :630-639.

 3. Kubo M, Hata J, Ninomiya T, Matsuda K, Yonemoto K, Nakano T, Matsushita T, Yamazaki K, Ohnishi Y, Saito S, Kitazono T, Ibayashi S, Sueishi K, Iida M, Nakamura Y, Kiyohara Y:  A non synonymous SNP in PRKCH  (protein kinase C  η ) increases the risk of cerebral infarction.  Nat Genet  2007,  39 :212-217.

 4. Matarin M, Brown WM, Scholz S, Simon-Sanchez J, Fung H, Hernandez D, Gibbs JR, DeVrieze FW, Crews C, Britton A, Langefeld CD, Brott TG, Brown RD Jr, Worrall BB, Frankel M, Silliman S, Case LD, Singleton A, Hardy JA, Rich SS, Meschia JF:  A genome-wide genotyping study in patients with ischaemic stroke: initial analysis and data release.  Lancet Neurol  2007,  6 :414-420.

 5. Bilguvar K, Yasuno K, Niemela M, Ruigrok YM, von Und Zu Fraunberg M, van Duijn CM, van den Berg LH, Mane S, Mason CE, Choi M, Gaal E, Bayri Y, Kolb L, Arlier Z, Ravuri S, Ronkainen A, Tajima A, Laakso A, Hata A, Kasuya H, Koivisto T, Rinne J, Ohman J, Breteler MM, Wijmenga C, State MW, Rinkel GJ, Herne s niemi J, J a ask elaine n JE, Palotie A,  et al :  Susceptibility loci for intra cranial aneurysm in European and Japanese populations.  Nat Genet  2008,  40 :1472-1477.

 6. Greta rs do ttir S, Thor leif s son G, Manolescu A, S tyr kars do ttir U, Helga do ttir A, G schw end t ner A, Kostulas K, Ku hlen baumer   $\mathsf{G},$   Bevan S, Jonsdottir T, Bjarnason H, S a emu nds do ttir J, Palsson S, Arnar DO, Holm H, Thor geir s son G, Vald i mars son EM, S vein bjorn s do ttir S, Gieger C, Berger K, Wichmann HE, Hillert J, Markus H, Gulcher JR, Ring el stein EB, Kong A, Dichgans M, Gud bj arts son DF, Thor steins do ttir U, Stefansson K:  Risk variants for atrial fibrillation on chromosome 4q25 associate with ischemic stroke.  Ann Neurol  2008,  64 :402-409.

 7. Gud bj arts son DF, Holm H, Greta rs do ttir S, Thor leif s son G, Walters GB, Thor geir s son G, Gulcher J, Mathiesen EB, Njolstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg   $\subset_{r}$   Kucera G, Stubble field T, Carter S, Roden D, Ng MC, Baum L, So WY, Wong KS, Chan JC, Gieger C, Wichmann HE, G schw end t ner A, Dichgans M, Ku hlen baumer G, Berger K, Ring el stein EB,  et al :  A sequence variant in zfhx3 on 16q22 associates with atrial fibrillation and ischemic stroke.  Nat Genet  2009,  41 :876-878.

 8. Ikram MA, Seshadri S, Bis JC, Fornage M, DeStefano AL, Aulchenko YS, Debette S, Lumley T, Folsom AR, van den Herik EG, Bos MJ, Beiser A, Cushman M, Launer LJ, Shahar E, Struchalin M, Du Y, Glazer NL, Rosamond WD, Riva den eira F, Kelly-Hayes M, Lopez OL, Coresh J, Hofman A, DeCarli  $\subset,$   Heckbert SR, Koudstaal PJ, Yang Q, Smith NL, Kase CS, et al :  Genomewide association studies of stroke.  N Engl J Med  2009, 360 :1718-1728.

 9. Yamada Y, Fuku N, Tanaka M, Aoyagi Y, Sawabe M, Metoki N, Yoshida H, Satoh K, Kato K, Watanabe S, Nozawa Y, Hasegawa A, Kojima T: Identification of celsr1 as a susceptibility gene for ischemic stroke in Japanese individuals by a genome-wide association study.  Atherosclerosis 2009,  207 :144-149.

 10. Lanktree MB, Dichgans M, Hegele RA:  Advances in genomic analysis of stroke: what have we learned and where are we headed?.  Stroke  2010, 41 :825-832.  

11. Meschia JF:  Stroke genome-wide association studies: the large numbers imperative.  Stroke  2010,  41 :579-580.

 12. Self SG, Longton  $\mathsf{G},$   Kopecky KJ, Liang KY:  On estimating HLA/disease association with application to a study of aplastic anemia.  Biometrics 1991,  47 :53-61.

 13. Piegorsch WW, Weinberg CR, Taylor JA:  Non-hierarchical logistic models and case-only designs for assessing susceptibility in population-based case-control studies.  Stat Med  1994,  13 :153-162.

 14. Vitti ngh off E, Bauer DC:  Case-only analysis of treatment-covariate iterations in clinical trials.  Biometrics  2006,  62 :769-776.

 15. Prentice RL, Huang Y, Hinds DA, Peters U, Pettinger M, Cox DR, Beilharz E, Chlebowski RT, Rossouw JE, Caan B, Ballinger DG:  Variation in the FGFR2 gene and the effects of post menopausal hormone therapy on invasive breast cancer.  Cancer Epidemiol Biomarker Prev  2009,  18 :3079-3085.

 16. Prentice RL, Huang Y, Hinds DA, Peters U, Cox DR, Beilharz E, Chlebowski RT, Rossouw JE, Caan B, Ballinger DG:  Variation in the FGFR2 gene and the effects of a low-fat dietary pattern on invasive breast cancer.  Cancer Epidemiol Biomarkers Prev  2010,  19 :74-79.

 17. Huang Y, Ballinger DG, Dai JY, Peters U, Hinds DA, Cox DR, Beilharz E, Chlebowski RT, Rossouw JE, McTiernan A, Rohan T, Prentice RL:  Genetic variants in the MRPS30 region and post menopausal breast cancer risk. Genome Med  2011,  3 :42.

 18. Prentice RL, Anderson GL:  The Women ’ s Health Initiative: lessons learned. Annu Rev Public Health  2007,  29 :131-150.

 19. The Women ’ s Health Initiative Study Group:  Design of the Women ’ s Health Initiative clinical trial and observational study.  Control Clin Trials 1998,  19 :61-109.

 20. Rossouw JE, Anderson GL, Prentice RL, LaCroix AZ, Kooperberg C, Stefanick ML, Jackson RD, Beresford SA, Howard BV, Johnson KC, Kotchen JM, Ockene J, Writing Group for the Women ’ s Health Initiative Investigators:  Risks and benefits of estrogen plus progestin in healthy post menopausal women: principal results from the Women ’ s Health Initiative randomized controlled trial.  JAMA  2002,  288 :321-333.

 21. Chlebowski RT, Hendrix SL, Langer RD, Stefanick ML, Gass M, Lane D, Rodabough RJ, Gilligan MA, Cyr MG, Thomson CA, Khandekar J, Petrovitch H, McTiernan A, WHI Investigators:  Influence of estrogen plus progestin on breast cancer and mammography in healthy post menopausal women: the Women ’ s Health Initiative randomized trial.  JAMA  2003,  289 :3243-3253.

 22. Wasser the il-Smoller S, Hendrix S, Limacher M, Heiss G, Kooperberg C, Baird A, Kotchen T, Curb D, Black H, Rossouw JE, Aragaki A, Safford M, Stein E, Laowattana S, Mysiw J:  Effect of estrogen plus progestin on stroke in post menopausal women: the Women ’ s Health Initiative.  JAMA 2003,  289 :2673-2684.

 23. Anderson GL, Limacher M, Assaf AR, Bassford T, Beresford SA, Black H, Bonds D, Brunner R, Brzyski R, Caan B, Chlebowski R, Curb D, Gass M, Hays J, Heiss G, Hendrix S, Howard BV, Hsia J, Hubbell A, Jackson R, Johnson KC, Judd H, Kotchen JM, Kuller L, LaCroix AZ, Lane D, Langer RD, Lasser N, Lewis CE, Manson J, Women ’ s Health Initiative Steering Committee,  et al :  Effects of conjugated equine estrogen in post menopausal women with hysterectomy: the Women ’ s Health Initiative randomized controlled trial.  JAMA  2004,  291 :1701-1712.

 24. Hendrix SL, Wasser the il-Smoller S, Johnson KC, Howard BV, Kooperberg C, Rossouw JE, Trevisan M, Aragaki A, Baird AE, Bray PF, Buring JE, Criqui MH, Herrington D, Lynch JK, Rapp SR, Torner J, WHI Investigators:  Effects of conjugated equine estrogen on stroke in the Women ’ s Health Initiative. Circulation  2006,  113 :2425-2434.

 25. Prentice RL, Manson JE, Langer RD, Anderson GL, Pettinger M, Jackson RD, Johnson KC, Kuller LH, Lane DS, Wactawski-Wende J, Brzysk R, Allison M, Ockene J, Sarto   $\mathsf{G},$   Rossouw JE:  Benefits and risks of post menopausal hormone therapy when it is initiated soon after menopause.  Am J Epidemiol  2009,  170 :12-23.

 26. Howard BV, Van Horn L, Hsia J, Manson JE, Stefanick ML, Wasser the ilSmoller S, Kuller LH, LaCroix AZ, Langer RD, Lasser NL, Lewis CE, Limacher MC, Margolis KL, Mysiw WJ, Ockene JK, Parker LM, Perri MG, Philips L, Prentice RL, Robbins J, Rossouw JE, Sarto GE, Schatz IJ, Snetselaar LG, Stevens VJ, Tinker LF, Trevisan M, Vitolins MZ, Anderson GL, Assaf AR,  et al :  Low-fat dietary pattern and risk of cardiovascular disease: the Women ’ s Health Initiative randomized controlled dietary modification trial.  JAMA  2006,  295 :655-666.  

27. Hsia J, Heiss G, Ren H, Allison M, Dolan NC, Greenland P, Heckbert SR, Johnson KC, Manson JE, Sidney S, Trevisan M, Women ’ s Health Initiative Investigators:  Calcium/vitamin D supplement ation and cardiovascular events.  Circulation  2007,  115 :846-854.

 28. Kooperberg C, LeBlanc M:  Increasing the power of identifying gene-gene interactions in genome-wide association studies.  Genet Epidemiol  2008, 32 :255-263.

 29. Dai J, LeBlanc M, Kooperberg C, Prentice RL:  On two-stage hypothesis testing procedures via asymptotically independent statistics.  Biometrika  .

 30. Dai JY, Logsdon B, Huang Y, Hsu L, Reiner A, Prentice RL, Kooperberg C: Simultaneous testing for marginal genetic association and geneenvironment interaction in genome-wide association studies.  Am J Epidemiol  176 :164-173.

 31. Hays JL, Hunt JR, Hubbell FA, Anderson GL, Limacher M, Allen C, Rossouw JE:  The Women ’ s Health Initiative recruitment methods and results.  Ann Epidmiol  2002,  13 :S18-S77.

 32. Saccone SS, Hinrich AL, Saccone NL, Chase GA, Konvicka K, Madden PA, Breslau N, Johnson EO, Hatsukami D, Pomerleau O, Swan GE, Goate AM, Rutter J, Bertelsen S, Fox L, Fugman D, Martin NG, Montgomery GW, Wang JC, Ballinger DG, Rice JP, Bierut LJ:  Choline r gic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs.  Hum Mol Genet  2006,  16 :36-49.

 33. Markus H:  Genes for stroke.  J Neurol Neurosurg Psychiatry  2004, 75 :1229-1231.

 34. Bersano A, Bresolin E, Bresolin N, Candelise L:  Genetic polymorphisms for the study of multi factorial stroke.  Hum Mutat  2008,  29 :776-795.

 35. Pruissen DMO, Slooter AJC, Rosendaal FR, van der Graaf Y, Algra A: Coagulation factor XIII gene variation, oral contraceptives, and risk of ischemic stroke.  Blood  2008,  111 :1282-1286.

 36. Katayama H, Pacznesny   $\mathsf{S}_{\prime}$   Prentice RL, Aragaki A, Faca VM, Pitteri SJ, Zhang Q, Wang H, Silva M, Kennedy J, Rossouw J, Jackson R, Hsia J, Chlebowski R, Manson J, Hanash S:  Application of serum proteomics to the Women ’ s Health Initiative conjugated equine estrogens trial reveals a multitude of effects relevant to clinical findings.  Genome Med  2009, 1 :47.1-47.16.

 37. Pitteri SJ, Hanash SM, Aragaki A, Amon L, Chen L, Busald Buson T, Paczesny S, Katayama H, Wang H, Johnson MM, Zhang Q, McIntosh M, Wang P, Kooperberg C, Rossouw JE, Jackson R, Manson JE, Hsia J, Liu S, Martin L, Prentice RL:  Post menopausal estrogen and progestin effects on the serum proteome.  Genome Med  2009,  1 :121.1-121.14.

 38. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D: Principal component analysis corrects for stratification in genome-wide association studies.  Nat Genet  2006,  38 :904-909.

 39. Benjamini Y, Hochberg Y:  Controlling the false discovery rate: a practical and powerful approach to multiple testing.  J R Stat Soc B  1995, 57 :289-300.

 40. Kooperberg C, Cushman M, Hsia J, Robinson JG, Aragaki AK, Lynch JK, Baird AE, Johnson KC, Kuller LH, Beresford SAA, Rodriguez B:  Can biomarkers identify women at increased stroke risk? The Women ’ s Health Initiative Hormone Trials.  PLoS Clin Trials  2007,  2 :e28.

 41. Abboud S, Karhunen PJ, Lütjohann D, Goebeler S, Luoto T, Friedrichs S, Lehtimaki T, Pandolfo M, and Laaksonen R:  Proprotein convertase subtilisin/kexin type 9 (PCSK9) gene is a risk factor of large-vessel atherosclerosis stroke.  PLoS ONE  2007,  2 :e1043.

 42. Costet P, Krempf M, Cariou B:  PCSK9 and LDL cholesterol: unraveling the target to design the bullet.  Trends Biochem Sci  2008,  33 :426-434.

 43. Peterson AS, Fong LG, Young SG:  PCSK9 function and physiology (commentary).  J Lipid Res  2008,  49 :1152-1156.  