# Article Obesity and Environmental Risk Factors Significantly Modify the Association between Ischemic Stroke and the Hero Chaperone  C19orf53  

Irina Shilenok   , Ksenia Kobzeva  $\mathbf{1}_{\bigcirc}$  , Alexey Deykin   , Vladimir Pokrovsky   5 , Evgeny Patra khan ov   5   and Olga Bushueva  $^{1,6,*\oplus}$  

1 Laboratory of Genomic Research, Research Institute for Genetic and Molecular Epidemiology, Kursk State Medical University, 305041 Kursk, Russia 2 Division of Neurology, Kursk Emergency Hospital, 305035 Kursk, Russia 3 Laboratory of Genome Editing for Bio medicine and Animal Health, Belgorod State National Research University, 308015 Belgorod, Russia 4 Department of Pharmacology and Clinical Pharmacology, Belgorod State National Research University, 308015 Belgorod, Russia 5 Laboratory of Genetic Technologies and Gene Editing for Bio medicine and Veterinary Medicine, Belgorod State National Research University, 308015 Belgorod, Russia 6 Department of Biology, Medical Genetics and Ecology, Kursk State Medical University, 305041 Kursk, Russia \* Correspondence: olga.bushueva@inbox.ru  

Citation:  Shilenok, I.; Kobzeva, K.; Deykin, A.; Pokrovsky, V.; Patra khan ov, E.; Bushueva, O. Obesityand Environmental Risk Factors Significantly Modify the Association between Ischemic Stroke and the Hero Chaperone  C19orf53 .  Life  2024 ,  14 , 1158. https://doi.org/10.3390/ life14091158  

Academic Editors: Irina-Iuliana Costache and Minerva Codruta Badescu  

Received: 31 July 2024 Revised: 7 September 2024 Accepted: 10 September 2024 Published: 12 September 2024  

Abstract:  The unique chaperone-like properties of C19orf53, discovered in 2020 as a “hero” protein, make it an intriguing subject for research in relation to ischemic stroke (IS). Our pilot study aimed to investigate whether C19orf53 SNPs are associated with IS. DNA samples from 2138 Russian subjects (947 IS and 1308 controls) were genotyped for 7  C19orf53  SNPs using probe-based PCR. Dominant (D), recessive (R), and log-additive (A) regression models in relation to the effect alleles (EA) were used to interpret associations. An increased risk of IS was associated with rs10104 (EA G  $;P_{\mathrm{{bond}(R)}}=0.0009,$  ;

  $\mathrm{P_{bonf(A)}=0.0004)}$  , rs11666524 (EA A;  $\mathrm{P_{bonf(R)}}=0.003$  ;  $\mathrm{P_{bonf(A)}}=0.02)$  ), rs346158 (EA C;   $\mathrm{P_{bonf(R)}}=0.006;$  ;

  $\mathrm{P_{bonf(A)}=0.045)}$  , and rs2277947 (EA A;  $\mathrm{P_{bonf(R)}=0.002;P_{b o n f(A)}=0.01)}$   in patients with obesity; with rs11666524 (EA A;  $\mathrm{P_{bonf(R)}}=0.02)$  , rs346157 (EA G  $\mathrm{{\tilde{g}};P_{b o n f(R)}=0.036)}$  , rs346158 (EA   $\mathcal{S};\mathrm{P_{bonf(R)}}=0.005)$  , and rs2277947 (EA A;  $\mathrm{P_{bonf(R)}}=0.02)$   in patients with low fruit and vegetable intake; and with rs10104 (EA G  $;P_{\mathrm{{bond}(R)}}=0.03)$   and rs11666524 (EA A;   $\mathrm{P_{bonf(R)}}=0.048)$   in patients with low physical activity. In conclusion, our pilot study provides comprehensive genetic and bio in format ic evidence of the involvement of  C19orf53  in IS risk.  

Keywords:  C19orf53; chromosome 19 open reading frame 53; SNP; obesity; low fruit and vegetable intake; low physical activity; chaperones; heat-resistant obscure; hero; ischemic stroke  

# 1. Introduction  

An ischemic stroke (IS) is recognized as a major contributor to morbidity and mortality worldwide [ 1 ]. An IS is a localized brain injury resulting from a substantial decrease in cerebral blood flow within major cerebral arteries caused by embolism or thrombosis usually following at hero sclerotic plaque rapture [ 2 ]. Acute ischemia leads to cell death directly via severe metabolic deficits or via secondary alteration as a result of inflammation and blood–brain barrier opening [ 3 ].  

A crucial mechanism in minimizing cellular damage in an ischemia–reperfusion injury involves the utilization of chaperone machinery [ 4 , 5 ]. Chaperones form a functionally linked network of proteins dedicated to ensuring the quality of proteome—a pivotal process for cellular protection against lethal stress [ 6 , 7 ]. Moreover, chaperones have been implicated in a wide range of processes markedly important for IS etiology, such as vascular such as vascular tone regulation [8–10], at hero sclerotic plaque formation and dynamics  tone regulation [ 8 – 10 ], at hero sclerotic plaque formation and dynamics [ 11 – 13 ], and platelet [11–13], and platelet aggregation [14,15].  aggregation [ 14 , 15 ]. C19orf53, also known as Chromosome 19 Open Reading Frame 53, plays a role in  

C19orf53, also known as Chromosome 19 Open Reading Frame 53, plays a role in mediating excessive cell proliferation, leading to metabolic imbalance and increased oxmediating excessive cell proliferation, leading to metabolic imbalance and increased oxidative stress in response to toxins [16]. Associated with various biological pathways, inidative stress in response to toxins [ 16 ]. Associated with various biological pathways, cluding rRNA biogenesis, ovarian cancer, and Zika virus replication [17,18], C19orf53  including rRNA biogenesis, ovarian cancer, and Zika virus replication [ 17 , 18 ], C19orf53 gained attention in 2020 when Tsuboyama K et al. identified it along with other small  gained attention in 2020 when Tsuboyama K et al. identified it along with other small heat-resistant proteins displaying chaperone-like activity in cell culture lysate [19]. Reheat-resistant proteins displaying chaperone-like activity in cell culture lysate [ 19 ]. Referred to as “heat-resistant obscure” or “Hero”, C19orf53 exhibited resilience, emphaferred to as “heat-resistant obscure” or “Hero”, C19orf53 exhibited resilience, emphasizing sizing its chaperone-like characteristics. Given the significance of chaperones in IS, exits chaperone-like characteristics. Given the significance of chaperones in IS, exploring ploring the newfound properties of C19orf53 protein is crucial for understanding its pothe newfound properties of C19orf53 protein is crucial for understanding its potential tential implications in the disease. All Hero genes previously studied by us showed sigimplications in the disease. All Hero genes previously studied by us showed significant asnificant associations with the risk of developing an IS [20–22], suggesting their potentially  sociations with the risk of developing an IS [ 20 – 22 ], suggesting their potentially significant significant role in the molecular mechanisms of IS development, most likely due to their  role in the molecular mechanisms of IS development, most likely due to their pronounced pronounced chaperone-like properties. In addition, a recent study has established the  chaperone-like properties. In addition, a recent study has established the role of C19orf53 in suppressing the pathological aggregation of TAR DNA-binding protein 43, which plays tein 43, which plays an important role in neuro degenerative diseases [23]. Given the sigan important role in neuro degenerative diseases [ 23 ]. Given the significance of chaperones in IS, exploring the newfound properties of C19orf53 protein is crucial for understanding crucial for understanding its potential implications in the disease.  its potential implications in the disease.  

The aim of our study was to investigate whether the polymorphism of the  C19orf53 gene, encoding a heat-resistant obscure (Hero) protein with chaperone-like activity, is associated with the risk and clinical features of ischemic stroke.  

# 2. Materials and Methods  

![](images/1a8561fcc37d875077ef6a34f0389be2776201fae17420e6505be45ded969f5b.jpg)  
Figure 1.  The outline of the study design.  

# 2.1. Experimental Design  

2.1. Experimental Design  The study included 2138 unrelated individuals from central Russia, comprising 947 IS patients and 1308 healthy individuals. The Ethical Review Committee of Kursk State Medical University approved the study protocol, and all participants provided written informed consent. Inclusion criteria for the study required participants to have self-declared Russian ancestry and to be born in central Russia.  

Table  1  provides general clinical characteristics of IS patients and controls.  

Table 1.  General clinical characteristics of IS patients and controls. 
![](images/ca645d7750e0b2d2ec08078a0fb09546e657fb6008c7151096e3ee4a1bb84bd0.jpg)  
Statistically significant differences between groups are indicated in bold; ND—no data.  

The patients were enrolled in the study during two distinct periods: from 2015 to 2017 at the Regional Vascular Center of Kursk Regional Clinical Hospital [ 24 , 25 ] and from 2010 to 2012 at the Neurology Clinics of Kursk Emergency Medicine Hospital [ 26 , 27 ].  

Inclusion criteria for the study required patients to have a diagnosis of ischemic stroke, confirmed by neurologists during the acute phase of the disease based on clinical and neurological evaluations and verified by computed tomography or magnetic resonance imaging of the brain performed within the first 48 hours following the stroke event. The study included the following subtypes of ischemic stroke: a the roth rom bot ic, card i oem boli c, and stroke of unknown etiology. Exclusion criteria included patients with liver or kidney failure, endocrine, autoimmune, on co logical, or other conditions that could lead to an acute cerebro vascular accident. Additionally, patients with hemorrhagic stroke, secondary intra cerebral hemorrhage, he mo dynamic, he mor heo logical, or dissection stroke as well as those with traumatic brain injury were excluded. All included IS patients had a documented history of hypertension and were receiving anti hypertensive treatment.  

The control group consisted of healthy volunteers who had normal blood pressure, did not receive anti hypertensive therapy, and showed no clinical symptoms of cardiovascular, cerebro vascular, or other significant illnesses. Healthy individuals were included in the control group if their systolic blood pressure was below  $130\:\mathrm{mm\:Hg}$   and diastolic blood pressure was below  $85\:\mathrm{mm\:Hg},$  , based on at least three separate measurements. Controls from the Kursk region were selected from hospitals during routine medical examinations conducted in public institutions and industrial establishments. This group was recruited from the same population and during the same time period.  

In accordance with WHO guidelines, low fruit and vegetable consumption was defined as consuming less than   $400\:\mathrm{g}$   per day. Adequate consumption of fresh vegetables and fruits was defined as consuming   $400\;\mathrm{g}$   or more, equivalent to 3–4 servings per day, excluding starchy tubers like potatoes.  

The following criteria were used to choose the SNPs: have a minor allele frequency of at least 0.05 in the European population and be distinguished by a high regulatory potential.  

The bio in format ic tools LD TAG SNP Selection (TagSNP) and SNPinfo Web Server ( https://snpinfo.niehs.nih.gov/  (accessed on 15 March 2022)) were utilized to choose SNPs based on the reference haplotypic structure of the Caucasian population (CEU) of the project HapMap. The  C19orf53  (Chromosome 19 Open Reading Frame 53, ID: 26135) gene contains seven SNPs (rs10104, rs11666524, rs346157, rs346158, rs2901077, rs2277947, and rs8107914). SNP rs10104 is a missense variant, SNPs rs11666524, rs346157, rs346158, rs2901077 are located in introns, and SNPs rs2277947 and rs8107914 are located in the 3 prime UTRs.  

# 2.2. Genetic Analysis  

Genotyping was carried out by the Laboratory of Genomic Research at the Research Institute for Genetic and Molecular Epidemiology. Venous blood samples (up to  $5\;\mathrm{mL}$  ) were drawn from each participant’s cubital vein and placed in EDTA-coated tubes, which were then stored at    $-20\,^{\circ}\mathrm{C}$   until processing. Genomic DNA was extracted from the thawed blood samples using standard phenol/chloroform extraction and ethanol precipitation methods. The purity, quality, and concentration of the extracted DNA were evaluated using a NanoDrop spectrophotometer (Thermo Fisher Scientific, Waltham, MA, USA).  

Detection of SNP variations in the  C19orf53  gene was conducted using Taq-Man-based PCR following protocols developed at the Research Institute for Genetic and Molecular Epidemiology. Primer design was carried out using Primer3 software version 4.1.0 [ 28 ], with the specific primers and probes listed in Table S1. The real-time PCR was performed in a  $25\;\upmu\mathrm{L}$   reaction mixture containing 1.5 units of Hot Start Taq DNA polymerase (Biolabmix, Novosibirsk, Russia), approximately   $10\;\mathrm{ng}$   of DNA, and the following reagent concentrations:   $0.25~\upmu\mathrm{M}$   of each primer;   $0.1~{\upmu\mathrm{M}}$   of each probe;   $250~\upmu\mathrm{M}$   of each dNTP;   $2.5\;\mathrm{mM}$   $\mathrm{MgCl}_{2}$  ; 1xPCR buffer (67 mM Tris-HCl,   $\mathrm{pH}\ 8.8.$  ,   $16.6~\mathrm{mM}$     $(\mathrm{NH_{4}})_{2}\mathrm{SO_{4}}.$  ,   $0.01\%$   Tween-20). The PCR procedure comprised an initial de natura tion for   $10\;\mathrm{{min}}$   at  $95~^{\circ}\mathrm{C},$   followed by 39 cycles of   $92\,^{\circ}\mathrm{C}$   for  $30\,\mathrm{s}$   and   $53\,^{\circ}\mathrm{C},$   $57.5\,^{\circ}\mathrm{C},$  ,   $59\,^{\circ}\mathrm{C}.$  ,  $61\ {}^{\circ}\mathbf{C},$  ,   $63\ ^{\circ}\mathrm{C},$   and  $65~^{\circ}\mathrm{C}$   for  $1\,\mathrm{{min}}$   (for rs346158, rs8107914, rs2901077 and rs11666524, rs10104, rs346157, and rs2277947, respectively). Supplementary Figure S1 shows allelic discrimination plots for  C19orf53  assays designed for this study. Ten percent of the DNA samples were genotyped twice, blinded to the case-control status, in order to assure quality control. Over   $99\%$   of the data were concordant.  

# 2.3. Statistical and Bio in format ic Analysis  

Statistical power was calculated prior to the study using genetic association study power calculator [ 29 ]. Association analysis between the  C19orf53  gene polymorphisms and IS risk could detect the genotype relative risk of 1.2–1.4, assuming 0.80 power and  $5\%$   type I error   $\alpha=0.05)$   on the sample size of 947 cases and 1308 controls.  

The STATISTICA software (v13.3, Santa Clara, California, USA) was utilized for statistical processing. The normality of the distribution for quantitative data was assessed using the Shapiro–Wilk’s test. Given that the majority of quantitative parameters exhibited deviations from normal distribution, they were presented as the median (Me) along with the first and third quartiles [Q1 and Q3]. To compare quantitative variables among two independent groups, the Mann–Whitney test was performed. For categorical variables, differences in statistical significance were evaluated using Pearson’s chi-squared test.  

The genotype distributions were assessed for compliance with Hardy–Weinberg equilibrium using Fisher’s exact test. The study groups’ genotype frequencies and their associations with disease risk were analyzed using SNPStats software ( https://www.snpstats. net/start.htm ) (accessed on 23 April 2024). Additive, dominant and recessive models were considered for the genotype association analysis. All calculations were made in relation to the effect allele (EA). The Bonferroni correction was applied additionally to account for multiple comparisons. Associations within the entire group of IS patients/controls were adjusted for age, gender, and smoking status. When information regarding the environmental risk factor was unavailable in the control group, associations were analyzed by comparing the presence or absence of the risk factor in the patient group to the overall control group. The functional effects of  C19orf53  SNPs were analyzed using the following bioinfor  

matics resources:  

• The GTExportal ( http://www.gtexportal.org/  (accessed on 14 May 2024)) was employed to analyze the expression levels of the studied genes in the brain, whole blood, and blood vessels as well as to examine the expression quantitative trait loci (eQTLs) [ 30 ].

 • For the examination of  C19orf53  SNPs binding to quantitative expression trait loci (eQTL) in peripheral blood, the eQTLGen resource available at  https://www.eqtlgen.org/ (accessed on 14 May 2024) was employed [ 31 ].

 • HaploReg (v4.2), a bioinformatics tool available at  https://pubs.broad institute.org/ mammals/haploreg/haploreg.php  (accessed on 14 May 2024), was utilized to assess the associations between  C19orf53  SNPs and specific histone modifications indicative of promoters and enhancers. These modifications included acetyl ation of lysine residues at positions 27 and 9 of the histone H3 protein as well as mono-methyl ation at position 4 (H3K4me1) and tri-methyl ation at position 4 (H3K4me3) of the histone H3 protein. Additionally, the tool was applied to investigate the positioning of SNPs in DNase hypersensitive regions, regulatory motif sites, and locations binding to regulatory proteins [ 32 ].

 • Bio in format ic tool STRING database ( https://string-db.org/  (accessed on 14 May 2024)) was employed to analyze the primary interaction partners of C19orf53. Additionally, the analysis of biological processes and molecular functions related to interactions with key functionally related proteins was conducted using the STRING database [ 33 ].

 • The mechanisms of interactions between  C19orf53  and cis-eQTL-associated genes were analyzed using the GeneMANIA tool [ 34 ].

 • The atSNP Function Prediction online tool ( http://atsnp.biostat.wisc.edu/search (accessed on 15 May 2024)) was utilized to assess how  C19orf53  SNPs affect the gene’s affinity for transcription factors (TFs) based on the presence of reference or alternative alleles [ 35 ]. This tool evaluated the impact of SNPs on TF-DNA interactions using a positional weight matrix to calculate the degree of influence.  

• The Gene Ontology online tool ( http://gene ontology.org/  (accessed on 15 May 2024)) allowed us to analyze the involvement of transcription factors associated with the reference and SNP alleles in overrepresented biological processes related to IS pathogenesis [ 36 ]. This tool helped identify biological functions regulated by TFs linked to C19orf53  SNPs.

 • The Cerebro vascular Disease Knowledge Portal (CDKP) ( https://cd.hugeamp.org/ (accessed on 16 May 2024)) and Cardiovascular Disease Knowledge Portal ( https://cvd.hugeamp.org/  (accessed on 16 May 2024)) integrate and analyze genetic association data from major consortia to study cardio- and cerebro vascular diseases, aiding in the exploration of associations between  C19orf53  SNPs and atherosclerosisrelated conditions and risk factors for IS (like low density lipids, body mass index, total cholesterol) [ 37 ].  

# 3. Results  

The  C19orf53  gene is expressed in various parts of the brain as well as in blood vessels and whole blood.  C19orf53  gene expression levels (MeTPM) range from 68.48 to 140.8 in brain tissues, range from 120.5 to 136.0 in blood vessels, and are 45.11 in whole blood (Supplementary Figure S2).  

# 3.1. C19orf53 SNPs and the Ischemic Stroke Risk: An Analysis of Associations  

The genotype frequencies of rs10104, rs11666524, rs346157, rs346158, rs2901077, rs2277947, and rs8107914  C19orf53  in study groups are presented in Table S2. In the control group, the genotype frequencies of all studied SNPs corresponded to the Hardy–Weinberg equilibrium   $(p>0.05)$  .  

The analysis of the total sample revealed associations between SNPs and the risk of an IS (Table S2): rs346157 (EA G;  $\mathrm{{OR}_{(R)}=1.30;95\%\,C I_{(R)}=1.03–1.64;P_{(R)}=0.03}$  , rs346158 (EA  $C;\mathrm{OR}_{(\mathrm{R})}=1.46$  ;   $95\%$   $\mathrm{CI}_{\mathrm{(R)}}=1.01–2.11$  ;  $\mathrm{P_{(R)}}=0.04)$  , and rs2901077 (EA T;   $\mathrm{{OR}_{(R)}=2.49}$  ;   $95\%$   $\mathrm{CI}_{\mathrm{(R)}}=1.11\mathrm{-}5.57;\mathrm{P}_{\mathrm{(R)}}=0.02$  significant after applying Bonferroni correction for the number of regression models (three) (Table S3).  

We analyzed the subgroups depending on smoking status, obesity, fresh fruit and vegetable intake, and physical activity level (Table S4). Our data reveal associations between specific genetic variants and the risk of an IS under the three following different conditions: obesity, low physical activity, and low fruit and vegetable intake (Table  2 ).  

rst, in obese patients   $(\mathrm{BMI}\geq30)$  ), the risk of an IS increases four SNPs  C19orf53 , with P being significant in both recessive (R) and log-additive models (A): rs10104 (EA G;

  $\mathrm{OR}_{\mathrm{(R)}}=4.7;$  ;  $95\%$     $\operatorname{CI}_{\mathrm{(R)}}=2.25–9.82;$  ;  $\mathrm{P_{R}}=0.0003$     $(\mathrm{P_{(R)\;b o n f}}=0.0009)$   and   $\mathrm{{OR}_{(A)}=1.82}$  ;   $95\%$   $\mathrm{CI_{(A)}}=1.22–2.72$  ;  $\mathrm{P_{(A)}}=0.004$     $\mathrm{(P_{(A)\;b o n f}}=0.01)$  ), rs11666524 (EA A;  $\mathrm{{OR}}_{(\mathrm{{R}})}=3.88$  ;   $95\%$   CI (R)

  $=1.88–8.02$  ;  $\mathrm{P_{(R)}}=0.001$     $(\mathrm{P_{(R)\;b o n f}}=0.003)$   and   $\mathrm{{OR}_{(A)}=1.76;\,95\%\ C I_{(A)}=1.19-2.6;\,P_{(A)}=}$  0.006   $(\mathrm{P_{(A)\;b o n f}}=0.02)$  ), rs346158 (  $\operatorname{EA}C;\mathrm{OR}_{(\mathrm{R})}=3.73.$  ;  $;95\%\,\mathrm{CI}_{(\mathrm{R})}=1.81-7.69;\mathrm{P}_{(\mathrm{R})}=0.002$   $(\mathrm{P_{(R)\;b o n f}}=0.006)$   and   $\mathrm{{DR}_{(A)}=1.65;95\%\,C I_{(A)}=1.11-2.44;\,P_{(A)}=0.015}$     $(\mathrm{P_{(A)\;bonf}}=0.045)$  ), and rs2277947 (EA A;  $\mathrm{OR}_{\mathrm{(R)}}=4.11$   $.11;95\%\,\mathrm{CI}_{\mathrm{(R)}}=1.98\-8.53;\mathrm{P}_{\mathrm{(R)}}=0.0008\,\left(\mathrm{P}_{\mathrm{(R)}\,\mathrm{bonf}}=0.002\right)$  and  $\mathrm{{OR}_{(A)}=1.82;95\%\,C I_{(A)}=1.22-2.7;\mathrm{{P}_{(A)}=0.004\,\left(\mathrm{{P}_{(A)\,b o n f}=0.01}\right)}}$   (Table  2 ).  

Table 2.  Statistically significant correlations between  C19orf53  SNPs and IS risk determined by BMI, fruit and vegetable intake, and physical activity. 
![](images/5d88e3abab7ef4efb24429b9da0e4aaa56a6bbd071dfa562a3e3cce353ad58f5.jpg)  
All calculations were based on the minor alleles (effect alleles);   1 —odds ratio and  $95\%$   confidence interval;  $p$  -value; statistically significant differences are marked in bold. Effect alleles (EA) are highlighted.  

Second, in subjects with low physical activity, the following genetic variants were discovered to be related to the increased risk of an IS: rs10104 (EA G;  $\mathrm{{OR}}_{\mathrm{{(R)}}}=1.91$   = 1.91; 95%   $95\%$   $\mathrm{CI}_{\mathrm{(R)}}=1.19{-}3.07;\mathrm{P}_{\mathrm{(R)}}=0.009~\mathrm{(P_{(R)\,bond}}$   $(\mathrm{P_{(R)}}\,\mathrm{bot})=0.03)$  ) and rs11666524 (EA A; OR  = 1.8; 95% CI  CI  =  (R)

  $=1.13{-}2.84;\,\mathrm{P_{(R)}}=0.016\;(\mathrm{P_{(R)\;b o n f}}=0.048))$   (Table  2 ).  

Third, under the condition of low fruit and vegetable consumption, the following genetic variants were associated with an increased risk of an IS: rs11666524 (EA A;

  $\mathrm{{OR}_{(R)}\,=\,1.81}$  ;   $\mathrm{CI}_{\mathrm{(R)}}\:=\:1.2–2.75;$     $\mathrm{P_{(R)}\,=\,0.006}$  ), rs346157 (EA   $\mathbf{G}$  ; (R)  = 1.2–2.75; P  = 0.02)), rs346157 (EA G; OR (R)

  $\mathrm{{OR}(R)\,=\,1.42;\,\,95\%\ C I_{(R)}\,=\,1.08–1.86;}$   = 0.036)), rs346158 (EA C; OR  $(\mathrm{P_{(R)\;b o n f}\,=\,0.036})$   = 1.93; 95% CI ), rs346158 (EA C;

  $\mathrm{{OR}_{(R)}=1.93;95\%\,C I_{(R)}=1.29-2.89;P_{(R)}=0.0017\left(P_{(R)\,b o n f}=0.005\right))}$  , and rs2277947 (EA A;

  $\mathrm{{OR}}_{(\mathrm{{R}})}=1.81$  (R) bonf ;   $95\%\,{\mathrm{CI}}_{\mathrm{(R)}}=1.18{-}2.76;$   $\mathrm{P_{(R)}}=0.0074$     $(\mathrm{P_{(R)\;b o n f}}=0.02)$  ) (Table  2 ).  

# 3.2. C19orf53 SNPs and Clinical Parameters  

Upon analyzing the clinical characteristics in patients with IS, it was observed that allele G rs346157 was associated with lower APTT in patients with low fruit/vegetable intake   $(p=0.03)$  ; allele A rs2277947 was associated with a higher BMI in patients with low fruit/vegetable intake   $(p=0.004)$   = 0.004) and in non-smokers ( p  $(p=0.01)$   = 0.01) (Figure 2, Table S5).  , Table S5).  

![](images/e6a0d48e01ac6e2e8fe043e638a0de246098d713cc65e6d420bef06c790f47b4.jpg)  

Figure 2.  Associations of  C19orf53  SNPs and clinical parameter. ( A )—APTT values for rs346157 in  Figure 2.  Associations of  C19orf53  SNPs and clinical parameter. ( A )—APTT values for rs346157 in patients with low fruit and vegetable consumption; ( B )—BMI data for rs2277947 in patients with  patients with low fruit and vegetable consumption; ( B )—BMI data for rs2277947 in patients with low low fruit/vegetable intake; ( C )—BMI values for rs2277947 in non-smoking patients.  fruit/vegetable intake; ( C )—BMI values for rs2277947 in non-smoking patients.  

3.3. The Role of Bio in format ic Tools in Substantiating the Results  

For the functional annotation of SNPs and the bio in format ic analysis of the role of the  the  C19orf53  gene in the path o genetic mechanisms of ischemic stroke, we used a range of tools aimed at analyzing the relationship of SNPs with transcription factors, expression quantitative trait loci, and histone modifications. Additionally, gene–gene and protein– protein interaction networks were constructed.  

# 3.3.1. QTL-Effects  

Based on data from the eQTLGene browser SNPs, linked to a higher risk of IS, specifically rs10104 (effect allele G), rs11666524 (effect allele A), rs346158 (effect allele C), and rs2277947 (effect allele A) are associated with reduced levels of expression of both C19orf53  MRI1  as well as increased levels of  as well as increased levels of  CCDC130  ZSWIM4  expression in peripheral ). On the other hand, rs346157 (effect allele G) is also linked to reduced reduced expression levels of  C19orf53  MRI1  and   and an increase in  and an increase in   CCDC130  expression but does but does not affect the expression levels of  ZSWIM4  ). (Table 3).  

GTEx Portal data reveal correlations between IS-associated SNPs and the reduced expression levels of  C19orf53  in brain and adipose tissues as well as in arteries through eQTL effects. Additionally, the expression of the  MRI1  gene shows a decrease in arteries, including the aorta and tibial arteries, as well as in the brain cortex and subcutaneous adipose tissue. These changes in  MRI1  expression are associated with the presence of rs10104, rs11666524, rs346158, and rs2277947 SNPs (Table  4 ).  

Table 3.  Relationship between  C19orf53  SNPs and cis-eQTL-mediated expression levels of genes in whole blood (according to browser eQTLGene) ( https://www.eqtlgen.org/ ) (accessed on 14 May 2024).  

![](images/3946f0bc175ba4b001f8ceafbf45a8b48b30af78f111333614960544342cd59b.jpg)  
Effect allele are marked in bold.  

Table 4.  The impact of SNPs in genes encoding  C19orf53 , gene expression (cis-eQTL) in blood vessels, and brain and blood cells (according to the GTEx Portal browser,  https://gtexportal.org ) (accessed on 14 May 2024).  

![](images/065124d3b2aaf89ecc95045d9e4ca6fd1b549dd6c0e0b93c69669adca472ae79.jpg)  

Table 4.  Cont.  

![](images/c716dd9ce0ea35d3a2b90f4b2e29717d649e88f7ef6d68455ccc6d35becddedd.jpg)  

Table 4.  Cont.  

![](images/bf1234ec9ff46886a376d21211f40e54753926ca635a1b762b0b2ab87c842847.jpg)  

Table 4.  Cont. 
![](images/e70e90ab931be1eb8c869901431b7dd1a3173277fc0ea350fe7a61c1a4d8ac23.jpg)  
Effect alleles are marked in bold.  

Using the GeneMania resource, an analysis of interactions between  C19orf53  and ciseQTL-associated genes was carried out ( MRI1 ,  CCDC130 ,  ZSWIM4 ). It turned out that they interact through 20 related genes and 125 total links, mainly through physical interactions , x FOR PEER REVIEW   $(77,64\%)$  ), co-expression   $\left(8,\!01\%\right)$  , prediction   $(5.37\%)$  , co-localization   $(3.63\%)$  , genetic interactions  $(2.87\%)$  , pathways   $(1.88\%)$  , and shared protein domains   $(0.60\%)$   (Figure  3 , Table S6).  

![](images/553f060166308800bad5d1ca156b9c05ec8295a1bf6b4245e76f4b8ae98313dc.jpg)  
Figure 3.  Interactions between Interactions between   C19orf53  and cis-eQTL-associated genes (24 total genes, 4 searched genes, 20 related genes, 125 total links). Note: Genes searched are indicated with stripes.  genes, 20 related genes, 125 total links). Note: Genes searched are indicated with stripes.  

The analysis of the joint functions of genes in a network of interactions of  C19orf53 and cis-eQTL-related genes showed their participation in such processes significant for the development of an IS as oli god end ro cyte differentiation, sulfur amino acid/aspartate family amino acid metabolic process (Table S7).  

# 3.3.2. Histone Modifications  

Utilizing the bioinformatics resource HaploReg (v4.2), we conducted an analysis of histone modifications associated with SNPs identified in our study as being linked to an increased risk of IS.  

Specifically, all SNPs linked to IS are located in the region of DNA binding to histone H3, characterized by mono-methyl ation at the 4th lysine residue of the histone H3 protein (H3K4me1) in both blood cells and brain tissues. Furthermore, SNPs rs10104, rs11666524, and rs2901077 are located in the region of DNA binding to H3K4me3 in both brain regions and peripheral blood.  

These histone modifications are further complemented by the effects of H3K27ac, marking enhancers, and the acetyl ation at the 9th lysine residues of the H3K9ac, marking promoters in both blood cells and brain tissues for the SNPs rs10104, rs11666524, rs346157, rs346158, and rs2901077.  

As for SNP rs2277947, it is localized within DNA region that binds to both H3K4me1 and H3K4me3 but exclusively in blood cells. The impact of these histone tags is further enhanced by the presence of H3K27ac, marking enhancer and H3K9ac, marking promoters specifically in blood cells.  

Notably, SNP rs10104 is additionally found within DNA regions that are hypersensitive to DNase-1 in blood cells (Table  5 ). The significant effect of rs10104  C19orf53  on DNA binding to regulatory proteins EGR1, POL24H8, TAF1, POL2, CFOS, ETS1, MAX, POL2B, ZBTB7A, and ZNF263 ( https://pubs.broad institute.org/mammals/haploreg/detail_v4.2 .php?query  $\leftleftarrows$  &id  $=$  rs10104 ) (accessed on 14 May 2024) should also be noted.  

Table 5.  The effect of  C19orf53  SNPs on histone modifications in different tissues. 
![](images/6927d06f88df29393ce76d40ef5b149657a3a26a828669520289d53eb3091aff.jpg)  

Table 5.  Cont. 
![](images/dccbff9d4362bd52d7edfd95e4146b0d5ab645ad6901d9ff41edb7d87ec0afe7.jpg)  
H3K4me1—mono-methyl ation at the 4th lysine residue of the histone H3 protein; H3K4me3—tri-methyl ation at the 4th lysine residue of the histone H3 protein; H3K9ac—acetyl ation at the 9th lysine residues of the histone H3 protein; H3K27ac—acetyl ation of the lysine residue at N-terminal position 27 of the histone H3 protein; effect alleles are marked in bold. Enh—histone modification associated with gene enhancers; Pro—histone modification associated with gene promoters; 1—Brain (hippocampus); 2—Brain (substantia nigra); 3—Brain (anterior caudate); 4—Brain (cingulate gyrus); 5—Brain (inferior temporal lobe); 6—Brain (angular gyrus); 7—Brain (dor so lateral prefrontal cortex); 8—Any cells from peripheral blood; 9—Adipose Nuclei.  

# 3.3.3. Analysis of Transcription Factors  

The risk allele G of rs10104  C19orf53  is associated with the generation of DNA-binding sites for 79 transcription factors (TFs) (Table S8). These TFs are involved in five overrepresented biological processes: the interleukin-9-mediated signaling pathway (GO:0038113;  $\mathrm{FDR}=0.0245)$  ), neuron fate determination (GO:0048664;   $\mathrm{FDR}=0.0375$  ), the positive regulation of neuron differentiation (GO:0045666;   $\mathrm{FDR}=0.0105,$  ), the regulation of endo the li al cell proliferation (GO:0001936;   $\mathrm{FDR}=0.0353)$  ), and the negative regulation of neuro genesis (GO:0050768;   $\mathrm{FDR}=0.0424$  ) (Table S8).  

Nineteen TFs, binding to the risk allele A of rs11666524  C19orf53 , are concurrently implicated in two overrepresented GO: the positive regulation of neuron differentiation

 (GO:0045666; F  ${\tt D R}=0.00146$  ) and astrocyte fate commitment (GO:0060018;   $\mathrm{FDR}=0.00306)$  )

 (Table S9).  

The risk allele G of rs346157  C19orf53  produces DNA-binding regions for 22 transcription factors, collectively engaged in four overrepresented GOs related to cell proliferation, oxidative stress, cell signaling, and apoptosis: positive regulation of glial cell proliferation

 (GO:0060252; F  $\mathrm{{FDR}=0.0259}$  ), cellular response to reactive oxygen species (GO:0034614;

  $\mathrm{FDR}=0.0248)$  ), response to hypoxia (GO:0001666;   $\mathrm{FDR}=0.0197)$  ), and mono nuclear cell differentiation (GO:1903131;   $\mathrm{FDR}=0.0371$  ) (Table S10). On the other hand, the protective allele A of rs346157  C19orf53  provides DNA-binding sites for 63 TFs, which are together participate in 11 GOs associated with the blood vessel morphogenesis (GO:0048514;  $\mathrm{FDR}=0.00756)$  , regulation of the vascular tone (cellular response to angiotensin (GO:1904385; FD  $\mathrm{~R~}=\ 0.00475)$  , neuro genesis (central nervous system development (GO:0007417;  $\mathrm{FDR}=0.0179)$  ), hypoxia (cellular response to hypoxia (GO:0071456;   $\mathrm{FDR}=0.00167)$  ), oxidative stress (cellular response to oxidative stress (GO:0034599;   $\mathrm{FDR}=0.0154)$  ), response to heat (regulation of cellular response to heat (GO:1900034;   $\mathrm{FDR}=0.0492)$  ), cell signaling (signal transduction by  $\mathsf{p}53$   class mediator (GO:0072331;  $\mathrm{FDR}=0.0071$  ), positive regulation of protein binding (GO:0032092;   $\mathrm{FDR}=0.00685)$  ), pro teo stasis (PERK-mediated unfolded protein response (GO:0036499;  $\mathrm{FDR}=0.015)$  ), and apoptosis (positive regulation of neuron apoptotic process (GO:0043525;   $\mathrm{FDR}=0.00157$  ), intrinsic apoptotic signaling pathway (GO:0097193;  $\mathrm{FDR}=0.0462)$  ) (Table S10).  

Lastly, the risk allele A of rs2277947  C19orf53  creates DNA-binding sites for 32 TFs, associated with overrepresented GO such as glial cell fate commitment (GO:0021781;  

$\mathrm{FDR}=0.0235)$  ), the positive regulation of neuron differentiation (GO:0045666;   $\mathrm{FDR}=0.0341$  ), and blood vessel development (GO:0001568;   $\mathrm{FDR}=0.0107$  ) (Table S11).  

3.3.4. Bio in format ic Analysis of the Associations of  C19orf53  SNPs with IS-Related Phenotypes  

According to the bio in format ic tools CDKP and CVDKP, the IS-associated  C19orf53 SNPs are linked with ischemic stroke risk and the severity of IS as well as with obesity and a number of laboratory parameters, characterizing the lipid metabolism (Table  6 ). Particularly noteworthy is the identification of a connection between these SNPs and arterial fibrillation along with indicators of dys lipid emi a (triglyceride-to-HDL ratio, serum ApoB),—significant risk factors for IS.  

Table 6.  Results of bio in format ic analyses for  C19orf53  SNPs using Cerebro vascular and Card i ovas cular Disease Knowledge Portals. 
![](images/92b6a9a1a52aa15305141b1e7db5ec161a57d2d45c932cda24b1e80c4d8bfe4a.jpg)  
1 —Cerebro vascular Disease Knowledge Portal data;  2 —Cardiovascular Disease Knowledge Portal data. Effect alleles are marked in bold.  

# 3.3.5. Protein–Protein Interactions  

Nine proteins having the most prominent interactions with  C19orf53  were found after searching for the primary functional partners of  C19orf53  using the STRING database data (proteins of the first shell of inter actors): POLR2J, C19orf67, CCDC124, COPS9, NDUFA11, NDUFB7, SERF2, RNF181, C9orf16 (PPI enrichment  $p$  -value  $=0.00201$  ) (Supplementary Figure S3, Table S13).  

Together,  C19orf53  and its main functional partners are involved in six biological processes, reflecting predominantly ATP synthesis (for example: «mitochondrial respiratory chain complex I assembly» (GO:0032981,   $\mathrm{FDR}=0.0032)$  , «mitochondrial elec- tron transport, NADH to ubiquinone» (GO:0006120,   $\mathrm{FDR}\,=\,0.0121)$  ); «proton motive force-driven mitochondrial ATP synthesis» (GO:0042776,   $\mathrm{FDR}=0.0032)$  ); pro teo stasis

 («proteasome-mediated ubiquitin-dependent protein catabolic process» (GO:0043161; FDR

  $=0.00083)$  ); and metabolic pathways («nucleobase-containing compound bio synthetic process» (GO:0034654;  $\mathrm{FDR}=0.0038)$  ), «nitrogen compound metabolic process» (GO:0006807;  $\mathrm{FDR}=0.0354)$  ) (Table S14).  

# 4. Discussion  

Here, providing genetic evidence that SNPs rs10104, rs11666524, rs346157, rs346158, and rs2277947  C19orf53  are associated with IS risk, we report that  C19orf53  is involved in the pathogen es is of stroke. In particular, BMI, physical activity, and the consumption of fruit and vegetables act as modifiers of the associations of SNPs rs10104, rs11666524, rs346157, rs346158, and rs2277947 (Figure  4 ).  

![](images/597cddf651d6e392fc2049e167faa31bfdcaf99f2cd23610f315b3198ceaf951.jpg)  
Figure 4.  The interplay of polymorphic variants of the  C19orf53  gene and environmental factors in the risk of IS development.  the risk of IS development.  

Moreover, genotypes A/A-A/G vs. G/G for rs346157 exhibited longer activated partial th rom bop last in time (APTT) in patients with insufficient fruit and vegetable intake, and genotypes G/G-G/A vs. A/A for rs2277947 were associated with a higher BMI in non-smokers and patients with low fruit and vegetable intake. Interestingly, another Hero protein member, SERBP1, has also been associated with shorter APTT and a higher BMI [ A reduction in APTT, which measures the time required for fibrin formation, is associated  

with an elevated risk of ischemic stroke, greater disease severity, and poorer outcomes [ 38 ]. Thus, rs346157 may be linked to excessive thrombosis.  

SNP rs10104 A/G is a missense variant leading to the substitution of Lys39Arg at codon 39 of the C19orf53 protein. It is noteworthy that rs10104 is an MRI1 Downstream Variant, significantly affecting the expression of  C19orf53  (Tables  4  and  5 ). SNPs rs11666524, rs346157, and rs346158 localized in introns, and rs2277947 is a  $3^{\prime}$  prime UTR variant.  

Due to the fact that the use of PubMed and Google Scholar search resources did not allow us to detect studies aimed at investigating the associations of the  C19orf53  SNPs with the development of IS/cardiovascular diseases, we used the bioinformatics analysis to interpret the effects of these genetic variants. Several compelling factors underscore IS-associated SNPs significance in unraveling the mechanisms behind IS.  

# 4.1. C19orf53 SNPs and the IS Risk: Underlying Mechanisms  

Based on cis-eQTL effects and the analysis of transcription factors of  C19orf53  SNPs, associated with IS risk, we distinguished the main mechanisms contributing to disease development (Figure  5 ).  

![](images/0dcff642f1cd82ada557fbe48347385689fc419f660845515822764e3cf03f4d.jpg)  
Figure 5.  The effects of  C19orf53  SNPs associated with IS.  

The IS-associated   C19orf53  SNPs exhibit a high level of regulatory potential, exerting  profound effects on histone modifications within brain and blood tissues. Notably, they  prominently influence histone H3 by inducing tri methyl ation at the 4th lysine residue,  they also impact the H3K27ac. and they also impact the  H3K27ac .  

Furthermore, it is worth highlighting that the risk alleles of the identified SNPs, which Furthermore, it is worth highlighting that the risk alleles of the identified SNPs,  were found to correlate significantly with IS in our study, exhibit associations beyond IS alone, which were found to correlate significantly with IS in our study, exhibit associations beaccording to the Cerebro vascular Disease Knowledge Portal. Specifically, these SNPs are associated with an increased risk of all ischemic strokes, as indicated by rs10104, rs11666524, rs346157, rs346158, and rs2277947. Moreover, rs10104, rs11666524, rs346158, and rs2277947 also play a role in determining the severity of stroke, as evidenced by their correlation with modified Rankin scale scores. Additionally, SNPs rs10104, rs11666524, rs346158, and rs2277947 are implicated in atrial fibrillation or flatter, further underscoring their relevance in IS pathogen es is [ 39 ]. Complementary, data collected from the Cardiovascular Disease Knowledge Portal reveal that protective alleles of  C19orf53  SNPs exhibit a distinct influence on lipid metabolism, increasing triglyceride to HDL ratio and lowering serum ApoB levels, all of which are critical factors in atherosclerosis and IS development [ 40 ].  

In ischemic conditions, established molecular mechanisms and cellular processes, including ex ci to toxicity, oxidative and nitrative stress, inflammation, and apoptosis, contribute to cell death [ 41 ]. Recognizing the potential involvement of  C19orf53  polymorphisms in these path o physiological processes becomes crucial. The IS-associated SNPs of  C19orf53 , which display complex interactions with TFs, are providing insights into their involvement in these important cellular processes. Notably, the risk allele G of rs10104  C19orf53  generates binding sites for TFs involved in inflammation, neuro genesis, and the regulation of endo the li al cell proliferation. Similarly, the risk allele A of rs11666524  C19orf53  creates binding sites for TFs primarily associated with neuron differentiation and astrocyte fate commitment. Moreover, the risk allele G of rs346157  C19orf53  provides binding sites for TFs with roles in glial cell proliferation, responses to hypoxia and oxidative stress response, cell signaling, and apoptosis, while its protective A allele introduces binding sites for TFs associated with vasculo- and neuro genesis, the regulation of the vascular tone, hypoxia and oxidative stress response, heat response, cell signaling, pro teo stasis, and apoptosis. It is worth noting that TFs binding to the DNA site created by protective allele A rs346157 participate in the PERK-mediated pathway. This pathway is involved in protecting the brain from an ischemia–reperfusion injury by inhibiting ER stress-induced apoptosis [ 42 ]. Additionally, the risk allele A of rs2277947  C19orf53  is linked to TFs associated with vasculoand neuro genesis. In summary, these allele-specific binding sites highlight  C19orf53 ’s involvement in pivotal biological processes related to neuron development and different i ation, vas cul o genesis, the regulation of vascular tone, and oxidative stress response as well as response to hypoxia, response to heat, pro teo stasis, inflammation, and the regulation of apoptosis pathways, playing a pivotal role in the molecular pathogen es is of IS [ 43 , 44 ]. Consequently, we can hypothesize that  C19orf53  not only increases the risk of an IS, as established by our genetic analysis, but is also involved in the complex mechanisms governing the survival of neurons, glia, and endo the li al cells by means of gains and losses of binding sites for TFs.  

To further expand on the involvement of C19orf53 within cellular functions as well as path o physiological processes, we analyzed its protein–protein interaction network, shedding light on its principal functional partners and their critical roles. First, C19orf53 and its principal functional partners participate in processes related to cellular energy production, such as “mitochondrial respiratory chain complex I assembly”, “mitochondrial electron transport, NADH to ubiquinone”, and “proton motive force-driven mitochondrial ATP synthesis”, emphasizing their significance in maintaining energy balance and mitochondrial functionality and potentially in ex ci to toxicity [ 45 ]. Second, their association with “proteasome-mediated ubiquitin-dependent protein catabolic process” signifies a role in maintaining protein homeostasis by aiding in the degradation of damaged proteins [ 46 ], a pathway triggered by ischemia [ 47 ]. Furthermore, proteins’ involvement in “nucleobasecontaining compound bio synthetic process” underscores their contributions to nucleotide and nucleic acid metabolism, crucial for DNA and RNA synthesis, ensuring proper cellular function. It is worth noting that C19orf53 interacts with C9orf16 and SERF2, both correlated with IS risk in our previous studies [ 20 , 21 ]. Collectively, these insights indicate that C19orf53 and its functional partners are involved in path o physiological cellular processes, thereby contributing to the underlying mechanisms of IS.  

Examining cis-eQTL effects of the IS-associated SNPs, a common theme emerges where these genetic variants (rs10104, rs11666524, rs346157, rs346158, and rs2277947) together contribute to the down regulation of  C19orf53  expression in various tissue types, including blood, brain, and adipose tissues and arteries. Moreover, these SNPs, with the exception of rs346157, exhibit a similar pattern of lowering the expression of the  MRI1  gene in arteries, adipose tissues, and the brain cortex. MRI1 plays a vital role as an enzyme involved in the methionine de novo and salvage pathway as well as the metabolism of sulfur amino acid ( https://www.genecards.org , assessed 25 May 2024). We hypothesize that the reduction in MRI1 production resulting from these SNP-driven alterations may have profound consequences, disrupting these essential metabolic pathways. Notably, the methionine cycle, in which MRI1 plays a role, holds significant importance in stroke and cardiovascular diseases through the regulation of homo cyst eine levels [ 48 ]. Additionally, sulfur-containing amino acids and their derivatives can regulate body weight, ad ipo genesis, lipolysis, and glucose metabolism [ 49 ], providing a link to associations between  C19orf53 SNPs and an increased BMI.  

In contrast,  ZSWIM4  displays an increased expression in blood affected by the risk alleles of rs10104, rs11666524, rs346158, and rs2277947. Expanding on the role of ZSWIM4 in the context of IS, it is important to highlight its involvement in the ubiquitin ation pathway, as demonstrated in studies [ 50 ]. Mechanically, ZSWIM4 has been found to have physical interactions with Smad1, leading to the ubiquitin ation and subsequent degradation of Smad1 [ 51 ]. This interaction holds significance within the broader context of IS, as Smad1 intersects with the regulation of ang io genesis following ischemia/reperfusion. Specifically, the TGFβ 1/ALK1/Smad1/5/8 signaling pathway has been identified as a key player in mediating angiogenic processes in response to ischemic events [ 52 , 53 ]. By influencing the ubiquitin ation and degradation of Smad1, the increased expression of  ZSWIM4,  caused by the effects of  C19orf53  SNPs, in blood may contribute to the down regulation of this critical pathway, potentially impacting the post-ischemic angiogenic response. Conversely, the presence of IS-associated alleles of  C19orf53  is associated with higher expression levels of  CCDC130  in blood. This observation is of particular significance since members of the CCDC protein family, to which  CCDC130  belongs, have been demonstrated to have roles in lipid metabolism and inflammation [ 54 ], atherosclerosis, and thrombus formation [ 55 , 56 ]. This intriguing association suggests a potential link between  C19orf53  and the development of IS through the modulation of  CCDC130  expression and its implications in pathological pathways.  

The analysis of the joint functions of genes in a network of interactions of  C19orf53  and cis-eQTL-related genes showed that their participation in such processes was significant for the development of IS as oli god end ro cyte differentiation, a sulfur amino acid metabolic process. Researchers have already shown the significant role of sulfur amino acids in atherosclerosis [ 57 ], and it was found that sulfur-containing amino acid metabolites are associated with the risk of ischemic stroke and fibrin clot properties [ 58 , 59 ] and affect the concentration of glut at hi one [ 60 ], a powerful antioxidant, the level of which can largely determine the risk of an IS [ 61 ]. Moreover, plasma sulfur amino acids are obesogenic [ 62 ].  

Additionally, to further emphasize  C19orf53 ’s potential impact on the risk of stroke, there exists a significant relationship between  C19orf53  and Heat Shock Factor 1 (HSF1) discovered by the authors [ 16 ]. Notably, experiments involving  HSF1  knockout demonstrated a substantial reduction in  C19orf53  expression, underscoring the regulatory influence of HSF1 on  C19orf53 . HSF1, known as a heat shock factor, plays a pivotal role in orchestrating stress-induced transcription and governing diverse physiological processes within cells. Its central position in maintaining cellular homeostasis is well-established, primarily through its ability to trans activate genes responsible for encoding heat shock proteins (HSPs). This newfound association underscores the implication of C19orf53 in the heat stress response, a pathway of paramount importance in the context of ischemia/reperfusion events and stroke pathogen es is.  

# 4.2. In sufi cie nt Fruit and Vegetable Intake-Linked Associations of C19orf53  

The correlation of SNPs rs10104, rs11666524, rs346157, rs346158, and rs2277947 with IS was significantly modified by the levels of fresh vegetable and fruit consumption. The link between a reduced intake of fresh vegetables and fruits and increased oxidative stress has been well established. Oxidative stress has been extensively studied and demonstrated to play a significant role in the risk of IS and related phenotypes in numerous investigations. Plant-rich diets, typically rich in phenolic acids, flavonoids, and carotenoids, are known for their potent antioxidant properties. These antioxidants effectively combat excessive reactive oxygen species in the body, safeguarding cells from their negative effect and consequently lowering the risk of cardiovascular diseases.  

The analysis of transcript omi c profiling data has unveiled a significant correlation between  C19orf53  and key genes involved in oxidative stress response, shedding light on its potential role in the risk of stroke. Notably,  C19orf53  exhibits strong correlations with  PARK7 , SOD1 , and  ROCK1 , all playing a crucial part in autophagic pro teo lysis and auto phagosome formation [ 63 – 65 ]. PARK7, in particular, is reported as a multifaceted player in the cellular defense against oxidative stress. Acting as a redox-sensitive chaperone and an oxidative stress sensor, PARK7 is instrumental in safeguarding neurons against oxidative stressinduced cells. Moreover, it prevents the formation of advanced glycation end-products, eliminates hydrogen peroxide, and maintains proper mitochondrial functions, including the autophagy of dysfunctional mitochondria. Additionally, PARK7 regulates inflammatory responses in astrocytes and neuronal cells.  SOD1 , another gene strongly correlated with C19orf53 , plays a key role in antioxidant defense [ 66 ]. Its protein product is a scavenger of harmful superoxide radicals by converting them into molecular oxygen and hydrogen peroxide. Lastly,  ROCK1 , while not directly involved in oxidative stress response, acts as a negative regulator of VEGF-induced angiogenic endo the li al cell activation, which can have implications in vascular health and stroke risk. This intricate web of correlations underscores the potential involvement of  C19orf53  in oxidative stress pathways and its connection to the risk of stroke.  

# 4.3. Low Physical Activity-Related Correlates of C19orf53  

Numerous studies have consistently demonstrated the benefits of physical exercise, establishing a positive connection to better heart and vessel health [ 67 ]. Exercising increases the expression levels of HSPs [ 68 ] as well as HSF1 [ 69 ]. As previously mentioned,  HSF1  regulates the expression of  C19orf53 . The up regulation of  C19orf53  levels can yield positive effects, as it participates in stress responses, leading to antioxidant [ 70 ] and anti-inflammatory effects [ 71 , 72 ] in brain tissues. Another protein linked to C19orf53, as discussed earlier, is VEGF. The expression of  VEGF  increases in response to exercise [ 73 – 77 ]. VEGF is significant for neuro vascular integrity, participating in endo the li al cell proliferation and ang io genesis and in securing neuro troph ic, neuro protective, and neurogenic effects [ 78 ]. The interplay of exercise-induced molecular responses involving HSPs, HSF1, C19orf53, and VEGF results in an increased risk of an IS for patients with low physical activity levels carrying rs10104 and rs1166524  C19orf53  SNPs.  

# 4.4. Obesity-Related Correlates of C19orf53  

Obesity had a substantial impact on the relationship between rs11666524, rs346158, and rs2277947 and the risk of an IS. Patients with a BMI of 30 or greater exhibited significant associations. The Bonferroni adjustment leveled associations in the subgroup with  $\mathrm{BM}<30\$  , except for rs10104  C19orf53 , which maintained a weak connection at  $p=0.046$  . First, overweight and obesity have been associated with a gradually greater risk of stroke, as revealed by a vast amount of research [ 79 – 81 ]. A bioinformatics study of PheWAS, carried out with the support of Knowledge Portals resources, indicated the influence of all IS-related SNPs on the abdominal fat ratio; rs346157 also exhibited a substantial risk effect in the phenotype “obese vs. controls” (Table  6 ). Furthermore, our research shows that these SNPs have significant cis-eQTL-mediated effects on both  C19orf53  and  MRI1  expression levels. MRI1 has been found to alter the metabolism of sulfur-containing amino acids and their derivatives, which play important roles in body weight regulation, ad ipo genesis, lipolysis, and glucose metabolism [ 49 ], implying a relationship between the  C19orf53  SNPs and an increased BMI. Additionally, the assessment of SNPs’ histone modifications shows that obesity-related SNPs are characterized by high levels of epigenetic modulation in adipose tissue. Further studies of  C19orf53  SNPs are required, primarily to analyze their associations with the risk of obesity and BMI, which will allow us to determine whether IS-related  C19orf53  SNPs are characterized by pronounced horizontal pleiotropic effects on the development of both IS and obesity or whether their effect on IS is due to mediators’ effects, reflecting their influence on the risk of IS through excess body weight.  

# 5. Conclusions  

The present study provides genetic evidence of associations between  C19orf53  polymorphisms and IS in the Caucasian population of central Russia. The results were supported with a bio in format ic assessment, revealing genome-, epigenome-, and proteome-wide effects of  C19orf53  polymorphisms as well as an established link between SNPs in the C19orf53  gene and cerebro vascular disease-related phenotypes.  

# 6. Limitations of the Study  

Our study has several limitations. First, we did not study the level of  C19orf53 expression, which does not allow us to establish the effect of the studied SNPs on the level of  C19orf53  mRNA. Second, we were limited in our analysis of environmental risk factor scores, especially in the control group, which did not allow us to conduct a model-based multi factor dimensionality reduction (MDR) analysis of gene–environment interactions. Third, we did not replicate our data in another independent population sample.  

Supplementary Materials:  The following supporting information can be downloaded at  https: //www.mdpi.com/article/10.3390/life14091158/s1 , Supplementary Figure S1: Allelic discrimination plots for  C19orf53  assays designed for this study. (A) rs10104  C19orf53.  (B) rs11666524  C19orf53. (C) rs346157 C19orf53. (D) rs346158 C19orf53. (E) rs2277947 C19orf53. (F) rs2901077 C19orf53. (G)rs8107914  C19orf53;  Supplementary Figure S2:  C19orf53  expression levels in vessels, brain, and peripheral blood; Supplementary Figure S3: Predicted interaction partners of  C19orf53 ; Table S1: Primers and probes designed for the study; Table S2: The analysis of the correspondence of the distribution of  C19orf53  genotypes to the Hardy–Weinberg equilibrium; Table S3: The relationship between  C19orf53  gene SNPs and ischemic stroke risk (entire group); Table S4: Smoking, BMI, fruit and vegetable intake, and physical activity level-depending associations between  C19orf53  SNPs and IS risk; Table S5: Statistically significant correlations between  C19orf53  SNPs and clinical parameter IS patients; Table S6: Mechanisms of interactions between  C19orf53  and cis-eQTL-related genes (results of analysis using the GeneMania resource); Table S7: Biological processes characterizing the joint functions of genes in a network of interactions of  C19orf53  and cis-eQTL-related genes; Table S8: Effects of rs10104  C19orf53  on the binding of DNA to TFs; Table S9: Effects of rs11666524  C19orf53  on the binding of DNA to TFs; Table S10: Effects of rs346157  C19orf53  on the binding of DNA to TFs; Table S11: Effects of rs2277947  C19orf53  on the binding of DNA to TFs; Table S12: Effects of rs346158 C19orf53  on the binding of DNA to TFs; Table S13: Basic molecular characteristics of predicted functional partners of  C19orf53;  Table S14: Functional enrichment s of C19orf53 network.  

Author Contributions:  Conceptualization, O.B.; methodology, O.B.; validation, A.D., V.P. and E.P.; formal analysis, I.S. and A.D.; investigation, I.S., V.P. and E.P.; data curation, O.B.; writing—original draft preparation, I.S., K.K. and O.B.; writing—review and editing, I.S. and O.B.; visualization, K.K.; supervision, O.B.; project administration, O.B.; funding acquisition, A.D. and O.B. All authors have read and agreed to the published version of the manuscript.  

Funding:  This study was funded by Russian Science Foundation   $\mathrm{\mathbf{N}^{\circ}}$  22-15-00288,  https://rscf.ru/ en/project/22-15-00288/ ).  

Institutional Review Board Statement:  The study was conducted according to the guidelines of the Declaration of Helsinki and was approved by the Ethical Review Committee of Kursk State Medical University, Russia (Protocol No. 12, accessed on 7 December 2015). All the participants gave written informed consent before the enrollment in this study.  

Informed Consent Statement:  Informed consent was obtained from all subjects involved in the study.  

Data Availability Statement:  The data presented in this study are available upon request from corresponding author.  

Conflicts of Interest:  The authors declare no conflicts of interest.  

# References  

1. Saini, V.; Guada, L.; Yavagal, D.R. Global Epidemiology of Stroke and Access to Acute Ischemic Stroke Interventions.  Neurology 2021 ,  97 , S6–S16. [ CrossRef ] [ PubMed ]

 2. Koyama, R.; Shichita, T. Glial Roles in Sterile Inflammation after Ischemic Stroke.  Neurosci. Res.  2023 ,  187 , 67–71. [ CrossRef ] [ PubMed ]

 3. Candelario-Jalil, E.; Dijkhuizen, R.M.; Magnus, T. Neuro inflammation, Stroke, Blood-Brain Barrier Dysfunction, and Imaging Modalities.  Stroke  2022 ,  53 , 1473–1486. [ CrossRef ] [ PubMed ]

 4. Giffard, R.G.; Yenari, M.A. Many Mechanisms for Hsp70 Protection From Cerebral Ischemia.  J. Neurosurg. An es the sio l.  2004 ,  16 , 53. [ CrossRef ]

 5. Kobzeva, K.A.; Soldatova, M.O.; Stetskaya, T.A.; Soldatov, V.O.; Deykin, A.V.; Freidin, M.B.; Bykanova, M.A.; Churnosov, M.I.; Polonikov, A.V.; Bushueva, O.Y. Association between HSPA8 Gene Variants and Ischemic Stroke: A Pilot Study Providing Additional Evidence for the Role of Heat Shock Proteins in Disease Pathogen es is.  Genes  2023 ,  14 , 1171. [ CrossRef ]

 6. Truettner, J.S.; Hu, K.; Liu, C.L.; Dietrich, W.D.; Hu, B. Subcellular Stress Response and Induction of Molecular Chaperones and Folding Proteins after Transient Global Ischemia in Rats.  Brain Res.  2009 ,  1249 , 9–18. [ CrossRef ]

 7. Venediktov, A.A.; Bushueva, O.Y.; Ku dry a vt seva, V.A.; Kuzmin, E.A.; Moiseeva, A.V.; Baldycheva, A.; Meglinski, I.; Piavchenko, G.A. Closest Horizons of Hsp70 Engagement to Manage Neuro degeneration.  Front. Mol. Neurosci.  2023 ,  16 , 1230436. [ CrossRef ]

 8. Jiang, J.; Cyr, D.; Babbitt, R.W.; Sessa, W.C.; Patterson, C. Chaperone-Dependent Regulation of Endo the li al Nitric-Oxide Synthase Intracellular Trafficking by the Co-Chaperone/Ubiquitin Ligase CHIP.  J. Biol. Chem.  2003 ,  278 , 49332–49341. [ CrossRef ]

 9. Bushueva, O.; Solodilova, M.; Ivanov, V.; Polonikov, A. Gender-Specific Protective Effect of the  − 463G>A Polymorphism of My elo per oxidase Gene against the Risk of Essential Hypertension in Russians.  J. Am. Soc. Hypertens.  2015 ,  9 , 902–906. [ CrossRef ]

 10. Bushueva, O. Single Nucleotide Polymorphisms in Genes Encoding Xenobiotic Metabolizing Enzymes Are Associated with Predisposition to Arterial Hypertension.  Res. Results Biomed.  2020 ,  6 , 447–456. [ CrossRef ]

 11. Madrigal-Matute, J.; de Bruijn, J.; van Kuijk, K.; Riascos-Bernal, D.F.; Diaz, A.; Tasset, I.; Mart í n-Segura, A.; Gijbels, M.J.J.; Sander, B.; Kaushik, S.; et al. Protective Role of Chaperone-Mediated Autophagy against Atherosclerosis.  Proc. Natl. Acad. Sci. USA  2022 , 119 , e2121133119. [ CrossRef ] [ PubMed ]

 12. Xu, Q.; Metzler, B.; Jahangiri, M.; Mandal, K. Molecular Chaperones and Heat Shock Proteins in Atherosclerosis.  Am. J. Physiol.-Heart Circ. Physiol.  2012 ,  302 , H506–H514. [ CrossRef ] [ PubMed ]

 13. Sorokin, A.; Kotani, K.; Bushueva, O.; Taniguchi, N.; Lazarenko, V. The Cardio-Ankle Vascular Index and Ankle-Brachial Index in Young Russians.  J. A the ros c ler. Thromb.  2015 ,  22 , 211–218. [ CrossRef ] [ PubMed ]

 14.Staron, M.; Wu, S.; Hong, F.; Stojanovic, A.; Du, X.; Bona, R.; Liu, B.; Li, Z. Heat-Shock Protein Gp96/Grp94 Is an EssentialChaperone for the Platelet Glycoprotein Ib-IX-V Complex.  Blood J. Am. Soc. Hematol.  2011 ,  117 , 7136–7144. [ CrossRef ]

 15. Nosaka, M.; Ishida, Y.; Kuninaka, Y.; Ishigami, A.; Taruya, A.; Shimada, E.; Hashizume, Y.; Yamamoto, H.; Kimura, A.; Furukawa, F. Relationship between Intra thr ombo tic Appearance of HSP27 and HSP70 and Thrombus Ages in a Murine Model of Deep Vein Thrombosis.  Sci. Rep.  2023 ,  13 , 22416. [ CrossRef ]

 16.Tang, Y.; Liao, S.; Liu, G.; Xiong, X.; Liu, H.; Li, F.; Tan, Z.; Kong, X.; Yin, Y.; Tan, B. Advanced Single-Cell Pooled CRISPRScreening Identifies C19orf53 Required for Cell Proliferation Based on mTORC1 Regulators.  Cell Biol. Toxicol.  2022 ,  38 , 43–68. [ CrossRef ]

 17. Zhang, Q.; Burdette, J.E.; Wang, J.-P. Integrative Network Analysis of TCGA Data for Ovarian Cancer.  BMC Syst. Biol.  2014 , 8 , 1338. [ CrossRef ]

 18. Kong, W.; Hayashi, T.; Fiches, G.; Xu, Q.; Li, M.Z.; Que, J.; Liu, S.; Zhang, W.; Qi, J.; Santoso, N. Diversified Application of Barcoded PLATO (PLATO-BC) Platform for Identification of Protein Interactions.  Genom. Proteom. Bioinform.  2019 ,  17 , 319–331. [ CrossRef ]

 19. Tsuboyama, K.; Osaki, T.; Matsuura-Suzuki, E.; Kozuka-Hata, H.; Okada, Y.; Oyama, M.; Ikeuchi, Y.; Iwasaki, S.; Tomari, Y. A Widespread Family of Heat-Resistant Obscure (Hero) Proteins Protect against Protein Instability and Aggregation.  PLoS Biol. 2020 ,  18 , e3000632. [ CrossRef ] [ PubMed ]

 20. Kobzeva, K.A.; Shilenok, I.V.; Belykh, A.E.; Gurtovoy, D.E.; Bobyleva, L.A. C9orf16 (BBLN) Gene, Encoding a Member of Hero Proteins, Is a Novel Marker in Ischemic Stroke Risk.  Res. Results Biomed.  2022 ,  8 , 278–292. [ CrossRef ]  

21. Belykh, A.E.; Soldatov, V.O.; Stetskaya, T.A.; Kobzeva, K.A.; Soldatova, M.O.; Polonikov, A.V.; Deykin, A.V.; Churnosov, M.I.; Freidin, M.B.; Bushueva, O.Y. Polymorphism of SERF2, the Gene Encoding a Heat-Resistant Obscure (Hero) Protein with Chaperone Activity, Is a Novel Link in Ischemic Stroke.  IBRO Neurosci. Rep.  2023 ,  14 , 453–461. [ CrossRef ] [ PubMed ]

 22. Shilenok, I.; Kobzeva, K.; Stetskaya, T.; Freidin, M.; Soldatova, M.; Deykin, A.; Soldatov, V.; Churnosov, M.; Polonikov, A.; Bushueva, O. SERPINE1 mRNA Binding Protein 1 Is Associated with Ischemic Stroke Risk: A Comprehensive Molecular–Genetic and Bioinformatics Analysis of SERBP1 SNPs.  Int. J. Mol. Sci.  2023 ,  24 , 8716. [ CrossRef ] [ PubMed ]

 23. Lam, A.Y.W.; Tsuboyama, K.; Tadakuma, H.; Tomari, Y. DNAJA2 and Hero11 Mediate Similar Conformation al Extension and Aggregation Suppression of TDP-43.  RNA  2024 ,  8 , rna.080165.124. [ CrossRef ] [ PubMed ]

 24. Stetskaya, T.A.; Kobzeva, K.A.; Zaytsev, S.M.; Shilenok, I.V.; Komkova, G.V.; Goryainova, N.V.; Bushueva, O.Y. HSPD1 Gene Polymorphism Is Associated with an Increased Risk of Ischemic Stroke in Smokers.  Res. Results Biomed.  2024 ,  10 , 175–186. [ CrossRef ]

 25. Bushueva, O.Y.; Bulgakova, I.V.; Ivanov, V.P.; Polonikov, A.V. Association of Flavin Mono oxygen as e Gene E158K Polymorphism with Chronic Heart Disease Risk.  Bull. Exp. Biol. Med.  2015 ,  159 , 776–778. [ CrossRef ]

 26. Vialykh, E.K.; Solidolova, M.A.; Bushueva, O.I.; Bulgakova, I.V.; Polonikov, A.V. [Catalase gene polymorphism is associated with increased risk of cerebral stroke in hypertensive patients].  Zhurnal Nevrol. P sikh i atri i Im. S.S. Korsakova  2012 ,  112 , 3–7.

 27. Polonikov, A.; Vialykh, E.; Vasil’eva, O.; Bulgakova, I.; Bushueva, O.; Illig, T.; Solodilova, M. Genetic Variation in Glut at hi one S-Transfer as e Genes and Risk of Nonfatal Cerebral Stroke in Patients Suffering from Essential Hypertension.  J. Mol. Neurosci. 2012 ,  47 , 511–513. [ CrossRef ]

 28. Koressaar, T.; Remm, M. Enhancements and Modifications of Primer Design Program Primer3.  Bioinformatics  2007 ,  23 , 1289–1291. [ CrossRef ]

 29. Johnson, J.L.; Abecasis, G.R. GAS Power Calculator: Web-Based Power Calculator for Genetic Association Studies.  BioRxiv  2017 . [ CrossRef ]

 30. THE GTEX CONSORTIUM. The GTEx Consortium Atlas of Genetic Regulatory Effects across Human Tissues.  Science  2020 ,  369 , 1318–1330. [ CrossRef ]

 31. V õ sa, U.; Clarin g bou ld, A.; Westra, H.-J.; Bonder, M.J.; Deelen, P.; Zeng, B.; Kirsten, H.; Saha, A.; Kreuzhuber, R.; Kasela, S. Unraveling the Polygenic Architecture of Complex Traits Using Blood eQTL Meta analysis.  BioRxiv  2018 , 447367. [ CrossRef ]

 32. Ward, L.D.; Kellis, M. HaploReg: A Resource for Exploring Chromatin States, Conservation, and Regulatory Motif Alterations within Sets of Genetically Linked Variants.  Nucleic Acids Res.  2012 ,  40 , D930–D934. [ CrossRef ] [ PubMed ]

 33. von Mering, C.; Jensen, L.J.; Snel, B.; Hooper, S.D.; Krupp, M.; Foglierini, M.; Jouffre, N.; Huynen, M.A.; Bork, P. STRING: Known and Predicted Protein–Protein Associations, Integrated and Transferred across Organisms.  Nucleic Acids Res.  2005 ,  33 , D433–D437. [ CrossRef ]

 34. Franz, M.; Rodriguez, H.; Lopes, C.; Zuberi, K.; Montojo, J.; Bader, G.D.; Morris, Q. GeneMANIA Update 2018.  Nucleic Acids Res. 2018 ,  46 , W60–W64. [ CrossRef ]

 35. Shin, S.; Hudson, R.; Harrison, C.; Craven, M.; Kele¸s, S. atSNP Search: A Web Resource for Statistically Evaluating Influence of Human Genetic Variation on Transcription Factor Binding.  Bioinformatics  2019 ,  35 , 2657–2659. [ CrossRef ]

 36. Consortium, G.O. The Gene Ontology Resource: 20 Years and Still GOing Strong.  Nucleic Acids Res.  2019 ,  47 , D330–D338. [ CrossRef ]

 37. Crawford, K.M.; Gallego-Fabrega, C.; Kourkoulis, C.; Miyares, L.; Marini, S.; Flannick, J.; Burtt, N.P.; von Grotthuss, M.; Alexander, B.; Costanzo, M.C.; et al. Cerebro vascular Disease Knowledge Portal.  Stroke  2018 ,  49 , 470–475. [ CrossRef ]

 38. Lin, C.-H.; Kuo, Y.-W.; Kuo, C.-Y.; Huang, Y.-C.; Hsu, C.-Y.; Hsu, H.-L.; Lin, Y.-H.; Wu, C.-Y.; Huang, Y.-C.; Lee, M.; et al. Shortened Activated Partial Th rom bop last in Time Is Associated With Acute Ischemic Stroke, Stroke Severity, and Neurological Worsening.  J. Stroke Cerebro vas c. Dis.  2015 ,  24 , 2270–2276. [ CrossRef ]

 39. Song, M.-S.; Choi, Y.J.; Kim, H.; Nam, M.J.; Lee, C.; Han, K.; Jung, J.-H.; Park, Y.-G.; Kim, D.-H.; Park, J.-H. Relationship between Blood Pressure Levels and Ischemic Stroke, Myocardial Infarction, and Mortality in Very Elderly Patients Taking Anti hypertensive s: A Nationwide Population-Based Cohort Study.  BMC Geriatr.  2021 ,  21 , 620. [ CrossRef ]

 40. Hoshino, T.; Ishizuka, K.; Toi, S.; Mizuno, T.; Nishimura, A.; Takahashi, S.; Wako, S.; Kitagawa, K. A the rogen ic Dys lipid emi a and Residual Vascular Risk After Stroke or Transient Ischemic Attack.  Stroke  2022 ,  53 , 79–86. [ CrossRef ]

 41. Khoshnam, S.E.; Winlow, W.; Farzaneh, M.; Farbood, Y.; Moghaddam, H.F. Pathogenic Mechanisms following Ischemic Stroke. Neurol. Sci.  2017 ,  38 , 1167–1186. [ CrossRef ] [ PubMed ]

 42. Hu, Y.-Q.; Chen, W.; Yan, M.-H.; Lai, J.-J.; Tang, N.; Wu, L. Ischemic Preconditioning Protects Brain from Ischemia/Reperfusion Injury by Attenuating Endoplasmic Reticulum Stress-Induced Apoptosis through PERK Pathway.  Eur. Rev. Med. Pharmacol. Sci. 2017 ,  21 , 5736–5744. [ PubMed ]

 43. Bushueva, O.; Barysheva, E.; Markov, A.; Belykh, A.; Koroleva, I.; Churkin, E.; Polonikov, A.; Ivanov, V.; Nazarenko, M. DNA Hypo methyl ation of the MPO Gene in Peripheral Blood Leukocytes Is Associated with Cerebral Stroke in the Acute Phase.  J. Mol. Neurosci.  2021 ,  71 , 1914–1932. [ CrossRef ]

 44. Soldatov, V.O. Hypoxia-Inducible Factor: Basic Biology and Involvement in Cardiovascular Pathology.  Asian J. Pharm. (AJP) 2018 ,  12 . [ CrossRef ]

 45. Doyle, K.P.; Simon, R.P.; Stenzel-Poore, M.P. Mechanisms of Ischemic Brain Damage.  Neuro pharmacology  2008 ,  55 , 310–318. [ CrossRef ] [ PubMed ]  

46. Jariel-Encontre, I.; Bossis, G.; Piechaczyk, M. Ubiquitin-Independent Degradation of Proteins by the Proteasome.  Biochim. Biophys. Acta (BBA)-Rev. Cancer  2008 ,  1786 , 153–177. [ CrossRef ]

 47. Thiebaut, A.M.; Hedou, E.; Marciniak, S.J.; Vivien, D.; Roussel, B.D. Pro teo stasis During Cerebral Ischemia.  Front. Neurosci. 2019 ,  13 . [ CrossRef ]

 48. Guiraud, S.P.; Montoliu, I.; Da Silva, L.; Dayon, L.; Galindo, A.N.; Corth é sy, J.; Kussmann, M.; Martin, F.-P. High-Throughput and Simultaneous Quantitative Analysis of Homo cyst eine–Methionine Cycle Metabolites and Co-Factors in Blood Plasma and Cerebrospinal Fluid by Isotope Dilution LC–MS/MS.  Anal. Bioanal. Chem.  2017 ,  409 , 295–305. [ CrossRef ]

 49. Zheng, J.; Xiao, H.; Duan, Y.; Song, B.; Zheng, C.; Guo, Q.; Li, F.; Li, T. Roles of Amino Acid Derivatives in the Regulation of Obesity.  Food Funct.  2021 ,  12 , 6214–6225. [ CrossRef ]

 50. Forde, N.; Duffy, G.B.; McGettigan, P.A.; Browne, J.A.; Mehta, J.P.; Kelly, A.K.; Mansouri-Attia, N.; Sandra, O.; Loftus, B.J.; Crowe, M.A.; et al. Evidence for an Early Endometrial Response to Pregnancy in Cattle: Both Dependent upon and Independent of Interferon Tau.  Physiol. Genom.  2012 ,  44 , 799–810. [ CrossRef ]

 51.Wang, C.; Liu, Z.; Zeng, Y.; Zhou, L.; Long, Q.; Hassan, I.U.; Zhang, Y.; Qi, X.; Cai, D.; Mao, B.; et al. ZSWIM4 RegulatesEmbryonic Patterning and BMP Signaling by Promoting Nuclear Smad1 Degradation.  EMBO Rep  2024 ,  25 , 646–671. [ CrossRef ] [ PubMed ]

 52. Du, J.; Yin, G.; Hu, Y.; Shi, S.; Jiang, J.; Song, X.; Zhang, Z.; Wei, Z.; Tang, C.; Lyu, H. Coicis Semen Protects against Focal Cerebral Ischemia-Reperfusion Injury by Inhibiting Oxidative Stress and Promoting Ang io genesis via the TGF β /ALK1/Smad1/5 Signaling Pathway.  Aging  2020 ,  13 , 877–893. [ CrossRef ] [ PubMed ]

 53. Gonz á lez-N ú ñez, M.; Muñoz-F é lix, J.M.; L ó pez-Novoa, J.M. The ALK-1/Smad1 Pathway in Cardiovascular Ph y sio pathology. A New Target for Therapy?  Biochim. Biophys. Acta  2013 ,  1832 , 1492–1510. [ CrossRef ] [ PubMed ]

 54. Azad, A.K.; Chakra bart i, S.; Xu, Z.; Davidge, S.T.; Fu, Y. Coiled-Coil Domain Containing 3 (CCDC3) Represses Tumor Necrosis Factorα /Nuclear Factor  κ B-Induced Endo the li al Inflammation.  Cell. Signal.  2014 ,  26 , 2793–2800. [ CrossRef ]

 55. Kobayashi, S.; Fukuhara, A.; Taguchi, T.; Matsuda, M.; Tochino, Y.; Otsuki, M.; Shimomura, I. Identification of a New Secretory Factor, CCDC3/Favine, in Adipocytes and Endo the li al Cells.  Biochem. Biophys. Res. Commun.  2010 ,  392 , 29–35. [ CrossRef ]

 56. Kobayashi, S.; Kita, S.; Okuzaki, D.; Fujishima, Y.; Otsuki, M.; Kato, H.; Nishizawa, Y.; Miyashita, K.; Yokoyama, C.; Fukuhara, A. Favine/CCDC3 Deficiency Accelerated Atherosclerosis and Thrombus Formation Is Associated with Decreased MEF2C-KLF2 Pathway.  iScience  2022 ,  25 , 105252. [ CrossRef ]

 57. Selhub, J.; Troen, A.M. Sulfur Amino Acids and Atherosclerosis: A Role for Excess Dietary Methionine.  Ann. N. Y. Acad. Sci.  2016 , 1363 , 18–25. [ CrossRef ]

 58. Vinknes, K.J.; Refsum, H.; Turner, C.; Khaw, K.-T.; Wareham, N.J.; Forouhi, N.G.; Imamura, F. Plasma Sulfur Amino Acids and Risk of Cerebro vascular Diseases: A Nested Case-Control Study in the EPIC-Norfolk Cohort.  Stroke  2021 ,  52 , 172–180. [ CrossRef ]

 59. Sikora, M.; Bretes, E.; Perła-Kaj á n, J.; Utyro, O.; Borowczyk, K.; Piechocka, J.; Głowacki, R.; Wojtasz, I.; Ka´zmierski, R.; Jakubowski, H. Homo cyst eine Thi ol act one and Other Sulfur-Containing Amino Acid Metabolites Are Associated with Fibrin Clot Properties and the Risk of Ischemic Stroke.  Sci. Rep.  2024 ,  14 , 11222. [ CrossRef ]

 60. Paterson, P.G.; Lyon, A.W.; Kamencic, H.; Andersen, L.B.; Juurlink, B.H. Sulfur Amino Acid Deficiency Depresses Brain Glut at hi one Concentration.  Nutr. Neurosci.  2001 ,  4 , 213–222. [ CrossRef ]

 61. Solodilova, M.; Drozdova, E.; Azarova, I.; Klyosova, E.; Bykanova, M.; Bushueva, O.; Polonikova, A.; Churnosov, M.; Polonikov, A. The Discovery of GGT1 as a Novel Gene for Ischemic Stroke Conferring Protection against Disease Risk in Non-Smokers and Non-Abusers of Alcohol.  J. Stroke Cerebro vas c. Dis.  2024 ,  33 , 107685. [ CrossRef ] [ PubMed ]

 62. Elshorbagy, A.K.; Kozich, V.; Smith, A.D.; Refsum, H. Cysteine and Obesity: Consistency of the Evidence across Epidemiologic, Animal and Cellular Studies.  Curr. Opin. Clin. Nutr. Metab. Care  2012 ,  15 , 49–57. [ CrossRef ] [ PubMed ]

 63. Lee, D.-H.; Kim, D.; Kim, S.T.; Jeong, S.; Kim, J.L.; Shim, S.M.; Heo, A.J.; Song, X.; Guo, Z.S.; Bartlett, D.L. PARK7 Modulates Autophagic Pro teo lysis through Binding to the N-Terminally Arg in y late d Form of the Molecular Chaperone HSPA5.  Autophagy 2018 ,  14 , 1870–1885. [ CrossRef ] [ PubMed ]

 64. Rudnick, N.D.; Griffey, C.J.; Guarnieri, P.; Gerbino, V.; Wang, X.; Piersaint, J.A.; Tapia, J.C.; Rich, M.M.; Maniatis, T. Distinct Roles for Motor Neuron Autophagy Early and Late in the SOD1G93A Mouse Model of ALS.  Proc. Natl. Acad. Sci. USA  2017 ,  114 , E8294–E8303. [ CrossRef ] [ PubMed ]

 65. Gurkar, A.U.; Chu, K.; Raj, L.; Bouley, R.; Lee, S.-H.; Kim, Y.-B.; Dunn, S.E.; Mandinova, A.; Lee, S.W. Identification of ROCK1 Kinase as a Critical Regulator of Beclin1-Mediated Autophagy during Metabolic Stress.  Nat. Commun.  2013 ,  4 , 2189. [ CrossRef ]

 66. Azarova, I.E.; Gureeva, A.V.; Postnikova, M.I.; Makarenko, V.V.; Klyosova, E.Y.; Polonikov, A.V. The Link of Single Nucleotide Polymorphism Rs4880 of the SOD2 Gene to the Development of Micro vascular Complications of Type 2 Diabetes Mellitus.  Res. Results Biomed.  2023 ,  9 , 461–473.

 67. Patel, H.; Alkhawam, H.; Madanieh, R.; Shah, N.; Kosmas, C.E.; Vittorio, T.J. Aerobic vs Anaerobic Exercise Training Effects on the Cardiovascular System.  World J. Cardiol.  2017 ,  9 , 134. [ CrossRef ]

 68. Szyller, J.; Bil-Lula, I. Heat Shock Proteins in Oxidative Stress and Ischemia/Reperfusion Injury and Benefits from Physical Exercises: A Review to the Current Knowledge.  Oxidative Med. Cell. Longev.  2021 ,  2021 , 6678457. [ CrossRef ]

 69. Koch an owicz, A.; Sawczyn, S.; Niespodzi´nski, B.; Mieszko w ski, J.; Koch an owicz, K.;  Zychowska, M. Cellular Stress Response Gene Expression During Upper and Lower Body High Intensity Exercises.  PLoS ONE  2017 ,  12 , e0171247. [ CrossRef ]  

70. Kalmar, B.; Greensmith, L. Induction of Heat Shock Proteins for Protection against Oxidative Stress.  Adv. Drug Deliv. Rev.  2009 , 61 , 310–318. [ CrossRef ]

 71. Yenari, M.A.; Liu, J.; Zheng, Z.; Vexler, Z.S.; Lee, J.E.; Giffard, R.G. Anti a pop to tic and Anti-inflammatory Mechanisms of Heat-shock Protein Protection.  Ann. N. Y. Acad. Sci.  2005 ,  1053 , 74–83. [ PubMed ]

 72. Chen, Y.; Voegeli, T.S.; Liu, P.P.; Noble, E.G.; Currie, R.W. Heat Shock Paradox and a New Role of Heat Shock Proteins and Their Receptors as Anti-Inflammation Targets.  Inflamm. Allergy-Drug Targets (Former. Curr. Drug Targets-Inflamm. Allergy)(Discontin.) 2007 ,  6 , 91–100.

 73. Cotman, C.W.; Berchtold, N.C.; Christie, L.-A. Exercise Builds Brain Health: Key Roles of Growth Factor Cascades and Inflammation.  Trends Neurosci.  2007 ,  30 , 464–472. [ CrossRef ] [ PubMed ]

 74. Taylor, J.M.; Montgomery, M.H.; Gregory, E.J.; Berman, N.E. Exercise Preconditioning Improves Traumatic Brain Injury Outcomes. Brain Res.  2015 ,  1622 , 414–429. [ CrossRef ]

 75. Carro, E.; Nuñez, A.; Busiguina, S.; Torres-Aleman, I. Circulating Insulin-like Growth Factor I Mediates Effects of Exercise on the Brain.  J. Neurosci.  2000 ,  20 , 2926–2933. [ CrossRef ]

 76. Zhang, Q.; Zhang, L.; Yang, X.; Wan, Y.; Jia, J. The Effects of Exercise Preconditioning on Cerebral Blood Flow Change and Endothelin-1 Expression after Cerebral Ischemia in Rats.  J. Stroke Cerebro vas c. Dis.  2014 ,  23 , 1696–1702. [ CrossRef ]

 77. Otsuka, S.; Sakakima, H.; Sumizono, M.; Takada, S.; Terashi, T.; Yoshida, Y. The Neuro protective Effects of Preconditioning Exercise on Brain Damage and Neuro troph ic Factors after Focal Brain Ischemia in Rats.  Behav. Brain Res.  2016 ,  303 , 9–18. [ CrossRef ]

 78. Pikula, A.; Beiser, A.S.; Chen, T.C.; Preis, S.R.; Vorgias, D.; DeCarli, C.; Au, R.; Kelly-Hayes, M.; Kase, C.S.; Wolf, P.A. Serum Brain–Derived Neuro troph ic Factor and Vascular Endo the li al Growth Factor Levels Are Associated with Risk of Stroke and Vascular Brain Injury: Framingham Study.  Stroke  2013 ,  44 , 2768–2775. [ CrossRef ]

 79. Strazzullo, P.; D’Elia, L.; Cairella, G.; Garbagnati, F.; Cappuccio, F.P.; Scalfi, L. Excess Body Weight and Incidence of Stroke: Meta-Analysis of Prospective Studies with 2 Million Participants.  Stroke  2010 ,  41 , e418–e426. [ CrossRef ]

 80. Wang, X.; Huang, Y.; Chen, Y.; Yang, T.; Su, W.; Chen, X.; Yan, F.; Han, L.; Ma, Y. The Relationship between Body Mass Index and Stroke: A Systemic Review and Meta-Analysis.  J. Neurol.  2022 ,  269 , 6279–6289. [ CrossRef ]

 81. Shiozawa, M.; Kaneko, H.; Itoh, H.; Morita, K.; Okada, A.; Matsuoka, S.; Kiriyama, H.; Kamon, T.; Fujiu, K.; Michihata, N.; et al. Association of Body Mass Index with Ischemic and Hemorrhagic Stroke.  Nutrients  2021 ,  13 , 2343. [ CrossRef ] [ PubMed ]  

Disclaimer/Publisher’s Note:  The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.  