# COL4A2  is associated with lacunar ischemic stroke and deep ICH  

Meta-analyses among 21,500 cases and 40,600 controls Kristiina Rannikmäe, MD, PhD Vhinoth Siva kumaran Henry Millar Rainer Malik, PhD Christopher D. Anderson, MD, MMSc Mike Chong, MSc Tushar Dave, MS Guido J. Falcone, MD, ScD, MPH Israel Fernandez-Cadenas, PhD Jordi Jimenez-Conde, MD, PhD Arne Lindgren, MD, PhD Joan Montaner, MD, PhD Martin O ’ Donnell, MD, PhD, MRCPI, BCh, BAO Guillaume Paré, MD, MSc, FRCPC Farid Radmanesh, MD, MPH Natalia S. Rost, MD, MPH Agnieszka Slowik, MD, PhD Martin Söderholm, MD Matthew Traylor, PhD Sara L. Pulit, PhD Sudha Seshadri, MD Brad B. Worrall, MD, MSc Daniel Woo, MD Hugh S. Markus, DM, FRCP Braxton D. Mitchell, MPH, PhD Martin Dichgans, MD, PhD Jonathan Rosand, MD, MsC  

# ABSTRACT  

Objective:  To determine whether common variants in familial cerebral small vessel disease (SVD) genes confer risk of sporadic cerebral SVD.  

Methods:  We meta-analyzed genotype data from individuals of European ancestry to determine associations of common single nucleotide polymorphisms (SNPs) in 6 familial cerebral SVD genes

 ( COL4A1 ,  COL4A2 ,  NOTCH3 ,  HTRA1 ,  TREX1 , and  CECR1 ) with intra cerebral hemorrhage (ICH)

 (deep, lobar, all; 1,878 cases, 2,830 controls) and ischemic stroke (IS) (lacunar, card i oem boli c, large vessel disease, all; 19,569 cases, 37,853 controls). We applied data quality filters and set statistical significance thresholds accounting for linkage d is equilibrium and multiple testing.  

Results:  A locus in  COL4A2  was associated (significance threshold  $\mathsf{p}<3.5\times\mathsf{10^{-4}})$   with both lacunar IS (lead SNP rs9515201: odds ratio [OR] 1.17,  $95\%$   confidence interval [CI] 1.11 – 1.24,

  $\mathsf{p}=6.6\mathsf{e}^{2}\times\mathsf{10^{-8}})$   and deep ICH (lead SNP rs4771674: OR 1.28,  $95\%\,{\sf C I}\,1.13–1.44,p=5.76\,\times$   $\underline{{1}}0^{-5}$  . A SNP in  HTRA1  was associated (significance threshold  $\mathsf{p}<5.5\times\mathsf{10^{-4}})$   with lacunar IS

 (rs79043147: OR 1.23,  $95\%$   CI 1.10 – 1.37,  $\mathsf{p}=1.90\times\mathsf{10^{-4}})$   and less robustly with deep ICH. There was no clear evidence for association of common variants in either  COL4A2  or  HTRA1  with non-SVD strokes or in any of the other genes with any stroke phenotype.  

Conclusions:  These results provide evidence of shared genetic determinants and suggest common path oph y sio logic mechanisms of distinct ischemic and hemorrhagic cerebral SVD stroke phenotypes, offering new insights into the causal mechanisms of cerebral SVD.  Neurology ®   2017;89:1829 – 1839  

# GLOSSARY  

${\mathsf{c l}}=$   confidence interval;  eQTL  $=$   expression quantitative trait loci;  GWS  $=$   genome-wide association study;  $\mathsf{I C H}=$   intracerebral hemorrhage;  $\mathtt{I S G C}=$   International Stroke Genetics Consortium;  LD  $=$   linkage d is equilibrium;  ${\tt O R}={\tt}$   odds ratio;  $\mathtt{s v D}=$  small vessel disease;  TOAST  $=$   Trial of Org 10172 in Acute Stroke Treatment;  WMH  $=$   white matter hyper intensities.  

Small vessel diseases (SVDs) of the brain include a subtype that affects the small, deep, penetrating arteries and arterioles in the brain, hereafter referred to as deep cerebral SVD. This deep cerebral SVD is thought to be responsible for most symptomatic lacunar ischemic strokes and deep intra cerebral hemorrhages (ICHs), as well as for substantial cognitive and physical disabilities, Cathie L.M. Sudlow, BMBCh, MSc, DPhil, FRCPE On behalf of the Stroke Genetics Network (SiGN), METASTROKE Collaboration, and International Stroke Genetics Consortium (ISGC)  

Correspondence to Dr. Sudlow: cathie.sudlow@ed.ac.uk  

Supplemental data at Neurology.org  

and to be a major pathologic substrate for brain MRI features, including white matter hyper intensities (WMH) and brain microbleeds.   Increasing evidence supports a distinct vascular pathology for deep cerebral SVD, but our knowledge of the underlying genes and path oph y sio logic mechanisms is limited, and specific treatment strategies are lacking.  

While the genetic determinants of common sporadic forms of cerebral SVD remain largely unknown, mutations in at least 6 genes ( COL4A1 ,  COL4A2 ,  HTRA1 ,  CECR1 , NOTCH3 ,  TREX1 ) are known to cause rare familial forms of deep cerebral SVD.   Such genes may also contain variants conferring risk for sporadic deep cerebral SVD. We previously investigated associations of common variation in the  COL4A1  and  COL4A2  genes with cerebro vascular phenotypes in a collaborative meta-analysis, demonstrating an association between an intronic  COL4A2  locus and sporadic deep ICH, and a suggestive association with other deep cerebral SVD phenotypes (lacunar ischemic stroke and WMH).   The same genetic locus has since been shown to be associated with WMH at genome-wide association study (GWAS) levels of significance.  

We aimed to extend this promising candidate gene approach to assess associations of common variants in all currently known familial deep cerebral SVD genes with stroke and its subtypes, investigating the hypothesis that associations would be specific to the 2 key sporadic deep cerebral SVD stroke phenotypes, lacunar ischemic stroke and deep ICH. We were able to take advantage of the increased sample sizes and more densely imputed genotype data now available through the International Stroke Genetics Consortium (ISGC) (http://www.stroke genetics.org/) and associated collaborative groups.  

METHODS  Identification of participating studies.  We identified most currently available large GWASs of stroke and stroke subtypes in individuals of European ancestry using a network of collaborations associated with the ISGC. – 11   The entire dataset comprised 20 case-control collections including 19,569 ischemic stroke cases and 37,853 controls, with information on Trial of  $\mathrm{Org~}10172$   in Acute Stroke Treatment (TOAST) subtypes (lacunar ischemic stroke, large vessel disease [LVD], cardioembolic),   and 5 case-control collections including 1,878 ICH cases and 2,830 controls, with information on the main ICH subtypes (table 1). For the majority of case collections, population-matched controls were recruited from studies with existing genotyping data (details of case-control collections found in references 7 – 11).  

Individual studies applied quality control measures before providing the data. All data were imputed with the 1000 Genomes Phase 1 reference dataset (or to a merged reference panel including the Genome of the Netherlands) using IMPUTE2 or MACH software 13   and provided with reference to Human Genome reference build 19.  

Data collection.  We collected genotype summary statistics from participating case-control collections for the  COL4A1 , COL4A2 ,  HTRA1 ,  CECR1 ,  NOTCH3 , and  TREX1  genes (en- compassing all known familial deep cerebral SVD genes), including a 10-kbp upstream and downstream flanking region for each gene (table 2).  

We focused on the lacunar ischemic stroke and deep ICH phenotypes, but we also assessed LVD, card i oem boli c and all ischemic stroke for ischemic stroke case-control collections, and lobar ICH and all ICH for ICH case-control collections to show specificity of the association. For each of these phenotypes, we collected summary data from each collection for all directly genotyped or imputed SNPs in genes of interest: SNP reference number and position, allele frequencies, association effect size ( b  coefficient) and its standard error, association    $\mathcal{p}$   value, and imputation quality measure and value.  

Data analysis.  Setting the significance threshold.  To allow multiple testing while accounting for linkage d is equilibrium (LD) between SNPs, we calculated significance  $\mathcal{p}$   values for each gene using a modified version of the Nyholt method (MeffLi), which controls accurately for error rate in evaluations of real and simulated data. – 17   We used the 1000 Genomes CEU dataset 18 SNP genotype information to calculate  $\mathcal{p}$   values for 5 genomic regions, treating the  COL4A1  and  COL4A2  genes as 1 region because they are located in tandem on chromosome 13q34 and share a promoter (table 2).  

Pre – meta-analysis data filtering.  We further filtered the data to include only SNPs with the following attributes: (1) imputation quality  ${\geq}0.3$  from MACH, IMPUTE2, or SNPTEST (because SNPs with very poor imputation quality may yield unreliable associations); (2) minor allele frequency  $\geq\!1\%$   (because we were investigating common SNPs); (3) absolute  $\upbeta$   value  ${<}100{,}000$   (because higher  $\upbeta$   values would generate implausible odds ratios [ORs], suggesting unreliable associations); and (4) biallelic SNPs (because the meta-analyses program could not process multi allelic SNPs).  

Meta-analyses of COL4A1, COL4A2, HTRA1, CECR1, NOTCH3, and TREX1 SNPs for each phenotype.  We metaanalyzed genotype summary data from each contributing casecontrol collection. We assessed associations of COL4A1 , COL4A2 ,  HTRA1 ,  CECR1 ,  NOTCH3 , and  TREX1  SNPs with each of the stroke phenotypes available, both those assumed to represent deep cerebral SVD specifically (lacunar ischemic stroke, deep ICH) and others (LVD ischemic stroke, card i oem boli c ischemic stroke, all ischemic stroke, lobar ICH, all ICH). Our hypothesis was that associations would be specific to (or at least strongest with) deep cerebral SVD phenotypes. We used a fixed-effects inverse-variance – based model in METAL genetic meta-analysis software, weighting the    $\upbeta$   coefficients by their estimated standard errors and generating, for each SNP, the OR per additional minor allele for being a case vs a control.  

Post – meta-analysis data filtering.  After the meta-analyses, we considered SNPs to be associated with the respective  

![](images/4caf802dddeaab31d278f3d7f5d997c1f26f362f0e87143e78614ca6da58d27f.jpg)  

![](images/b1ec9e654b8d309ab7e9e9ca446e7de93072ba29214f3776dc808a1fd353db09.jpg)  

phenotype if the relevant associations passed the relevant modified Nyholt-corrected  $\mathcal{p}$   threshold, were based on data from  ${\geq}50\%$   of cases contributing to the analyses, and did not demonstrate substantial heterogeneity (requiring  $P<50\%$   and  $\rlap/p>$  0.001 from    $\chi^{2}$    test).  

We chose our filtering thresholds from those most commonly used and accepted 6 –   with the aim of ensuring that any associations deemed significant would be based on SNPs with sufficient, reliable, and consistent data.  

Further exploration of associated SNPs.  When  ${>}1$   SNP in the same gene was associated with any given phenotype, we used the 1000 Genomes project CEU population data to investigate the LD between the lead SNP (with the lowest  $\mathcal{P}$   value) and all other associated SNPs for the relevant gene-phenotype association. SNPs in moderate or strong LD (defined respectively as  $r^{2}$  and/or  $\mathrm{D^{\prime}~}{\geq}0.6$   or    ${\geq}0.8$  ) with the lead SNP were considered likely to represent a signal from the same locus.  

We examined associations of all lead SNPs across all casecontrol collections included in the respective meta-analyses and of all lead SNPs with all other phenotypes, comparing findings for deep cerebral SVD stroke phenotypes and non-SVD stroke phenotypes.  

Finally, we sought functional annotation data from the Haploreg version 2 database (http://www.broad institute.org/ mammals/haploreg/haploreg.php),   genotype-tissue expression portal expression quantitative trait loci (eQTL) browser (http:// www.broad institute.org/gtex/), and the RegulomeDB database (http://regulome.stanford.edu/) for all associated SNPs.  

RESULTS  Meta-analyses of COL4A1 , COL4A2 , HTRA1 ,  CECR1 ,  NOTCH3 , and  TREX1  SNPs for each phenotype.  Modified Nyholt significance thresholds for the 5 genomic regions are shown in table 2. Using our preset data filtering criteria, we found associations of 18 SNPs in  COL4A2  with lacunar ischemic stroke, 9 SNPs in  COL4A2  with deep ICH, and 1 SNP in HTRA1  with lacunar ischemic stroke (table 3 and figure 1). Two of the SNPs in the  COL4A2  gene (rs4771674 and rs9515199) were associated with both lacunar ischemic stroke and deep ICH. There were no associations of common SNPs in  COL4A2  or HTRA1  with any of the non cerebral SVD or combined stroke phenotypes or of common variants in COL4A1 ,  CECR1 ,  NOTCH3 , or  TREX1  with any of the stroke phenotypes.  

Further exploration of associated SNPs.  LD between COL4A2 SNPs.  The lead SNPs were rs9515201 for lacunar ischemic stroke (OR per additional A allele

 1.17,   $95\%$   confidence interval [CI] 1.11 – 1.24,  $\mathscr{p}=$   $6.62\ \times\ 10^{-8})$  ) and rs4771674 for deep ICH (OR per additional A allele 1.28,   $95\%$   CI 1.13 – 1.44,  $\mathcal{p}$   $=5.76\times10^{-5})$  ). These 2 lead SNPs are in strong LD with each other   $\left(r^{2}\,=\,0.9/\mathrm{D}^{\prime}\,=\,1\right)$  ), suggesting that this represents the same genetic signal.  

We investigated the LD between the lead SNP rs9515201 (most strongly associated SNP in the locus) and all other associated SNPs in  COL4A2 . Of the other 24 SNPs in  COL4A2  associated with lacunar ischemic stroke and/or deep ICH, 19 were in  

![](images/2a92cfdce2884a2eb747a83c47e805a4b047438fb93d3442932d6f72368d53ad.jpg)  
Abbreviation:  ${\mathsf{S N P}}=$   single nucleotide polymorphism. a  Based on the Human Genome reference build 19.  

moderate to strong LD with the lead SNP, suggesting that their signal may be from the same  COL4A2 locus. The remaining 4 SNPs showed minimal LD with the lead SNP, while data for 1 SNP were not available, suggesting that there might possibly be additional relevant loci (table e-1 at Neurology.org).  

Associations across individual case-control collections in the meta-analyses.  The associations for  COL4A2  SNPs showed minimal to moderate heterogeneity across individual case-control collections in the lacunar ischemic stroke and deep ICH meta-analyses (  ${\mathit{I}}^{2}\;=\;$   $0\%–49\%$  , heterogeneity    $\phi\ =\ 0.7{-}0.01$  ; and    ${\mathit{I}}^{2}\ =$   $0\%–42\%$  , heterogeneity  $\phi\,=\,0.1{-}0.5$   respectively), while the associations across individual collections for rs79043147 in  HTRA1  in the lacunar ischemic stroke meta-analysis showed only minimal heterogeneity (  ${{\mathit{I}}^{2}}\,=\,4\%$  , heterogeneity  $\rlap{\,/}p=0.41\,\$  ), suggesting consistent results (figure e-1). All imputed SNPs showed a good imputation quality of  ${\tt>}0.7$  .  

Associations with other phenotypes of the lead COL4A2 SNPs.  Figure 2 shows association results for the lead COL4A2  SNPs (rs9515201 and rs4771674) associated with lacunar ischemic stroke and deep ICH across all 7 phenotypes assessed. Although rs9515201 was associated only with lacunar ischemic stroke (OR 1.17,   $95\%$   CI 1.11 – 1.24,  $p\,=\,6.62\,\times$   $10^{-8}$  ), there was a suggestive association of similar magnitude with deep ICH (OR 1.21,   $95\%$   CI 1.07 – 1.37,    $\mathit{p}=2.15\times10^{-3})$  ).  $\mathrm{R}s4771674$   was asso- ciated with both cerebral SVD phenotypes: lacunar ischemic stroke (OR 1.14,   $95\%$   CI 1.07 – 2.10,  $\mathscr{p}=$   $1.6\times10^{-5})$   and deep ICH (OR 1.28,  $95\%$   CI 1.13 –

 1.44,  $\rlap/p=5.76\times10^{-5})$  .  

There were no associations with non-SVD stroke or combined SVD and non-SVD phenotypes. ORs for the all ischemic stroke and all ICH phenotypes were intermediate between those for SVD and those for non-SVD phenotypes, suggesting that associations with these combined phenotypes were driven by results for lacunar ischemic stroke and deep ICH.  

Associations with other cerebro vascular phenotypes of the HTRA1 SNP.  Rs79043147 was associated only with lacunar ischemic stroke (OR 1.23,   $95\%$   CI 1.10 –  $1.37,p=1.90\times10^{-4})$   but also showed a suggestive association with deep ICH (OR 1.56,  $95\%$   CI   $1.24-$   $1.97,p=1.71\times10^{-4})$   (figure 2). In fact, the  $\mathcal{p}$   value for deep ICH passed the significance threshold, but the SNP did not pass our preset heterogeneity filter and was therefore not considered associated overall. There were no associations with non-SVD stroke or combined SVD and non-SVD phenotypes.  

Functional annotation.  All  COL4A2  and  HTRA1 SNPs associated with lacunar ischemic stroke or deep ICH were intronic. The GTEx eQTL browser search revealed no significant eQTLs for any of these SNPs. However, the RegulomeDB database revealed that 2 COL4A2  SNPs were in an area likely to affect binding, 2  COL4A2  SNP were in an area less likely to affect binding, and 17  COL4A2  SNP showed minimal binding evidence, suggesting that these SNPs are located in areas of the genome that may have regulatory functions (table e-2).  

DISCUSSION  Our results demonstrate an association of an intronic, possibly regulatory locus in COL4A2  with 2 distinct deep cerebral SVD phenotypes, lacunar ischemic stroke and deep ICH. We also found an association of deep cerebral SVD with HTRA1 , demonstrating an association with lacunar ischemic stroke and a suggestive association with deep ICH. Finding the same genetic signal associated with both ischemic and hemorrhagic sporadic stroke confirms the usefulness of a joint exploration of cerebro vascular phenotypes and the potential for genetic studies to shed light on common underlying mechanisms.  

Our findings for  COL4A2  are supported by previous work showing the relevance of this genomic region in sporadic deep cerebral SVD. A sequence analysis of  COL4A1/COL4A2  found missense mutations in sporadic ICH cases.   In addition, our  

![](images/efa6bdaffdaff35ea3aeb1d7c8f6e19a70f481ecddef3edc2b8a9c25cf7801dd.jpg)  

![](images/a76daecd08c3df4e38278479e83e70815182b4dc2fa67366fc360321e78f3d36.jpg)  

previous meta-analyses in a smaller, partly overlapping sample already demonstrated an association of this  COL4A2  locus with deep ICH and a suggestive association with other cerebral SVD phenotypes.   By increasing the sample size (by   $40\%$   for lacunar ischemic stroke and by   $15\%$   for deep ICH) and density of coverage in the present study, we have now established a substantial association of the same locus with lacunar ischemic stroke and confirmed the association with deep ICH. Furthermore, a recently published GWAS identified our lead SNP for lacunar ischemic stroke to be associated with another deep cerebral SVD phenotype, WMH.  

While the association with  COL4A2  is supported by previous data and a convincing signal for both ischemic and hemorrhagic phenotypes, the association with  HTRA1  is suggestive but less certain. On the basis of our    $\mathcal{p}$   threshold, the  HTRA1  SNP was associated with both lacunar ischemic stroke and deep ICH, but there was significant heterogeneity in the deep ICH meta-analyses. We are also aware of a previous report suggesting an association of rare variation in  HTRA1  with more extreme sporadic deep cerebral SVD phenotypes.   Thus, this finding should be pursued in independent, large samples to replicate the association.  

From a biological point of view, our strategy of investigating these familial genes jointly is supported by an emerging view that the resulting familial deep cerebral SVDs have similar disease mechanisms involving disruption of the cerebro vascular matrisome. Familial mutations leading to alterations of matrisome proteins and function could be a convergent pathway driving the functional and structural alterations of small brain vessels and disease manifestations, and similar mechanisms could play a role in sporadic disease.  

COL4A1  and  COL4A2  genes encode the collagen protein chains, a major component of the vascular basement membrane.   Their dominant missense mutations are associated with basement membrane defects and endoplasmic reticulum stress and cause rare familial SVDs. – 30   Recent data suggest that manipulation of endoplasmic reticulum stress (e.g., with 4-phenyl butyric acid) is a potential therapeutic option for collagen IV diseases, including hemorrhagic stroke.   Mutations in  HTRA1  gene are associated with cerebral autosomal recessive arte rio path y with sub cortical infarcts and leuk o encephalopathy. While the majority of  HTRA1  mutations cause this autosomal recessive cerebral SVD, heterozygous HTRA1  mutations associated with cerebral SVD have also recently been reported.   HTRA1  encodes the HTRA1 enzyme, which, through regulating transforming growth factor $\cdot\upbeta$   signaling, plays an important role in the formation of blood vessels.  

![](images/59c27a2e49a75dc14a4d79bef0c6f3e755c81aedef06b8183089db0983290129.jpg)  
Only SNPs passing the post – meta-analysis filters (heterogeneity    $I^{\mathcal{C}}<50\%,$  ,  $p>0.001$  ,  $\geq\!50\%$   cases contributing data) are displayed. Red dashed lines mark the relevant Nyholt significance  $p$   thresholds. Dots mark individual SNPs with respect to their chromosomal position  $(\times$  -axis) and  $p$   value for association between each SNP and phenotype (left y-axis). The SNP in purple is the most strongly associated (lead) SNP; linkage d is equilibrium with this lead SNP determines the colors for other SNPs, as seen from the  $r^{\flat}$    color coding on figure. Recombination rates (right  $\curlyvee$  -axis), shown by the continuous blue lines, are measured as frequency of exchange per unit physical distance (centi morgan [cM]/mega base pair [Mb]).  $\mathsf{I C H}=$   intra cerebral hemorrhage; SNP  $=$   single nucleotide polymorphism.  

Possible reasons for an apparent lack of an association with  COL4A1 ,  CECR1 ,  NOTCH3 , and  TREX1 include a genuine lack of association in our study population; weaker association not detected because of sample size; suboptimal diagnosis of SVD phenotypes in the original studies, resulting in reduced power; and variability in the density and quality of genotyping across different genes. In addition, we used a 10-kbp flanking region to cover regulatory areas for all genes; therefore, relatively more conservative    $\mathcal{p}$   values may have been derived for smaller genes ( TREX1 ) after adjustment for the number of  

![](images/0721177ee8e6e92efb243ad4c31736ca912719d8f6e8f10d2ae91678463162eb.jpg)  
Diamonds represent pooled ORs across all case-control collections for each phenotype, with the line through the diamond showing its  $95\%$   CI. Associations significant at relevant Nyholt threshold are shown in red; non significant associations with SVD phenotypes are shown in black; and non significant associations with non-SVD phenotypes are shown in gray. CE  $=$   card i oem boli c;  ${\mathsf{C}}{\mathsf{I}}=$   confidence interval;   $\mathsf{I C H}=$   intra cerebral hemorrhage;  $|S=$   ischemic stroke; LVD  $=$  large vessel disease; OR  $=$   odds ratio;  ${\mathsf{S N P}}=$   single nucleotide polymorphism.  

SNPs tested. In addition, because we treated the COL4A1/COL4A2  region as one, more conservative  $\mathcal{p}$   values were derived for the  COL4A1  gene than if we had treated it as a separate region.  

Our study has several strengths. We investigated a specific, pre specified hypothesis, clearly defining the phenotypes and candidate genes of interest on the basis of preexisting supporting data. Through a network of collaborative groups, we could include the majority of currently available data from stroke genetics studies of individuals of European ancestry. We used appropriate methods to correct for multiple testing.  

There were some limitations. First, while we have shown that SNPs in  COL4A2  are associated with lacunar ischemic stroke and deep ICH through analyzing data for the specific candidate region, the associations did not reach GWAS significance   $(p\le5\times$   $10^{-8}.$  ), most likely because of limited sample size. However, the lead lacunar ischemic stroke SNP (rs9515201) had a value of    $\mathit{p}=6.6\times10^{-8}$  , and it has been shown that a substantial proportion of SNPs with a  $\mathcal{p}$   value in this  “ borderline ”  GWAS significance range   $(\phi>5\times10^{-8}$    and  $\rlap/p\leq1\times10^{-7}\,$  ) represent genuine, replicable associations.   Second, we did not adjust the statistical threshold for the number of genomic regions and phenotypes investigated, considering this overly conservative because we were investigating a series of specific related hypotheses rather than a single hypothesis. However, had we further adjusted the  COL4A1 / COL4A2  region  $\mathcal{p}$   value for the number of tests   $(3.5~\times~10^{-4}/25~=~1.4~\times$   $10^{-5}\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!$  ), the association for the lead SNP with lacunar ischemic stroke would have remained significant. Third, our analyses found a locus in  COL4A2  containing several SNPs associated with deep cerebral SVD, most (but not all) of which were in moderate to strong LD with the lead SNP. This suggests that the association was likely driven by the lead SNP, but the possibility remains that independently significant signals in the locus may emerge.   Further investigation of this would require additional analyses adjusted for the lead SNP, requiring genome-wide genetic data that were not sought for this study, given its targeted hypothesis-driven approach. Fourth, the diagnostic workup leading to TOAST subtype classification was study specific, which may introduce some heterogeneity. Fifth, not all studies controlled for age in their statistical analyses before inclusion in the metaanalyses, and this may decrease the study power. Finally, we were not able to include data for additional relevant phenotypes such as WMH and brain micro bleeds in the present study.  

While genetic studies of ischemic stroke and ICH have generally been pursued separately, these findings emphasize the mechanistic insights that can be gained from joint analyses of cerebro vascular phenotypes. We have shown that the same genetic signal is associated with clinically evident sporadic ischemic and hemorrhagic stroke, but the joint exploration approach is further supported by previous GWASs showing a locus on chromosome 1q22 to be associated with both deep ICH and WMH.   In addition, it has recently been shown that a locus on chromosome 6p25, near the FOXF2  gene (also associated with familial deep SVD), is associated with all stroke (likely driven by SVD stroke phenotypes) and suggestively with WMH.  

Follow-up studies should further explore potential common genetic signals for deep cerebro vascular ischemic and hemorrhagic SVD phenotypes in larger sample sizes and for additional relevant phenotypes such as WMH and brain micro bleeds and should include non-European ethnic groups. Future studies could also assess the potential contribution of rare variants to common cerebral SVD phenotypes in these mendelian genes. In addition, the robust findings for  COL4A2  now merit further deep sequencing of the entire genomic region among sporadic deep cerebral SVD cases, with detailed functional studies of promising variants thus identified.  

# AUTHOR CONTRIBUTIONS  

K.R., R.M., and C.L.M.S. contributed to the conception and design of the study. K.R., V.S., and H.M. conducted the meta-analyses. K.R., V.S., H.M., R.M., and C.L.M.S. all contributed by drafting a significant portion of the manuscript. K.R., R.M., V.S., H.M., C.D.A., M.C., J.L.P., S.L.P., G.J.F., T.D., I.F.-C., J.J.-C., A.L., J.M., M.O., G.P., F.R., A.S., N.S.R., M.S., M.T., S.S., B.B.W., D.W., H.S.M., B.D.M., M.D., J.R., and C.L.M.S. contributed to the acquisition and analysis of the data. All authors reviewed and approved the final version.  

# STUDY FUNDING  

This work was funded in part by NIH grants U01 NS069208 and P30 DK072488 (SiGN). M.D. and R.M. were supported by grants from the Deutsche For s chung s gemeinschaft (CRC 1123 [B3] and Munich Cluster for Systems Neurology [SyNergy]), the German Federal Ministry of Education and Research (BMBF, e:Med programme e:A the roSy s Med), the FP7/2007 – 2103 European Union project CVgenes@target (grant agreement Health-F2-2013-601456), the European Union Horizon 2020 projects   $\mathrm{SVD}s@$  target (grant agreement 66688) and CoSTREAM (grant agreement 667375), the Fondation Leducq (Transatlantic Network of Excellence on the Pathogen es is of Small Vessel Disease of the Brain), the Vascular Dementia Research Foundation, and the Jackstaedt Foundation. G.P. and M.C. were supported by the Canadian Stroke Network, Canadian Institutes of Health Research, and Heart & Stroke Foundation. N.S.R. acknowledges the National Institute of Neurologic Disorders and Stroke K23NS064052, R01NS086905, and R01NS082285. J.R. acknowledges the National Institute of Neurologic Disorders and Stroke R 01 NS 059727 and Keane Stroke Genetics Fund. H.S.M. is supported by a National Institute for Health Research Senior Investigator award and support from the Cambridge University Hospitals Comprehensive Biomedical Research Centre. C.D.A. acknowledges funding from NIH K23 NS086873. A.L. is supported by Region Skåne, Lund University, the Swedish Heart and Lung Foundation, the Freemasons Lodge of Instructions EOS Lund, and the Swedish Stroke Association.  

# DISCLOSURE  

The authors report no disclosures relevant to the manuscript. Go to Neurology.org for full disclosures.  

Received December 15, 2016. Accepted in final form July 31, 2017.  

1. Pantoni L. Cerebral small vessel disease: from pathogen es is and clinical characteristics to therapeutic challenges. Lancet Neurol 2010;9:689 – 701.

 2. Wardlaw JM, Smith EE, Biessels GJ, et al. Neuro imaging standards for research into small vessel disease and its contribution to ageing and neuro degeneration. Lancet Neurol 2013;12:822 – 838.

 3. Tan RY, Markus HS. Monogenic causes of stroke: now and the future. J Neurol 2015;262:2601 – 2616.

 4. Haffner C, Malik R, Dichgans M. Genetic factors in cerebral small vessel disease and their impact on stroke and dementia. J Cereb Blood Flow Metab 2015;36:158 – 171.

 5. Rannikmäe K, Davies G, Thomson PA, et al. Common variation in  COL4A1/COL4A2  is associated with sporadic cerebral small vessel disease. Neurology 2015;84:918 – 926.

 6. Traylor M, Zhang CR, Adib-Samii P, et al. Genome-wide meta-analysis of cerebral white matter hyper intensities in patients with stroke. Neurology 2016;86:146 – 153.

 7. Malik R, Traylor M, Pulit SL, et al. Low-frequency and common genetic variation in ischemic stroke: the METASTROKE collaboration. Neurology 2016;86:1217 – 1226.

 8. NINDS Stroke Genetics Network (SiGN) & International Stroke Genetics Consortium (ISGC). Loci associated with ischaemic stroke and its subtypes (SiGN): a genome-wide association study. Lancet Neurol 2016;15:174 – 184.

 9. O ’ Donnell M, Xavier D, Diener C, et al. Rationale and design of INTER STROKE: a global case-control study of risk factors for stroke. Neuro epidemiology 2010;35: 36 – 44.

 10. Woo D, Falcone GJ, Devan WJ, et al. Meta-analysis of genome-wide association studies identifies 1q22 as a susce pti bil it y locus for intra cerebral hemorrhage. Am J Hum Genet 2014;94:511 – 521.

 11. Woo D, Sauerbeck LR, Kissela BM, et al. Genetic and environmental risk factors for intra cerebral hemorrhage: preliminary results of a population-based study. Stroke 2002;33:1190 – 1195.

 12. Adams HP, Bendixen BH, Kappelle LJ, et al. Classification of subtype of acute ischemic stroke: definitions for use in a multi center clinical trial: TOAST: Trial of Org 10172 in Acute Stroke Treatment. Stroke 1993;24:35 – 41.

 13. Marchini J, Howie B. Genotype imputation for genome-wide association studies. Nat Rev Genet 2010;11:499 – 511.

 14. Li J, Ji L. Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix. Heredity (Edinb) 2005;95:221 – 227.

 15. Nyholt DR. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage d is equilibrium with each other. Am J Hum Genet 2004;74:765 – 769.

 16. Nyholt DR. Evaluation of Nyholt ’ s procedure for multiple testing correction: author ’ s reply. Hum Hered 2005;60: 61 – 62.

 17. Salyakina D, Seaman SR, Browning BL, et al. Evaluation of Nyholt ’ s procedure for multiple testing correction. Hum Hered 2005;60:19 – 25.

 18. Auton A, Brooks LD, Durbin RM, et al. A global ref- erence for human genetic variation. Nature 2015;526: 68 – 74.

 19. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 2010;26:2190 – 2191.  

20. Rutten-Jacobs LC, Traylor M, Adib-Samii P, et al. Common NOTCH3 variants and cerebral small-vessel disease. Stroke 2015;46:1482 – 1487.

 21. Mueller JC. Linkage d is equilibrium for different scales and applications. Brief Bioinform 2004;5:355 – 364.

 22. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res 2012;40:D930 – D934.

 23. Jeanne M, Labelle-Dumais C, Jorgensen J, et al. COL4A2 mutations impair COL4A1 and COL4A2 secretion and cause hemorrhagic stroke. Am J Hum Genet 2012;90:91 – 101.

 24. Weng YC, Sonni A, Labelle-Dumais C, et al. COL4A1 mutations in patients with sporadic late-onset intra cerebral hemorrhage. Ann Neurol 2012;71:470 – 477.

 25. Nozaki H, Kato T, Nihonmatsu M, et al. Distinct molecular mechanisms of HTRA1 mutants in manifesting heterozygotes with CARASIL. Neurology 2016;86: 1964 – 1974.

 26. Joutel A, Haddad I, Ratelade J, et al. Perturbations of the cerebro vascular matrisome: a convergent mechanism in small vessel disease of the brain? J Cereb Blood Flow Metab 2015;36:143 – 157.

 27. Khoshnoodi J, Pedchenko V, Hudson BG. Mammalian collagen IV. Microsc Res Tech 2008;71:357 – 370.

 28. Murray LS, Lu Y, Taggart A, et al. Chemical chaperone treatment reduces intracellular accumulation of mutant collagen IV and ameliorates the cellular phenotype of a COL4A2 mutation that causes hae mor rha gic stroke. Hum Mol Genet 2014;23:283 – 292.

 29. Van Agtmael T, Schlötzer-Schrehardt U, McKie L, et al. Dominant mutations of Col4a1 result in basement membrane defects which lead to anterior segment dysgenesis and g lo merulo path y. Hum Mol Genet 2005;14:3161 – 3168.

 30. Van Agtmael T, Bailey MA, Schlötzer-Schrehardt U, et al. Col4a1 mutation in mice causes defects in vascular function and low blood pressure associated with reduced red blood cell volume. Hum Mol Genet 2010;19:1119 – 1128.

 31. Hara K, Shiga A, Fukutake T, et al. Association of HTRA1 mutations and familial ischemic cerebral smallvessel disease. N Engl J Med 2009;360:1729 – 1739.

 32. Verdura E, Hervé D, Scharrer E, et al. Heterozygous HTRA1 mutations are associated with autosomal dominant cerebral small vessel disease. Brain 2015;138:2347 – 2358.

 33. Panagiotou OA, Ioannidis JP; Genome-Wide Significance Project. What should the genome-wide significance threshold be? Empirical replication of borderline genetic associations. Int J Epidemiol 2012;41:273 – 286.

 34. Yang J, Ferreira T, Morris AP, et al. Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. Nat Genet 2012;44:369 – 375, S361 – S363.

 35. Verhaaren BF, Debette S, Bis JC, et al. Multi ethnic genome-wide association study of cerebral white matter hyper intensities on MRI. Circ Cardiovasc Genet 2015;8: 398 – 409.

 36. Chauhan G, Arnold CR, Chu AY, et al. Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies. Lancet Neurol 2016;15:695 – 707.  