# Siblings with Ischemic Stroke Study (SWISS): Results of a Genome-wide Scan for Stroke Loci  

James F. Meschia, MD ,  Michael Nalls, PhD ,  Mar Matarin, PhD ,  Thomas G. Brott, MD ,  Robert D. Brown Jr., MD ,  John Hardy, PhD ,  Brett Kissela, MD ,  Stephen S. Rich, PhD ,  Andrew Singleton, PhD ,  Dena Hernandez, MS ,  Luigi Ferrucci, MD ,  Kerra Pearce, MSc ,  Margaret Keller, PhD ,  Bradford B. Worrall, MD , and  for the Siblings with Ischemic Stroke Study (SWISS) Investigators\*  

Dept. of Neurology (JFM; TGB), Mayo Clinic, Jacksonville, FL; Dept. of Neurology (RDB Jr.), Mayo Clinic, Rochester, MN; Dept. of Molecular Neuroscience Institute of Neurology University College London (JH), London, UK; Dept. of Neurology (BK), University of Cincinnati, OH; National Institute on Aging (MN; AS; DH; LF), University College London Genomics and Institute of Child Health (KP), London, UK; Bethesda, MD; Coriell Institute for Medical Research (MK), Camden, NJ; Center for Public Health Genomics University of Virginia (SSR) Charlottesville, VA; Depts. of Neurology and Public Health Sciences University of Virginia (BBW) Charlottesville, VA; Department of Clinical and Experimental Epilepsy (MM), UCL Institute of Neurology, London, UK  

# Abstract  

Background and Purpose— Ischemic stroke has a strong familial component to risk. The Siblings with Ischemic Stroke Study (SWISS) is a genome-wide family-based analysis that included use of imputed genotypes. SWISS was conducted to examine associations between SNPs and risk of stroke and stroke subtypes within pairs.  

Methods— SWISS enrolled 312 probands with ischemic stroke across 70 US and Canadian centers. Affected siblings were ascertained by centers and confirmed by central record review; unaffected siblings were ascertained by telephone contact. Ischemic stroke was subtyped using TOAST criteria. Genotyping was performed using an Illumina 610 quad array (probands) and an Illumina linkage  $\mathrm{v}$   array (affected siblings). SNPs were imputed using 1000 Genomes Project data and MACH software. Family-based association analyses were conducted using the siblingtransmission d is equilibrium test.  

Results— For all pairs, the correlation of age at stroke within pairs of affected siblings was  $\mathbf{r}=$  0.83 (  $95\%\mathrm{CI}$  , 0.78 to 0.86;  $\mathbf{P}<2.2{\times}10^{-16}$  ). The correlation did not differ substantially by subtype. The concordance of stroke subtypes among affected pairs was  $33.8\%$     $(\mathrm{\kappa=0.13}$  ;  $\mathrm{P}=$   $5.06\!\times\!10^{-4})$  ) and did not differ by age at stroke in the proband. Although no SNP achieved genomewide significance for risk of ischemic stroke, there was clustering of the most associated SNPs on chromosomes 3p ( NOS1 ) and 6p.  

Conclusions— Stroke subtype and age at stroke in affected sibling pairs exhibit significant clustering. No individual SNP reached genome-wide significance. However, two promising candidate loci were identified, including one that contains  NOS1 , though these risk loci warrant further examination in larger sample collections.  

Ischemic stroke has long been recognized to cluster in families. This clustering has been attributed to both genetic factors and common environmental exposures. Gene mutations have been identified that lead to rare ischemic stroke syndromes like CADASIL and MELAS. However, search for genetic loci associated with ischemic stroke risk has yielded meager results thus far despite several candidate gene [1] and large-scale genome-wide association studies. [2] [3] The Siblings with Ischemic Stroke Study (SWISS) provides a different and promising approach to discover novel risk factors for ischemic stroke through the study of unrelated families of affected and unaffected siblings. Here we present the results of the family-based genome-wide scan of SWISS after achieving the original recruitment goal.  

# Methods  

# Study Subjects  

Probands were recruited at 70 US medical centers and Canadian medical centers. Probands were adult (>18 years old) men and women presenting to a participating center with a study neurologist-confirmed ischemic stroke. Stroke was defined as rapidly developing signs of a focal or global disturbance of cerebral function, with symptoms lasting at least 24 hours or leading to death with no apparent cause other than vascular origin (WHO definition). [4] Stroke was defined as ischemic if CT or MR imaging of the brain was performed within 7 days of onset of stroke symptoms and identified the symptomatic cerebral infarct or failed to identify an alternative cause of symptoms. Probands were required to have reported at least one living full sibling with a history of stroke. No probands were enrolled with iatrogenic va so spastic, or vasculitic stroke or if the stroke occurred in the setting of a mechanical heart valve or in the setting of untreated or actively treated bacterial endocarditis. Probands were also excluded if they were known to have CADASIL, Fabry disease, homo c ystein uri a, MELAS, or sickle cell anemia. Study neurologists at each center assigned to the qualifying ischemic stroke of each proband a TOAST subtype diagnosis.[5]  

Stroke-affected siblings of the proband (concordant siblings) were recruited using probandinitiated contact.[6] Telephone interviews were performed to obtain demographic and clinical information and to gain permission for obtaining medical records pertaining to treatment for stroke. Medical records were compiled and adjudicated by a central committee (JFM; TGB), to verify the diagnosis of ischemic stroke and to assign a TOAST subtype diagnosis. Assignment of TOAST subtype diagnoses to SWISS concordant siblings has moderate inter-rater reliability.[7] Unaffected siblings were ascertained by telephone contact and interview.  

# Genotyping Considerations  

The establishment of lymph oblast oid cell lines, quality control of genomic DNA, acquisition of genetic data, and genotyping quality control metrics were performed using standard procedures. Please see http://stroke.aha journals.org for these details.  

Consensus SNPs that passed QC in both phases (genome-wide association and family-based phases) were merged for all available sibships (2239 SNPs were imputed in the probands). Using all 5612 SNPs in the merged dataset, reported relationships were verified using pi_hat estimates. Sibships were confirmed if pairwise pi_hats values were between 0.35 and 0.65; samples were removed from a sibship if estimated pi_hat was not in this range. This dataset of the combined genotyping phases represents the final dataset for all subsequently described analyses. The flow of patients in the study is shown in Figure 1.  

# Genetic Data Analysis  

All family-based analyses were conducted using PLINK 1.07 software. [8] The dFam utility within PLINK implements a siblings-based transmission-d is equilibrium test and was used to conduct these analyses. The dFam option is a powerful test for sibling-only datasets, incorporating data across sibships as well as using data from estimated parental genotypes to calculate expected allele frequencies for comparison to observed allele frequencies. The association test is based on the Cochran-Mantel-Haenszel test. Bonferroni correction for the number of tested SNPs corresponds to a minimum p-value for genome-wide significance of  $\mathbf{P}<8.91\!\times\!10^{-6}$  .  

# Additional Statistical Analyses  

Frequencies of stroke risk factors (hypertension, hyper lipid emi a and diabetes) between affected and unaffected participants were compared using chi-squared tests. The correlation between affected sibling age at stroke was estimated using the Pearson test of correlation. These analyses were conducted across all TOAST subtypes as well as following stratification by concordant and discordant subtypes among affected sibling pairs. Linear regression was used to determine the confidence intervals and linear fit of the age association as show in Figure 2 Kappa statistics were calculated to quantify concordance of phenotypes of interest within sibling pairs for all ages and stroke subtypes as well as models stratified by age (  $\zeta<65$   year proband as defining age strata) and stroke subtype. All analyses that did not include genetic data were conducted using scripts written in R (R Development Core Team (2008)). [9]  

# Results  

A total of 312 affected sibling pairs (312 probands) were enrolled at 70 centers across the US and Canada. After quality control filtering, the final study population consisted of 223 probands, 248 stroke-affected siblings, and 84 stroke-unaffected siblings (total sample size, 555). Ischemic stroke-affected individuals had expected high rates of conventional at hero sclerotic risk factors (Table 1). Stroke-affected individuals (probands and affectedsiblings) were significantly more likely to have hypertension   $(\mathrm{P}<0.0001)$  ), hyper lipid emi a  $(\mathrm{P}{=}\,0.002)$  ) and diabetes   $\mathrm{P}=0.008)$  ) than stroke-unaffected individuals. Stroke-affected siblings were somewhat older than the probands. This difference of 2 years   $\mathrm{(P}=0.057)$  ) is expected as an older sibling of the proband would be more likely to have a stroke than a younger sibling.  

Sibling age at time of stroke strongly correlated with proband age at time of stroke, despite the sibling being older. As shown in Figure 2 for all the sibling pairs, the correlation coefficient was  $\mathrm{r}=0.83$   (  $95\%\mathrm{CI}$  , 0.78 to 0.86;  $\mathbf{P}<2.2{\times}10^{-16}$  ). For affected sibling pairs who have the same stroke subtype, the correlation coefficient was not different from all pairs,  $\mathrm{r}=0.83$     $95\%\mathrm{CI}$  , 0.75 to 0.89;  $\mathrm{P}<2.2{\times}10^{-16}$  ). This was the same for sibling pairs in which the affected siblings had different stroke subtypes,  $\mathrm{r}=0.83$   (  $95\%\mathrm{CI}$  , 0.77 to 0.87;  $\mathrm{P}<$   $2.2{\times}10^{-16}$  ). Over   $50\%$   of the variance in age at stroke onset in siblings could be predicted by the age of proband at time of stroke. As shown in Table 2, there was significant concordance with affected siblings for TOAST subtype   $(\mathrm{\kappa=0.13}$  ;  $\mathbf{P}=5.06{\times}10^{-4})$  ); this relationship remained significant for sibling pairs where the proband was less than 65 years old at time of stroke and for sibling pairs in which the proband was 65 years or older.  

Results of the genome-wide analyses are shown in Figure 3. Although no SNP association with ischemic stroke achieved genome-wide level of significance, the ten most associated SNPs exhibited clustering in several genomic regions. These ten most significantly associated SNPs, their locations, frequencies and effect estimates are shown in Table 3. The ten SNPs represent 8 genomic loci, with minor allele frequencies ranging from 0.38–0.48 (common alleles). The effects for each are small, with odds ratios ranging from 0.96 to 1.04. The location of the most significantly associated SNPs (as well as others within  $2.5\,\mathrm{Mb}$  ) is shown in Figure 3 (indicated by blue shading). There are clusters of associated SNPs on chromosomes 3p and 6p. The SNPs on chromosome 3p lie in a strong candidate gene, NOS1 .  

# Discussion  

Our genome-wide scan for risk factors for ischemic stroke was performed in the largest collection of affected sibling pairs to date and showed potential loci of interest including a locus on  NOS1 . We are not aware of other genetic studies showing  NOS1  variants to be associated with human ischemic stroke risk. However, knock-out mouse models show that NOS1  gene deficiency causes worsening neointimal formation and constrictive vascular remodeling. [10] Genetic variants theoretically could impart elevated ischemic stroke risk by a vascular mechanism, by a thrombotic mechanism or even by a p are nch y mal mechanism where brain tissue is sensitized to focal ischemia. In the case of  NOS1 , it seems that variants would be more likely to impart risk by a vascular mechanism than by a p are nch y mal mechanism, since  NOS1  knockout mice have smaller infarcts than control wild-type mice in an MCA occlusion model. [11] It is important not to speculate beyond the strength of our observations, as no locus achieved genome-wide statistical significance.  

Genome-wide studies have had mixed results in ischemic stroke. When SWISS was initiated, the human genome had only been sequenced in draft form. [12] SWISS was predicated on the hypothesis that ischemic stroke obeyed the common disease, common variant hypothesis, which states that the genetic influences on many common disease are attributable to a limited number of allelic variants present in  $>1–5\%$   of the population. [13] It has since become less clear that the hypothesis holds for ischemic stroke. No single locus has been identified in two genome-wide association studies at a genome-wide level of significance. [14] Our study supports the idea that no single locus substantially contributes to ischemic stroke risk from the perspective of common variants contributing to disease risk, although future sequencing-based studies of rare variants may meet with substantially more success.  

SWISS was designed to treat all types of ischemic stroke as a single phenotype. The phenotypic heterogeneity of ischemic stroke has long been appreciated, but categorizing subtypes of ischemic stroke historically has been done with little consistency in genetic research.[15] Despite this methodological limitation, genetic risk factors have been identified that appear to be specific for certain ischemic stroke subtypes. For example, the chromosome 9p21 locus appears to impart risk for so-called large vessel at hero sclerotic stroke. [16, 17] The atrial fibrillation locus 4q25 appears to impart risk for card i oem boli c stroke. [18, 19] Collaborating with investigators from Sweden, we previously assessed whether ischemic stroke subtypes clustered among affected sibships, showing low aggregation rates. [20] We continue to see low aggregation rates, but the relationship is significant. In addition to having larger numbers, the current analysis is restricted to those affected sibling pairs confirmed to be full siblings through genomic analysis. It is not known whether more complex systems of classifying stroke also show a tendency toward aggregation within families.[21]  

Age at onset of stroke may be a quantitative phenotype more tractable to genomic analysis. In an interim analysis, we had observed a significant association of proband age at stroke onset and sibling age at stroke onset. [22] As with the subtype aggregation reanalysis, the current analysis involves a larger sample size and is restricted to those affected sibling pairs confirmed to be full siblings through genomic analysis. As a phenotype, age at stroke onset has the limitation that it does not necessarily reflect the burden of ischemic disease at any given moment in the lifespan of a patient. Some cerebral infarcts are a symptomatic [23] while other cerebral infarcts may be symptomatic but undiagnosed. [24]  

In summary, we have described here an affected relative-based genetic analysis of ischemic stroke. This work provides preliminary evidence for the involvement of several loci in risk for this disease, and these loci certainly warrant follow-up. This work also suggests that any individual risk variants involved in ischemic stroke are likely to have a low populationattributable risk. Attributable risk could be low if the risk conferred is relatively low; it could also be low if there is extensive allelic and/or genetic heterogeneity in stroke, with no single locus being a common, high risk conferring locus. Clearly we can hope that the future application of now- and next-generation technologies in large and extremely well characterized cohorts will enable identifying genetic risks for ischemic stroke.  

# Supplementary Material  

Refer to Web version on PubMed Central for supplementary material.  

# Acknowledgments  

This study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD. (http://biowulf.nih.gov).  

This work used samples and clinical data from the NINDS Human Genetics Resource Center DNA and Cell Line Repository (http://ccr.coriell.org/ninds), human subjects protocol numbers 2003-081 and 2004-147.  

# Sources of Funding  

SWISS is funded by the National Institute for Neurological Disorders and Stroke (NINDS) through grant number R01 NS39987 (J.F.M. PI), IRB 1082-99 (Mayo Clinic). The inclusion of BLSA samples was supported in part by the Intramural Research Program of the National Institute on Aging, National Institutes of Health, Department of Health and Human Services; project number Z01 AG000015-50, human subjects protocol number 2003-078. Dr. Hardy is supported by an MRC Returning scientist award. The Intramural Research Program of the National Institute on Aging, National Institutes of Health, Department of Health and Human Services; project number Z01 AG000954-06.  

# References  

1. Wang X, Cheng S, Brophy VH, Erlich HA, Mannhalter C, Berger K, et al. A meta-analysis of candidate gene polymorphisms and ischemic stroke in 6 study populations: association of lymph o toxin-alpha in non hypertensive patients. Stroke. 2009; 40:683–695. [PubMed: 19131662]

 2. Ikram MA, Seshadri S, Bis JC, Fornage M, DeStefano AL, Aulchenko YS, et al. Genomewide association studies of stroke. N Engl J Med. 2009; 360:1718–1728. [PubMed: 19369658]

 3. Debette S, Bis JC, Fornage M, Schmidt H, Ikram MA, Sigurdsson S, et al. Genome-wide association studies of MRI-defined brain infarcts: meta-analysis from the CHARGE Consortium. Stroke. 2010; 41:210–217. [PubMed: 20044523]

 4. The World Health Organization MONICA Project (monitoring trends and determinants in cardiovascular disease): a major international collaboration. WHO MONICA Project Principal Investigators. J Clin Epidemiol. 1988; 41:105–114. [PubMed: 3335877]

 5. Adams HP Jr, Bendixen BH, Kappelle LJ, Biller J, Love BB, Gordon DL 3rd, et al. Classification of subtype of acute ischemic stroke. Definitions for use in a multi center clinical trial. TOAST. Trial of Org 10172 in Acute Stroke Treatment. Stroke. 1993; 24:35–41. [PubMed: 7678184]  

6. Worrall BB, Chen DT, Meschia JF. Ethical and methodological issues in pedigree stroke research. Stroke. 2001; 32:1242–1249. [PubMed: 11387482]

 7. Meschia JF, Barrett KM, Chu kw ude lun zu F, Brown WM, Case LD, Kissela BM, et al. Inter observer Agreement in the Trial of Org 10172 in Acute Stroke Treatment Classification of Stroke Based on Retrospective Medical Record Review. Journal of Stroke and Cerebro vascular Diseases. 2006; 15:266–272. [PubMed: 17904086]

 8. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007; 81:559– 575. [PubMed: 17701901]

 9. R Development Core Team. Vienna, Austria: 2008. R: A language and environment for statistical computing. R Foundation for Statistical Computing. http://www.R-project.org.

 10. Morishita T, Tsutsui M, Shimokawa H, Horiuchi M, Tanimoto A, Suda O, et al. Vas cul o protective roles of neuronal nitric oxide synthase. FASEB J. 2002; 16:1994–1996. [PubMed: 12397095]

 11. Huang Z, Huang PL, Panahian N, Dalkara T, Fishman MC, Moskowitz MA. Effects of cerebral ischemia in mice deficient in neuronal nitric oxide synthase. Science. 1994; 265:1883–1885. [PubMed: 7522345]

 12. Meschia JF, Brown RD Jr, Brott TG, Chu kw ude lun zu FE, Hardy J, Rich SS. The Siblings With Ischemic Stroke Study (SWISS) Protocol. BMC Med Genet. 2002; 3:1. [PubMed: 11882254]

 13. Manolio TA, Brooks LD, Collins FS. A HapMap harvest of insights into the genetics of common disease. J Clin Invest. 2008; 118:1590–1605. [PubMed: 18451988]

 14. Lanktree MB, Dichgans M, Hegele RA. Advances in genomic analysis of stroke: what have we learned and where are we headed? Stroke. 2010; 41:825–832. [PubMed: 20167918]

 15. Meschia JF. Addressing the heterogeneity of the ischemic stroke phenotype in human genetics research. Stroke. 2002; 33:2770–2774. [PubMed: 12468768]

 16. G schw end t ner A, Bevan S, Cole JW, Plourde A, Matarin M, Ross-Adams, et al. Sequence variants on chromosome 9p21.3 confer risk for at hero sclerotic stroke. Ann Neurol. 2009; 65:531–539. [PubMed: 19475673]

 17. Smith JG, Melander O, Lovkvist H, Hedblad B, Engstrom G, Nilsson P, et al. Common genetic variants on chromosome 9p21 confers risk of ischemic stroke: a large-scale genetic association study. Circ Cardiovasc Genet. 2009; 2:159–164. [PubMed: 20031580]

 18. Greta rs do ttir S, Thor leif s son G, Manolescu A, S tyr kars do ttir U, Helga do ttir A, G schw end t ner A, et al. Risk variants for atrial fibrillation on chromosome 4q25 associate with ischemic stroke. Ann Neurol. 2008; 64:402–409. [PubMed: 18991354]

 19. Lemmens R, Buys sch a ert I, Geelen V, Fernandez-Cadenas I, Montaner J, Schmidt H, et al. The association of the 4q25 susceptibility variant for atrial fibrillation with stroke is limited to stroke of cardioembolic etiology. Stroke. 2010; 41:1850–1857. [PubMed: 20671249]

 20. Wiklund PG, Brown WM, Brott TG, Stegmayr B, Brown RD Jr, Nilsson-Ardnor S, et al. Lack of aggregation of ischemic stroke subtypes within affected sibling pairs. Neurology. 2007; 68:427– 431. [PubMed: 17283317]

 21. Arsava EM, Ballabio E, Benner T, Cole JW, Delgado-Martinez MP, Dichgans M, et al. The Causative Classification of Stroke system: an international reliability and optimization study. Neurology. 2010; 75:1277–1284. [PubMed: 20921513]

 22. Meschia JF, Brott TG, Brown RD Jr, Kissela BM, Hardy JA, Brown WM, et al. Correlation of proband and sibling stroke latency: the SWISS Study. Neurology. 2005; 64:1061–1063. [PubMed: 15781828]

 23. Vermeer SE, Longstreth WT Jr, Koudstaal PJ. Silent brain infarcts: a systematic review. Lancet Neurol. 2007; 6:611–619. [PubMed: 17582361]

 24. Howard VJ, McClure LA, Meschia JF, Pulley L, Orr SC, Friday GH. High prevalence of stroke symptoms among persons without a diagnosis of stroke or transient ischemic attack in a general population: the REasons for Geographic And Racial Differences in Stroke (REGARDS) study. Arch Intern Med. 2006; 166:1952–1958. [PubMed: 17030827]  

# Appendix 1. SWISS Investigative Team  

Executive Committee: James F. Meschia, chair; Thomas G. Brott, Robert D. Brown, Jr., Brett Kissela, John Hardy, Stephen S. Rich, Andrew Singleton, Bradford Worrall. Statistical Center: University of Virginia, Charlottesville, VA. Stephen S. Rich, Director of Center for Public Health Genomics. DNA Repository: Coriell Institute for Medical Research, Camden, New Jersey. Margaret Keller, PhD, Director. Data Management: Mayo Clinic, Rochester, Minnesota. Barry Bisbee, Data Management Specialist. Centers and Investigators (listed by proband enrollment):  Neurological Associates, Inc, Richmond, Virginia  (probands enrolled, 72): Principal Investigators (PI): Alan Schulman, MD; Francis McGee, MD; coordinators: Gwendolyn H. Darby; Julie Thornton.  Mayo Clinic, Rochester, Minnesota  (67): PI: Robert D. Brown, Jr., MD; coordinator: Colleen S. Albers.  Mayo Clinic, Jacksonville, Florida  (64): PI: Thomas G. Brott, MD; coordinators: Alexa Richie; Dale Gamble; Sothear Luke. University of Virginia Health System, Charlottesville, Virginia  (59): PI: Bradford Worrall, MD, MSc; coordinators: Colleen Harman; Martha Davis; Kay Maupin.  University of Cincinnati, Cincinnati, Ohio  (51): PI: Brett Kissela, MD; coordinator: Kathleen Alwell. Shands Jacksonville, affiliated with the University of Florida, Jacksonville, Florida  (50): PI: Scott Silliman, MD; coordinator: Rhonda Calhoun.  Mercy Ruan Neurology Clinic, Des Moines, Iowa  (46): PI: Michael Jacoby, MD; coordinator: Cheryl Inman.  Washington University School of Medicine, St. Louis,  Missouri (33): PI: Jin-Moo Lee, MD, PhD; coordinators: Julie Rickmann; Jill Newgent.  University of Maryland, Baltimore, Maryland (27): PI: John Cole, MD; coordinators: Mark Dobbins; Mary J. Sparks; Nancy Zappala. University of Pennsylvania Medical Center, Philadelphia, Pennsylvania  (23): PI: Scott Kasner, MD; Co-Investigators: Brett Cucchiara, MD; Steven Messe, MD; Michael McGarvey, MD; coordinators: Mary Liz DeSanto; Jean Luciano.  Lankenau Institute for Medical Research, Bryn Mawr, Pennsylvania  (22): PI: Gary H. Friday, MD; coordinators: Joan Brown; Angela Gordon.  Kaleida Stroke Center, Millard Fillmore Gates Hospital, Buffalo, New York  (22): PI: Richard Ferguson, MD; coordinator: Kathy Parkes. Centre Hospital i er Affilie Universitaire de Quebec, Quebec City, Province of Quebec, Canada  (21): PI: Ariane Mackey MD; Co-Investigator: Steve Verrault, MD; coordinator: Annette Hache. Metro Health Medical Center, Cleveland, Ohio  (21): PI: Joseph Hanna, MD; coordinator: Dana Cook.  Mayo Clinic, Scottsdale, Arizona  (20): PI: Bart Dem a ers chalk, MD; coordinator: Erica L. Boyd.  Ohio State University, Columbus, Ohio  (19): PI: Andrew Slivka, MD; coordinator: Peggy Notestine.  Mercy General Hospital, Sacramento, California  (19): PI: Paul Akins; coordinators: Deidre Wentworth; Laura Newman.  Luther-Midelfort Hospital, Eau Claire, Wisconsin  (18): PI: Felix Chu kw ude lun zu, MD; coordinator: Karen Snobl.  OSF Stroke Center, Peoria, Illinois  (17): PI: Arun Talkad, MD; Co-Investigators: David Wang, DO; Maureen Matthews, APN, CNP, CNRN; coordinator: Mary Buttice. Rochester General Hospital, Rochester, New York  (17): PIs: W. Scott Burgin, MD; Joshua Hollander, MD; coordinator: Cheryl Weber.  University of Iowa Hospital, Iowa City, Iowa (15): PI: Patricia Davis, MD; coordinator: Jeri Sieren.  University of South Alabama, Mobile, Alabama  (15): PI: J. Ivan Lopez, MD; coordinator: Mel Parnell.  Hôspital Charles Le Moyne, Greenfield Park, Province of Quebec, Canada  (14): PI: Leo Berger, MD; coordinators: Martine Mainville; Louise-Marie Béliveau.  Marshfield Clinic, Marshfield, Wisconsin  (14): PI: Percy Karanjia, MD; Co-Investigator: Kenneth Madden, MD, PhD; coordinators: Kathy Mancl; Rob Bohl.  University of California, Davis School of Medicine, Sacramento, California  (13): PI: Piero Verro, MD; coordinators: John Bautista; Roxana Hupcey.  Mercy Hospital, Sioux City, Iowa  (13): PI: Jennifer Pary, MD; coordinators: Deb Motz; Darla Bauer. University of Texas Southwestern Medical Center at Dallas, Dallas, Texas (12): PI: Mark Johnson, MD; Co-Investigator: Jessica Lee, MD; coordinator: April Blair.  Maine Medical Center, Scarborough, Maine  (12): PI: John Belden; coordinator: Diane DiconzoFanning.  Cleveland Clinic Florida, Weston, Florida  (11): PI: Virgilio Salanga, MD; coordinator: Kilaun Thompson.  Wake Forest University School of Medicine, Winston-Salem, North Carolina  (11): PI: David Lefkowitz, MD; Co-Investigators: Patrick Reynolds, MD; Charles Tegeler, MD; coordinator: Emily Smith.  University of Utah Stroke Center, Salt Lake City, Utah  (10). PI: Elaine J. Skalabrin, MD; coordinator: Jill Austin.  Helen Hayes Hospital, West Haverstraw, New York  (10): PIs: Jason Greenberg, MD; Laura Lennihan, MD; coordinator: Laura Tente romano.  Stroke Prevention and Atherosclerosis Research Centre, London, Ontario Canada  (10): PI: J. David Spence, MD; coordinator: Rose Freitas.  Indiana University School of Medicine, Indianapolis, Indiana  (9): PI: Linda Williams, MD; coordinator: Flossy Lincoln.  Scripps Clinic, La Jolla, California  (9): PI: Mary Kalafut, MD; coordinators: Crystal Sanchez; Gail Moore.  Thomas Jefferson University Hospital, Philadelphia, Pennsylvania  (8): PI: Rodney Bell, MD; coordinator: Prema Krishna. University of Kentucky, Lexington, Kentucky  (7): PI: L. Creed Pettigrew, MD; coordinator: Deborah Taylor.  University of Wisconsin, Madison, Wisconsin  (7): PI: Robert J. Dempsey, MD; coordinator: Pam Winne.  East Bay Region Associates in Neurology, Berkeley, California  (7): PI: Brian Richardson, MD; coordinator: Lauren McCormick.  Emory University, Atlanta, Georgia  (7): PI: Barney J. Stern, MD; coordinator: Betty Jo Shipp. Inova Fairfax Hospital, Falls Church, Virginia  (7): PI: Paul Nyquist, MD; coordinator: Barbara Farmer.  Massachusetts General Hospital, Boston, Massachusetts  (6): PI: Karen Furie, MD; coordinator: Ashley Birch.  University of California Los Angeles Stroke Center, Los Angeles, California  (6): PI: Jeffrey Saver, MD; coordinator: Hannah Smith.  Florida Neuro vascular Institute, Tampa, Florida  (5): PI: Erfan Albakri, MD; coordinator: Valerie Walters.  Seton Brain and Spine Institute, Seton Family of Hospitals, Austin, Texas  (5): PI: Darryl Camp, MD; coordinators: Lisa Houy; Michele Ajgaonkar.  Morton Plant Hospital - Neuroscience Institute, Clearwater, Florida  (5): PI: Ajay Arora, MD; coordinators: Jo Simpson; Teresa Jones; Victoria Lindeman.  Yale University School of Medicine, New Haven, Connecticut  (5): PI: Lawrence Brass, MD; coordinators: Renee Dobos; Janet Halliday.  The Stroke Center at Hartford Hospital, Hartford, Connecticut  (4): PI: Isaac Silverman, MD; coordinators: Martha Ahlquist; Jennifer Blum.  Southtowns Neurology, West Seneca, New York  (4): PI: Peterkin Lee Kwen, MD; coordinator: Sarfaraz Semy.  Vanderbilt Stroke Center, Nashville, Tennessee  (4): PI: Anne E. O'Duffy, MD; coordinator: Diane Brown.  University of North Carolina Chapel Hill, Chapel Hill, North Carolina  (4): PI: Souvik Sen, MD; Co-Investigator: Natalie Aucutt-Walter, MD; coordinators: Roxanne Poole; Douglas Parker; Omid Akhavan.  Elkhart Clinic, Elkhart, Indiana  (3): PI: Thomas Vidic, MD; coordinator: Randall Gibson, PhD.  USCD Stroke Center, San Diego, California (3): PI: Patrick Lyden, MD; coordinator: Nancy Kelly.  Rush Presbyterian St. Luke’s Medical Center, Chicago, Illinois  (3): PI: Sean Ruland, DO; coordinator: Karen Whited. University of Illinois at Chicago, Chicago, Illinois  (3): PI: Cathy Helgason, MD; coordinator: Joan Martell otto.  Johns Hopkins Bayview Medical Center, Baltimore, Maryland  (3): PI: Rafael H. Llinas, MD; coordinator: Janice Alt.  Field Neuroscience s Institute, Saginaw, Michigan  (3): PI: Faith Abbott, DO; coordinator: Richard Herm.  Medical University of South Carolina, Charleston, South Carolina  (3): PI: Timothy Carter, MD; coordinator: Feng Liu.  Royal University Hospital, Saskatoon, Province of Saskatchewan , Canada (3): PI: Ali Rajput, MD; coordinator: Theresa Shirley.  Chattanooga Center for Neurologic Research, Chattanooga, Tennessee  (3): PI: Thomas Devlin, MD; coordinators: Katrina Baron; Tammi Owens.  Lancaster General Hospital, Lancaster, Pennsylvania  (2): PI: Ven kata chal am Mange sh kumar, MD; coordinator: Holly Snyder.  St. Mary’s Medical Center, Huntington, West Virginia  (2): PI: Carl McComas, MD; coordinators: Jennifer Edwards; Cheryl Kane.  Albert Einstein Medical Center, Philadelphia, Pennsylvania  (2): PI: Jonathan Dissin, MD; coordinator: Rohini Bhole.  Froedtert Memorial Lutheran, Milwaukee, Wisconsin  (1): PI: Michel Torbey, MD; coordinator: Erin Brandenburg.  Sparks Neurology Center, Fort Smith, Arkansas  (1): PI: Margaret Tremwel, MD; coordinators: Cheryl Hyde; Mary Rambin.  Syracuse VAMC, Syracuse, New York  (1): PI: Antonio Culebras, MD; coordinator: Therese Dean.  Rhode Island Hospital, Providence, Rhode Island  (1): PI: Janed Wilterdink, MD; coordinator: Carol Cirillo.  Charles R. Drew University of Medicine and  

Science, Los Angeles, California  (1): PI: Lowell Nelson, MD; coordinator: Marcia Montenegro.  Saint Joseph Mercy Oakland, Pontiac, Michigan  (1): PI: Lionel Glass, MD; coordinator: Susan Oblak.  Legacy Health System, Tualatin, Oregon  (0): PI: Paul Ash, MD; coordinator: Taryn Lust.  

![](images/070c4c1e1bf000ecc01bc2c5fcae2ad53e5fd36821077cbdf390e42e004cd4ff.jpg)  

Figure 1. Flow of participants in the study.  

![](images/436bf451435f3d429c087dabbdce33df7ee236af633a68a748f26a48d64a3f9c.jpg)  

# Figure 2.  

Correlation between proband and sibling age at stroke. Correlation coefficient  $=0.83$  . Pvalue   $<0.0001$  . Pairs are points, Blue line  $=$   linear model, grey shading  $=95\%$   CI.  

![](images/48bb859864ab09a25fb93cc70857e0b6f488a058b257749641e61147f921ab0e.jpg)  

# Figure 3.  

Summary of transmission-d is equilibrium test results. All SNPs within 2.5 megabases of SNPs shown in table 1 are highlighted in blue.  

# Table 1  

Characteristics of the Study Population after passing genomic quality controls. 
![](images/c522cae5a0d35de0ca45e700678b487a3b8ebff7a1150b0f2c72995f50e5f06c.jpg)  
\* n.a.=Not Applicable  

![](images/02f13ed6f16dc357bab2b8cc1a2171ca85a7f995fed190c8d93ef87c8f7c9a15.jpg)  
Note: Kappa statistics calculated for all pairs by age strata  

![](images/f1d5e261ea9d68e243cad077612dc9b0ccee2ed1693b84cc6893a4234cec7192.jpg)  
SNP denotes single nucleotide polymorphism; CHR, chromosome; MB, megabase; cM, centi Morgan; MAF, minor allele frequency.  