# Ph os ph odi est erase 4D polymorphisms and the risk of cerebral infarction in a biracial population: the Stroke Prevention in Young Women Study  

Qing Song 1,† ,  John W.  $\mathsf{c o l e}^{2,3,\star,\dagger}$  ,  Jeffrey R. O’Connell 2 ,  Oscar C. Stine 2 ,  Margaret 4 4 2 2,3 Gallagher ,  Wayne H. Giles ,  Braxton D. Mitchell ,  Marcella A. Wozniak ,  Barney J. Stern 2,3 ,  John D. Sorkin 2,3 ,  Patrick F. McArdle 2 ,  Adam C. Naj 2,5 ,  Qin  $\pmb{\chi}\mathfrak{u}^{1}$  ,  Gary H. 1 2,3 Gibbons , and  Steven J. Kittner  

1  Morehouse School of Medicine, Atlanta, GA 30314, USA, 2  University of Maryland School of Medicine, Baltimore, MD 21201, USA, 3  Veterans Administration Medical Center, Baltimore, MD 21201, USA, 4  Centers for Disease Control, Atlanta, GA 30333, USA and 5  Johns Hopkins Bloomberg School of Public Health, Baltimore, MD 21205, USA  

An association between polymorphisms within the ph os ph odi est erase 4D gene ( PDE4D ) and ischemic stroke was initially reported in older adults from Iceland and has been supported by studies in several other primarily elderly populations. In the present study, we examined the association between  PDE4D  polymorphisms and early-onset ischemic stroke in a biracial female population. A systematic search for polymorphisms in the highly evolutionary conserved regions of  PDE4D  was performed on 48 African-American and 48 Caucasian participants. Novel and known polymorphisms were then prioritized and genotyped in the entire study population of 224 cases of first ischemic stroke among women aged 15–49 and 211 age- and ethnicity-balanced control subjects. Forty novel and previously reported polymorphisms with a minor allele frequency greater than 0.05 were determined, with 23 polymorphisms selected for analysis in the full case–control sample. Single nucleotide polymorphism (SNP), linkage d is equilibrium and haplotype analyses were performed. SNP rs918592, found in an intron near the  $5^{\prime}$   end of the gene, was significantly associated with stroke (age- and race-adjusted odds ratio   $(\mathrm{OR}=1.5$  ,  $P\,{=}\,0.007)$  ), with four other SNPs showing significant, albeit less strong, associations. The magnitude of association was similar across African-Americans and Caucasians and across multiple stroke subtypes (e.g. at hero sclerotic, lacunar and non-lacunar of undetermined etiology). The association of rs918592 with stroke was confined exclusively to current smokers   $(\mathrm{OR}=3.2$  ,  $P=0.00014\$  ), with no association observed among never-smokers   $(\mathrm{OR}=0.9$  ,  $P=0.75$  ) or former smokers   $(\mathrm{OR}=1.2$  ,  $P\,{=}\,0.66)$  , demonstrating a gene–environment interaction   $\scriptstyle(P\,=\,0.03)$  ). A strong dose–response relationship was also seen among current smokers. No specific risk haplotypes were identified.  

# INTRODUCTION  

The ph os ph odi est erase 4D gene ( PDE4D ) has been implicated in the etiology of stroke, particularly the at hero sclerotic and card i oem boli c subtypes, based on linkage and association analyses carried out by deCODE Genetics in the Icelandic population (1). The human PDE4D  gene spans a   $1.6\,\mathbf{Mb}$   region on chromosome 5q12 and contains 24 exons (OMIM 600129). Through differential promoter and alternative splicing in gene expression, the gene can express nine different functional protein isoforms (Fig. 1). The associated single nucleotide polymorphisms (SNPs) are located among the first exons at the  $5^{\prime}$   end of the gene. The association with stroke is hypothesized to be mediated through cAMP, a key signal transduction molecule in multiple tissues and cell types, including vascular cells (2).  PDE4D  selectively degrades cAMP, which exerts protean effects on the vas cula ture and nervous system.  

Other linkage studies to the  PDE4D  region of chromosome 5q12 have been mixed, with a Swedish study (3) replicating the Icelandic finding, but an American study (4) reporting a negative result. Several studies have examined the deCODE markers in the region of PDE4D  where associations were found in Iceland. Some (3–8) but not all (9,10) studies have found associations in this region to all ischemic stroke or some stroke subtypes. Most (3,4,6, 9) but not all (5,8,10) studies have examined predominantly older and Caucasian populations and, with one exception (3), have examined only deCODE markers. Furthermore, no prior study had tested for gene–environment interactions. Therefore, we sought to extend the genetic epidemiology of  $P D E4D$   and stroke risk by examining a biracial case–control population of early-onset stroke, by performing novel SNP discovery and by assessing interactions with environmental exposures.  

# RESULTS  

# Subject characteristics  

Demographic and risk factor characteristics by case–control status are described in Table 1. The mean age of the cases and control subjects was 41.7 and 39.6 years, respectively. Cases were significantly more likely to have a history of hypertension   $(P<0.0001)$  ), diabetes   $P=$  0.0002) and angina/MI   $\!P=0.0005\!,$  ) and to currently smoke cigarettes   $(P<0.0001)$   and to report the use of oral contraceptive pills (OCP) within the month (31 days) prior to their stroke  $(P=0.032)$  ).  

# Novel SNP discovery on the non-coding conserved segments of the PDE4D gene  

We identified a total of 71 polymorphisms (66 SNPs and five insertion/deletion polymorphisms). Of these, 40 were confirmed to have a minor allele frequency  ${\geq}0.05$   in controls of at least one population with 19 observed in African-Americans only, four in Caucasians only and 17 in both races. Many of these SNPs were not listed in public or private databases [National Center for Biotechnology Information (NCBI), Celera’s Human SNP Reference Database, The SNP Consortium, Human Genome Variation Database and others].  

Twenty-three SNPs were genotyped in the entire case–control sample based upon the SNP selection criteria detailed in Materials and Methods. In brief, SNPs were chosen based upon their location within the  $P D E4D$   gene, with a special emphasis on the regions around the first exons at the  $5^{\prime}$   end of the gene, and based upon an MAF  ${\geq}0.05$   among control subjects. Table 2 lists the 23 genotyped SNPs as ordered by their physical position within the gene, and additionally provides the allelic variants, genotype call rates and minor allele frequencies among cases and controls stratified by race. Seventeen SNPs had a minor allele frequency  ${\geq}0.05$   among controls of both races. There were three SNPs unique to Caucasians and three SNPs unique to African-Americans that satisfied this criterion. All genotyped SNPs were in Hardy–Weinberg equilibrium in the combined case/control sample stratified by race.  

# SNPs associated with stroke  

Minimally adjusted model— Figure 1 illustrates the  $P$  -values from the additive model (primary analysis) of the 23 SNPs and their position within  $P D E4D$  . Results are shown for the entire population with adjustment for age and race and for African-American and Caucasian women separately with adjustment for age only. In the entire population, one SNP was significantly associated with stroke, rs918592   $\mathrm{{OR}}=1.49$  ,  $P=0.0077$  ). Age-adjusted racestratified analyses were not consistently significant, although the risk allele for SNP rs918592

 (the A allele) showed a similar degree of association in both races (African-American   $\mathrm{OR}=$  1.50,  $P=0.029$   and Caucasian  $\mathrm{OR}=1.52$  ,  $P=0.08J$  ). The A allele occurred more frequently in African-Americans (frequencies among cases and controls are 0.66 and 0.55, respectively) than in Caucasians (frequencies among cases and controls are 0.25 and 0.18, respectively). These findings are detailed in Tables 2 and 3. Using the GG genotype as the reference group, the age- and race-adjusted genotype effect of the AG heterozygotes risk allele demonstrated no stroke risk   $(\mathrm{OR}=1.21\$  ,  $P=0.51$  ) but significant stroke risk among AA homozygotes (OR  $=2.22$  ,  $P\,{=}\,0.014$  ). Age-adjusted race-specific genotype risk estimates demonstrated increased risk among African-Americans (AG:   $\mathrm{OR}=1.18$  ,  $P=0.63$  ; AA:  $\mathrm{OR}=2.73$  ,  $P=0.024)$  ) but not among Caucasians (AG:   $\mathrm{OR}=1.08$  ,  $P=0.91$  ; AA:  $\mathrm{OR}=1.77$  ,  $P=0.39$  ).  

As detailed in Materials and Methods, the permutation method was used to determine the multiple comparison-corrected  $P_{-}$  -value for the age–race-adjusted association of rs918592 with stroke. When the 17 SNPs with minor allele frequency greater than 0.05 in both races were considered, the empirically determined ‘experiment-wide’  $P_{-}$  -value was 0.08. When this subset was further reduced to include only our eight genotyped SNPs that were within the stroke associated Icelandic haplotype block (block B) (1), the empirically determined experimentwide  $P$  -value was 0.03. A similar analysis that included our four genotyped SNPs in Icelandic block A (1) also yielded an experiment-wide  $P_{-}$  -value was 0.03.  

Vascular model— Further analyses evaluating SNP rs918592 in an additive model controlling for age, smoking, OCP, HTN, DM and angina/MI (see Materials and Methods section) are shown in Table 3. The relationship was slightly attenuated   $(\mathrm{OR}=1.38,P=0.052)$  .  

Table 4 shows the risks among all participants (Caucasian, African-American and all others) for SNP rs918592, as stratified by age and other stroke risk factors with associated adjusted ORs,  $P$  -values and  $95\%$   confidence intervals. We found that this SNP was not associated with stroke among never-smokers   $(\mathrm{OR}=0.93$  ,  $P=0.75$  ) or former smokers   $(\mathrm{OR}=1.16$  ,  $P=0.66)$  , but was highly associated with stroke among smokers   $\mathrm{{OR}}=3.22$  ,  $P=0.00014\$  ). Further analyses demonstrated a strong dose–response relationship between stroke risk and smoking quantity. Among light smokers (1–10 cigarettes per day), the risk genotype had a weaker association with stroke   $\mathrm{OR}\,{=}\,2.27$  ,  $P\,{=}\,0.033)$  ), whereas among heavy smokers (  $11+$   cigarettes per day), the risk genotype has a stronger association with stroke   $\mathrm{{OR}}=8.39$  ,  $P=0.0004$  ). These results are demonstrated in Figure 2. The smoking by genotype interaction term was significant   $(P=0.03)$   in a model including the covariates of age, race, hypertension, diabetes mellitus and oral contraception. The risk genotype was not significantly associated with any of the stroke risk factors.  

Of the remaining 22 SNPs evaluated using these and other models, we found that several additional SNPs were associated with stroke, including: rs966221 (SNP 83) and rs1396476 (SNP 89) under an additive model, rs153031 (SNP 42) and rs918592 under a dominant model and rs1498606 under a recessive model. Details regarding these results are provided in  

Supplementary Material, Table S1. Risk factor stratified analyses of these associated SNPs are provided in Supplementary Material, Table S2.  

# Stroke subtype analyses  

In an exploratory analysis, stroke subtypes were analyzed separately. Table 5 demonstrates the results for SNP rs918592 as stratified by stroke subtype with associated adjusted ORs,  $P.$  -values and  $95\%$   confidence intervals. Similar analyses regarding the other associated SNPs are shown in Supplementary Material, Table S3. In brief, the at hero sclerotic group included 27 cases with either probable or possible at hero sclerotic mechanism, the cardiac group included 14 cases with a probable cardiac source of embolism, the probable dissection group included 13 cases confirmed by neuro imaging, the lacunar group included 45 cases of symptomatic small deep lesions on neuro imaging studies or classic lacunar syndromes regardless of other potential causes and the probable he mato logic group included nine cases. These categories were not mutually exclusive. There were 125 non-lacunar stroke cases of undetermined etiology. SNP rs918592 was significantly associated with all stroke subtypes except the cardiac and he mato logic subtypes. The analysis for the dissection subgroup was indeterminate as the model estimates did not converge.  

# Haplotype analyses  

Figures 3 and 4 show the haplotype block structure among African-Americans and Caucasians, respectively. High correlations existed within both races between eight SNPs (excluding SNP PDE4D-229902), which were in the promoter 1a isoform region of the gene, between SNP rs152312 (SNP 41) and rs918592. Of note, these figures demonstrate that linkage d is equilibrium exists between SNPs rs918592 [associated with stroke in the Stroke Prevention in Young Women Study 2 (SPYW2) population] and rs152312 (SNP 41, associated with stroke in the Iceland population) that is stronger among African-Americans   $(\mathrm{LD}=1.0)$  ) than Caucasians   $(\mathrm{LD}=0.66)$  . Our stroke-associated SNPs rs918592 and rs153031 (SNP 42) (Supplementary Material, Tables) (Figs 3 and 4) also demonstrate varying linkage d is equilibrium among African-Americans   $(\mathrm{LD}=0.59)$  ) and Caucasians (  $(\mathrm{LD}=0.95)$  ).  

Haplotype analyses including the eight SNPs identified through Haploview (as described in preceding paragraph) were performed among all SPYW2 participants, among the Caucasian and African-American subgroups and by stroke subtype. After correction for multiple comparisons testing, these analyses failed to demonstrate additional associations.  

# DISCUSSION  

Replication studies of the Icelandic findings have yielded mixed results. Figure 5 shows the SNPs examined by all published studies of  PDE4D  and stroke, demonstrating that work has primarily focussed on the portion of the gene where deCODE associations were found.  

Four European studies exclusively examined the strongest genotype and haplotype associations with all ischemic stroke or the combined at hero sclerotic/card i oem boli c group from the Iceland study and failed to replicate these findings (3,6,7,9). A German study examined SNP 41 (rs152312), SNP 45(rs12188950) and one micro satellite marker (AC008818-1), which had shown the most significant association with stroke in Iceland (9). They further examined a set of haplotype tagging SNPs that distinguished  $95\%$   of all chromosomes within linkage d is equilibrium blocks ‘B’ and   $\mathbf{\partialC}^{\prime}$  , which contained the haplotypes with the strongest association with stroke in Iceland. No positive associations with ischemic stroke were found.  

An English study (6) examined the positively associated Icelandic SNPs and haplotypes in relationship to carotid intima-media thickness and carotid plaque as well as risk of ischemic stroke and ischemic stroke subtypes. No SNPs or haplotyes were associated with all ischemic stroke or with the combined at hero sclerotic/card i oem boli c groups. Two SNPs, SNP 19 (rs4133470) and SNP 87(rs2910829) were found to be associated with large vessel stroke after adjustment for other risk factors and one of these two, SNP 87(rs2910829), was associated with carotid intima-media thickness at the carotid bulb. Six deCODE SNPs [SNP2(rs152341), SNP13(rs26949), SNP 14(rs26950), SNP 15(rs35382), SNP 20(rs16878206), SNP 26 (rs40512)] showed significant association with card i oem boli c stroke after multivariate adjustment, and haplotypes derived from these SNPs were less strongly associated than the individual SNPs.  

A Swedish study (3) of predominantly ischemic strokes found no association with the deCODE-associated variants, the A allele of AC008818-1 marker and deCODE SNPs 45 (rs12188950) and 41(rs152312) near PDE4D7 region. However, this study found associations with the B allele of AC008818-1 and an SNP near PDE4D3 region.  

A study in a geographically isolated population in the Netherlands (7) examined deCODE SNPs 39(rs3887175), 45(rs12188950), 83(rs966221) and their derived haplotypes. For the overall ischemic stroke group, no associations were found either in the total population or within inbred individuals. The subgroup of inbred individuals with small vessel disease did show an association with SNPs 39(rs3887175) and 45(rs12188950).  

In contrast, recent American (4,5) and Pakistani (8) studies found significant associations between deCODE SNPs and all ischemic stroke. One American (4) study, predominately consisting of Caucasians, found associations with two of four deCODE-associated variants, SNP 83(rs966221) and SNP 56(rs702553), with the most significant SNP 83(rs966221) having a stronger association with large artery at hero sclerotic stroke. The other American (5) study found SNP 87(rs2910829) to be associated with card i oem boli c stroke among both whites and blacks and SNP 41(rs152312) to be associated with card i oem boli c stroke among whites, all after multiple comparison corrections. The Pakistani study (8) examined SNP 32(rs456009), 83(rs966221) and 87(rs2910829) and found an association with SNP 83(rs966221).  

A recent Japanese (10) study examined three micro satellites, one VNTR and 31 deCODE SNPs, finding no associations with any of the polymorphisms examined after correcting for multiple comparisons. However, haplotype analyses within two separate blocks revealed significant association with cerebral infarction. The first block, consistent in location with the deCODE findings, revealed a significant haplotype   $P\!=\!0.002)$  ) comprising SNP83(rs966221), rs153031 and VNTR(AC008818). The second block in a different  PDE4D  gene region revealed a significant haplotype   $(P<0.001)$  ) comprising of rs878567, rs3756739, rs6875372 and rs1423351.  

Using existing information from databases, our SNP discovery efforts, as well as the Iceland results, we examined SNPs throughout the entire  PDE4D  gene with greater emphasis on the regions around the first exons at the  $5^{\prime}$   end of the gene. Among the deCODE stroke-associated variants evaluated in our study, several SNPs were found to be associated with ischemic stroke including SNP 83(rs966221) among Caucasians in an additive model, SNP 89(rs1396476) among African-Americans using an additive model and SNP 42(rs153031) in AfricanAmericans using a dominant model. Additionally, rs918592, which was most associated with stroke in our study, was in linkage d is equilibrium with rs152312(SNP 41), one of the polymorphisms more strongly associated with stroke in the Iceland population. Furthermore, the risk haplotype in the Iceland study contained rs152312 and had lower expression of the PDE4D 7 isoform (1).  

Although many of the studies of the  PDE4D  gene focussed on replicating the Icelandic findings, the association between specific SNPs and stroke have been inconsistent. Only the Japanese study (10) and our study have examined the  PDE4D  gene more extensively, but neither study has comprehensively examined the genetic variation within the  PDE4D  gene. The Japanese study identified two stroke-associated haplotype blocks. The more strongly associated haplotype was towards the  $3^{\prime}$   end of the gene. The other associated haplotype was in the  $5^{\prime}$   region of  $P D E4D$  , near the first alternative exon for  $P D E4D7$  , found to be associated with stroke in Iceland. Our strongest associated SNP was also in the  $5^{\prime}$   region of  PDE4D . Thus, our study adds independent support to the findings of the Iceland investigators and extends these findings to early-onset stroke among African-Americans as well as Caucasians.  

Our findings do not support the hypothesis that at hero genesis mediates the association. We have demonstrated an effect of the  PDE4D  locus on stroke risk among young adults, a population with a very low prevalence of at hero sclerotic disease. In contrast to the findings from Iceland (1) and the recent American (4) study, we found that our most strongly associated SNP, rs918592, was associated with multiple ischemic stroke subtypes. Significant findings were found not only for at hero sclerotic stroke, but also for lacunar stroke and non-lacunar stroke of undetermined etiology. This later group included large artery strokes without evidence for a significant degree of proximal atherosclerosis. Although our findings differed from the Iceland results in that there was no association with cardiac embolism, the underlying cardiac conditions associated with stroke in a young population are quite different from those among the elderly.  

Ours is the first study to examine the possibility of a  PDE4D  gene–environment interaction with smoking. The risk genotype was not associated with stroke among never-smokers or former smokers, but highly associated with stroke among current smokers. Additionally, a strong dose–response relationship was seen among current smokers. Replication of these findings, including studies of males and older stroke patients, will be essential because of the small sample size, the number of risk factors tested for interaction and the lack of a similar interaction among the other stroke-associated  PDE4D  SNPs. However, an interaction is biologically plausible. Cigarette smoking causes endo the li al dysfunction (11) and is known to modify the expression of many genes in endo the li al cells, including cAMP response element binding protein (CREB) (12). CREB is a transcription factor that induces the expression of a panel of genes with established roles in cell survival, metabolism and plasticity in the nervous system, including a potential role in ischemic preconditioning (13). Thus, both  PDE4D  and smoking may mediate effects via CREB providing a potential mechanism for the smoking interaction. An alternative mechanism could be that smoking may potentiate the effect of increased  PDE4D  expression. In a mouse model, pre-natal exposure to cigarette smoking increased  PDE4D  messenger RNA expression and decreased cAMP in the lung (14).  

In conclusion, we identified several  PDE4D  SNPs associated with stroke in both AfricanAmerican and Caucasian females, providing support for association of this gene with stroke in non-Icelandic populations and among young adults. Our most significantly associated SNP was found to be in strong linkage d is equilibrium with another previously associated with stroke in Iceland. The effect of the highly prevalent risk allele in this SNP was not subtype-specific but was specific to smokers. More comprehensive studies of the association between PDE4D  genetic variation and both functional correlates and stroke risk in larger, ethnically diverse populations are needed.  

# MATERIALS AND METHODS  

# Study subjects  

The SPYW2 is a population-based case–control study that was initiated to examine genetic risk factors for ischemic stroke in young women. The term ‘population-based’ indicates that the cases and their comparison group were identified from the same defined population, which included all of Maryland (except the far Western panhandle), Washington DC and the southern portions of both Pennsylvania and Delaware. Cases were 239 female patients 15–49 years of age with a first cerebral infarction as identified by discharge surveillance at 51 regional hospitals and through direct referral by regional neurologists. The methods for discharge surveillance, chart abstraction and case adjudication have been described previously (15–17). The adjudication of stroke cases was performed blinded to genetic information. Stroke cases were classified as having a probable, possible or undetermined etiology as per prior description (15,16). Using predetermined exclusions modified from the Siblings With Ischemic Stroke Study (SWISS) protocol (18), we excluded 15 (no. 15) cases with the following characteristics: sickle cell disease (no. 1), CNS vasculitis by angiogram and clinical criteria (no. 3), postradiation arte rio path y (no. 1), endocarditis (no. 3), neuro syphillis (no. 1), mechanical prosthetic heart valves (no. 2), left atrial myxoma (no. 1) and cocaine use in the  $48\,\mathrm{h}$   prior to their stroke (no. 3). Control subjects were 212 women without a history of stroke, identified by random digit dialing and were frequency matched to the cases by age and geographic region of residence. One control was excluded from analyses based on a history of sickle cell disease. Thus, the sample for genetic analyses consisted of 224 cases and 211 controls.  

Cases and controls were grouped into the following race/ethnic categories: Caucasian (nonHispanic)   $(n=95$   cases and 99 controls), African-American (  $(n=105$   cases and 91 controls) and others (including Hispanic, Asian, American-Indian, etc.)   $\acute{n}=24$   cases and 21 controls). Because of the small size and heterogeneity of the latter group, it was not analyzed separately, but was included within the combined total study group   $(n=224$   cases and 211 controls). Haplotype analyses were conducted only on the Caucasian (non-Hispanic) and the AfricanAmerican groups.  

# Previously identified SNPs  

We evaluated stroke risk within our population for several previously reported positively associated Icelandic deCODE Genetics SNPs (1). The following SNPs were evaluated: rs12188950 (deCODE SNP 45), rs152312 (SNP 41), rs153031 (SNP 42), rs966221 (SNP 83) and rs1396476 (SNP 89). All of these SNPs are in the region near the alternative forms of exon 1. SNPs were also chosen from the NCBI database based upon their location in the  PDE4D gene and their allele frequencies (minor allele frequency  ${\tt>}0.05$  ). Thus, rs27224, rs153067, rs42222, rs918590, rs918592, rs294496, rs525099, rs929820 and rs1498606 were included in this study.  

# Novel SNP discovery  

Screening for novel polymorphisms was performed on a subset of stroke cases and controls including both African-Americans and Caucasians (24 unrelated individuals for each group, in total 96 individuals). Because the  PDE4D  gene locus is very large and coding SNPs tend to occur very infrequently in exons (1), we performed a computational VISTA analysis prior to SNP discovery. Aiming to identify potentially functional non-coding variants, we selected highly evolutionarily conserved regions throughout the entire human  PDE4D  gene that were revealed in the VISTA map as the targets of our novel SNP discovery effort. Special focus was paid to the  PDE4D 7 isoforms and  PDE4D 1/2 isoforms, the  $5^{\prime}$   flanking regions   $(5\,\mathrm{k}\mathrm{b})$   for all PDE4D  isoform-specific promoters and the exon–intron junctions.  

VISTA alignments— Postulating that highly evolutionary-conserved genomic regions among species are likely to be functionally important, we used the VISTA web server (19) (http://www-gsd.lbl.gov/VISTA/) to align genomic sequences in our  in silico  analysis of global sequence conservation on the  PDE4D  gene locus. We aligned human (UCSC human genome assembly 2004 May chr5: 59968082.58282468), mouse (UCSC mouse genome assembly 2004 May chr13: 104923788.106765297) and rat (UCSC rat genome assembly 2003 June chr2:  

39230274.41294264) genomic sequences containing the entire  PDE4D  gene and its upstream

  $(100\,\mathrm{kb)}$   and downstream   $(20\,\mathrm{k\mathbf{b})}$   intergenic sequences. Trans pos able elements in the mammalian sequences were selected to be masked. Human sequence and its annotation were always used as the base sequence. Pairwise sequence comparisons were calculated with a threshold of  $75\%$   identity in a 100 bp window. All segments with a conservation  $-80\%$   over a 100 bp window or a conservation  $-75\%$   over 100 bp but within proximal  $5\,\mathrm{k}\mathrm{b}$   of the   $5^{\prime}$  - flanking regions for each  $P D E4D$   isoform-specific promoter were selected as the targets for the denaturing high-performance liquid chromatography (DHPLC) analysis.  

DHPLC analysis— PCR primers were designed to amplify the targeted regions. Genomic DNA (  $.10\,\mathrm{ng})$   was amplified by PCR using AmpliTaq Gold (Applied Biosystems, Foster City, CA, USA), denatured at   $94^{\circ}\mathrm{C}$   for   $5\;\mathrm{min}$   and slowly re-annealed at room temperature for he t ero duplex formation. Samples were then analyzed in the WAVE DNA Fragment Analysis System (Trans genomic Inc., Omaha, NE, USA) based on DHPLC, with the buffer gradient and oven temperature controlled by the WAVEMaker software. Heterozygous amplicons were identified by elution profiles. For each type of elution profile, three representative heterozygous samples were further analyzed on an ABI 3100 automatic sequencer (Applied Biosystems) in the forward and reverse directions. Sequence trace files were analyzed using the Mutation Surveyor software (Soft Genetics, State College, PA, USA) to locate nucleotide variants and were verified by manual inspections. In this study, only those polymorphic PCR amplicons (with minimal of three individuals for a DHPLC he t ero duplex elution profile) were subjected to further sequence analysis.  

# Genotyping methods for the case/control population  

After removing markers with minor allele frequency  ${<}0.05$   by race and among controls, we prioritized the remaining markers as follows: (i) greater minor allele frequencies, (ii) distribution throughout the gene, (iii) VISTA peaks and (iv) deCODE positive SNPs. Twentythree SNPs were genotyped in our stroke cohort (Table 2).  

Genotyping was conducted using DNA isolated from whole blood using the QIAamp DNA Blood Maxi Kit (Qiagen, Valencia, CA, USA). SNP typing was performed by one of two methods. The first method types 12 SNPs simultaneously and was developed for an SNPstream Ultra-High Throughput machine (Beckman Coulter, Inc., Fullerton, CA, USA) Sequences surrounding the SNPs were obtained from GenBank  

(http://www.ncbi.nlm.nih.gov/Genbank/index.html) and submitted to Autoprimer.com

 (Beckman Coulter, Inc.). For each SNP, three primers were designed, two for PCR amplification and an internal primer with a  $5^{\prime}\,\mathrm{DN}\mathrm{A}$   sequence tag. Twelve pairs of primers were used to initiate PCR amplification. The free primers were removed by enzymatic digestion using Exon ucl ease I and Shrimp Alkaline Ph os p hot as e (Beckman Coulter, Inc.). The internal primers were used to initiate a sequencing reaction that adds one labeled base for the alternative nucleotides of each SNP to have distinct labels. The labeled products are separated on a SNPIT plate consisting of 384 mini-arrays with 16 spots each (Beckman Coulter, Inc.). For each individual DNA sample, 16 spots hybridized to the two homozygotes, the heterozygote, a negative control and the 12 labeled primers associated with the 12 SNPs. Thus, every PCR and labeling reaction has internal controls to confirm the success of the reactions and the appropriateness of the fluorescent outputs for each DNA sample.  

The second method was the Taqman method (Applied Bio-systems). This method is based on four primers, two flanking the SNP that are used to amplify the DNA surrounding the SNP and two, one for each alternative allele, that were labeled with different fluorescent dyes. The original form of the labeled primer has a quencher in close proximity to the dye. However, when the exon ucl ease activity of DNA polymerase disrupts the primer hybridized to the singlestrand DNA during the PCR, the quencher and the dye are released and the fluorescence can be measured. The reaction itself follows manufacturer’s instructions included with each individual primer set.  

All statistical analyses were performed using   $\mathrm{SAS^{\mathrm{\textregistered}}}$  , Version 9.1 (SAS Institute, Cary, NC, USA). We compared means by   $t$  -tests and proportions by  $\chi^{2}$   tests using two-sided  $P_{-}$  -values. For the primary analysis, we considered an age-adjusted additive model to test the effect of genotype of stroke. Analyses were conducted in the total population and in African-American and Caucasian women separately. As secondary analyses, dominant and recessive genetic models were also considered. Genotype-specific ORs were also computed.  

Adjusted ORs derived by logistic regression were used to determine whether the presence of the risk allele was associated with an increased risk for stroke after controlling for potential con founders. As previously mentioned, the primary analysis or minimally adjusted model included age and race. A secondary vascular risk factor model was also implemented that included age, race, OCP use, hypertension, diabetes mellitus, current cigarette smoking and history of angina or myocardial infarction (angina/MI). Age, race, cigarette smoking status and OCP use were determined by subject reports (or proxy report, if a participant was unable to answer). Hypertension and diabetes mellitus were determined by asking study participants (or a proxy) whether a physician had ever told them that they had the condition.  

Significant SNPs underwent additional race-specific stratified analyses to evaluate groups stratified by standard risk factors (age, OCP use, current cigarette smoking, hypertension, diabetes mellitus and history of angina or myocardial infarction). Additionally, significant SNPs underwent a stroke subtype (at hero sclerotic, cardiac, dissection, lacunar, he mato logic and all other stroke) analyses adjusting for age and race. Tests for interaction were performed using logistic regression.  

Haplotype block structure among African-Americans and Caucasians was determined by implementing Haploview (http://www.broad.mit.edu/mpg/haploview/). Haplotypes were estimated and haplotypic association analyses were performed using the HaploStats package (20). Haplotypes based on eight SNPs demonstrating high pairwise correlation, as demonstrated through Haploview, were studied for association with stroke adjusting for multiple comparisons. The eight SNPs included were deCODE SNPs 41, 42 and 45, as well as rs27224, rs153067, rs42222, rs918590 and rs918592. Analyses were performed for all SPYW2 participants and stratified by race and stroke subtype.  

Permutation testing was used to empirically determine the multiple comparison-corrected  $P$  - value for the age–race-adjusted association of rs918592 with stroke.  

Because we considered multiple SNPs in our analyses, we are sensitive to the fact that a ‘significant’ association may have been observed with any one of them by chance alone. However, because many of the SNPs are correlated with each other, a Bonferroni-type adjustment would be too conservative. To account for the non-independence of SNPs, we therefore carried out a permutation test in which we permuted the genotype vector among subjects. In this analysis, we included all of our genotyped SNPs with a minor allele frequency greater than 0.05 in controls of both races; 17 SNPs (of the 23 analyzed) satisfied these criteria. Restricting the permutation to the genotype vector preserves the original allelic correlations between the SNPs, but disassociates any phenotype–genotype correlation that may have existed. To construct the distribution of expected  $P$  -values under the null hypothesis, we performed 5000 permutations, taking the smallest  $P$  -value from the association analysis of the 17 SNPs each time. We then defined our multiple comparison-corrected  $P$  -value as the  

proportion of the   $5000\,P$  -values obtained from the permutation that were lower than the  $P_{-}$  - value observed from our analysis of the real data. Following these analyses, we then evaluated our results in the context of the deCODE findings using a similar permutation testing methodology. In this case, the empiric experiment-wide  $P$  -value was based upon a reduced subset of SNPs which included only the eight SNPs (listed earlier) genotyped by our group that were within the stroke-associated Icelandic haplotype block (block B) (1). A similar analysis was performed on our four genotyped SNPs in Icelandic Haplotype block A (1), which included SNPs PDE4D-249037, PDE4D-327964 and deCODE SNPs 83 and 89.  

# Supplementary Material  

Refer to Web version on PubMed Central for supplementary material.  

# Acknowledgements  

The authors are highly indebted to Laurie Reinhart, MS for her contribution to data analyses reported in this manuscript and the SPYW2 study coordinator, Mary J. Sparks, RN. This project was partially supported through a grant from More-house Cardiovascular Research Institute (Enhancement of Cardiovascular and Related Research Areas, NIH/ NHLBI 5 UH1 HL003676), Morehouse Clinical Research Center (Center of Clinical Research Excellence, NIH/NCRR 5 U 54 RR 014758-05), Morehouse School of Medicine (Enhancement of the Capacity of Biomedical Research-RCMI, 5 G 12 RR 003034-18), Morehouse School of Medicine (Stroke Prevention/Intervention Research Program, NIH/ NINDS 5 U 54 NS 046798-02) and a Beginning-Grant-in-Aid of American Heart Association. Additionally, this project was partially supported by the Department of Veterans Affairs, Baltimore, Office of Research and Development, Medical Research Service, by the Geriatrics Research, Education and Clinical Center and by the Stroke Research Enhancement Award Program. Additional support was provided by the University of Maryland, Other TobaccoRelated Diseases Research Grant through the Maryland Cigarette Restitution Fund Program. Additional support was provided by a Cooperative Agreement with the Division of Adult and Community Health, Centers for Disease Control and Prevention; the National Institute of Neurological Disorders and Stroke and the NIH Office of Research on Women’s Health; the National Institute on Aging Pepper Center Grant P60 12583 and the University of Maryland General Clinical Research Center (Grant M01 RR 165001), General Clinical Research Centers Program, National Center for Research Resources, NIH.  

# References  

1. Greta rs do ttir S, S vein bjorn s do ttir S, Jonsson HH, Jakobsson F, E in ars do ttir E, Agnarsson U, Shkolny D, Einarsson G, Gud jons do ttir HM, Vald i mars son EM, et al. The gene encoding ph os ph odi est erase 4D confers risk of ischemic stroke. Nat Genet 2003;35:131–138. [PubMed: 14517540]

 2. Dominiczak AF, McBride MW. Genetics of common polygenic stroke. Nat Genet 2003;35:116–117. [PubMed: 14517535]

 3. Nilsson-Ardnor S, Wiklund PG, Lindgren P, Nilsson AK, Janunger T, Escher SA, Hallbeck B, Stegmayr B, Asplund K, Holmberg D. Linkage of ischemic stroke to the PDE4D region on 5q in a Swedish population. Stroke 2005;36:1666–1671. [PubMed: 16020760]

 4. Meschia JF, Brott TG, Brown RD Jr, Crook R, Worrall BB, Kissela B, Brown WM, Rich SS, Case LD, Evans EW, et al. Ph os ph odi est erase 4D and 5-lip oxygen as e activating protein in ischemic stroke. Ann Neurol 2005;58:351–361. [PubMed: 16130105]

 5. Woo D, Kaushal R, Kissela B, Sekar P, Wolujewicz M, Pal P, Alwell K, Haverbusch M, Ewing I, Miller R, et al. Association of Ph os ph odi est erase 4D with ischemic stroke: a population-based case– control study. Stroke 2005;37:371–376. [PubMed: 16373644]

 6. Bevan S, Porteous L, Sitzer M, Markus HS. Ph os ph odi est erase 4D gene, ischemic stroke, and a symptomatic carotid atherosclerosis. Stroke 2005;36:949–953. [PubMed: 15802632]

 7. van Rijn MJ, Slooter AJ, Schut AF, Isaacs A, Aulchenko YS, Snijders PJ, Kappelle LJ, van Swieten JC, Oostra BA, van Duijn CM. Familial aggregation, the PDE4D gene, and ischemic stroke in a genetically isolated population. Neurology 2005;65:1203–1209. [PubMed: 16162858]

 8. Saleheen D, Bukhari S, Haider SR, Nazir A, Khanum S, Shafqat S, Anis MK, Frossard P. Association of ph os ph odi est erase 4D gene with ischemic stroke in a Pakistani population. Stroke 2005;36:2275– 2277. [PubMed: 16166573]  

9. Lohmussaar E, G schw end t ner A, Mueller JC, Org T, Wichmann E, Hamann G, Meitinger T, Dichgans M. ALOX5AP gene and the PDE4D gene in a central European population of stroke patients. Stroke 2005;36:731–736. [PubMed: 15731479]

 10. Nakayama T, Asai S, Sato N, Soma M. Genotype and haplotype association study of the  STRK1 region on 5q12 among Japanese: a case–control study. Stroke 2006;37:69–76. [PubMed: 16322495]

 11. Celermajer DS, Sorensen KE, Georg a kop ou los D, Bull C, Thomas O, Robinson J, Deanfield JE. Cigarette smoking is associated with dose-related and potentially reversible impairment of endothelium-dependent dilatation in healthy young adults. Circulation 1993;88:2149–2155. [PubMed: 8222109]

 12. Zhang S, Day IN, Ye S. Microarray analysis of nicotine-induced changes in gene expression in endo the li al cells. Physiol Genomics 2001;5:187–192. [PubMed: 11328964]

 13. Ratan RR. cAMP response element binding protein family transcription factors: the Holy Grail of neurological therapeutics? Ann Neurol 2004;56:607–608. [PubMed: 15505827]

 14. Singh SP, Barrett EG, Kalra R, Razani-Boroujerdi S, Langley RJ, Kurup V, Tesfaigzi Y, Sopori ML. Prenatal cigarette smoke decreases lung cAMP and increases airway hyper responsiveness. Am J Respir Crit Care Med 2003;168:342–347. [PubMed: 12791581]

 15. Johnson CJ, Kittner SJ, McCarter RJ, Sloan MA, Stern BJ, Buchholz D, Price TR. Interrater reliability of an etiologic classification of ischemic stroke. Stroke 1995;26:46–51. [PubMed: 7839396]

 16. Kittner SJ, Stern BJ, Feeser BR, Hebel R, Nagey DA, Buchholz DW, Earley CJ, Johnson CJ, Macko RF, Sloan MA, et al. Pregnancy and the risk of stroke. N Engl J Med 1996;335:768–774. [PubMed: 8703181]

 17. Kittner SJ, Stern BJ, Wozniak M, Buchholz DW, Earley CJ, Feeser BR, Johnson CJ, Macko RF, McCarter RJ, Price TR, et al. Cerebral infarction in young adults: the Baltimore-Washington Cooperative Young Stroke Study. Neurology 1998;50:890–894. [PubMed: 9566368]

 18. Mes chia J F Brown RD Jr Brot tT G Chu kw ude lun zu FE Hardy J Rich S S(2002) The Siblings With Ischemic Stroke Study (SWISS) protocol.  BMC Med. Genet. ,  3 , Epub.

 19. Mayor C, Brudno M, Schwartz JR, Poliakov A, Rubin EM, Frazer KA, Pachter LS, Dubchak I. VISTA: visualizing global DNA sequence alignments of arbitrary length. Bioinformatics 2000;16:1046–1047. [PubMed: 11159318]

 20. Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. Am J Hum Genet 2002;70:425–434. [PubMed: 11791212]  

![](images/8ec904675a0ade3139aa76f02afddfd88ede553c1cec33a37264992fa0e42eff.jpg)  

# Figure 1.  

Association mapping of the PDE4D gene polymorphisms with ischemic stroke in the SPYW. Age-adjusted  $P_{-}$  -values among the overall population (race adjusted), Caucasians and AfricanAmericans for all SNPs analyzed using an additive model. The PDE4D gene structure is shown at the bottom. Solid boxes indicate the first exons for various isoforms, other exons are indicated by open boxes. Exons 2–5 are shown together, and exons 7–15 are shown together. Sideways arrows indicate promoters. Parenthesis indicate isoforms. The upward arrows on the gene demonstrate the SNP position within the gene as listed in Table 2.  

![](images/12a4953302ecf46075f97769a7e4eaca70cead724ae2e3b3bb703638dd1b6ba4.jpg)  

# Figure 2.  

ORs and   $P$  -values for SNP rs918592 among never-, former-, all current-smokers and among light smokers (1–10 per day) and heavy smokers (  $11+$   per day) demonstrating a strong dose– response relationship.  

![](images/2265376b367192fd08e3e6ea3c836544d1372121413dc8fee2b40eea6322de63.jpg)  

# Figure 3.  

Haplotype block structure among African-Americans. Arrows indicate SNPs rs918592 (associated with stroke in the SPYW2 population) and rs152312 (deCODE SNP 41, associated with stroke in the Iceland population) are in strong linkage d is equilibrium as indicated by the circle.  

![](images/d8aa0e7ea175371f071b17439be38dfadfac3d3b27240c174aca63589899a706.jpg)  

Figure 4. Haplotype block structure among Caucasians.  

![](images/8dc7190174e5b76eee0d495b242f2f330e05beb2e56f0b271ef5f7bcd4e90670.jpg)  

# Figure 5.  

A summary of   $P D E4D$   replication association studies in various populations. ( A ) The  $P D E4D$   gene structure. Exons are indicated by boxes. Differential promoters for  PDE4D isoforms are indicated by horizontal arrows on the gene. The alternative splicing patterns of  $P D E4D$   isoforms are shown below the gene structure. The SNPs that were associated with stroke in the loelandic population are shown above the gene structure by the vertical arrows.  $(\mathbf{B})$   The SNP position throughout the  PDE4D  gene used in the replication studies. ( C ) A summary of these replication studies with asterisks  $(^{*})$   representing SNPs that reached statistical significance in association with stroke.  

Characteristics by case–control status  

![](images/85b70f7f335729a0d86b4c477733bb37fc0f9ffce1ae5c5d76145b70b0544dbf.jpg)  

![](images/99a1b4e95e7fcf4b05b96e63dd6a1f5e4c6f656bbaab00f01d8e61cdae5fa27f.jpg)  

Relative position (148436–148557 exon 1a) (7). As listed by minor allele frequency in African-Americans (bolded in column 1). In race-stratified age-adjusted additive model, (primary analyses) as denoted in Fig. 1. SNP genotyped using the UHT system. a b c d  

SNP rs918592—additive model  

![](images/a7ac8039296f3153d739c6446bc04996222e934d34cb2edc9949d812980e3c5f.jpg)  

Table 4 Among all participants, SNP rs918592—additive model, stratified by age and other stroke risk factors demonstrating adjusted ORs,  $95\%$   confidence intervals and  $P_{-}$  -values 
![](images/4eb44ff32fbf97d99c265177af233c6688619165a28129d8e27e76fa49b4ab92.jpg)  
Significant results   $(P<0.05)$  ) are in bold. NC indicates insufficient sample size to perform calculation; model did not converge.  

Table 5 Among all participants, SNP rs918582—additive model, stratified by stroke subtype demonstrating adjusted ORs,  $95\%$   confidence intervals and  $P$  -values 
![](images/854b3a448335ae95449a046d5a4c85f750ab65f4695ea97bd62800661cc05307.jpg)  
Significant results   $(P<0.05)$  ) are in bold. NC indicates insufficient sample size to perform calculation; model did not converge.  