# Structural genomic variation in ischemic stroke  

# Mar Matarin ,  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

# Javier Simon-Sanchez ,  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

Unitat de Genética Molecular, Depart a men to de Genómica y Proteómica, Instituto de Biome dic in a de Valencia-CSIC, 46010 Valencia, Spain  

# Hon-Chung Fung ,  

Laboratory of Neuro genetics, National Institute on Aging, National Institutes of Health, 35 Convent Dr, Bethesda, MD 20892, USA  

Department of Neurology, Chang Gung Memorial Hospital and College of Medicine, Chang Gung University, Taipei, Taiwan  

Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, University College London, London, UK  

# Sonja Scholz ,  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

# J. Raphael Gibbs ,  

Computational Biology Core, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

# Dena G. Hernandez ,  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

# Cynthia Crews ,  

Laboratory of Neuro genetics, National Institute on Aging, National Institutes of Health, 35 Convent Dr, Bethesda, MD 20892, USA  

# Angela Britton ,  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

# Fabienne Wavrant De Vrieze ,  

Laboratory of Neuro genetics, National Institute on Aging, National Institutes of Health, 35 Convent Dr, Bethesda, MD 20892, USA  

# Thomas G. Brott ,  

Department of Neurology, Mayo Clinic, Jacksonville, FL 32256, USA  

# Robert D. Brown Jr. ,  

Department of Neurology, Mayo Clinic, Rochester, MN 55905, USA  

# Bradford B. Worrall ,  

Departments of Neurology and Public Health Sciences, University of Virginia, Char lott s ville, VA 22908, USA  

# Scott Silliman  

Department of Neurology, University of Florida College of Medicine, Jacksonville, FL 32256, USA  

# L. Douglas Case ,  

Section on Biostatistics, Division of Public Health Sciences, Wake Forest University Health Sciences, Medical Center Boulevard, Winston-Salem, NC 27157-1063, USA  

# John A. Hardy ,  

Reta Lila Weston Institute of Neurological Studies, Institute of Neurology, University College London, London, UK  

# Stephen S. Rich ,  

Center for Public Health Genomics, University of Virginia, Char lott s ville, VA 22908, USA  

# James F. Meschia , and  

Department of Neurology, Mayo Clinic, Jacksonville, FL 32256, USA  

# Andrew B. Singleton  

Molecular Genetics Section, National Institute on Aging, National Institutes of Health, Bethesda, MD 20892, USA  

Laboratory of Neuro genetics, National Institute on Aging, National Institutes of Health, 35 Convent Dr, Bethesda, MD 20892, USA  

Center for Public Health Genomics, University of Virginia, Char lott s ville, VA 22908, USA  

Mar Matarin: ; Javier Simon-Sanchez: ; Hon-Chung Fung: ; Sonja Scholz: ; J. Raphael Gibbs: ; Dena G. Hernandez: ; Cynthia Crews: ; Angela Britton: ; Fabienne Wavrant De Vrieze: ; Thomas G. Brott: ; Robert D. Brown: ; Bradford B. Worrall: ; Scott Silliman: ; L. Douglas Case: ; John A. Hardy: ; Stephen S. Rich: ; James F. Meschia: ; Andrew B. Singleton: Singleta@mail.nih.gov  

# Abstract  

Technological advances in molecular genetics allow rapid and sensitive identification of genomic copy number variants (CNVs). This, in turn, has sparked interest in the function such variation may play in disease. While a role for copy number mutations as a cause of Mendelian disorders is well established, it is unclear whether CNVs may affect risk for common complex disorders. We sought to investigate whether CNVs may modulate risk for ischemic stroke (IS) and to provide a catalog of CNVs in patients with this disorder by analyzing copy number metrics produced as a part of our previous genome-wide single-nucleotide polymorphism (SNP)-based association study of ischemic stroke in a North American white population. We examined CNVs in 263 patients with ischemic stroke (IS). Each identified CNV was compared with changes identified in 275 neurologic ally normal controls. Our analysis identified 247 CNVs, corresponding to 187 insertions   $76\%$  ; 135 heterozygous; 25 homozygous duplications or trip li cations; 2 heteros omi c) and 60 deletions   $(24\%$  ; 40 heterozygous deletions;3 homozygous deletions; 14 heteros omi c deletions). Most alterations  $(81\%)$   were the same as, or overlapped with, previously reported CNVs. We report here the first genome-wide analysis of CNVs in IS patients. In summary, our study did not detect any common genomic structural variation unequivocally linked to IS, although we cannot exclude that smaller CNVs or CNVs in genomic regions poorly covered by this methodology may confer risk for IS. The application of genome-wide SNP arrays now facilitates the evaluation of structural changes through the entire genome as part of a genome-wide genetic association study.  

# Keywords  

# Introduction  

Ischemic stroke (IS) is a heterogeneous multi factorial disorder. Studies in twins, families, and animal models provide substantial evidence for a genetic contribution to this disease [1,2]. Some conditions where stroke occurs are inherited in a classical Mendelian pattern; for example, mutations in  NOTCH3  underlie cerebral autosomal dominant arte rio path y with sub cortical infarcts and leuk o encephalopathy (CADASIL) [3], and mutation of  BRI  causes familial British dementia, an amyloid angiopathy, with white matter lesions without hemorrhage [4].  

Identifying individual causative mutations when stroke is the prominent feature remains problematic due to the complexity of genetic and environmental risk factors in this disease. While numerous genetic risk loci have been reported for IS, the majority of these have not been replicated. The most prominent exception is the reported association between  PDE4D  variants and ischemic stroke, first identified using an extended family based approach [5]. This work has been confirmed by several groups [6–13], although the variants associated at this locus are not consistent across populations [14].  

Application of genome-wide single-nucleotide polymorphism (SNP) assays in the context of case-control, rather than family-based studies, holds considerable promise in the identification of novel susceptibility loci, even in complex genetic disorders. SNP-based genome-wide association studies have proven successful in diseases with high-risk conferring alleles, such as variability in   $C F H$   in age-related macular degeneration and for lower risk alleles in type II diabetes (odds ratio of  ${\sim}1.1$   to 1.4), albeit with extremely large cohorts [15–17]. In addition to providing SNP association data, the genotyping assay also generates metrics that allow detection of copy number variants (CNVs), i.e., segments of the genome that may have been deleted or duplicated [18].  

The CNV distribution in the human genome and its potential association with risk for complex human disease have been the center of much discussion [19]. Large deletions and duplications have previously been associated with rare familial diseases [20,21], in addition to common traits such as  $\upalpha$  -thal as semi a [22], and color blindness [23]. We have recently performed a pilot study of genome-wide association in a North American cohort of ischemic stroke patients and controls [24]. We have now analyzed these data specifically to assess the role of CNVs in an attempt to determine whether structural genomic variation may contribute to risk for IS.  

# Materials and methods  

# Subject collection  

The Ischemic Stroke Genetics Study (ISGS) supplied all the stroke samples for the current study. ISGS is a prospective five-center North American case-control study. The protocol for ISGS has been reported previously [25]. For the stroke cohort, all cases had recent (within 30 days) firstever IS confirmed by history, physical examination, and head imaging (CT or MRI). Stroke was defined according to the World Health Organization (WHO) definition [26]. Iatrogenic, septic embolic, va so spastic, and vasculitic stroke cases were excluded.  

A single neurologist rater (RDB) classified ischemic strokes according to the Trial of Org 10172 in Acute Stroke Treatment (TOAST) [27], Oxfordshire [28], and Baltimore [29] criteria based on medical record review. Video-certified examiners assessed neurological impairment using the National Institutes of Health (NIH) Stroke Scale [30]. Functional status was assessed using the Barthel Index [31], Oxford Handicap Scale [32], and the Glasgow Outcome Score [33].  

The control cohort used here and the identification of structural alterations in this cohort have been previously described [18,34]. Neurologic ally normal subjects were briefly derived from three panels ( NDPT002, NDPT006 , and  NDPT009 ) containing DNA from 275 unrelated individuals from North America and one replicate sample (133 males and 142 females). Each panel contains DNA from 92 unrelated individuals without a history of Alzheimer disease, amyotrophic lateral sclerosis, ataxia, autism, bipolar disorder, brain aneurysm, dementia, dystonia, Parkinson’s disease, or stroke. None had any first-degree relative with a known primary neurological disorder, and the mean age of participants at sample collection was 68 years, ranging from 55 to 88 years (for more details, see http://ccr.coriell.org/ninds/catalog/panel/).  

# Sample preparation  

All individuals provided written consent for the genetic analysis. Epstein–Barr virus (EBV) immortal iz ation was performed as previously described [35,36]. DNA for the experiments was extracted from the EBV-immortalized lymphocyte cell lines (LCL); as we have previously shown, these LCLs remain highly faithful to the genotype of source [18].  

# Genotyping  

All samples were assayed with the Illumina Infinium Human-1 and Human Hap 300 SNP chips (Illumina Inc, San Diego, CA, USA). The Human-1 product assays 109,365 gene-centric SNPs and the Human Hap 300 product assays 317,511 haplotype tagging SNPs derived from phase I of the International HapMap Project (http://www.hapmap.org). There are 18,073 SNPs in common between the two arrays; thus, the assays combined provide data on 408,803 unique SNPs. Any assay with a call rate below  $95\%$   was repeated on a fresh DNA aliquot; if the call rate persisted below  $95\%$   the sample was excluded from further analysis.  

# CNV detection  

Data were analyzed using BeadStudio v2.1.10.0 (Illumina Inc., San Diego, CA). Two metrics were visualized using this tool: B allele frequency and log  $R$   ratio.  

The B allele frequency is the theta value for an individual SNP corrected for cluster position. This parameter provides an estimate of the proportion of times an individual allele at each polymorphism was called A or B. In this setting, an individual who is homozygous for the B allele (BB genotype) would have a score close to 1, an individual homozygous for the A allele (AA) would have a score close to 0, and an individual who is heterozygous (AB) would have a score of approximately 0.5. Significant deviations from these figures in contiguous SNPs are indicative of a CNV. The  $\log{\tt R}$   ratio is defined as the log (base 2) ratio of the observed normalized  $R$   value for the SNP divided by the expected normalized  $R$   value for the SNPs theta value. The expected  $R$   value is calculated from the values theta and  $R$  , where  $R$   is the intensity of dyelabeled molecules that have hybridized to the beads on the array and theta is the ratio of signal at each polymorphism for beads recognizing an A allele to beads recognizing a B allele. The expected  $R$   value for any individual at any typed SNP is calculated using a large population of typed individuals. Therefore, the ratio of observed  $R$   to expected  $R$   in any individual at any SNP gives an indirect measure of genomic copy number. An  $R$   value above 1 is indicative of an increase in copy number, and an  $R$   value below 1 suggests a decrease (deletion) in copy number. While this metric exhibits a high level of variance for individual SNPs, it does provide a measure of copy number when log  $R$   ratio values for numerous contiguous SNPs are visualized.  

Based on our own experience with this technology, we have established that the smallest copy number variation that can be reliably detected is  ${\sim}50\,\mathrm{k\mathbf{b}}$   [18,37]. We evaluated both the log  $R$   ratio and the B allele frequency plots across the genome in all samples. Each identified CNV was compared with changes identified in our neurologic ally normal control population [18] and those published in the Database for Genomic Variants (http://projects.tcag.ca/variation/). We calculated that our study had good power  $(\mathord{>}90\%)$   to detect rare variants conferring a genetic risk ratio of 3.75 or greater at an alpha of 0.05.  

# Results  

Analysis of CNVs in the control cohort was previously performed by us in a manner identical to that described here [18] and deposited publicly (http://projects.tcag.ca/variation/). Our previous work has demonstrated that the majority (10 of 10 alterations examined) of simple copy number changes  ${\leq}1$   Mb in size that were observed in LCLs are also apparent in the blood sample used for immortal iz ation. In contrast, those copy number changes larger than 1Mb (or present as heteros omi c alterations) appear to represent artifacts of the LCL creation and culture process (10 of 12 alterations examined), or correspond to V(D)J-like recombination events (two of two alterations examined) [18].  

Within the stroke cohort we identified a total of 231 CNVs that were simple deletions or duplications, corresponding to 185 insertions   $(80\%)$   and 46 deletions   $(20\%)$  , ranging in size from   $1.7\,\mathrm{k\mathbf{b}}$   to  $2.1\,\mathbf{M}\mathbf{b}$   (ESM-Tables 1 and 2). Most of the 231 simple CNVs have been previously reported in healthy individuals or overlap with previously reported CNVs (ESMTables 1 and 2). Forty-five of the 231 simple CNVs   $(19.5\%)$   are unique. Of these potential new sites of structural variation, only one genomic region, on chromosome 1, contained recurrent CNVs in three individuals with IS ( IS-14 ,  IS-236  and  IS-553 ). The three individuals showed an apparently identical duplication spanning the genes SPRY domain-containing SOCS box protein 1 ( SPSB1)  and hexose-6-phosphate dehydrogenase ( H6PD) .  

Because of the potential disease relevance of these alterations, we examined copy-number metrics at this locus in an additional 450 neurologic ally normal controls samples ( NDPT019, NDPT020, NDPT022, NDPT023 , and  NDPT024  from the NINDS neuro genetics repository at the Coriell Institute) using Illumina Infinium Human Hap 550 SNP chips (unpublished data). These data showed the presence of CNVs at this locus in five of these samples  $(\sim\!1\%)$  .  

We also identified 14 deletions and two duplications that were consistent with heteros omi c copy number changes (those where not all of the cellular population examined carry the CNV) ranging from  $30\,\mathrm{k\mathbf{b}}$   to an entire chromosome (E-Table 3). Of the 14 heteros omi c deletions,  $50\%$   spanned the immunoglobulin lambda gene cluster located at chromosome 22q11.22 and likely reflect normal   $\mathrm{V(D)J}$  -type recombination [38]. Thus, in the IS patients, 146 of the 263 samples demonstrated some form of copy number variation. Forty-nine of these samples have more than one CNV, with a maximum of five within two samples ( IS-1  and  IS-55) . Figure 1 shows the CNVs detected sorting by chromosome.  

# Discussion  

There is increasing discussion of the impact structural genomic alterations are likely to have in common diseases [19]. Cataloging CNVs using a methodology that assesses this variation in a genome-wide manner is critical for the identification of disease-associated genetic variability. With recent technologies such as the high-density SNP-based assays used here, an abundance of genomic copy number variations have been reported, ranging from kilobases to megabases in size. Further, this variation is readily identified in apparently healthy individuals [18,38–40].  

We report here the first genome-wide analysis of CNVs in IS patients. The CNVs identified in the current study were widely distributed throughout the genome. The majority of CNVs were rare (or orphan) changes. While single sample to group comparisons using the current methodology may under-represent common CNVs, the observation that the majority of alterations are rare is consistent with previous reports. A clear limitation of a SNP-array based approach for CNV observation is based in coverage. Many regions of the genome are poorly covered with SNPs using these technologies and thus many CNVs will be missed, in addition SNPs were previously excluded from inclusion in such arrays based on apparent Mendelian errors within families such an exclusion would clearly lead to the removal of SNPs in CNV regions. While this issue would be less of a problem for rare or orphan CNVs, such an exclusion leads to an under estimation of common CNVs.  

The accuracy of the current platform for identifying CNVs was previously evaluated [18,41]. The EBV immortal iz ation process and clonal nature of LCL culture has been shown to lead to structural genomic variation that is not detectable in the source tissue used for immortal iz ation. In our previous work, concordance rates between DNA derived from LCLs and DNA extracted from source tissue were  $100\%$   for  $\mathrm{{CNVs}\!\leq\!1}$   MB and  $17\%$   for  $\mathrm{CNVs\!>\!1\,MB}$   (excluding apparent CNVs resulting from V(D)J type recombination) [18]. Given this observation, we concentrated analyses on CNVs less than that   $1\,\mathbf{M}\mathbf{b}$   in size. Of the 45 CNVs identified that did not overlap with previously identified CNVs (Table 1), only 1 is recurrent in more than one IS sample; a duplication across  SPSB1  and  H6PD  identified in 3 individuals. However the presence of similar CNVs over this locus in 5 of an additional 460 controls suggests that this variant in not a risk factor for IS. The remaining CNVs may be of importance in the path o biology of IS; however, given the low frequency of each individual alteration, screening of these variants in a very large cohort (1000’s of cases and controls) would be required to make any unequivocal conclusions.  

In summary, our study did not detect any common genomic structural variation unequivocally linked to IS. We cannot exclude the possibility that smaller CNVs or CNVs in genomic regions poorly covered by this methodology may confer risk for IS. The recent availability of higher density CNV-directed arrays from both Affymetrix and Illumina will increase the number of CNVs that can be detected and thus may go some way toward addressing this question; however, because some studies have reported linkage d is equilibrium (LD) between CNVs and proximal SNPs [42,43], we would predict that our LD-based whole-genome association study [24] would have detected common CNVs linked to disease, whether detected directly or not in this study, by showing association at SNPs tagging such changes. The application of genomewide SNP arrays now facilitates the evaluation of structural changes through the entire genome as part of a genome-wide genetic association study.  

# Supplementary Material  

Refer to Web version on PubMed Central for supplementary material.  

# References  

1. Hassan A, Markus HS. Genetics and ischaemic stroke. Brain 2000;123(Pt 9):1784–1812. [PubMed: 10960044]

 2. Tournier-Lasserve E. New players in the genetics of stroke. N Engl J Med 2002;347:1711–1712. [PubMed: 12444190]

 3. Joutel A, Corpechot C, Ducros A, Vahedi K, Chabriat H, Mouton P, Alamowitch S, Domenga V, Cecillion M, Marechal E, Maciazek J, Vayssiere C, Cruaud C, Cabanis EA, Ruchoux MM, Weiss en bach J, Bach JF, Bousser MG, Tournier-Lasserve E. Notch3 mutations in CADASIL, a hereditary adult-onset condition causing stroke and dementia. Nature 1996;383:707–710. [PubMed:  

8878478]

 4. Vidal R, Frangione B, Rostagno A, Mead S, Revesz T, Plant G, Ghiso J. A stop-codon mutation in the BRI gene associated with familial British dementia. Nature 1999;399:776–781. [PubMed: 10391242]

 5. Greta rs do ttir S, Thor leif s son G, Rey n is do ttir ST, Manolescu A, Jonsdottir S, Jonsdottir T, Gud mun ds do ttir T, Bj arna do ttir SM, Einarsson OB, Gud jons do ttir HM, Hawkins M, Gud mun dss on G, Gud mun ds do ttir H, Andrason H, Gud mun ds do ttir AS, Sigurd ar do ttir M, Chou TT, Nahmias J, Goss S, S vein bjorn s do ttir S, Vald i mars son EM, Jakobsson F, Agnarsson U, Gudnason V, Thor geir s son G, Fingerle J, Gurney M, Gud bj arts son D, Frigge ML, Kong A, Stefansson K, Gulcher JR. The gene encoding ph os ph odi est erase 4D confers risk of ischemic stroke. Nat Genet 2003;35:131–138. [PubMed: 14517540]

 6. Meschia JF, Brott TG, Brown RD Jr, Crook R, Worrall BB, Kissela B, Brown WM, Rich SS, Case LD, Evans EW, Hague S, Singleton A, Hardy J. Ph os ph odi est erase 4D and 5-lip oxygen as e activating protein in ischemic stroke. Ann Neurol 2005;58:351–361. [PubMed: 16130105]

 7. Nilsson-Ardnor S, Wiklund PG, Lindgren P, Nilsson AK, Janunger T, Escher SA, Hallbeck B, Stegmayr B, Asplund K, Holmberg D. Linkage of ischemic stroke to the PDE4D region on 5q in a Swedish population. Stroke 2005;36:1666–1671. [PubMed: 16020760]

 8. van Rijn MJ, Slooter AJ, Schut AF, Isaacs A, Aulchenko YS, Snijders PJ, Kappelle LJ, van Swieten JC, Oostra BA, van Duijn CM. Familial aggregation, the PDE4D gene, and ischemic stroke in a genetically isolated population. Neurology 2005;65:1203–1209. [PubMed: 16162858]

 9. Saleheen D, Bukhari S, Haider SR, Nazir A, Khanum S, Shafqat S, Anis MK, Frossard P. Association of ph os ph odi est erase 4D gene with ischemic stroke in a Pakistani population. Stroke 2005;36:2275– 2277. [PubMed: 16166573]

 10. Nakayama T, Asai S, Sato N, Soma M. Genotype and haplotype association study of the STRK1 region on 5q12 among Japanese: a case-control study. Stroke 2006;37:69–76. [PubMed: 16322495]

 11. Woo D, Kaushal R, Kissela B, Sekar P, Wolujewicz M, Pal P, Alwell K, Haverbusch M, Ewing I, Miller R, Klein dor fer D, Flaherty M, Chakra bor ty R, Deka R, Broderick J. Association of Ph os ph odi est erase 4D with ischemic stroke: a population-based case-control study. Stroke 2006;37:371–376. [PubMed: 16373644]

 12. Brophy VH, Ro SK, Rhees BK, Lui LY, Lee JM, Umblas N, Bentley LG, Li J, Cheng S, Browner WS, Erlich HA. Association of ph os ph odi est erase 4D polymorphisms with ischemic stroke in a US population stratified by hypertension status. Stroke 2006;37:1385–1390. [PubMed: 16675738]

 13. Zee RY, Brophy VH, Cheng S, Hegener HH, Erlich HA, Ridker PM. Polymorphisms of the ph os ph odi est erase 4D, cAMP-specific (PDE4D) gene and risk of ischemic stroke: a prospective, nested case-control evaluation. Stroke 2006;37:2012–2017. [PubMed: 16825591]

 14. Rosand J, Bayley N, Rost N, de Bakker PI. Many hypotheses but no replication for the association between PDE4D and stroke. Nat Genet 2006;38:1091–1092. [PubMed: 17006457]author reply 1092– 1093

 15. Zeggini E, Weedon MN, Lindgren CM, Frayling TM, Elliott KS, Lango H, Timpson NJ, Perry JR, Rayner NW, Freathy RM, Barrett JC, Shields B, Morris AP, Ellard S, Groves CJ, Harries LW, Marchini JL, Owen KR, Knight B, Cardon LR, Walker M, Hitman GA, Morris AD, Doney AS, Burton PR, Clayton DG, Craddock N, Deloukas P, Duncanson A, Kwiatkowski DP, Ouwehand WH, Samani NJ, Todd JA, Donnelly P, Davison D, Easton D, Evans D, Leung HT, Spencer CC, Tobin MD, Attwood AP, Boorman JP, Cant B, Everson U, Hussey JM, Jolley JD, Knight AS, Koch K, Meech E, Nutland S, Prowse CV, Stevens HE, Taylor NC, Walters GR, Walker NM, Watkins NA, Winzer T, Jones RW, McArdle WL, Ring SM, Strachan DP, Pembrey M, Breen G, St Clair D, Caesar S, Gordon-Smith K, Jones L, Fraser C, Green EK, Grozeva D, Hamshere ML, Holmans PA, Jones IR, Kirov G, Moskvina V, Nikolov I, O'Donovan MC, Owen MJ, Collier DA, Elkin A, Farmer A, Williamson R, McGuffin P, Young AH, Ferrier IN, Ball SG, Balmforth AJ, Barrett JH, Bishop DT, Iles MM, Maqbool A, Yuldasheva N, Hall AS, Braund PS, Dixon RJ, Mangino M, Stevens S, Thompson JR, Bredin F, Tremelling M, et al. Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. Science 2007;316:1336–1341. [PubMed: 17463249]

 16. Saxena R, Voight BF, Lyssenko V, Burtt NP, de Bakker PI, Chen H, Roix JJ, Kathiresan S, Hirschhorn JN, Daly MJ, Hughes TE, Groop L, Altshuler D, Almgren P, Florez JC, Meyer J, Ardlie K, Bengtsson Bostrom K, Isomaa B, Lettre G, Lindblad U, Lyon HN, Melander O, Newton-Cheh C, Nilsson P, Orho-Melander M, Rastam L, Speliotes EK, Taskinen MR, Tuomi T, Guiducci C, Berglund A, Carlson J, Gianniny L, Hackett R, Hall L, Holmkvist J, Laurila E, Sjogren M, Sterner M, Surti A, Svensson M, Tewhey R, Blu mens tiel B, Parkin M, Defelice M, Barry R, Brodeur W, Camarata J, Chia N, Fava M, Gibbons J, Handsaker B, Healy C, Nguyen K, Gates C, Sougnez C, Gage D, Nizzari M, Gabriel SB, Chirn GW, Ma Q, Parikh H, Richardson D, Ricke D, Purcell S. Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. Science 2007;316:1331– 1336. [PubMed: 17463246]  

17. Scott LJ, Mohlke KL, Bonny castle LL, Willer CJ, Li Y, Duren WL, Erdos MR, Stringham HM, Chines PS, Jackson AU, Prokunina-Olsson L, Ding CJ, Swift AJ, Narisu N, Hu T, Pruim R, Xiao R, Li XY, Conneely KN, Riebow NL, Sprau AG, Tong M, White PP, Hetrick KN, Barnhart MW, Bark CW, Goldstein JL, Watkins L, Xiang F, Saramies J, Buchanan TA, Watanabe RM, Valle TT, Kinnunen L, Abecasis GR, Pugh EW, Doheny KF, Bergman RN, Tuomilehto J, Collins FS, Boehnke M. A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. Science 2007;316:1341–1345. [PubMed: 17463248]

 18. Simon-Sanchez J, Scholz S, Fung HC, Matarin M, Hernandez D, Gibbs JR, Britton A, de Vrieze FW, Peckham E, Gwinn-Hardy K, Crawley A, Keen JC, Nash J, Borgaonkar D, Hardy J, Singleton A. Genome-wide SNP assay reveals structural genomic variation, extended homo zy gos it y and cell-line induced alterations in normal individuals. Hum Mol Genet 2007;16:1–14. [PubMed: 17116639]

 19. Lupski JR. Structural variation in the human genome. N Engl J Med 2007;356:1169–1171. [PubMed: 17360997]

 20. Inoue K, Osaka H, Sugiyama N, Kawanishi C, Onishi H, Nezu A, Kimura K, Yamada Y, Kosaka K. A duplicated PLP gene causing Pelizaeus–Merzbacher disease detected by comparative multiplex PCR. Am J Hum Genet 1996;59:32–39. [PubMed: 8659540]

 21. Singleton AB, Farrer M, Johnson J, Singleton A, Hague S, Kachergus J, Hulihan M, Peuralinna T, Dutra A, Nussbaum R, Lincoln S, Crawley A, Hanson M, Maraganore D, Adler C, Cookson MR, Muenter M, Baptista M, Miller D, Blancato J, Hardy J, Gwinn-Hardy K. alpha-Synuclein locus trip li cation causes Parkinson's disease. Science 2003;302:841. [PubMed: 14593171]

 22. Lauer J, Shen CK, Maniatis T. The chromosomal arrangement of human alpha-like globin genes: sequence homology and alpha-globin gene deletions. Cell 1980;20:119–130. [PubMed: 6446404]

 23. Vollrath D, Nathans J, Davis RW. Tandem array of human visual pigment genes at Xq28. Science 1988;240:1669–1672. [PubMed: 2837827]

 24. Matarin M, Brown WM, Scholz S, Simon-Sanchez J, Fung HC, Hernandez D, Gibbs JR, De Vrieze FW, Crews C, Britton A, Langefeld CD, Brott TG, Brown RD Jr, Worrall BB, Frankel M, Silliman S, Case LD, Singleton A, Hardy JA, Rich SS, Meschia JF. A genome-wide genotyping study in patients with ischaemic stroke: initial analysis and data release. Lancet Neurol 2007;6:414–420. [PubMed: 17434096]

 25. Meschia JF, Brott TG, Brown RD Jr, Crook RJ, Frankel M, Hardy J, Merino JG, Rich SS, Silliman S, Worrall BB. The Ischemic Stroke Genetics Study (ISGS) Protocol. BMC Neurol 2003;3:4. [PubMed: 12848902]

 26. The World Health Organization MONICA. Project (monitoring trends and determinants in cardiovascular disease): a major international collaboration. WHO MONICA Project Principal Investigators. J clin Epidemiol 1988;41:105–114. [PubMed: 3335877]

 27. Adams HP Jr, Bendixen BH, Kappelle LJ, Biller J, Love BB, Gordon DL, Marsh EE 3rd. Classification of subtype of acute ischemic stroke. Definitions for use in a multi center clinical trial. TOAST. Trial of Org 10172 in Acute Stroke Treatment. Stroke 1993;24:35–41. [PubMed: 7678184]

 28. Bamford J, Sandercock P, Dennis M, Burn J, Warlow C. Classification and natural history of clinically identifiable subtypes of cerebral infarction. Lancet 1991;337:1521–1526. [PubMed: 1675378]

 29. Johnson CJ, Kittner SJ, McCarter RJ, Sloan MA, Stern BJ, Buchholz D, Price TR. Interrater reliability of an etiologic classification of ischemic stroke. Stroke 1995;26:46–51. [PubMed: 7839396]

 30. Lyden P, Brott T, Tilley B, Welch KM, Mascha EJ, Levine S, Haley EC, Grotta J, Marler J. NINDS TPA Stroke Study Group. Improved reliability of the NIH Stroke Scale using video training. Stroke 1994;25:2220–2226. [PubMed: 7974549]

 31. Collin C, Wade DT, Davies S, Horne V. The Barthel ADL Index: a reliability study. Int Disabil Stud 1988;10:61–63. [PubMed: 3403500]  

32. Bamford JM, Sandercock PA, Warlow CP, Slattery J. Inter observer agreement for the assessment of handicap in stroke patients. Stroke 1989;20:828. [PubMed: 2728057]

 33. Jennett B, Bond M. Assessment of outcome after severe brain damage. Lancet 1975;1:480–484. [PubMed: 46957]

 34. Fung HC, Scholz S, Matarin M, Simon-Sanchez J, Hernandez D, Britton A, Gibbs JR, Langefeld C, Stiegert ML, Schymick J, Okun MS, Mandel RJ, Fernandez HH, Foote KD, Rodriguez RL, Peckham E, De Vrieze FW, Gwinn-Hardy K, Hardy JA, Singleton A. Genome-wide genotyping in Parkinson’s disease and neurologic ally normal controls: first stage analysis and public release of data. Lancet Neurol 2006;5:911–916. [PubMed: 17052657]

 35. Miller G, Shope T, Lisco H, Stitt D, Lipman M. Epstein–Barr virus: transformation, cytopathic changes, and viral antigens in squirrel monkey and marmoset leukocytes. Proc Natl Acad Sci U S A 1972;69:383–387. [PubMed: 4333982]

 36. Tumilowicz JJ, Gallick GE, East JL, Pathak S, Trentin JJ, Arlinghaus RB. Presence of retrovirus in the B95-8 Epstein-Barr virus-producing cell line from different sources. In Vitro 1984;20:486–492. [PubMed: 6086497]

 37. Gibbs JR, Singleton A. Application of genome-wide single nucleotide polymorphism typing: simple association and beyond. PLoS Genet 2006;2:e150. [PubMed: 17029559]

 38. Sebat J, Lakshmi B, Troge J, Alexander J, Young J, Lundin P, Maner S, Massa H, Walker M, Chi M, Navin N, Lucito R, Healy J, Hicks J, Ye K, Reiner A, Gilliam TC, Trask B, Patterson N, Zetterberg A, Wigler M. Large-scale copy number polymorphism in the human genome. Science 2004;305:525– 528. [PubMed: 15273396]

 39. Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, Andrews TD, Fiegler H, Shapero MH, Carson AR, Chen W, Cho EK, Dallaire S, Freeman JL, Gonzalez JR, Gratacos M, Huang J, Kala it zo poul os D, Komura D, MacDonald JR, Marshall CR, Mei R, Montgomery L, Nishimura K, Okamura K, Shen F, Somerville MJ, Tchinda J, Valsesia A, Woodwark C, Yang F, Zhang J, Zerjal T, Armengol L, Conrad DF, Estivill X, Tyler-Smith C, Carter NP, Aburatani H, Lee C, Jones KW, Scherer SW, Hurles ME. Global variation in copy number in the human genome. Nature 2006;444:444–454. [PubMed: 17122850]

 40. Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y, Scherer SW, Lee C. Detection of large-scale variation in the human genome. Nat Genet 2004;36:949–951. [PubMed: 15286789]

 41. Huang J, Wei W, Zhang J, Liu G, Bignell GR, Stratton MR, Futreal PA, Wooster R, Jones KW, Shapero MH. Whole genome DNA copy number changes identified by high density oli go nucleotide arrays. Hum Genomics 2004;1:287–299. [PubMed: 15588488]

 42. McCarroll SA, Hadnott TN, Perry GH, Sabeti PC, Zody MC, Barrett JC, Dallaire S, Gabriel SB, Lee C, Daly MJ, Altshuler DM. Common deletion polymorphisms in the human genome. Nat Genet 2006;38:86–92. [PubMed: 16468122]

 43. Locke DP, Sharp AJ, McCarroll SA, McGrath SD, Newman TL, Cheng Z, Schwartz S, Albertson DG, Pinkel D, Altshuler DM, Eichler EE. Linkage d is equilibrium and heritability of copy-number polymorphisms within duplicated regions of the human genome. Am J Hum Genet 2006;79:275– 290. [PubMed: 16826518]  

![](images/d02915368b77ce5eaea9a8cf3822004e2e341e17e9bb9d2350b4d6191f7cd0c5.jpg)  

# Fig. 1.  

Simple CNVs detected sorting by chromosome in 263 patients with ischemic stroke. The red and the black stars indicate regions with insertions and deletions respectively. Regions where we found insertions and deletions are showed with violet starts. Numbers in parenthesis are the total number of CNVs (insertions in red, deletions in black) found in each chromosome. Chromosome Y was excluded from the analysis  

Structural changes found in ischemic stroke that have not been previously reported in healthy controls  

![](images/359a4dc9ab0e7435179895dd0c130a1fe692fffef4675446e6faf422b1b79578.jpg)  

![](images/9285ef8c9cf28478140743014bdb5dbce4d5f119ada730a13ecb5f0c33e6e77f.jpg)  