# Impact of carotid atherosclerosis loci on cardiovascular events  

Daiane Hemerich   a , , Sander W. van der Laan   c , Vinicius Tragante   a , Hester M. den Ruijter   c , , Gert Jan de Borst   d , Gerard Pasterkamp   a , , Paul I.W. de Bakker   f , , Folkert W. Asselbergs   a , , ,  

a  Department of Cardiology, Division of Heart and Lungs, University Medical Center Utrecht, Utrecht, The Netherlands b  CAPES Foundation, Ministry of Education of Brazil, Brasília, Brazil c  Laboratory of Experimental Cardiology, Division of Heart and Lungs, University Medical Center Utrecht, Utrecht, The Netherlands d  Department of Vascular Surgery, Division of Surgical Specialties, University Medical Center Utrecht, Utrecht, The Netherlands e  Laboratory of Clinical Chemistry and Hematology, Division Laboratories and Pharmacy, University Medical Center Utrecht, Utrecht, the Netherlands f  Department of Medical Genetics, Center for Molecular Medicine, University Medical Center Utrecht, Utrecht, The Netherlands g  Julius Center for Health Sciences and Primary Care, University Medical Center Utrecht, The Netherlands h  Durrer Center for Car dio genetic Research, ICIN-Netherlands Heart Institute, Utrecht, The Netherlands i  Institute of Cardiovascular Science, Faculty of Population Health Sciences, University College London, London, United Kingdom  

# a b s t r a c t  

# a r t i c l e i n f o  

Background:  Genome-wide association studies (GWAS) have identi ﬁ ed six single-nucleotide polymorphisms (SNPs) related to carotid intima media thickness (cIMT) or plaque. However, whether these loci relate to other vascular diseases and subsequent vascular events is unclear. Methods and results:  We tested six SNPs (rs4888378, rs11781551, rs445925, rs6601530, rs17398575 and rs1878406) for association with sub clinical at hero sclerotic measures (cIMT, plaque presence and anklebrachial index), as well as ischemic stroke, abdominal aortic aneurysm, peripheral or coronary artery disease (CAD) in the Second Manifestations of ARTerial disease (SMART) cohort. Four SNPs were associated with cIMT and two with plaque   $({\tt p}\,<\,0.05)$  . One SNP was also signi ﬁ cantly associated to CAD  ${\mathsf{O R}}=1.24,95\%\,{\mathsf{C I}}=1.08\mathrm{-}1.42,{\mathsf{p}}=2\times10^{-3}\mathrm{)}$  ). A genetic risk score (GRS) based on the cIMTrelated SNPs was associated to increased risk of cIMT itself   $({\mathfrak{p}}=1\times10^{-3})$  ), but not to other secondary outcomes or vascular events during follow-up   $\mathbf{\tau}(\mathbf{p}=0.86)$  ). Conclusions:  In addition to replicating previously published associations for cIMT, we con ﬁ rmed a nominally signi ﬁ cant effect between the GRS and cIMT.  

# Article history:  

Received 3 July 2015 Received in revised form 21 September 2015 Accepted 12 October 2015 Available online 19 October 2015  

Keywords: Atherosclerosis Single-nucleotide polymorphisms Carotid intima-media thickness Plaque  

$^\copyright$   2015 Elsevier Ireland Ltd. All rights reserved.  

# 1. Introduction  

pathways involved in the initiation and early phases of these diseases.  

Recently, genome-wide association studies (GWAS) identi ﬁ ed 6 single-nucleotide polymorphisms (SNPs)  [7,8]  associated to cIMT or plaque. The extent to which these loci are related to other subclinical phenotypes, clinically manifested vascular diseases and subsequent cardiovascular events is unclear. Therefore, we aimed to demonstrate the external validity of these  ﬁ ndings by testing whether these SNPs relate to cIMT, plaque, ankle-brachial index (ABI), ischaemic stroke (IS), abdominal aortic aneurysm (AAA), peripheral artery disease (PAD), and CAD in the SMART (Second Manifestations of ARTerial disease) cohort. We also modeled the 3 cIMT-related SNPs as a multilocus genetic risk score (GRS) and tested for association with CAD, IS, AAA, PAD, ABI, and cIMT itself, as well as with recurrent vascular events.  

Coronary artery disease (CAD) and stroke are leading causes of death worldwide  [1] , in ﬂ uenced by common genetic factors. Subclinical atherosclerosis, a thickening of the artery wall caused by the deposition of cholesterol material, often precedes these events [2,3] . Carotid intima-media thickness (cIMT) and plaque, measures of sub clinical atherosclerosis, have been shown to predict incident atherosclerosis-related cardiovascular disease  [4 e 6] . In this sense, genetic association studies may identify susceptibility genes and  

# 2. Methods  

# 2.1. Study populations and phe no typing  

We used data from the SMART cohort, consisting of patients from the University Medical Center Utrecht (UMCU), the Netherlands, included on the basis of manifest at hero sclerotic vascular disease or the treatment of at hero sclerotic risk factors [9,10]  ( Supplemental Table S1 ). From the 8210 patients included in the study, 3743 had CAD, 640 had AAA, 1726 had PAD, and 1764 had IS, with overlap among traits. Patients free of cardiovascular disease who had one or more risk factors for cardiovascular disease and were included in the SMART follow-up phase, served as a control group   ${\bf\tilde{n}}=1981$  ). All patients provided informed consent, and the Medical Ethics Committee of the UMCU approved the study.  

# 2.2. SNP selection, genotyping and quality control  

Based on a GWAS meta-analysis for cIMT and plaque  [7] , we selected 3 cIMT-associated SNPs (rs11781551, rs445925, and rs6601530), and 2 plaque-associated SNPs (rs17398575 and rs1878406). A second study associated rs4888378 with cIMT  [8] and was added to this analysis, resulting in a total of 4 cIMTassociated and 2 plaque-associated SNPs. Community standard quality control (QC)  [11]  was applied in all 6 SNPs. We excluded rs445925 from further analysis as it was out of Hardy e Weinberg Equilibrium   $({\bf p}=4.14\,\times\,10^{-9})$  .  

Wet-lab genotyping was carried out at K Bioscience s (Hertfordshire, UK,  www.k bioscience.co.uk ) whose personnel were blinded to patient status, using the proprietary KASPar PCR technique and TaqMan. Genotype calling was done using an automated system, with results checked manually using SNPviewer software.  

# 2.3. Statistical analysis  

Single SNP and GRS analyses were performed using linear and logistic regression models where appropriate, adjusting for sex and age. We considered the presence of plaque when cIMT > 1.1, following the same criteria as in Bis et al.  [7] .  

For each individual in our cohort we constructed an unweighted GRS using PASW Statistics 21 (SPSS, Inc., 2012, Chicago, IL, USA, www.spss.com ). The GRS was calculated as the sum of the number of risk alleles at the 3 cIMT-related SNPs (rs11781551, rs6601530, and rs4888378).  

We used Cox proportional hazards model to analyze the association between GRS and clinical events during follow-up, considering age, sex and inclusion criteria as covariates. To estimate quantitative effect sizes of the GRS on subsequent disease risk, we divided individuals into quartiles according to the GRS distribution and computed hazard ratios (HR) between the quartiles, with the ﬁ rst quartile as reference.  

Since all 5 SNPs that passed QC were previously associated with cIMT or plaque at genome-wide signi ﬁ cance   $({\mathfrak{p}}^{.}<5\,\times\,10^{-8})$  ), we considered a  $\mathsf{p}\!<\!0.05$   threshold (for the same risk allele in the same direction previously reported) as signi ﬁ cant for these two phenotypes. Correcting for multiple testing, the Bonferroni threshold de ﬁ ned for the other  ﬁ ve tested vascular beds (IS, AAA, CAD, PAD, and ABI) was   $\mathsf{p}<2\times10^{-3}$  .  

# 3. Results and discussion  

In this study SMART comprised a total of 8210 individuals (6229 cases and 1981 controls), aged 17 e 83 years, whose majority   $(67.5\%)$  is male.  

The single-SNP analysis for association with cIMT and plaque in SMART ( Table 1 ) resulted in 4 SNPs associated with cIMT and 2 with plaque   $\left({\bf p}<0.05\right)$  , all showing a concordant direction of effect with the originally reported associations (binomial   ${\tt p}=0.031$  ). This independent replication supports the role of these loci as genetic determinants of cIMT and plaque.  

In order to test the relation between these loci and other vascular beds, we tested their association with IS, AAA, CAD, PAD, and ABI ( Table 2 ). One variant (rs1878406) was signi ﬁ cantly associated with CAD (odds ratio   $[0\mathsf{R}]=1.24$  ,  $95\%$   con ﬁ dence interval  $[{\mathsf{C}}.{\mathsf{I}}.]=1.08\!-\!1.42$  ,  ${\mathfrak{p}}=2\times10^{-3}.$  . This SNP is located   $8.5~\mathrm{kb}$   from EDNRA , a gene related to endo the li al dysfunction. The endothelin receptor is a target to reduce blood pressure, given the vasoconstrictor role of endo the lins in blood pressure elevation and vascular hypertrophy  [12] . Interestingly, genetic variations in this gene have also been associated with atherosclerosis in hypertensive patients [13]  and with ambulatory blood pressure  [14] . Convincingly, previous reports also linked the same locus to CAD  [7,15] . This supports the hypothesis that  EDNRA  might affect atherosclerosis causing changes in blood pressure and thereby increasing the risk on CAD. SNP rs6601530 showed a nominally signi ﬁ cant association to CAD  $\mathbf{\dot{p}}=0.04)$   and ABI   $\mathbf{\check{p}}=0.006\mathbf{\check{p}}$  ). This SNP locates in an intron of PINX1, encoding Pin2-interacting protein 1, a telomerase inhibitor [16]  that relates to chromosomal segregation in mitosis  [17]  and has been implicated in cancer  [18,19] . A recent study  [20]  found that SNP rs7840785, also located in the intron of PINX1, was signi ﬁ - cantly associated with right carotid IMT   $\mathbf{\langlep=0.0003\rangle}$  ) in a nonEuropean population. The same study also conducted a genebased analysis in which PINX1 was signi ﬁ cantly associated with right carotid IMT. Even after removing the signi ﬁ cant SNP rs7840785, PINX1 was still signi ﬁ cantly associated in the overall sample   $({\sf p}\,=\,1\,\mathrm{~\times~}\,10^{-7})$  ). However, after correcting for multiple testing, we did not  ﬁ nd signi ﬁ cant associations with IS, AAA, PAD, or ABI.  

We further tested the combined effects of the cIMT-related SNPs in a multilocus GRS for association with disease risk ( Supplemental Table S2 ). We con ﬁ rmed the association between the GRS and cIMT  $\mathbf{\tau}(\mathbf{p}=0.04)$  )  [7] . We did not  ﬁ nd a signi ﬁ cant association between the  

Table 1 cIMT/plaque associated SNPs reported by literature and the association results for cIMT/plaque in SMART. 
![](images/38a916d36165c630031cecef236e3003f25fdcd28e60b706e8074e5563018e35.jpg)  
SNP: single-nucleotide polymorphism. Alleles: effect and non-effect alleles. EAF: effect allele frequency. Results listed in bold are nominally signi ﬁ cant   $\mathbf{\Psi}(\mathbf{p}=0.05)$  . HWE: Hardy e Weinberg Equilibrium.  

Table 2 Single SNP association results for IS, AAA, CAD, PAD and ABI in SMART. 
![](images/d6f6c1129c5b895a8133ae29c21a6d435c0abdc0d6153e1222eebabf0f828d74.jpg)  
IS, ischaemic stroke; AAA, abdominal aortic aneurysm; CAD, coronary artery disease; PAD, peripheral artery disease; ABI, ankle-brachial index; SNP, single-nucleotide polymorphism. N refers to the number of cases. Controls  $=1981$  . Bold values signi ﬁ es that the Bonferroni corrected threshold de ﬁ ned for the  ﬁ ve tested vascular beds (IS, AAA, CAD, PAD, and ABI) was   $\mathsf{p}<2\times10^{-3}$  .  

risk score and IS, AAA, CAD, PAD, or ABI. This can be due to the small effect the combined SNPs confer on the susceptibility of disease, or cardiovascular traits.  

We also analyzed the association between the GRS and vascular events during follow-up using Cox proportional hazard models  $({\mathrm{HR}}\,=\,0.977,$  ,   $95\%$   C.I. 0.93 e 1.07, survival curves in  Supplemental Figure S1 ). We found no increased risk of vascular events during follow-up when comparing individuals in the highest GRS quartile with those in the lowest quartile   $\mathbf{\check{p}}=0.865$  ). We may have had limited power due to the modest number of secondary events to ﬁ nd such an association, if it exists.  

The results showed in this study con ﬁ rm the original  ﬁ ndings [7,8] , adding to the evidence of the effect these loci have on cIMT or plaque presence. The GRS based on cIMT-related SNPs did not show a signi ﬁ cant effect on other vascular beds, or secondary vascular events, suggesting against pleiotropy. An explanation for this ﬁ nding could be that risk factors and their underlying genetic background may impact differently on each vascular bed in the a the rogen ic process. In conclusion, our  ﬁ ndings provide support for previously claimed SNP associations for cIMT and plaque, specifically highlighting the role of rs1878406 in both atherosclerosis and CAD. This should motivate further research focused on the underlying mechanisms involved.  

# Acknowledgments  

DH is supported by the National Council for the Improvement of Higher Education (CAPES) and Science without Borders Project, process no 13259/13-0. SWvdL is funded through grants from the Netherlands CardioVascular Research Initiative ( “ GENIUS ” , CVON2011-19), the Inter university Cardiology Institute of the Netherlands (ICIN, 09.001) and the FP7 EU project  CVgenes@target (HEALTH-F2-2013-601456). FWA is supported by the UCL Hospitals NIHR Biomedical Research Centre and by a Dekker Scholarship (Junior Staff Member 2014T001) from the Dutch Heart Foundation.  

We acknowledge the support from the  “ Netherlands CardioVascular Research Initiative ” , which includes the Dutch Heart Foundation, Dutch Federation of University Medical Centres, the Netherlands Organisation for Health Research and Development and the Royal Netherlands Academy of Sciences, for the GENIUS project  “ GENerating the best evIdence-based pharmaceUtical targetS for atherosclerosis ”  (CVON2011-19).  

We gratefully acknowledge the contribution of the SMART research nurses; R.van Petersen (data-manager); B.G.F. Dinther (vascular manager) and the participants of the SMART Study Group: A. Algra MD,PhD; Y. van der Graaf, MD,PhD; D.E. Grobbee, MD,PhD; G.E.H.M. Rutten, MD,PhD, Julius Center for Health Sciences and Primary care; F.L.J.Visseren, MD,PhD, Department of Internal Medicine; G.J. de Borst, MD,PhD, Department of Vascular Surgery; L.J. Kappelle, MD,PhD, Department of Neurology; T. Leiner, MD,PhD, Department of Radiology; P.A. Doevendans, MD,PhD, Department of Cardiology.  

# Appendix A. Supplementary data  

Supplementary data related to this article can be found at  http:// dx.doi.org/10.1016/j.atherosclerosis.2015.10.017 .  

# References  

[1]  D. Lloyd-Jones, et al., Executive summary: heart disease and stroke statistics e 2010 update: a report from the American heart association, Circulation 121 (7) (2010) 948 e 954 .

 [2]  E. Falk, Pathogen es is of atherosclerosis, J. Am. Coll. Cardiol. 47 (8 Suppl. l) (2006) C7 e C12 .

 [3]  C.G. Santos-Gallego, B. Picatoste, J.J. Badim on, Path o physiology of acute coronary syndrome, Curr. A the ros c ler. Rep. 16 (4) (2014) 1 e 9 .

 [4]  M. Amato, et al., Carotid intima-media thickness by B-mode ultrasound as surrogate of coronary atherosclerosis: correlation with quantitative coronary angiography and coronary intra vascular ultrasound  ﬁ ndings, Eur. Heart J. 28 (17) (2007) 2094 e 2101 .

 [5]  M.W. Lorenz, et al., Prediction of clinical cardiovascular events with carotid intima-media thickness: a systematic review and meta-analysis, Circulation 115 (4) (2007) 459 e 467 .

 [6]  I.M. van der Meer, et al., Predictive value of non invasive measures of atherosclerosis for incident myocardial infarction: the Rotterdam Study, Circulation 109 (9) (2004) 1089 e 1094 .

 [7]  J.C. Bis, et al., Meta-analysis of genome-wide association studies from the CHARGE consortium identi ﬁ es common variants associated with carotid intima media thickness and plaque, Nat. Genet. 43 (10) (2011) 940 e 947 .

 [8]  K. Gertow, et al., Identi ﬁ cation of the BCAR1-CFDP1-TMEM170A locus as a determinant of carotid intima-media thickness and coronary artery disease risk, Circ. Cardiovasc Genet. 5 (6) (2012) 656 e 665 .

 [9]  P.C. Simons, et al., Second manifestations of ARTerial disease (SMART) study: rationale and design, Eur. J. Epidemiol. 15 (9) (1999) 773 e 781 .

 [10]  S. Achterberg, et al., Patients with coronary, cerebro vascular or peripheral arterial obstructive disease differ in risk for new vascular events and mortality: the SMART study, Eur. J. Cardiovasc Prev. Rehabil. 17 (4) (2010) 424 e 430 .

 [11]  C.C. Laurie, et al., Quality control and quality assurance in genotypic data for genome-wide association studies, Genet. Epidemiol. 34 (6) (2010) 591 e 602 .

 [12]  R. Nakov, et al., Darusentan: an effective endo the linA receptor antagonist for treatment of hypertension, Am. J. Hypertens. 15 (7 Pt 1) (2002) 583 e 589 .

 [13]  H. Yasuda, et al., Association of single nucleotide polymorphisms in endothelin family genes with the progression of atherosclerosis in patients with essential hypertension, J. Hum. Hypertens. 21 (11) (2007) 883 e 892 .

 [14]  T. Rahman, et al., Common genetic variation in the type A endothelin-1 receptor is associated with ambulatory blood pressure: a family study, J. Hum. Hypertens. 22 (4) (2008) 282 e 288 .

 [15]  C.A.D. Consortium, et al., Large-scale association analysis identi ﬁ es new risk loci for coronary artery disease, Nat. Genet. 45 (1) (2013) 25 e 33 .

 [16]  X.Z. Zhou, K.P. Lu, The Pin2/TRF1-interacting protein PinX1 is a potent telomerase inhibitor, Cell 107 (3) (2001) 347 e 359 .

 [17]  K. Yuan, et al., PinX1 is a novel micro tubule-binding protein essential for accurate chromosome segregation, J. Biol. Chem. 284 (34) (2009) 23072 e 23082 .

 [18]  B. Zhang, et al., Silencing PinX1 compromises telomere length maintenance as well as tumor i geni city in telomerase-positive human cancer cells, Cancer Res. 69 (1) (2009) 75 e 83 .

 [19]  M.Y. Cai, et al., Decreased expression of PinX1 protein is correlated with tumor development and is a new independent poor prognostic factor in ovarian carcinoma, Cancer Sci. 101 (6) (2010) 1543 e 1549 .

 [20]  C. Li, et al., Genetic association and gene-smoking interaction study of carotid intima-media thickness at  ﬁ ve GWAS-indicated genes: the Bogalusa Heart Study, Gene 562 (2) (2015) 226 e 231 .  