# The association between mitochondrial DNA abundance and stroke: A  combination of multivariable-adjusted survival and Mendelian  random iz ation analyses  

Leon G. Martens   a , , Jiao Luo   a , , Marieke J.H. Wermer   c , Ko Willems van Dijk   d , , , Sara H agg   g ,  Felix Grassmann   g , , Raymond Noordam   a , Diana van Heemst   a  

a  Department of Internal Medicine, Section of Gerontology and Geriatrics, Leiden University Medical Center, Leiden, the Netherlands  b  Department of Clinical Epidemiology, Leiden University Medical Center, Leiden, the Netherlands  c  Department of Neurology, Leiden University Medical Center, Leiden, the Netherlands  d  Department of Human Genetics, Leiden University Medical Center, Leiden, the Netherlands  e  Department of Internal Medicine, Division of Endocrinology, Leiden University Medical Center, Leiden, the Netherlands  f  Einthoven Laboratory for Experimental Vascular Medicine, Leiden University Medical Center, Leiden, the Netherlands  g  Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden  h  Institute of Medical Sciences, University of Aberdeen, Forester hill, Aberdeen AB25 2ZD, UK  

# A R T I C L E  I N F O  

# Keywords:  

Ischemic stroke  Hemorrhagic stroke  Mitochondrial DNA copy numbers  Reactive oxygen species  Mitochondrial function  Cardiovascular diseases  Mendelian random iz ation  

# A B S T R A C T  

Background and aims:  Mitochondrial dysfunction is associated with increased reactive oxygen species (ROS) that  are thought to drive disease risk, including stroke. We investigated the association between mtDNA abundance,  as a proxy measure of mitochondrial function, and incident stroke, using multivariable-adjusted survival and  Mendelian Random iz ation (MR) analyses.  

Methods:  Cox-proportional hazard model analyses were conducted to assess the association between mtDNA  abundance, and incident ischemic and hemorrhagic stroke over a maximum of 14-year follow-up in European-ancestry participants from UK Biobank. MR was conducted using independent   $(\mathbf{R}^{2}<0.001)$  ) lead variants for  mtDNA abundance   $(p\,<\,5\,\times\,10^{-8})$   as instrumental variables. Single-nucleotide polymorphism (SNP)-ischemic  stroke associations were derived from three published open source European-ancestry results databases (cases/  controls): MEGASTROKE (60,341/454,450), UK Biobank (2404/368,771) and FinnGen (10,551/202,223). MR  was performed per study, and results were subsequently meta-analyzed.  

Results:  In total, 288,572 unrelated participants (  $.46\%$   men) with mean (SD) age of 57 (8) years were included in  the Cox-proportional hazard analyses. After correction for considered con founders (BMI, hypertension, choles­ terol, T2D), no association was found between low  versus  high mtDNA abundance and ischemic (HR: 1.06   $[95\%$  CI: 0.95, 1.18]) or hemorrhagic   $(\mathrm{HR};\,0.97$     $[95\%$   CI: 0.82, 1.15]) stroke. However, in the MR analyses after  removal of platelet count-associated SNPs, we found evidence for an association between genetically-influenced  mtDNA abundance and ischemic stroke (odds ratio, 1.17; confidence interval, 1.03, 1.32).  

Conclusions:  Although the results from both multivariable-adjusted prospective and basis MR analyses did not  show an association between low mtDNA and increased risk of ischemic stroke, in-depth MR sensitivity analyses  may suggest evidence for a causal relationship.  

# 1. Introduction  

Stroke is the second leading cause of death and loss of disability-adjusted life years worldwide [ 1 ]. Oxidative stress has been  hypothesized to play an important role in the path o physiology of stroke  by aggravating secondary damage and increases reperfusion injury after  ischemic stroke [ 2 – 4 ]. As a result of direct or indirect reactive oxygen  species (ROS)-induced damage to the (cerebral) vascular wall, multiple  

aspects of the vascular system are affected including platelet aggrega­ tion, endo the li al function, vascular permeability and va so dilation [ 3 ].  These local vessel changes induced by oxidative stress can also gradually  develop before stroke onset, and therefore may also lead to an increased  risk of stroke incidence [ 5 ].  

Mitochondria are a major source of ROS production [ 6 ]. Mitochon­ drial dysfunction leads to an increase in ROS production due to a change  in redox homeostasis [ 7 ]. Additionally, impaired mitochondrial  dysfunction, frequently proxied by the mitochondrial copy number  (mtDNA-CN) [ 8 , 9 ], has been associated with diseases such as diabetes,  heart failure, and neurological defects [ 10 ]. mtDNA-CN can be assessed  relatively easily in large populations by estimating mtDNA abundance  from the intensities of genotyping probes representing mitochondrial  DNA on genotyping arrays [ 9 , 11 , 12 ]. Increased ROS production drives  mitochondrial dysfunction causing increased defects in mitochondrial  fusion, fission, and mitophagy activation [ 13 ], which subsequently lead  to subsequent excessive ROS production [ 13 ].  

Although a relatively small study was not able to provide evidence of  an association between low mtDNA-CN and increased stroke risk [ 14 ],  we hypothesized that leukocyte mtDNA might affect brain path o logie s,  given the available biological data. Based on the combination of the  postulated detrimental biological effect of blood oxidative stress on the  (cerebro)vascular endo the li al system and its role in secondary damage  after stroke occurrence, we investigated the prospective association  between mtDNA abundance and incident ischemic and hemorrhagic  stroke in a large cohort of European-ancestry participants from the UK  Biobank. In addition, we applied Mendelian Random iz ation (MR) to  provide evidence for possible causality [ 15 , 16 ] as a way to tria ngu late  the results from prospective analyses by obtaining results from two  analysis methods, both with different assumptions and limitations [ 15 ].  

# 2. Materials and methods  

# 2.1. Population description  

The UK Biobank cohort is a prospective general population cohort  with 502,628 participants between the age of 40 and 70 years recruited  from the general population between 2006 and 2010 [ 17 ] (more in­ formation can be found online  https://www.ukbiobank.ac.uk ). Blood  samples were collected for genotyping. Access for information to invite  participants was approved by the Patient Information Advisory Group  (PIAG) from England and Wales. All participants in the UK Biobank  provided a written informed consent and local research ethics commit­ tees and institutional review boards approved the study. The present  study was accepted under project number 56340.  

In the present study, genotyped European-ancestry participants were  followed   $(\Nu=488{,}377)$  ). Exclusion criteria included: 1) non-European  ancestry; 2) participants who failed genotyping quality control and/or  with low call rate; 3) related individuals defined by principal compo­ nents (PCs); 4) participants with high SD of autosomal probes; 5) history  of any stroke at baseline; 6) missing covariates. After exclusion, the final  analyses were performed in 288,572 participants.  

# 2.2. Mitochondrial DNA abundance  

We used somatic mtDNA abundance as a proxy measure of mtDNA-CN, as the exposure, which is determined from the intensities of geno­ typing probes on the mitochondrial chromosome on the Affymetrix  Array. The method for computing mtDNA abundance has been described  previously [ 11 ]. In brief, the relative amount of mtDNA hybridized to  the array at each probe was the log2 transformed ratio (L2R) of the  observed genotyping probe intensity divided by the intensity at the same  probe observed in a set of reference samples. We used the median L2R  values across all 265 variants passing quality control on the MT chro­ mosome as an initial raw measure of mtDNA abundance. To correct for  confounding induced by poorly performing probes, we weighted L2R  values of each probe by multiplying the weight of the probe generated  from a multivariate linear regression model in which those intensities  statistically significantly predicted normalized mitochondrial coverage  from exome sequencing data, resulting in a single mtDNA abundance  estimate for each individual. To eliminate the plate effect, we subse­ quently normalized the abundance to mean of zero and SD of one within  each genotyping plate consisting of 96 wells [ 9 ].  

# 2.3. Covariates  

In addition to age and sex, we took into account data based on self-reported questionnaires (smoking, alcohol consumption, disease status,  medication use), blood cell counts (white blood cell counts and platelet  counts), body mass index (BMI) in  $\mathbf{kg}/\mathbf{m}^{2}$  , serum lipid levels (total and  LDL cholesterol) in mmol/L, and systolic and diastolic blood pressure in  mmHg.  

# 2.4. Outcome  

The outcome in the analysis was ischemic and hemorrhagic stroke  separately, as well as combined, in the time period August  2006 – January 2021. Stroke incidence was obtained via hospital  admission data and national health register data and used to identify the  date of the first stroke or stroke-related death after baseline assessment.  The primary outcomes were any stroke incidence and further specified  ischemic and hemorrhagic stroke incidence. Incident disease diagnoses  are coded according to the International Classification of Diseases edi­ tion 10 (ICD-10). Ischemic stroke was defined as I63 and hemorrhagic  stroke as I61. Any stroke was defined as the combination of I63 and I61.  Follow-up time is computed from baseline visit to diagnosis of incident  disease, loss-to-follow-up or death, or the end of the study period,  whichever came first.  

# 2.5. Data required for Mendelian Random iz ation analyses  

For MR, genetic variants of mtDNA abundance were used as instru­ ment variables. In a previous study, 129 independent single-nucleotide  polymorphisms (SNPs) as genetic variants were found to be indepen­ dently associated with mtDNA abundance at a genome-wide significance  threshold   $(p<5\times10^{-8})$  ; SNPs were additionally pruned to an LD   $\mathbb{R}^{2}<$  0.0001 [ 18 ]. The study was performed in a total of 465,809 individuals  using a combined population of the Cohorts for Heart and Aging  Research in Genomic Epidemiology (CHARGE) consortium and the UK  Biobank.  

# 2.6. Mendelian Random iz ation outcome datasets  

For the extraction of summary statistics on the associations of the  mtDNA abundance related SNPs with ischemic stroke, which was  defined as any ischemic stroke (I63), three large studies were used: the  MEGASTROKE consortium, the UK Biobank, and the FinnGen study [ 17 ,  19 ]. Both UK Biobank and FinnGen were not part of the main analyses of  the MEGASTROKE consortium preventing inclusion of overlapping  samples in the analyses. In the three studies insufficient data on hem­ orrhagic stroke was available.  

The  trans -ancestry meta-analysis from the MEGASTROKE con­ sortium was used to retrieve the ischemic stroke SNP-outcome data and  was based on 60,341 cases and 454,450 controls collected from 29  studies of predominantly European ancestry   $(86\%)$   [ 19 ].  

For the MR analyses in UK Biobank, cases developed before and after  enrolment were considered. Follow-up information that included  ischemic stroke occurrence was retrieved through the routinely avail­ able NHS database. In the European-ancestry dataset with full genomics  data available, we had data on 2404 cases of ischemic stroke and  368,771 controls. We performed new genome-wide association analyses  using linear mixed models to assess the associations between genetic  instruments and ischemic stroke, adjusted for age, sex and 10 principal  components, and corrected for familial relationships using BOLT_LMM  (v2.3.2).  

Data from FinnGen (Freeze 5;  https://www.finngen.fi/en/ ), which is  an ongoing cohort study launched in 2017, and analyses were based on  10,551 cases of ischemic stroke, and 202,223 controls.  

Although with lower numbers, we additionally performed MR on  subtypes of ischemic stroke (Cardio Embolic Stroke: 7193 cases,  406,111 controls, Large Artery Atherosclerosis: 4373 cases, 406,111  controls, Small Vessel Stroke: 5386 cases, 192,662 controls) using data  from MEGASTROKE and hemorrhagic stroke (1,687cases, 201,146  controls) from FinnGen.  

# 2.7. Statistical analysis  

# 2.7.1. Multivariable-adjusted analyses  

For the analyses and presentation purposes, we divided the study  population in 5 equally-sized groups based on mtDNA abundance, with  the first quintile containing the group with the lowest levels of mtDNA  abundance and the fifth quintile containing the highest levels (used as  reference).  

Baseline characteristics of the study population were presented  separately per quintile of mtDNA abundance, as mean (SD) for contin­ uous variables if they followed a normal distribution, or as median  (Interquartile range) otherwise, and frequency (proportion) for cate­ goric variables.  

The cumulative incidence for competing risk (CICR) was used to plot  the cumulative incidence of ischemic and hemorrhagic stroke against  follow-up time separately using a Kaplan-Meier survival curve by  mtDNA abundance quintiles, where death was accounted for as a  competing event. For any ischemic and hemorrhagic stroke, a Cox  proportional hazards model was used to estimate the hazard ratio (HR)  and   $95\%$   confidence interval (CI) presented as stroke incidence,  comparing the lowest   $20\%$   mtDNA abundance with the highest   $20\%$  .  Analyses were additionally done stratified by sex. Two multivariate  regression models were fitted, where for model 2 covariates were first  added individually:  

- Model 1: age, sex, batch, the first 10 genetic principal components,  white blood cell counts, platelet count   - Model 2: Model   $1~+~\mathtt{B M I}$  , smoking, alcohol consumption, total  cholesterol, hypertension, diabetes, cholesterol lowering medica­ tion, blood pressure lowering medication  

Covariates were included in regression models given their known  relation with both exposure and outcome (age, sex, smoking, alcohol  consumption, total cholesterol, disease status, medication status), or  were included as a technical correction due to measurement composi­ tion (batch, white blood cell count, platelet count). Participants were  censored in the event of loss-to-follow-up or death. To check whether the  proportional hazards assumption was fulfilled, a Cox proportional haz­ ard assumption test ( “ cox.zph ”  from R package  “ Survival ” ) was per­ formed. Additionally, mtDNA-CN was assessed continuously as a one-SD  lower mtDNA-CN on stroke incidence. Analyses were performed using  the  “ Survival ”  (cran.r-project.org/web/packages/survival) package in R  (v4.1.0)  

# 2.7.2. Mendelian Random iz ation  

All the analyses were done using R (v4.1.0) statistical software (The  R Foundation for Statistical Computing, Vienna, Austria). MR analyses  were performed using the R-based package  “ Two Sample MR ”  ( https://  mrcieu.github.io/Two Sample MR/ ) [ 20 ].  

For our primary MR analysis, Inverse-Variance weighted (IVW)  regression analyses were performed [ 16 ]. Estimates were calculated for  each genetic instrument using the Wald ratio (SNP  –  outcome associa­ tion divided by the SNP  –  exposure association) and subsequently  meta-analyzed using the inverse-weighted meta-analyses weighted on  the standard error of the SNP-outcome association (assuming no mea­ surement error [NOME] in the exposure) [ 21 ]. The calculated estimates  were expressed as odds ratios (OR) on ischemic stroke per SD (obtained  from the exposure data) difference in mtDNA abundance.  

To ensure that the results obtained from the IVW analyses were not  biased due to directional pleiotropy, we performed MR-Egger regression  analysis and Weighted-Median Estimator [ 21 ]. Although MR-Egger is  considered as a relatively inefficient approach (e.g., large confidence  intervals), this method does not force the regression line to go through  the intercept. The intercept depicts the estimated average pleiotropic  effect across the genetic variants, and a value that differs from zero in­ dicates that the IVW estimate is biased [ 22 ]. The Weighted-Median  estimator analysis can provide a consistent valid estimate if at least  half of the instrumental variables are valid [ 23 ]. In addition,  MR-PRESSO (MR Pleiotropy RESidual Sum and Outlier) was applied to  detect and correct for horizontal pleiotropy through removing outlying  causal estimates based on individual instruments [ 24 ], as implemented  in the R-based package  “ MR-PRESSO ”  ( https://github.com/rondolab/  MR-PRESSO ) .  The Cochran ’ s Q statistic was performed in order to test  the heterogeneity between the estimated Wald ratios from different  genetic variants [ 25 ]. Additionally, a Steiger directional it y test was  performed to ensure consistent causal direction-of-effect. A power  calculation was performed with mRnd ( https://shiny.cns genomics.  com/mRnd/ ) [ 26 ]. With power  $=0.80$  , minimal effect size (OR) was  1.076.  

Recent research has proven that two-sample MR methods can safely  be used for one-sample MR in large databases [ 27 ]. This allows us to use  the UK Biobank database in our sample set despite also being used as our  exposure dataset. As a limitation to this method, results of MR-Egger  analyses are to be interpreted with caution when used to check for  pleiotropy [ 27 ].  

The main MR analyses were performed in the individual datasets,  and subsequently meta-analyzed to derive the pooled estimates for the  exposure on the risk of ischemic stroke using a fixed-effect model.  Heterogeneity testing of the estimates across three datasets was per­ formed by   $\mathrm{I}^{2}$  , and corresponding p-value was obtained from the  Cochran ’ s Q test. All meta-analyses were performed in the R-based  “ meta ”  package ( https://cran.r-project.org/web/packages/meta/index.  html ).  

2.7.3. Sensitivity analysis after stratification of genetic instruments  

SNPs identified in relation to mtDNA-CN have been found in relation  to platelet activation and mega kary o cyte proliferation [ 18 ], which both  could affect stroke risk and could potentially lead to biased results, we  first examined the associations between the SNPs and platelet count in  our study sample (adjusted for age, sex, and the first 10 genetic principal  components); all SNPs with  $p{<}(0.05/123)$   in its association with platelet  count were excluded from further MR sensitivity analyses.  

# 3. Results  

# 3.1. Baseline characteristics of the study population  

A total of 288,572 participants were included in the final study  sample (see full procedure in  Supplementary Fig. 1 ) for multivariable-adjusted survival analyses. Participants with unavailable genetic data 

  $\mathbf{\Gamma}(\mathbf{N}=14{,}251)$  ), not used to compute the genetic principal components (N 

  $=81{,}623)$  ), or having unrealistic SD of autosomal probes   $\left(\Nu\,=\,9440\right)$  )  were excluded according to standard UK Biobank quality control rec­ om mend at ions. Subsequently, we excluded related participants   $(\Nu\,=$  38,642), and those with a non-white British ancestry   $(\Nu\,=\,65,498)$  .  Finally, 4602 participants were excluded due to a history of stroke  before study enrollment. Participants in the lower mtDNA abundance  quintile ( Table 1 ) had a mean age of 57.5  versus  56.1 year in the highest  quintile, a mean BMI of 27.7  versus  $27.0\,\mathrm{kg}/\mathrm{m}^{2}$  , T2D prevalence of   $2.8\%$  

Table 1  Baseline characteristics of the study participants stratified by quintiles of mtDNA abundance.    
![](images/1508d0db5728465622f31806a9195795325a68d73c85b27179d417ed32859f8e.jpg)  

Data are mean (SD) for continuous variables or percentages for dichotomous variables. mtDNA abundance is presented as normalized in unit standard deviations. BMI,  Body Mass Index.  

versus   $2.0\%$  , and   $11.3\%$   were current smokers compared with   $8.4\%$   in  the highest quintile.  

# 3.2. Multivariable-adjusted survival analyses mtDNA abundance and  stroke  

A total of 6218 of the 288,572 participants   $(2.15\%)$   had a stroke  incidence, of which 3994   $(1.38\%)$   were ischemic and 1883   $(0.65\%)$  hemorrhagic over a median (IQR) follow-up of 11.8 (11.1 – 12.5) years.  The incidence of ischemic stroke was higher in the lower mtDNA-CN  quintiles than in the higher quintiles ( Fig. 1 A), while hemorrhagic  stroke incidence was similar in all mtDNA-CN quintiles ( Fig. 1 B); in both  cases the analyses fulfilled the proportional hazard assumption   $\acute{p}$  - value :  0.84 and 0.88).  

After stratification based on mtDNA-CN ( Table 2 ), in model 1,  mtDNA abundance was associated with any stroke and ischemic stroke  incidence, when comparing the first quintile with the highest   $20\%$  mtDNA abundance (any stroke: hazard ratio (HR), 1.11;  $95\%$   confidence  interval (CI): 1.02 to 1.20; ischemic stroke: HR, 1.15;   $95\%$   CI: 1.04 to  1.27). Similarly, a one-SD increase in mtDNA abundance was associated  with lower risk of incident ischemic stroke (HR, 0.96;   $95\%$   CI: 0.93 to  0.99). No association was found between mtDNA abundance and inci­ dent hemorrhagic stroke.  

After correcting for other con founders, the associations with stroke  and ischemic stroke attenuated (any stroke: HR, 1.06;   $95\%$   CI: 0.97 to  1.16; ischemic stroke: HR, 1.07;   $95\%$   CI: 0.95 to 1.19), as did the  continuous model on ischemic stroke (HR, 0.98;   $95\%$   CI: 0.94 to 1.01).  

# 3.3. Mendelian Random iz ation on mtDNA abundance and ischemic  stroke  

# 3.3.1. Main analyses  

We did not observe evidence favoring an association between  genetically-influenced lower mtDNA-CN and ischemic stroke ( Fig. 2 ).  

![](images/099d6d000cbd697b60a8697d10fa7229aed9af95cc44bd9d647bccc5b0506f48.jpg)  
Fig. 1.  Cumulative incidence of ischemic (A) and hemorrhagic (B) stroke by quintiles of mtDNA abundance.  We calculated the cumulative incidence for ischemic and hemorrhagic stroke, accounting for death as a competing event. Differences in cumulative incidence  between groups were assessed using Gray ’ s test.  

Table 2  The multivariable-adjusted association between mtDNA abundance and incident stroke in European-ancestry participants from UK Biobank.     
![](images/e1c4c7136037005c7ec5a18ee39505187ec9c75c0ea0ca71e9225ad64cffbd68.jpg)  
Estimated hazard ratios per-SD increase in mtDNA abundance (continuous), or for the 1st to the 4th quintile compared with the 5th (reference) quintile (categorical) on  any, ischemic, and hemorrhagic stroke. Model 1 includes age, sex, batch, PCs, white blood cell count, platelet. Model 2 includes model 1, BMI, smoking, total  cholesterol, hypertension, diabetes, cholesterol lowering medication, blood pressure lowering medication. CI, confidence interval; HR, hazard ratio.  

![](images/cb67184177ab0199ad500a4b03486124716748dca1d303b3cee7b5e96d55f6ad.jpg)  
Fig. 2.  Causal association between mtDNA abundance and ischemic stroke occurrence.  

The odds ratios per 1 SD less mtDNA-CN were 1.07 (  $95\%\mathrm{CI}$  : 0.95, 1.20)  in MEGASTROKE, 1.04 (  $95\%\mathrm{CI}$  : 0.79, 1.37) in the UK Biobank, and 0.99   $\scriptstyle{.95\%C I}$  : 0.82, 1.20) in FinnGen. After meta-analysis, in a combined  sample size of 1,098,740 (of which 73,296 cases), the pooled odds ratio  was 1.04   $.95\%\mathrm{CI}$  : 0.95 to 1.15) per 1-SD decrease in genetically-influenced mtDNA abundance.  

The exact set of variants, their corresponding coefficients, standard  errors, and  $p$  -values are presented in  Supplementary Table 1 . Variance  explained   $(\mathbf{R}^{2})$   was  $2.0\%$   and calculated based on the derived summary  statistics. The MR-Egger intercept indicated no pleiotropy   $(p>0.05)$  .  Although several outliers were identified with MR-PRESSO in MEGA­ STROKE and FinnGen, results remained similar after removal of these  outlying SNPs. The Steiger test of directional it y showed a correct causal  direction, indicating that there is no evidence for reverse causation, and  no different results were observed with MR-sensitivity analyses, MR-Egger and weighted median ( Supplementary Table 2 ).  

Sub-analyses performed with separate outcomes, card i oem boli c,  large artery atherosclerosis, small-vessel, and hemorrhagic stroke  ( Supplementary Fig. 2  and  3 ), showed no evidence favoring a different  result.  

# 3.3.2. Additional sensitivity analyses  

A total of 61 SNPS were associated with platelet count, which were  subsequently excluded from additional sensitivity analyses. In the full  sample, a 1-SD genetically-determined lower mtDNA abundance was  associated with a higher risk of ischemic stroke (OR: 1.165;   $95\%$   CI:  1.026 to 1.323), although results from FinnGen did not align with those  obtained in UK Biobank and MEGASTROKE ( Supplementary Fig. 4 ).  

# 4. Discussion  

In the UK Biobank cohort, consisting of 288,572 participants after  exclusion, an initial association was found between mtDNA abundance  and incident ischemic stroke, which attenuated after adjustment for  con founders. Consistent with the prospective analyses, MR analyses,  using a total sample size of 73,296 cases and 1,025,444 controls, showed  no evidence for an association between genetically-predicted mtDNA  abundance and ischemic stroke. However, some in-depth sensitivity  analyses in which SNPs associated with platelet count were excluded,  did provide some preliminary evidence for low mtDNA-CN as possible  causal driver for ischemic stroke.  

Although of specific interest, caution in these results is warranted  given that they were mainly driven by results derived from MEGA­ STROKE, and to a lesser extent by UK Biobank. Furthermore, results  from these additional MR sensitivity analyses deviated significantly  from those observed in the prospective multivariable-adjusted analyses,  and therefore do not meet the requirements for triangulation [ 15 ].  Collectively, our results indicate that there is only weak evidence for a  causal association between mtDNA abundance and ischemic stroke, and  more studies are required to elucidate the nature of the pleiotropy  identified in our study, which goes beyond the current scope.  

Previously, an association was observed between low mtDNA-CN  and increased risk of incident stroke in 20,162 participants, followed  over a 13.5-year period, during which 1584 stroke events occurred [ 28 ],  and therefore deviate from our study done in a larger sample of 288,752  participants with 6218 stroke cases. Difference in baseline health  characteristics are possible reasons explaining the observed differences  in results.  

Recent studies showed that mtDNA-CN could be a marker of stroke  prognosis after hospitalization [ 29 , 30 ]. By analyzing mtDNA-CN, and  consequently oxidative stress, our findings did provide some, albeit  circumstantial, evidence for a relationship between oxidative stress and  stroke occurrence, although this association attenuated after adjustment  for con founders. In the Mendelian Random iz ation analysis, after  excluding SNPs associated with platelet count, we also found an asso­ ciation between genetically determined mtDNA abundance and  ischemic stroke risk. In contrast to ischemic stroke, we did not find an  association between mtDNA abundance and hemorrhagic stroke in  univariate or MR analyses. This difference might be explained because  hemorrhagic stroke, in contrast to ischemic stroke, is also often caused  by non-classic cardiovascular mechanisms such as vascular amyloid  deposition in cerebral amyloid angiopathy [ 31 ].  

Our data on mtDNA abundance was obtained from leukocytes.  Although some of the leukocytes may be directly involved in the pa­ thology of stroke, additional cell types such as endo the li al and smooth  muscle cells, that we did not query for mitochondrial abundance, are  clearly more directly involved. This could potentially explain our overall  null findings. Studies on the differences in mitochondrial function  within an individual between cell groups are largely non-existent.  However, mtDNA-CN measured in blood has been associated with  gene expression in other tissues, which suggests mtDNA-CN derived  from leukocytes can reflect metabolic health across multiple tissues  [ 32 ]. Thus, the evidence so far indicates that mitochondrial dysfunction,  as measured with leukocyte mtDNA-CN, is systemic. Of interest, using  similar methodology as in our study, low genetically-influenced mtDNA  has recently been associated with increased dementia risk [ 12 ]. This  would further indicate that lower mtDNA-CN, although measured in  leukocytes, can reflect processes of a systemic increase in disease risk.  

A key strength of this study is the statistical power of the analyses of  the association between stroke and mitochondrial abundance (288,572  participants for the multivariable survival analysis and 1,098,740 for the  MR, respectively). Additionally, we adopted the triangulation of causal  inference [ 15 ]. By using two different approaches in observational  research to study the association between low mtDNA abundance and  (ischemic) stroke risk, we increased the credibility of our results.  Although results from both our used approaches were not exactly  similar, they were direction ally consistent.  

Some limitations are to be considered. First, mtDNA abundance was  determined from intensities of genotyping probes on mitochondrial DNA,  whereas the assessment with whole-exome sequencing is generally  considered to result in more reliable mtDNA abundance estimates [ 33 , 34 ].  Although H agg et al. showed a moderate correlation between mtDNA  based on SNP array intensities and exome sequencing of 0.33 [ 11 ], ana­ lyses still indicated the measurements of SNP array intensities reflect un­ derlying biology of mtDNA abundance. For this reason, the increased  variance is most likely the result of non differential measurement error, and  therefore considered to mainly cause a reduction in statistical power. As a  consequence, the true associations, particularly those from the  multivariable-adjusted prospective analyses, are most likely larger than  observed. Second, our study population consists of predominantly Cauca­ sian participants, limiting the general iz ability of the results to other  ancestry groups. Third, Mendelian Random iz ation functions on several  assumptions. However, using several sensitivity analyses such as MR-Egger  and MR-PRESSO, we can establish with some conviction that these are  fulfilled. In addition, although in a one-sample MR (as conducted in the UK  Biobank) the assumption of independence does not hold up, previous  studies have shown that two-sample MR methods can be used reliably with  large enough biobanks [ 27 ]. Last, despite a large sample size in the  multivariable adjusted analysis, stroke, especially hemorrhagic stroke  incidence, was relatively few. However, as an association was found before  correction, we think our analyses had enough power to detect a difference  between groups. In addition, we used one of the larger data sets available.  

In conclusion, despite a large sample size, our prospective study did not  find evidence for an association between mtDNA abundance and ischemic  or hemorrhagic stroke. After exclusion of pleiotropic SNPs associated with  platelet count, we found some preliminary evidence for an association  between genetically determined lower mtDNA-CN and ischemic stroke risk  using MR. However, further studies are required for validation and to  examine the nature of this type of pleiotropy.  

# Financial support  

This work was supported by the VELUX Stiftung [grant number  1156] to DvH and RN, and JL was supported by the China Scholarship  Counsel [No. 201808500155]. RN was supported by an innovation grant  from the Dutch Heart Foundation [grant number 2019T103 to R.N.].  Parts of this work were funded by the Åke Wibergs Foundation (grant  number M19-0294 to F.G).  

# Author contributions  

LGM, JL, RN and DvH designed research; LGM and JL conducted  research; LGM and JL performed statistical analysis; LGM, JL, RN, and  MJHW wrote paper; LGM had primary responsibility for final content.  DvH, KWvD, SH, FG and MJHW contributed to the data interpretation  and commented on initial versions of the manuscript; All authors read  and approved the final manuscript.  

# Declaration of interests  

The authors declare that they have no known competing financial  interests or personal relationships that could have appeared to influence  the work reported in this paper.  

# Acknowledgements  

The authors are grateful to the UK Biobank for allowing us the use of  their data. The analyses done in UK Biobank were done under project  number 56340. Furthermore, the authors acknowledge the participants  and investigators of the MEGASTROKE consortium and the FinnGen  Biobank who contributed to the summary statistics data which are made  available for further studies.  

# Appendix A. Supplementary data  

Supplementary data to this article can be found online at  https://doi.  org/10.1016/j.atherosclerosis.2022.06.1012 .  

# References  

[1]  H. Wang, Global age-sex-specific fertility, mortality, healthy life expectancy  (HALE), and population estimates in 204 countries and territories, 1950-2019 : a  comprehensive demographic analysis for the Global Burden of Disease Study 2019 ,  Lancet 396 (2020) 1160 – 1203 . 

 [2]  R. Rodrigo, R. Fernandez-Gajardo, R. Gutierrez, J.M. Matamala, R. Carrasco,  A. Miranda-Merchak, W. Feuerhake, Oxidative stress and path o physiology of  ischemic stroke: novel therapeutic opportunities, CNS Neurol. Disord.: Drug  Targets 12 (2013) 698 – 714 . 

 [3]  C.L. Allen, U. Bay rak tut an, Oxidative stress and its role in the pathogen es is of  ischaemic stroke, Int. J. Stroke 4 (2009) 461 – 470 . 

 [4]  X. Duan, Z. Wen, H. Shen, M. Shen, G. Chen, Intra cerebral hemorrhage, oxidative  stress, and antioxidant therapy, Oxid. Med. Cell. Longev. 2016 (2016), 1203285 . 

 [5]  R. Shirley, E.N. Ord, L.M. Work, Oxidative stress and the use of antioxidants in  stroke, Antioxidants 3 (2014) 472 – 501 . 

 [6]  L.L. Lackner, J.M. Nunnari, The molecular mechanism and cellular functions of  mitochondrial division, Biochim. Biophys. Acta 1792 (2009) 1138 – 1144 . 

 [7]  B. Bandy, A.J. Davison, Mitochondrial mutations may increase oxidative stress:  implications for carcinogen es is and aging? Free Radic. Biol. Med. 8 (1990)  523 – 539 . 

 [8]  J.K. Chung, S.Y. Lee, M. Park, E.J. Joo, S.A. Kim, Investigation of mitochondrial  DNA copy number in patients with major depressive disorder, Psychiatr. Res. 282  (2019), 112616 . 

 [9]  R.J. Longchamps, C.A. Castellani, S.Y. Yang, C.E. Newcomb, J.A. Sumpter, J. Lane,  M.L. Grove, E. Guallar, N. Pankratz, K.D. Taylor, et al., Evaluation of mitochondrial  DNA copy number estimation techniques, PLoS One 15 (2020) e0228166 . 

 [10]  J.L. Yang, S. Mukda, S.D. Chen, Diverse roles of mitochondria in ischemic stroke,  Redox Biol. 16 (2018) 263 – 275 . 

 [11]  S. Hagg, J. Jylhava, Y. Wang, K. Czene, F. Grassmann, Deciphering the genetic and  epidemiological landscape of mitochondrial DNA abundance, Hum. Genet. 140  (2021) 849 – 861 . 

 [12]  M. Chong, P. Mohammadi-Shemirani, N. Perrot, W. Nelson, R. Morton, S. Narula,  R. Lali, I. Khan, M. Khan, C. Judge, et al., GWAS and ExWAS of blood  Mitochondrial DNA copy number identifies 71 loci and highlights a potential  causal role in dementia, Elife (2022) 11 . 

 [13]  F. Liu, J. Lu, A. Manaenko, J. Tang, Q. Hu, Mitochondria in ischemic stroke: new  insight and implications, Aging Dis 9 (2018) 924 – 937 . 

 [14]  M. Wachsmuth, A. Hubner, M. Li, B. Madea, M. Stoneking, Age-related and  he ter op las my-related variation in human mtDNA copy number, PLoS Genet. 12  (2016) e1005939 . 

 [15]  D.A. Lawlor, K. Tilling, G. Davey Smith, Triangulation in a e tio logical  epidemiology, Int. J. Epidemiol. 45 (2016) 1866 – 1886 . 

 [16]  S. Burgess, A. Butterworth, S.G. Thompson, Mendelian random iz ation analysis  with multiple genetic variants using summarized data, Genet. Epidemiol. 37  (2013) 658 – 665 . 

 [17]  C. Sudlow, J. Gallacher, N. Allen, V. Beral, P. Burton, J. Danesh, P. Downey,  P. Elliott, J. Green, M. Landray, et al., UK biobank: an open access resource for  identifying the causes of a wide range of complex diseases of middle and old age,  PLoS Med. 12 (2015) e1001779 .  

[18]  R.J. Longchamps, S.Y. Yang, C.A. Castellani, W. Shi, J. Lane, M.L. Grove, T.  M. Bartz, C. Sarnowski, C. Liu, K. Burrows, et al., Genome-wide analysis of  mitochondrial DNA copy number reveals loci implicated in nucleotide metabolism,  platelet activation, and mega kary o cyte proliferation, Hum. Genet. 141 (2022)  127 – 146 . 

 [19]  R. Malik, G. Chauhan, M. Traylor, M. Sar guru prem raj, Y. Okada, A. Mishra,  L. Rutten-Jacobs, A.K. Giese, S.W. van der Laan, S. Greta rs do ttir, et al.,  Multi ancestry genome-wide association study of 520,000 subjects identifies 32 loci  associated with stroke and stroke subtypes, Nat. Genet. 50 (2018) 524 – 537 . 

 [20]  G. Hemani, J. Zheng, B. Elsworth, K.H. Wade, V. Haberland, D. Baird, C. Laurin,  S. Burgess, J. Bowden, R. Langdon, et al., The MR-Base Platform Supports  Systematic Causal Inference across the Human Phenome, vol. 7, Elife, 2018 . 

 [21]  S. Burgess, S.G. Thompson, Interpreting findings from Mendelian random iz ation  using the MR-Egger method, Eur. J. Epidemiol. 32 (2017) 377 – 389 . 

 [22]  J. Bowden, G. Davey Smith, S. Burgess, Mendelian random iz ation with invalid  instruments: effect estimation and bias detection through Egger regression, Int. J.  Epidemiol. 44 (2015) 512 – 525 . 

 [23]  J. Bowden, G. Davey Smith, P.C. Haycock, S. Burgess, Consistent estimation in  mendelian random iz ation with some invalid instruments using a weighted median  estimator, Genet. Epidemiol. 40 (2016) 304 – 314 . 

 [24]  M. Verbanck, C.Y. Chen, B. Neale, R. Do, Detection of widespread horizontal  pleiotropy in causal relationships inferred from Mendelian random iz ation between  complex traits and diseases, Nat. Genet. 50 (2018) 693 – 698 . 

 [25]  M.F. Greco, C. Minelli, N.A. Sheehan, J.R. Thompson, Detecting pleiotropy in  Mendelian random is ation studies with summary data and a continuous outcome,  Stat. Med. 34 (2015) 2926 – 2940 .  

[26]  M.J. Brion, K. Shakhbazov, P.M. Visscher, Calculating statistical power in  Mendelian random iz ation studies, Int. J. Epidemiol. 42 (2013) 1497 – 1501 . 

 [27]  C. Minelli, M.F. Del Greco, D.A. van der Plaat, J. Bowden, N.A. Sheehan,  J. Thompson, The use of two-sample methods for Mendelian random iz ation  analyses on single large datasets, Int. J. Epidemiol. (2021) . 

 [28]  F.N. Ashar, Y. Zhang, R.J. Longchamps, J. Lane, A. Moes, M.L. Grove, J.  C. My chal eck yj, K.D. Taylor, J. Coresh, J.I. Rotter, et al., Association of  mitochondrial DNA copy number with cardiovascular disease, JAMA Cardiol 2  (2017) 1247 – 1255 . 

 [29]  L. Song, T. Liu, Y. Song, Y. Sun, H. Li, N. Xiao, H. Xu, J. Ge, C. Bai, H. Wen, et al.,  mtDNA copy number contributes to all-cause mortality of lacunar infarct in a  Chinese prospective stroke population, J Cardiovasc Transl Res 13 (2020)  783 – 789 . 

 [30]  M.R. Chong, S. Narula, R. Morton, C. Judge, L. Akhabir, N. Cawte, N. Pathan,  R. Lali, P. Mohammadi-Shemirani, A. Shoamanesh, et al., Mitochondrial DNA copy  number as a marker and mediator of stroke prognosis: observational and  mendelian random iz ation analyses, Neurology 98 (2022) e470 – e482 . 

 [31]  R.A. Grysiewicz, K. Thomas, D.K. Pandey, Epidemiology of ischemic and  hemorrhagic stroke: incidence, prevalence, mortality, and risk factors, Neurol.  Clin. 26 (2008) 871 – 895, vii . 

 [32]  S.Y. Yang, C.A. Castellani, R.J. Longchamps, V.K. Pill a lamarr i, B. O ’ Rourke,  E. Guallar, D.E. Arking, Blood-derived mitochondrial DNA copy number is  associated with gene expression across multiple tissues and is predictive for  incident neuro degenerative disease, Genome Res. 31 (2021) 349 – 358 . 

 [33]  P. Zhang, B.D. Lehmann, D.C. Samuels, S. Zhao, Y.Y. Zhao, Y. Shyr, Y. Guo,  Estimating relative mitochondrial DNA copy number using high throughput  sequencing data, Genomics 109 (2017) 457 – 462 . 

 [34]  E. Picardi, G. Pesole, Mitochondrial genomes gleaned from human whole-exome  sequencing, Nat. Methods 9 (2012) 523 – 524 .  