# Genetic variant at coronary artery disease and ischemic stroke locus 1p32.2 regulates endo the li al responses to he mo dynamics  

Matthew D. Krause a , Ru-Ting Huang a , David Wu a , Tzu-Pin Shentu a , Devin L. Harrison a , Michael B. Whalen b , Lindsey K. Stolze b , Anna Di Rienzo c , Ivan P. Moskowitz , Mete Civelek f , Casey E. Romanoski b , and Yun Fang  

a Department of Medicine, The University of Chicago, Chicago, IL 60637;  b Department of Cellular and Molecular Medicine, The University of Arizona, Tucson, AZ 85721;   c Department of Human Genetics, The University of Chicago, Chicago, IL 60637;   d Department of Pediatrics, The University of Chicago, Chicago, IL 60637;   e Department of Pathology, The University of Chicago, Chicago, IL 60637; and   f Department of Biomedical Engineering, The University of Virginia, Charlottesville, VA 22908  

Edited by Shu Chien, University of California, San Diego, La Jolla, CA, and approved October 19, 2018 (received for review June 25, 2018)  

Bio mechanical cues dynamically control major cellular processes, but whether genetic variants actively participate in mecha no sensing mechanisms remains unexplored. Vascular homeostasis is tightly regulated by he mo dynamics. Exposure to disturbed blood flow at arterial sites of branching and bifurcation causes constitutive activation of vascular endothelium contributing to atherosclerosis, the major cause of coronary artery disease (CAD) and ischemic stroke (IS). Conversely, unidirectional flow promotes quiescent endothelium. Genome-wide association studies (GWAS) have identified chromosome   $\pmb{1\uprho32.2}$   as strongly associated with CAD/IS; however, the causal mechanism related to this locus remains unknown. Using statistical analyses, assay of trans posa se accessible chromatin with whole-genome sequencing (ATAC-seq), H3K27ac/H3K4me2 ChIP with whole-genome sequencing (ChIP-seq), and CRISPR interference in human aortic endo the li al cells (HAECs), our results demonstrate that rs17114036, a common noncoding polymorphism at 1p32.2, is located in an endo the li al enhancer dynamically regulated by he mo dynamics. CRISPR-Cas9 – based genome editing shows that rs17114036-containing region promotes endo the li al quiescence under unidirectional shear stress by regulating phospholipid phosphatase 3 (PLPP3). Chromatin accessibility quantitative trait locus (caQTL) mapping using HAECs from 56 donors, allelic imbalance assay from 7 donors, and luciferase assays demonstrate that CAD/ IS-protective allele at rs17114036 in PLPP3 intron 5 confers increased endo the li al enhancer activity. ChIP-PCR and luciferase assays show that CAD/IS-protective allele at rs17114036 creates a binding site for transcription factor Krüppel-like factor 2 (KLF2), which increases the enhancer activity under unidirectional flow. These results demonstrate that a human SNP contributes to critical endo the li al mechanotransduction mechanisms and suggest that human haplotypes and related  cis -regulatory elements provide a previously unappreciated layer of regulatory control in cellular mecha no sensing mechanisms.  

GWAS  |  mecha no transduction  |  coronary artery disease  |  endo the li al cells  | shear stress  

M echanical stimuli regulate major cellular functions and play critical roles in the pathogen es is of diverse human diseases (1). This is especially important in the vas cula ture, where endothelial cells are activated by local disturbed flow in arterial regions prone to atherosclerosis (2 – 5), the major cause of coronary artery disease (CAD) and ischemic stroke (IS). The role of bio mechanical forces on the noncoding and regulatory regions of the human genome is unexplored. Recent studies demonstrated that the noncoding, non transcribed human genome is enriched in  cis -regulatory elements (6). In particular, enhancers are distinct genomic regions that contain binding sites for sequence-specific transcription factors (7). Enhancers spatially and temporally control gene expression with cell type- and cell state-specific patterns (8). Notably, topassociated human disease-associated SNPs are frequently located within enhancers that explicitly activate genes in disease-relevant cell types (9). The nature of mecha no sensitive enhancers and their biological roles in vascular functions have not been identified. At hero sclerotic disease is the leading cause of morbidity and mortality worldwide. Genome-wide association studies (GWAS) identified chromosome   $1\mathbf{p}32.2$   as one of the most strongly associated loci with susceptibility to CAD and IS (10 – 12). One candidate gene in this locus is phospholipid phosphatase 3 (PLPP3; also known as ph os phat i dic acid phosphatase-type 2B), which inhibits endo the li al inflammation and promotes monolayer integrity by hydro ly zing lys oph os phat i dic acid (LPA) that activates endothelium (13, 14). Our recent study demonstrated that PLPP3 expression is significantly increased in vascular endothelium by unidirectional flow in vitro and in vivo (13). Moreover, expression quantitative trait locus (eQTL) mapping showed that CAD-protective allele at  $1\mathbf{p}32.2$   is associated with increased PLPP3 expression in an endothelium-specific manner  

# Significance  

Bio mechanical stimuli control major cellular functions and play critical roles in human diseases. Although studies have implicated genetic variation in regulating key biological functions, whether human genetic variants participate in the processes by which cells sense and respond to bio mechanical cues remains unclear. This study provides a line of evidence supporting an under appreciated role of genetic predisposition in cellular mecha no transduction. Using genetics approaches and genome editing, our data demonstrate that rs17114036, a common noncoding polymorphism implicated in coronary artery disease and ischemic stroke by genome-wide association studies, dynamically regulates endo the li al responses to atherosclerosis-related blood flow (he mo dynamics) via a noncoding DNA region important for transcription activation (enhancer). These results provide molecular insights linking diseaseassociated genetic variants to cellular mecha no biology.  

Author contributions: M.D.K., R.-T.H., D.W., T.-P.S., D.L.H., M.B.W., L.K.S., A.D.R., I.P.M., M.C., C.E.R., and Y.F. designed research; M.D.K., R.-T.H., D.W., T.-P.S., D.L.H., M.B.W., L.K.S., A.D.R., I.P.M., M.C., C.E.R., and Y.F. performed research; M.D.K., R.-T.H., T.-P.S., D.L.H., M.B.W., L.K.S., A.D.R., I.P.M., M.C., C.E.R., and Y.F. analyzed data; and M.D.K., R.-T.H., D.W., T.-P.S., M.C., C.E.R., and Y.F. wrote the paper.  

The authors declare no conflict of interest.  

(13). However, whether genetic variants and mecha no sensing mechanisms converge on PLPP3 expression is unclear. In addition, causal SNPs at locus 1p32.2 remain unknown.  

Using statistical analyses, assay of trans posa se accessible chromatin with whole-genome sequencing (ATAC-seq), H3K27ac ChIP with whole-genome sequencing (ChIP-seq), H3K4me2 ChIP-seq, luciferase assays, and CRISPR-based approaches, we report that rs17114036-containing genomic region at 1p32.2 causatively promotes endo the li al expression of PLPP3 and governs the at hero resistant endo the li al phenotype under unidirectional shear stress by functioning as a mecha no sensitive endo the li al enhancer. Using human aortic endo the li al cells (HAECs) isolated from a cohort of human subjects, we performed transcript ome analyses and chromatin accessibility quantitative trait locus (caQTL) mapping showing nucleotide-specific epigenetic and transcript omi c effects of rs17114036 in humans. Allelic imbalance (AI) assays, ChIP-PCR, and luciferase assays collectively demonstrate that, due to a single base pair change, the CAD/IS-protective allele at rs17114036 confers increased activity of an endo the li al intronic enhancer that is dynamically activated by unidirectional blood flow and transcription factor Krüppel-like factor 2 (KLF2). This report elucidates underlying molecular mechanisms related to CAD/IS locus   $1\mathbf{p}32.2$  and linking human disease-associated genetic variants to critical mecha no transduction mechanisms. The molecular insights suggest that human genetic variants provide a layer of molecular control by which cells convert physical stimuli into biological signaling via tissue-specific enhancers.  

# Results  

Bayesian Refinement and Conditional and Joint Multiple-SNP Analyses Predict That rs17114036 and rs2184104 Are Possible Causal SNPs Located in CAD/IS Locus 1p32.2.  rs17114036 is the tag SNP used in most CAD/IS GWAS (10 – 12) and in our eQTL mapping (13); however, there are 44 common SNPs in high linkage disequilibrium (LD;  $r^{2}\,>\,0.8$  ) with rs17114036, and any of these SNPs could conceivably be a causal variant. To predict possible causal SNPs at the 1p32.2 locus, we conducted two statistical analyses. First, we used a Bayesian statistical approach to assign posterior probabilities and credible sets of SNPs that refine the association signals of GWAS-detected loci (15). Second, we applied conditional and joint association analyses using summary-level statistics of GWAS data to predict causal variants (16). Using Bayes ’ theorem in the cohort of 45 SNPs at 1p32.2, we identified 15 SNPs with  ${\tt>}95\%$   posterior probability to be causal (Fig. 1 A  and  SI Appendix , Table S1 ). Using the approximate conditional and joint association analyses, we identified seven   $1{\mathrm{p}}32.2$  -associated SNPs to be possible causal (Fig. 1 A  and  SI Appendix , Table S1 ). Only two SNPs, rs17114036 and rs2184104, were predicted to be causal by both methods.  

CAD/IS-Associated SNP rs17114036 Is Located in an Enhancer Element (chr1:56962213 – 56963412) in HAECs.  Both rs17114036 and rs2184104 are located in noncoding regions. To probe the regulatory functions of these two SNPs in vascular endothelium, we performed ATAC-seq as well as H3K27ac and H3K4me2 ChIP-seq in HAECs. ATAC-seq is a high-throughput, genome-wide method to define chromatin accessibility that correlates with precise measures of transcription factor binding (17). The combination of H3K27ac and H3K4me2 ChIP-seq marks was used to identify active enhancers. Since the human PLPP3 gene is expressed from the minus strand in the annotated human genome, we use alleles in the minus strand at rs17114036 and rs2184104 in this manuscript. It is important to note that, because CAD/IS risk alleles at rs17114036 (T) and rs2184104 (A) are major alleles in all ethnic groups   $(70{-}99\%$   frequency) (18), our experiments, unless specified otherwise, were conducted in HAEC lines from donors who carry major alleles at rs17114036 and rs2184104. As demonstrated in Fig. 1 B , rs17114036 in the intron 5 of the PLPP3 resides in an enhancer-like element [chr1:56962213 – 56963412, University of California, Santa Cruz (UCSC) version hg19] identified by ATAC-seq and H3K27ac/H3K4me2 ChIP-seq in HAECs. Encyclopedia of DNA Elements (ENCODE) also reported a DNase hypersensitive site and an H3K27ac/H3K4me1 peak in an  ∼ 1-kb region enclosing rs17114036 in human umbilical vein endo the li al cells (HUVECs) (6) ( SI Appendix , Fig. S1 ). Notably, this region does not exhibit enhancer-like marks in other ENCODE cell types, such as K562, GM12878, and NHEK cells ( SI Appendix , Fig. S1 ). In contrast, the other putative causal SNP, rs2184104, is located  ${\sim}120~\mathrm{kb}$   downstream of the PLPP3 transcription start site at a location that lacks enhancer-like features (Fig. 1 B ). ENCODE data also signify an inactive chromatin domain surrounding rs2184104 ( SI Appendix , Fig. S1 ).  SI Appendix , Fig. S2  shows the ATAC-seq and H3K27ac/H3K4me2 tracks in HAECs at 1p32.2 locus. The enhancer activity of chr1:56962213 – 56963412 was experimentally demonstrated by a luciferase reporter assay (Fig. 1 C ). Plasmid transfection was first validated in HAECs using electro p oration of pmax green fluorescent protein (pmaxGFP)- expressing constructs ( SI Appendix , Fig. S3 ). A 1,200-bp DNA sequence corresponding to human chr1:56962213 – 56963412 was cloned upstream of firefly luciferase that was driven by a minimal promoter. Reporter assays demonstrated that insertion of this putative enhancer region with major allele T at rs17114036 significantly increased the luciferase activity in HAECs (Fig. 1 C ). No significant enhancer activity was implicated when rs2184104-containing region (chr1:56911623 – 56912823) was cloned into the same reporter vector compared with the rs17114036-containing region ( SI Appendix , Fig. S4 ). We further cloned the rs17114036-containing region into a luciferase vector that contains the human PLPP3 promoter. Endogenous human PLPP3 promoter led to a 7.9-fold higher luciferase activity in HAECs compared with the vector with minimal promoter (Fig. 1 C ). Moreover, insertion of chr1:56962213 – 56963412 resulted in a 2.14-fold increase in luciferase activity compared with the vector with only PLPP3 promoter (Fig. 1 C ). Minimal enhancer activities chr1:56962213 – 56963412 were detected when the constructs were expressed in the non endothelium cell line HEK 293 ( SI Appendix , Fig. S5 ). ATAC-seq, H3K27ac/H3K4me2 ChIP-seq, and luciferase assays altogether demonstrate that chr1:56962213 – 56963412 functions as an enhancer in HAECs.  

CRISPR-Based Approaches Identified rs17114036-Containing Region as a  cis -Regulatory Element for Endo the li al PLPP3 Expression.  To determine the causal role of rs17114036-containing genomic locus in regulating endo the li al PLPP3 expression, the bacterial CRISPR-Cas9 system was used to selectively delete an  ${\sim}66$  -bp genomic region (chr1:56962783 – 56962849) enclosing rs17114036 in HAECs. A pair of guide RNAs (Fig. 2 A ) was designed according to the method described by Ran et al. (19). We preassembled the Cas9 – guide RNA rib on u cleo protein (RNP) complex by incubating guide RNAs with recombinant  Streptococcus pyogenes Cas9 followed by the delivery to cells using cationic liposome transfection reagents. To improve efficiency of the CRISPR-Cas9 – mediated deletion, cells were reverse trans fec ted with the RNP complex four times before flow cytometry to sort single cells (Fig. 2 B ). Immortalized HAECs (carrying major alleles at rs17114036) with high proliferating capacity were used for single-cell clonal isolation to select a genetically identical cell line. Among 459 HAEC colonies that we grew to confluency, PCR assays detected 17 lines with  ${\sim}66$  -bp genetic deletion in PLPP3 intron 5 enclosing rs17114036 ( SI Appendix , Fig. S6 A ). DNA deletion was further confirmed by cloning and Sanger sequencing ( SI Appendix , Fig. S6 B ). Endo the li al PLPP3 expression is significantly reduced in the genome-edited cells compared with teloHAECs that underwent CRISPR-Cas9 treatment and single-cell clonal isolation but showed no sign of deletion at chr1:56962783 – 56962849 (Fig. 2 C ). In addition, deletion of this putative enhancer in PLPP3 intron 5 resulted in an increase of LPA-induced E-selectin expression (Fig. 2 D ) and leukocyte adhesion (Fig. 2 E ), in agreement with the anti inflammatory/ adhesive role of endo the li al PLPP3 (13, 14). Moreover, transendo the li al electrical resistance detected increased monolayer  

![](images/db0546eceffe80e795edf5bd3801836d594055d555a2627e3598858c524ebaf6.jpg)  
Fig. 1. CAD-associated SNP rs17114036 is located in an enhancer element (chr1:56962213 – 56963412) in HAECs. ( A ) In silico prediction of causal SNPs in the CAD locus 1p32.2. Diagrams demonstrate the association  $(-{\mathsf{l o g}}_{10}{\mathsf{P}})$   and LD pattern of a total of 45 common SNPs in the 1p32.2 locus. Green circles indicate possible causal SNPs predicted by the Bayesian Statistical Approach. Red circles indicate possible causal SNPs predicted by the Approximate Conditional and Joint Association Analysis. Purple circles indicate putative causal SNPs predicted by both statistical analyses. ( B ) Chromatin accessibility and canonical enhancer marks in chr1:56962213 – 56963412 region enclosing rs17114036 in HAEC. ATAC-seq and H3K27ac/H3K4me2 ChIP-seq collectively identified an enhancer-like region in chr1:56962213 – 56963412 in HAECs. All sequencing experiments were performed in duplicate, and the merged tracks are shown. ( C ) Enhancer activity of chr1:56962213 – 56963412 in vascular endothelium. DNA sequences of chr1:56962213 – 56963412 were cloned into luciferase reporters (firefly luciferase construct pGL4) that contain a minimal promoter or human PLPP3 promoter. The red asterisks denote the relative position of rs17114036 in the luciferase construct. Dual luciferase reporter assays were conducted in HAECs  $24{\textsf{h}}$   after the electro p oration-based transfection (using pRL-TK plasmid carrying Renilla luciferase as transfection controls) in HAECs, detecting increased firefly luciferase as the result of insertion of chr1:56962213 – 56963412. Data represent mean  $\pm$   SEM.  $***P<0.0005$   as determined by Student ’ s  t  test.  

permeability in rs17114036-deleted HAECs (Fig. 2 F ), consistent with PLPP3 ’ s role in maintaining endo the li al monolayer integrity (13, 14). CRISPR interference was recently developed to suppress the activity of  cis -regulatory elements (20). Here, we showed that rs17114036-targeted (Fig. 2 G ) but not rs2184104-targeted ( SI Appendix , Fig. S7 ) guide RNAs significantly reduce PLPP3 mRNA expression in HAECs. These results demonstrate that rs17114036- containing region causative ly regulates PLPP3 expression and endo the li al functions.  

Unidirectional Flow Increases the Enhancer Activity at chr1:56962213 – 56963412 in Vascular Endothelium.  Given the critical role of hemodynamics in controlling endo the li al PLPP3 transcription (13), we tested whether shear stresses regulate the enhancer activity of chr1:56962213 – 56963412. ATAC-seq and H3K27ac ChIP-seq were conducted in HAECs subjected to  “ at hero protective ”  unidirectional flow representing wall shear stress in human distal internal carotid artery or  “ at hero susceptible ”  flow mimicking he mo dynamics in human carotid sinus (21). ATAC-seq captured increased open chromatin at chr1:56962213 – 56963412 in HAECs under unidirectional flow compared with cells under disturbed flow (Fig. 3 A ). H3K27ac ChIP-seq indicated an increased enhancer activity of chr1:56962213 – 56963412 in HAECs under unidirectional flow (Fig. 3 A ). In addition, genetic deletion of the rs17114036-containing region by CRISPR-Cas9 significantly  

![](images/6cd141e171de71c780eb7416245c779a0736e810aa0c9cd68223c81785159f9b.jpg)  
Fig. 2. CRISPR-based approaches identify rs17114036-containing genomic locus as a  cis -regulatory element regulating endo the li al PLPP3 expression. ( A ) RNA RNP complex that contains recombinant  S. pyogenes  Cas9 and two sgRNAs flanking rs17114036. ( B ) Experimental overview of CRISPR-Cas9 – mediated genomic deletion in HAECs. TeloHAECs were treated with RNP complex four times before single-cell sorting and isogenic clone selection. ( C ) Reduced PLPP3 expression,  $\left(D\right)$  elevated E-selectin expression, ( E ) increased leukocyte adhesion, and ( F ) higher monolayer permeability in teloHAECs with genomic deletion at rs17114036- containing region. ( G ) Reduced PLPP3 expression in HAECs treated with CRISPR interference targeting rs17114036-containing region.  $n=3–8$  . Data represent mean  $\pm~{\sf S E M}$  .  $^{\star}P<0.05$   as determined by Student ’ s  t  test;  $\star\star P<0.005$   as determined by Student ’ s  t  test;  $\star\star\star P<0.0005$   as determined by Student ’ s  t  test.  

impaired unidirectional flow-induced PLPP3 expression in HAECs (Fig. 3 B ). Moreover, enhancer activity at chr1:56962213 – 56963412, measured by H3K27ac ChIP-PCR, was increased in control HAEC but not in rs17114036-(biallelic) deleted cells when subjected to  $24\,\mathrm{h}$  of unidirectional flow (Fig. 3 C ). These results collectively demonstrate that enhancer activity of chr1:56962213 – 56963412 is dynamically activated by the at hero protective unidirectional flow to regulate endo the li al PLPP3.  

CAD/IS-Protective Allele C at rs1711403 Confers a Higher Enhancer Activity of chr1:56962213 – 56963412.  GWAS have linked the minor allele C at rs17114036 at  $1\mathrm{p}32.2$   to reduced CAD/IS susceptibility (10 – 12), and our eQTL mapping described increased PLPP3 expression in HAECs with minor allele C (13). Here, we investigated the genotype-dependent effect of rs17114036 on the enhancer activity of chr1:56962213 – 56963412 by ATAC-seq and luciferase assays. In addition to HAEC lines carrying major (risk) allele  $\mathrm{T}$   at rs17114036, we conducted ATAC-seq in HAECs isolated from donors who are heterozygous (T/C;  ${\sim}20\%$   of Europeans) at rs17114036, allowing us to perform caQTL mapping. caQTL was recently developed to detect between-individual signaling in  cis -regulatory elements as a function of genetic variants (22). ATAC-seq detected significantly increased numbers of reads corresponding to rs17114036- containing region in HAEC lines that contain one CAD-protective allele (T/C) compared with HAECs from donors homozygous of CAD risk allele (T/T) (Fig. 4 A ), supporting increased chromatin acces sibi lit y associated with C allele at rs17114036. In addition, we conducted RNA-seq analysis in these cells, demonstrating that there  

![](images/858e3f1ac63a8c9a3c11c7a3580945ce6f17f76325e9f023b9a235f94f91b349.jpg)  
Fig. 3. Unidirectional flow (UF) increases enhancer activity at chr1:56962213 – 56963412 that transcription ally activates PLPP3 in human aortic endothelium. ( A ) Increased chromatin accessibility and H3K27ac mark at chr1:56962213 – 56963412 in HAECs subjected to 24-h at hero protective UF compared with cells under  $^{24-\mathsf{h}}$   at hero susceptible disturbed flow. The PLPP3 locus is shown and zoomed in to demonstrate details around the enhancer region of interest, with rs17114036 highlighted with a vertical blue line. ( B ) Reduced UF-induced PLPP3 expression in HAECs with genomic deletion at rs17114036-containing genomic locus. Control and genome-edited ( ∼ 66-bp deletion) HAECs were subjected to 24-h UF. The  $y$   axis represents the fold change of PLPP3 mRNA quantities between the static conditions and UF for each individual clone. Non deletion clones,  $n=3,$  ; deletion clones,  $n=4$  . ( C ) H3K27ac ChIP-PCR performed in two CRISPR clones; one non deletion clone and one biallelic deletion clone harboring the 66-bp deletion near rs17114036. The cells were subjected to static or UF conditions before cross-linking and ChIP. PCR primers were designed to amplify a region within the enhancer but not overlapping the deleted region. Data are shown as fold change of the UF-treated samples compared with static conditions. Data represent mean  $\pm\,\mathsf{S E M}$   $^{\star}P<0.05$   as determined by Student ’ s  t  test;  $\star\star\star P<0.0005$   as determined by Student ’ s  t  test.  

is a strong correlation between enhanced chromatin accessibility in rs17114036-containing region and increased mRNA levels of PLPP3 in HAECs (Fig. 4 B ) and further suggesting that chr1:56962213 – 56963412 functions as an enhancer in promoting endo the li al PLPP3 transcription. Moreover, ATAC-seq experiments in HAEC lines heterozygous at rs17114036 further allow us to determine whether the chromosome with C at rs17114036 exhibits higher chromatin accessibility at chr1:56962213 – 56963412 compared with the chromosome with T allele. This is achieved by the AI analysis, which assigns next generation sequencing reads overlapping heterozygous sites to one chromosome or the other for allele-specific signals (23). ATAC-seq detected reads enriched from the  $\mathrm{C}\mathrm{-}$  containing chromosome compared with that from   $\mathrm{T}$   allele in HAECs heterozygous at rs1711403 (Fig. 4 C ), further supporting the increased chromosome accessibility associated with C allele at rs17114036. Lastly, luciferase assays were conducted to support the genotype-dependent enhancer activity of chr1:56962213 – 56963412. Replacement of  $\mathrm{T}$   allele with C allele led to a much higher luciferase activity   $\sim\!\!5.2$  -fold, C vs. T) in endothelium (Fig.  $4D_{-}$  ). Polymorphisms (A or G) at rs2184104 had no effect of the chr1:56911623 – 56912823 in the luciferase assay ( SI Appendix , Fig. S8 ). Taken together, these results demonstrate that CADprotective allele C at rs17114036 confers a higher enhancer activity of chr1:56962213 – 56963412 to promote PLPP3 expression in vascular endothelium.  

# CAD/IS-Protective C Allele at rs1711403 Promotes Flow-Induced, KLF2- Mediated Enhancer Activity of chr1:56962213 – 56963412.  We further examined whether the genetic variants at rs17114036 modulate  

![](images/9c6bad5afafdb599d52c1271b974215fa7ca8d39d062a3cc30511614b5a3e996.jpg)  
Fig. 4. CAD-protective allele C at rs17114036 confers higher enhancer activity of chr1:56962213 – 56963412. ( A ) Increased ATAC-seq reads in chr1:56962213 – 56963412 region from HAECs isolated from people heterogeneous (T/C) at rs17114036 compared with HAECs from people heterozygous (T/T) at rs17114036. ( B ) A positive correlation   $(R=0.6)$  ,  $P$   value  $=6.23\mathrm{e}.06)$  ) between ATAC-seq reads at chr1:56962213 – 56963412 and PLPP3 mRNA detected by RNA-seq in 56 HAEC lines. ( C ) Increased ATAC-seq reads at rs17114036-containing genomic locus from C (rs17114036)-containing chromosome compared with T-containing chromosome in HAEC lines heterozygous (T/C) at rs17114036. ( D ) Increased enhancer activity of chr1:56962213 – 56963412 with C allele at rs17114036 compared with T allele. Dual luciferase reporter assays were conducted in teloHAEC. The red and green asterisks denote the relative position of rs17114036 polymorphisms in the luciferase construct.  $n=4{-}6$   $\pm$   SEM.  $\star\star\star P<0.0005$   as determined by Student ’ s  t  test.  

the flow-induced enhancer activity of chr1:56962213 – 56963412. Luciferase assays detected an increase of the enhancer activity of chr1:56962213 – 56963412 (with protective C allele at rs17114036) in cells under 18-h unidirectional flow compared with disturbed flow (Fig. 5 A ), while no significant increase of the enhancer activity by unidirectional flow was detected with the risk T allele ( SI Appendix , Fig. S9 ). In addition to the ATAC-seq experiments in HAECs homozygous at rs17114036 under flow (Fig. 3), we performed ATAC-seq analysis in four HAEC lines heterozygous at rs17114036 under 24-h unidirectional flow to perform open chromatin AI analysis.  $S I$   Appendix , Fig. S10  demonstrates that, in all four selected HAEC lines heterozygous at rs17114036, unidirectional flow increases ATAC-seq peaks in the proposed enhancer region in PLPP3 intron 5, in agreement with increased ATAC-seq reads in rs17114036-containing region (Fig. 5 B ). Moreover, AI analysis showed an enrichment of ATAC reads from the chromosome harboring the protective C allele (Fig.  $5B$  ). In contrast, ATAC-seq detected no AI at rs6421497, a common SNP in high LD with rs17114036 ( SI Appendix , Fig. S11 ). Indeed, the protective allele C at rs17114036 creates a CACC box that is a binding site for KLF2, which mediates the flow sensitivity of a cohort of endo the li al genes, including PLPP3 (13, 24 – 27). We then tested whether KLF2 dynamically activates this rs17114036-containing enhancer and if rs17114036 alleles impact KLF2-mediated enhancer activity. The affinity of KLF2 to the rs17114036-containing locus was determined by KLF2 ChIP-PCR assays in HAECs carrying a protective allele at rs17114036 (T/C), showing a physical binding of KLF2 to the rs17114036-containing region and the CACC site in the PLPP3 promoter (Fig. 5 C ). Enhancer activities of chr1:56962213 – 56963412 were further determined in HAECs as a function of KLF2 expression. Constructs of enhancer (chr1:56962213 – 56963412) and PLPP3 promoter were co trans fec ted with KLF2- over expressing plasmids. Luciferase assays detected a 2.9-fold increase of luciferase activity in the T allele-containing construct as the result of KLF2 over expression (Fig. 5 D ). Moreover, KLF2 over expression led to a 4.7-fold increase of luciferase activity when T allele was substituted by the protective allele C at rs17114036 (Fig.   $5D$  ). Collectively, KLF2 ChIP-PCR and luciferase assays demonstrate that CAD/IS-protective allele C at rs17114036 confers a higher KLF2-dependent enhancer activity of chr1:56962213 – 56963412 in vascular endothelium.  

# Discussion  

Although it is proposed that genetic and environmental factors jointly influence the risk of most common human diseases, the interplay between genetic predisposition and bio mechanical cues at the molecular level is poorly understood. The biology underlying the majority of CAD and IS GWAS loci remains to be elucidated (28). Most of the CAD and IS SNPs reside in the noncoding genome. Gupta et al. (29) recently reported that the noncoding common variant at rs9349379, implicated in CAD by GWAS, regulates endothelin 1 expression in endothelium. At hero sclerotic lesions preferentially develop in elastic arteries where vascular endo the li al cells are activated by local disturbed flow (2 – 5). As of now, it remains unknown whether disease-associated genetic variants contribute to mecha no sensing mechanisms by which cells sense and convert bio mechanical stimuli to biological signaling. Our results here elucidate the convergence of CAD/IS genetic predisposition and mechanotransduction mechanisms in endo the li al PLPP3 expression. Statistical analyses, whole-genome chromatin accessibility/enhancer marks, CRISPR interference (CRISPRi), genome editing, enhancer assays, caQTL mapping, and AI assay collectively demonstrate that CAD/IS locus  $1\mathbf{p}32.2$   harbors a mecha no sensitive endothelial enhancer that regulates PLPP3 expression. Moreover, CAD/ IS-protective allele at rs17114036 confers an increased enhancer activity that is dynamically regulated by unidirectional flow and transcription factor KLF2 (Fig.  $5E$  ).  

Dys regulation of mecha no sensing mechanisms contributes to the etiology of a wide range of human diseases in cardiovascular, pulmonary, orthopedic, muscular, and reproductive systems (1). The genetic basis of these complex human diseases has been strongly suggested by GWAS, but the interplay between genetic variants and mecha no sensing mechanisms has not been investigated. Our data provide a line of evidence supporting the genetic regulation of mecha no transduction mechanisms in complex human diseases and suggest an under appreciated role of genetic predisposition in cellular mecha no sensing processes.  

Transcriptional enhancers orchestrate the majority of cell type-specific patterns of gene expression and play key roles in development, evolution, and disease (7), which are tightly regulated by mechanical cues (1). Our data provide molecular evidence that the noncoding genome actively participates in cellular mecha no transduction mechanisms that are influenced by human genetic variances. In addition to the flow regulation of the specific locus 1p32.2, our results provide a dataset to systematically determine the mecha no sensitive chromatin accessibility and putative enhancer regions at the whole-genome scale in vascular endothelium. It is important to note that most of the epigenome studies including ENCODE were conducted in cells without physiological or path o physiological mechanical stimuli, such as HUVECs under static (no flow) conditions (6). Since major endo the li al functions are tightly and dynamically regulated by he mo dynamics flow, our whole-genome epigenome profiling in HAECs under at hero relevant flows may benefit future studies to investigate mechanical regulation of the noncoding genome in vascular cells. Indeed, we have applied model-based analysis of ChIP-seq (30) and HOMER differential analysis (31), which unbiasedly identified rs17114036-containing locus as 1 of 36,965 open chromatin sites that are activated by unidirectional flow ( SI Appendix , Fig. S12 ).  

Mecha no sensitive transcription factors have been proposed as major regulators to determine endo the li al functions relevant to at hero genesis. For instance, nuclear factor  $\mathbf{\cdot}\kappa\mathbf{B}$   and HIF  $\cdot1\upalpha$   me- diate gene sets associated with pro inflammatory, pro coagulant, and glycolytic endo the li al phenotypes under disturbed flow (32 – 35), while KLFs and nuclear factor erythroid 2-like 2 regulate gene networks promoting the quiescent endo the li al phenotype under unidirectional flow (25 – 27, 36 – 39). However, the interaction between flow-sensitive transcription factors and disease-associated genetic predisposition in vascular functions has not been suggested. Our results demonstrate that a genetic variant can influence important endo the li al functions via a noncoding enhancer region recognized by the mecha no sensitive transcription factor KLF2. These results are consistent with emerging evidence showing that top-scoring disease-associated SNPs are frequently located within enhancers explicitly active in disease-relevant cell types (9, 40). Moreover, the data suggest that disease-associated genetic variants, via modulation of transcription factor binding, may regulate the enhancer activities dynamically responding to bio mechanical cues that are instrumental to key cellular processes.  

GWAS related to at hero sclerotic diseases have suggested previously unsuspected loci, genes, and biology involved in lipoprotein metabolism (28), resulting in the development of new cholesterol-lowering therapies (41). Despite that dys lipid emi a is a major systemic risk factor of CAD and IS, at hero sclerotic lesions largely initiate and develop at arterial regions of atypical vascular geometry associated with disturbed flow. Previous studies demonstrated that cellular mecha no transduction mechanisms, particularly endo the li al responses to he mo dynamics, causative ly contribute to the focal nature of at hero sclerotic lesions (2 – 5, 42). Our studies here demonstrate that genetic variants contribute to not only inter individual variation in plasma  

![](images/f321392cecb8f466e21e9fb114fb688e626abec79b11a446c7bfb71b66f1f2b0.jpg)  
Fig. 5. CAD-protective C allele at rs17114036 promotes flow-induced, KLF2-mediated enhancer activity of chr1:56962213 – 56963412. ( A ) Increased enhancer activity of chr1:56962213 – 56963412 (with C allele at rs17114036) under unidirectional flow (UF) but not disturbed flow (DF). Experiment was performed in biological triplicate and technical triplicate. Data represent mean  $\pm\,\mathsf{S E M}$  .  $^{\star}P<0.05$   as determined by two-way ANOVA;  $^{\star\star}\!{\cal P}<0.005$   as determined by two-way ANOVA. The green asterisk denotes the relative position of rs17114036 in the luciferase construct. ( B ) Increased ATAC-seq reads in rs17114036-containing region in HAECs under UF compared with DF. ATAC-seq was conducted in four HAEC lines heterozygous at rs17114036 under 24-h at hero relevant flows, detecting increased ATAC reads in cells under UF and higher reads from the C allele-containing chromosome compared with  $\intercal$   allele. ( C ) KLF2 affinity to CACC sites in human PLPP3 promoter and intron 5. ChIP-qPCR was performed with either a control IgG antibody or the antibody against HA followed by qPCR using primers detecting CACC sites in PLPP3 promoter or rs17114036-enclosing region from control HAECs (Ctrl) or HAECs trans fec ted with KLF2 transcripts with HA tag. Primers that detect a site  ${\sim}200$   bp from the CACC at rs17114036 were used as a negative control.  $n=4$  .   $\left(D\right)$   Increased enhancer activity of chr1:56962213 – 56963412 by KLF2 over expression. Dual luciferase reporter assays were conducted in HAECx trans fec ted with luciferase constructs containing the PLPP3 promoter and enhancer with either the major (T) or minor ( C ) allele at s17114036 along with KLF2-over expressing or control plasmids. KLF2 over expression resulted in a 2.9-fold increased luciferase activity in HAECs trans fec ted with T allele-containing construct and a 4.7-fold increase in cells trans fec ted with C allele-containing construct.  $n=3$  . Data represent mean  $\pm\,\mathsf{S E M}$  .  $^{\star}P<0.05$   as determined by Student ’ s  t  test. ( E ) The interplay between he mo dynamic forces, chromatin landscapes at PLPP3 intron 5, and rs17114036 at the molecular level in regulating endo the li al PLPP3 expression and vascular functions.  

lipid concentrations (43) but also, endo the li al responses to blood flow. Indeed, genetic variants at rs17114036 predict CAD susce pti bil it y independent of traditional systemic risk factors, such as cholesterol and diabetes mellitus (10, 11). Recent GWAS identified 15 new CAD risk loci near genes of key functions in endo the li al, smooth muscle, and white blood cells (44), further highlighting the potential importance of genetic contribution to the arterial wall-specific mechanisms in at hero genesis. Our results indicate that CAD genetic predisposition and disturbed flow converge to inhibit endo the li al PLPP3 expression and that restoration of endo the li al PLPP3 in at hero susceptible regions may provide an attractive approach for future arterial wall-based atherosclerosis therapy complementary to current pharmacological treatments targeting systemic risk factors.  

Our studies demonstrate that the latest human genetics approaches, such as caQTL mapping (22), AI assay (23), CRISPRi (20), and CRISPR-based assays (19), are powerful tools to investigate possible genetic contributions to cellular mechanotransduction. Miao et al. (45) recently applied CRISPR-Cas9 to achieve high efficiency of a  $\cdot_{10-\mathrm{k}\mathfrak{b}}$   deletion of an enhancer region in bulk HUVECs. Here, we expanded the applications of CRISPR-based techniques to investigate key vascular functions. Isogenic adult aortic endo the li al lines subjected to CRISPRbased deletion were successfully selected to determine the causal role of    ${\sim}66$  -bp genomic region in regulating endo the li al PLPP3. Nevertheless, one limitation here is that we are still unable to replace this human SNP at rs17114036 in adult aortic endothelium, although we have tried various methods to promote homology-directed repair. This will be the subject of a future study. Nevertheless, caQTL mapping (22) and AI assay (23) provide complementary approaches detecting at the singlenucleotide resolution that CAD/IS-protective allele at rs17114036 confers a higher enhancer activity at the PLPP3 intron 5.  

Cellular mecha no transduction is required for physiological control of tissue homeostasis, while abnormal cell response to mechanical forces promotes path o logie s of numerous human diseases. Although investigations, such as GWAS, have suggested the genetic basis of complex human diseases, the interplay between genetic predispositions of mecha no sensing mechanisms remains virtually unknown. Our results identified that CADassociated genetic variant at rs17114036 interacts with hemodynamics in concert to regulate endo the li al PLPP3 expression and consequently, key vascular functions. Moreover, our experiments  

1. Jaalouk DE, Lammerding J (2009) Mecha no transduction gone awry.  Nat Rev Mol Cell Biol  10:63 – 73.

 2. Davies PF, Civelek M, Fang Y, Fleming I (2013) The at hero susceptible endothelium: Endo the li al phenotypes in complex ha emo dynamic shear stress regions in vivo. Cardiovasc Res  99:315 – 327.

 3. Hahn C, Schwartz MA (2009) Mecha no transduction in vascular physiology and atherogenesis.  Nat Rev Mol Cell Biol  10:53 – 62.

 4. Zhou J, Li YS, Chien S (2014) Shear stress-initiated signaling and its regulation of endo the li al function.  Arte rios c ler Thromb Vasc Biol  34:2191 – 2198.

 5. Gimbrone MA, Jr, García-Cardeña G (2016) Endo the li al cell dysfunction and the path o biology of atherosclerosis.  Circ Res  118:620 – 636.

 6. Consortium EP; ENCODE Project Consortium (2012) An integrated encyclopedia of DNA elements in the human genome.  Nature  489:57 – 74.

 7. Ong CT, Corces VG (2011) Enhancer function: New insights into the regulation of tissue-specific gene expression.  Nat Rev Genet  12:283 – 293.

 8. Heinz S, Romanoski CE, Benner C, Glass CK (2015) The selection and function of cell type-specific enhancers.  Nat Rev Mol Cell Biol  16:144 – 154.

 9. Ernst J, et al. (2011) Mapping and analysis of chromatin state dynamics in nine human cell types.  Nature  473:43 – 49.

 10. Schunkert H, et al.; Car dio ge nics; CARDIoGRAM Consortium (2011) Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease.  Nat Genet  43:333 – 338.

 11. Deloukas P, et al.; CARDIoGRAM plus C 4 D Consortium; DIAGRAM Consortium; CAR DIO GE NICS Consortium; MuTHER Consortium; Wellcome Trust Case Control Consortium (2013) Large-scale association analysis identifies new risk loci for coronary artery disease.  Nat Genet  45:25 – 33.

 12. Dichgans M, et al.; METASTROKE Consortium; CARDIoGRAM Consortium; C4D Consortium; International Stroke Genetics Consortium (2014) Shared genetic susceptiprovide evidence supporting the regulatory role of the noncoding, non transcribed genome in mecha no transduction mechanisms. In summary, this study demonstrates that human haplotypes and related  cis -regulatory elements provide an important layer of molecular control by which cells convert physical stimuli into biological signaling.  

# Methods  

ATAC-Seq.  ATAC-seq was performed as previously described (17) using Tn5 trans posa se (Illumina). Libraries were sequenced on an Illumina HiSeq 4000 according to the manufacturer ’ s specifications by the Genomics Core Facility at the University of Chicago. The reads were aligned to the UCSC hg19 genome using Bowtie2 (46). ATAC-seq was conducted in HAECs under static conditions or subjected to 24-h unidirectional flow or disturbed flow.  

caQTL Mapping and AI.  caQTL mapping was performed to test for association between genotype at rs17114036 and chromatin accessibility measured by ATAC-seq. We pulled genotypes for HAEC donors from our previous study (47) and imputed linked alleles using IMPUTE2 and SHAPEIT as we published previously (40). Association testing between ATAC-seq tags at the rs17114036 enhancer and genotype was performed using the Combined Haplotype Test in WASP (23).  

To perform AI analysis that assigns next generation sequencing reads overlapping heterozygous sites to one chromosome or the other, we quantified ATAC-seq tags at the rs17114036 enhancer using HOMER ’ s annotate Peaks function to express the log2-normalized tags in this region.  

CRISPR-Cas9 – Mediated Deletion of Enhancer in TeloHAECs.  The CRISPR reagents were adapted from the Alt-R system from IDT. The guide RNAs were designed using an online tool at  crispr.mit.edu/  to minimize off-targeting effects using two guides to create an  $\mathord{\sim}66$  -bp deletion. The guide RNAs were made by annealing the trans activating CRISPR RNA (tracrRNA) to the single guide RNA (sgRNA). The RNP complex of  S. Pyogenes  Cas9 and sgRNA was formed by placing the components together at room temperature and then immediately trans fec ted into cells using Lip of ect amine RNAiMAX (Thermo). For each successive treatment, the reagent amounts were scaled relative to the size of the destination vessel to compensate for the number of cells in the reaction. The volumes for each part of the reaction were increased  $4\times$   when treating cells from the 96-well to the 6-well and  $16\times$   when moving from the 6-well to the T75 flask.  

Detailed methods are available in  SI Appendix .  

ACKNOWLEDGMENTS.  This work was funded by NIH Grants T32 HL007381

 (to M.D.K.), F32 HL134288 (to D.W.), T32 EB009412 (to D.L.H.), R00 HL121172

 (to M.C.), R00 HL123485 (to C.E.R.), R01 HL136765 (to Y.F.), and R01 HL138223 (to Y.F.) as well as American Heart Association Grant B GIA 7080012

 (to Y.F.)  

bility to ischemic stroke and coronary artery disease: A genome-wide analysis of common variants.  Stroke  45:24 – 36.

 13. Wu C, et al. (2015) Mecha no sensitive PPAP2B regulates endo the li al responses to at hero relevant he mo dynamic forces.  Circ Res  117:e41 – e53.

 14. Panch atc haram M, et al. (2014) Mice with targeted in activation of ppap2b in endothelial and he ma top oie tic cells display enhanced vascular inflammation and permeability.  Arte rios c ler Thromb Vasc Biol  34:837 – 845.

 15. Maller JB, et al.; Wellcome Trust Case Control Consortium (2012) Bayesian refinement of association signals for 14 loci in 3 common diseases.  Nat Genet  44:1294 – 1301.

 16. Yang J, et al. (2012) Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.  Nat Genet  44: 369 – 375.

 17. Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ (2013) Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNAbinding proteins and nucleosome position.  Nat Methods  10:1213 – 1218.

 18. Abecasis GR, et al.; 1000 Genomes Project Consortium (2012) An integrated map of genetic variation from 1,092 human genomes.  Nature  491:56 – 65.

 19. Ran FA, et al. (2013) Genome engineering using the CRISPR-Cas9 system.  Nat Protoc  8: 2281 – 2308.

 20. Qi LS, et al. (2013) Re purposing CRISPR as an RNA-guided platform for sequencespecific control of gene expression.  Cell  152:1173 – 1183.

 21. Dai G, et al. (2004) Distinct endo the li al phenotypes evoked by arterial waveforms derived from atherosclerosis-susceptible and -resistant regions of human vas cula ture. Proc Natl Acad Sci USA  101:14871 – 14876.

 22. Kumasaka N, Knights AJ, Gaffney DJ (2016) Fine-mapping cellular QTLs with RASQUAL and ATAC-seq.  Nat Genet  48:206 – 213.

 23. van de Geijn B, McVicker G, Gilad Y, Pritchard JK (2015) WASP: Allele-specific software for robust molecular quantitative trait locus discovery.  Nat Methods  12:1061 – 1063.  

24. Dekker RJ, et al. (2006) KLF2 provokes a gene expression pattern that establishes functional quiescent differentiation of the endothelium.  Blood  107:4354 – 4363.

 25. Sen Banerjee S, et al. (2004) KLF2 is a novel transcriptional regulator of endo the li al pro inflammatory activation.  J Exp Med  199:1305 – 1315.

 26. Parmar KM, et al. (2006) Integration of flow-dependent endo the li al phenotypes by Kruppel-like factor 2.  J Clin Invest  116:49 – 58.

 27. Dekker RJ, et al. (2005) Endo the li al KLF2 links local arterial shear stress levels to the expression of vascular tone-regulating genes.  Am J Pathol  167:609 – 618.

 28. Nurnberg ST, et al. (2016) From loci to biology: Functional genomics of genome-wide association for coronary disease.  Circ Res  118:586 – 606.

 29. Gupta RM, et al. (2017) A genetic variant associated with five vascular diseases is a distal regulator of endothelin-1 gene expression.  Cell  170:522 – 533.e15.

 30. Zhang Y, et al. (2008) Model-based analysis of ChIP-seq (MACS).  Genome Biol  9:R137.

 31. Heinz S, et al. (2010) Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.  Mol Cell  38:576 – 589.

 32. Lan Q, Mercurius KO, Davies PF (1994) Stimulation of transcription factors NF kappa B and AP1 in endo the li al cells subjected to shear stress.  Biochem Biophys Res Commun 201:950 – 956.

 33. Khachigian LM, Resnick N, Gimbrone MA, Jr, Collins T (1995) Nuclear factor-kappa B interacts functionally with the platelet-derived growth factor B-chain shear-stress response element in vascular endo the li al cells exposed to fluid shear stress.  J Clin Invest  96:1169 – 1175.

 34. Wu D, et al. (2017)  HIF-1 α  is required for disturbed flow-induced metabolic reprogramming in human and porcine vascular endothelium.  eLife  6:e25217.

 35. Feng S, et al. (2017) Mechanical activation of hypoxia-inducible factor 1 α  drives endothelial dysfunction at at hero prone sites.  Arte rios c ler Thromb Vasc Biol  37: 2087 – 2101.  

36. Huang RT, et al. (2017) Experimental lung injury reduces Krüppel-like factor 2 to increase endo the li al permeability via regulation of RAPGEF3-Rac1 signaling.  Am J Respir Crit Care Med  195:639 – 651.

 37. Fang Y, Davies PF (2012) Site-specific microRNA-92a regulation of Kruppel-like factors 4 and 2 in at hero susceptible endothelium.  Arte rios c ler Thromb Vasc Biol  32:979 – 987.

 38. Li Z, et al. (2017) Krüppel-like factor 4 regulation of cholesterol-25-hydroxyl as e and liver X receptor mitigates atherosclerosis susceptibility.  Circulation  136:1315 – 1330.

 39. Zhou G, et al. (2012) Endo the li al Kruppel-like factor 4 protects against atherothrombosis in mice.  J Clin Invest  122:4727 – 4731.

 40. Hogan NT, et al. (2017) Transcriptional networks specifying homeostatic and inflammatory programs of gene expression in human aortic endo the li al cells.  eLife  6: e22536.

 41. Sabatine MS, et al.; FOURIER Steering Committee and Investigators (2017) Evolocumab and clinical outcomes in patients with cardiovascular disease.  N Engl J Med  376: 1713 – 1722.

 42. Nam D, et al. (2009) Partial carotid ligation is a model of acutely induced disturbed flow, leading to rapid endo the li al dysfunction and atherosclerosis.  Am J Physiol Heart Circ Physiol  297:H1535 – H1543.

 43. Willer CJ, et al.; Global Lipids Genetics Consortium (2013) Discovery and refinement of loci associated with lipid levels.  Nat Genet  45:1274 – 1283.

 44. Howson JMM, et al.; CARDIoGRAM plus C 4 D; EPIC-CVD (2017) Fifteen new risk loci for coronary artery disease highlight arterial-wall-specific mechanisms.  Nat Genet  49: 1113 – 1119.

 45. Miao Y, et al. (2018) Enhancer-associated long non-coding RNA LEENE regulates endo the li al nitric oxide synthase and endo the li al function.  Nat Commun  9:292.

 46. Langmead B, Salzberg SL (2012) Fast gapped-read alignment with Bowtie 2.  Nat Methods  9:357 – 359.

 47. Romanoski CE, et al. (2010) Systems genetics analysis of gene-by-environment interactions in human cells.  Am J Hum Genet  86:399 – 410.  