# Identification of Common Variants within KCNK17 in Chinese Han  \* Population  

1 2 2 1# Zhouping TANG  (唐洲平) ,  Hu DING   (丁 虎) ,  Yujun XU  (徐宇军) ,  Shabei XU  (徐沙贝)   1 2 Department of Neurology,  The Institute of Hypertension and Department of Internal Medicine, Tongji Hospital,Tongji Medical  College, Huazhong University of Science and Technology, Wuhan 430030, China  

$\copyright$  Huazhong University of Science and Technology and Springer-Verlag Berlin Heidelberg 2010  

Summary:  KCNK17 is a member of the acid-sensitive subfamily of tandem pore   $\mathbf{K}^{+}$  channels, which  are open at all membrane potentials and contribute to cellular resting membrane potential. Recent genome-wide study (GWA) has shown that variants within KCNK17 confer genetic susceptibility for increasing ischemic stroke. In an effort to discover additional polymorphism(s), we scrutinized the genetic  polymorphisms in the KCNK17. By direct DNA sequencing in 32 individuals, we identified nine sequence variants within the   $16~\mathrm{kb}$   of whole KCNK17 gene: one in exon1, one in intron and seven in the  promoter region. Haplotypes, their frequencies and linkage d is equilibrium coefficients (D′), among  polymorphisms were estimated. All the polymorphisms in the   $5\,^{\prime}$  ′ - flanking region (SNP2–SNP7) being  in complete (or nearly complete) association with each other in the promoter region maybe produce  synergistic effect to regulate the expression of KCNK17 gene and then have an influence on the pathogenesis of cerebro vascular diseases. The common haplotypes were observed comprising   $88.9\%$   of the  total haplotypes in the same block. Bio in format ic analysis predicted several potential transcriptional  factors binding sites by SNP -95, -134, -596 and -846. However, these binding sites need to be experimentally verified. The information concerning genetic polymorphisms of KCNK17 gene might provide  valuable information for future genetic studies of diseases.  

Key words : polymorphism; haplotypes; KCNK17; ischemic stroke; Chinese Han population    Stroke is a leading cause of death worldwide . In  China, the incidence of stroke is much higher than that of  coronary heart disease compared with incidence recorded  for Western countries . These different epi de mio logical patterns may be due to differences in genetic backgrounds of the populations. The search of the disease  susceptibility genes has become the research frontier.   Stroke is a complex trait which   is assumed to be caused  by both genetic and environmental   factors as well as  their interaction with each other . Genetic factors have  been defined as an important risk contributor for the  pathogen es is of ischemic stroke , but the responsible  molecular and genetic determinants remain largely unidentified . Genome-wide SNP association studies are a  recently developed strategy for identifying genetic factors for common complex disease traits . Very recently,  independent genome-wide SNP association studies identified three SNPs within KCNK17 that were associated  with ischemic stroke in White cohorts . KCNK17 is a  member of the acid-sensitive subfamily of tandem pore  

$\mathbf{K}^{+}$   channels, which are open at all membrane potentials  and contribute to cellular resting membrane potential.  KCNK17 transcripts are widely expressed in humans,  with highest levels in the liver, lung, pancreas, placenta,  aorta, and heart .  

Despite the functional importance of the KCNK17  gene, polymorphisms of KCNK17 have not been fully  examined in Chinese Han population. In an effort to discover polymorphism(s) in gene whose variant(s) might  be implicated in clinical manifestations of stroke, we  scrutinized the genetic polymorphisms in KCNK17 by  using direct PCR-sequencing. Here we present 9 genetic  variants found in KCNK17 and identified potential relevant polymorphism by bioinformatics analysis.  

# 1 MATERIALS AND METHODS  

# 1.1 Subjects  

A total of 32 unrelated healthy subjects were randomly recruited from an unselected population at Tongji  Hospital (in routine health check-ups). The study protocol was approved by the Institutional Review Board of  the Clinical Research Institute at Tongji Hospital. Informed consent was obtained from all subjects before  drawing blood. All study subjects were examined in the  morning after overnight fasting. Height and weight were  recorded and used for calculation of BMI. Blood samples  were drawn for both biochemical measurements of total  cholesterol and DNA extraction. The clinical character istics of the subjects were shown in table 1.  

Table 1 Baseline characteristics of the study   Subjects ( ± s ,  $\scriptstyle{\pmb{n}}={\pmb{32}})$  )  
![](images/3a03506a73b6a62b05ea4f6006012d0c3555dc076e2a40b785594d56c755d38c.jpg)  
BMI: body mass index; SBP: systolic blood pressure; DBP:  diastolic blood pressure; HDL-C: high-density lipoprotein  cholesterol; TC: total cholesterol  

# 1.2 DNA Isolation  

Genomic DNA was isolated from whole blood collected in   $\mathrm{K}_{3}$  -EDTA tubes using the QG-Mini80 workflow with a DB-S kit (FUJIFILM Corporation, Tokyo,  Japan) as instructed. DNA was quantified and diluted to  a final concentration of   $10\,\mathrm{{ng}/\upmu L}$  .  

We first obtained the candidate gene KCNK17 genomic sequences from the published database of the National Center for Biotechnology Information (GenBank  Accession: NC_000006). We designed PCR arrays to  amplify regions up to   $1~\mathrm{\bf~kb~}$   upstream from transcription-initiation sites, i.e., putative promoter regions, all  exons, and adjacent noncoding regions. All PCRs were  performed in plate thermal cyclers (GeneAmp PCR System 9700; PE Applied Biosystems, USA). The amplification was performed with a   $50~\mathrm{\textmuL}$   reaction mixture  containing   $10~\mathrm{{ng}}$   of DNA,   $0.1\ \upmu\mathrm{mol}/\mathrm{L}$   of each primer, 

  $0.2\ \mathrm{mmol/L}$   de oxy nu cleo side triphosphate s,   $1.5\ \mathrm{mmol/L}$   $\mathrm{{MgCl}}_{2}$  , and   $0.5\mathrm{~U~}$  of DNA polymerase with   $1\times$   reaction  buffer (TaKaRa Code: DR005A, Takara Biotech Co.  Ltd., China). Conditions of amplification and primers  designed for the target regions using the web-based  software Primer 3.0 were available from table 2. We amplified and purified DNA samples from 32 apparently  normal Chinese individuals. These samples included 64  chromosomes, providing at least a   $95\%$   confident level to  detect alleles with frequencies  $<5\%$  . We then sequenced  the PCR products by using ABI PRISM Dye Terminator  Sequencing kits (Applied Biosystems, USA) and loading  the samples onto an ABI   $3130\mathrm{x}1$   sequencer. Finally, we  used the PolyPhred program to identify SNP candidates  that were then confirmed by two independent observers.  We further confirmed these SNP positions and individual  genotypes by re-amplifying and re-sequencing the SNP  site from the opposite strand.  

![](images/a44e523bf2405109630114c7564aa82bc640fac127213ad609f017fcf3a66ac8.jpg)  

# 1.4 Linkage D is equilibrium and Haplotype Structure  

The D’(D-Prime) measurement was used to determine linkage d is equilibrium (LD) with the software  Haploview   $\breve{4}.0^{[10]}$   and double checked by EMLD (http://  request.mdacc.tmc.edu/qhuang/Software/pub.htm). Haplotype frequencies for various SNP combinations were  first estimated by  haplo.stats  (version 1.2.1) for the R  statistical package  and then verified using Haploview  4.0. Both of the software above used the Expectation-M-a xi miz ation (EM) algorithm when constructing the haplotypes. To reduce ret i cul ation, only haplotypes with  estimated frequency  ${>}1\%$   were represented.  

# 1.5 In Silico KCNK17 Promoter Analysis for Transcription Factor Binding  

The identified SNPs were interrogated for the presence of regulatory and transcription factor binding sites  to the KCNK17 promoter binding for all the variants by   $\mathbf{M}\mathbf{A}\mathbf{T}\mathbf{C}\mathbf{H}^{\mathrm{TM}}$   software, which interconnected and distributed together with the TRANSFAC database . All the  prediction was performed under default settings following the instructions.  

# 1.6 Statistical Analysis  

Descriptive statistical analyses were performed with  SAS software (SAS Institute Inc., USA). Data were ex  

pressed as   $\overline{{x}}{\pm}s$  . For each polymorphism, deviation of the  genotype frequencies in the control subjects from those  expected under Hardy-Weinberg equilibrium was assessed by using the standard   $\chi^{2}$   test by  SNPassoc  for R  language . Bonferroni corrections were applied to adjust the multiple testing. A 2-tailed   $P{<}0.05$   was considered statistically significant.  

# 2 RESULTS  

We detected a total of 9 polymorphisms in KCNK1-  7 gene. Seven polymorphisms were identified in the   $5^{\prime}$  -flanking region, at position -95, -134, -511, -569, -846,  -1539 and -1556 (SNP3 – SNP9)   upstream from the first  transcribed nucleotide. A non synonymous base substitution in exon 1 was identified (rs10947804, Ser21Gly)  and no polymorphism was detected in the other exons  and   $_3\prime$  -UTR. One polymorphism was identified within  intron. Position information and mutation types on all  these SNPs were presented in table 3 and fig. 1, along  with the minor allele frequencies. No significant deviation from Hardy–Weinberg equilibrium was found for  any polymorphism, after Bonferroni corrections were  applied.  

Table 3 KCNK17 gene sequence variations  
![](images/3a57c263c882ee15060765f2ff6add3611bbab7d31f316a8822879f64ce16eb5.jpg)  
a  b  c If present in database;  SNPs are numbered relative to gene start site;  With major allele given first and minor allele given   d  e second; Position pairbase is based on NCBI 36.1;  Minor allele frequency using 32 Chinese Han healthy subjects  

![](images/843460f1b929650b09fa7c620c988ea4c876050c715a442fd297e4281defb0c0.jpg)  
Fig. 1  Identification of a SNP in the KCNK17 gene  

Pairwise LD coefficients   $\mathrm{(D^{\prime})}$   of the 9 SNPs were  displayed in fig. 2. It was noted that all the polymorphisms in the   $5^{\prime}$  -flanking region (SNP2–SNP7) were in  complete (or nearly complete) association with one another. Fig. 3 shows the inferred haplotypes (frequency   ${>}0.01_{,}$  ) of the 6 SNPs in the 32 individual health subjects.  Only three truly common haplotypes were observed  comprising   $88.9\%$   of the total. Five additional haplotypes  had frequencies around   $10.4\%$   whereas the remaining  haplotypes were rare  $(<\!2\%)$  .  

![](images/1f2c8b78fa602e1823e7662bb86865c93e7ac7f542d3c2b83c2a03647674b2ec.jpg)  
Fig. 2  LD structure and haplotype blocks of the KCNK17 gene  The above image is the linkage d is equilibrium structure of  the locus (D′), as generated by Haploview 4.0 from the  genotype of KCNK17 polymorphisms in 32 random health  subjects. Haplotype blocks derived from these genotypes  using the solid spine LD setting are outline in black in the   $\mathbf{D}^{\prime}$    chart  

![](images/4930f6189534d751448c40ed34c96ed00895a85f70271326756759102ecb4bd1.jpg)  

Fig. 3  Haplotypes of KCNK17   Only those with frequency  ${\tt>}0.05$   are shown  

To determinate the difference of genetic background in different ethnic groups, we took a comparison  of four ethnic (CEU, CHB, JPT and YRI) data of this  region from HapMap Project (www.hapmap.org) and  identified notable ethnic difference in allelic distributions  and LD structure (table 4). Not surprisingly, YRI had  much more alleles and blocks in this region and most of  them were less common or rare than CEU. Thus, more  tag SNPs will be required for capturing common variation in African-derived samples in a case-control study.  This can be attributed to higher genetic diversity and  lower extent of LD in the African populations . Interestingly, the genetic background was remarkably similar  

Table 4 Genetic diversity in different ethnic populations on KCNK17  
![](images/8f3394157f5f09d8d5ac4ab9c70bf4cd7a85340e7097154240630719de6efb27.jpg)  

a SNPs spanning 18 kb region (NCBI 36.1 from 39, 374, 755 to 39, 392, 214) of KCNK17 also with   $5^{\prime}$  flanking sequence   $(2{\mathrm{\kb}})$   were     $^{\mathrm{b}}\mathrm{MAF}$   c chosen for this study;  : minor allele frequency; Tag SNPs picked out for different populations using the algo d rithm-tagger-pairwise tagging from HapMap; Haplotype blocks were inferred under the algorithm of Gabriel  et al  in Haploview 4.1.  CEU, U.S. residents with northern and western European Ancestry; CHB, Han Chinese in Beijing; JPT, Japanese in Tokyo; YRI,  Yoruba people of Ibadan, Nigeria  

We performed in silico analysis of the KCNK17  promoter to determine the putative effects of polymorphisms -95, -134, -511, -569 -846, -1539 and -1556  (SNP3-SNP9)   on transcription factor binding in table 5.  One transcription factor, ETF (GT GC GG G GT GC,  T00270) was predicted to bind the G del but not the G  ins at SNP -95. Four transcription factors, c-Ets-2 

 (AAGGAGGAA, T00113), STAT4 (GGAAGC, T01577 

 ), c-Ets-1 (GAGGAAG, T00112), Elk-1 (AGGAGGAAG, T00250), were all predicted to bind to -134A but  not the -134G. Minor allele of SNP -596 and SNP -846  have created three predicted binding sites, NF-kappaB  (GG GG A AGT CC TG, T00590), RelA (TGGGGAAGTC  C, T00594), YY1 (ATGG, T00915); VDR (GTTCATGGA, T00885), FOXP3 (GTTCAT, T04280), PXR1  (GACGTTCA, T05671), respectively. No transcription  factors were predicted to bind either nucleotide at -511,  -1539 and -1556.  

Table 5 The predicted regulatory and transcription factor binding sites by Mat inspector software (version 2.2)   
![](images/98b876b7de20b8d765b0412dac2c9ecd37f6f3ebaae42b30b5e9f78c7c25b853.jpg)  
TM Different transcription factors combined with the promoter region according to various allele which is predicted from MATCH , a  tool interconnected and distributed together with the TRANSFAC database  a  $\mathrm{^bN o}$  Gene start site;  additional transcription factor binding site prediction referring to major or minor allele  

# 3 DISCUSSION  

Over the past 20 years, genetic association studies—mainly of the candidate gene variety carried out in  small single samples using a few SNPs—have been disappointing, yielding largely inconsistent and nonreplicable findings . Studies of candidate genes or studies  that used classical linkage approaches have yielded inconsistent findings . The genetic story-of-the-year for  2007 is undoubtedly the dramatic and surprising success  of large-scale genomewide association (GWA) studies in  finding consistent and replicable genetic markers of several complex diseases of adulthood. Of important note,  common variants within KCNK17 have recently been  proved to confer genetic susceptibility for increasing  ischemic stroke from the first European origin ischemic  stroke genome-wide study . By genomic sequence  analysis, Girard  et al  determined that the KCNK16,  KCNK17, and KCNK5 genes form a tight cluster of 25  kb on chromosome 6p21.2–p21.1 . KCNK17 is a  member of the 2-pore domain superfamily of background   $\mathbf{K}^{+}$   channels, which are open at all membrane potentials  and contribute to cellular resting membrane potential,  and its gating is variously regulated by free fatty acids,  membrane tension, G protein-generated signaling, and  extracellular   $\mathrm{pH}$  . Highest expression was in the liver,  lung, placenta, pancreas, small intestine, and aorta .  Given that the functional importance of the KCNK 17  gene potential involving in the path o physiology of  ischemic stoke, we performed the re-sequencing analysis  to identify common variants in Chinese Han population.  Nine polymorphic sites were identified in the human  KCNK17 gene. Haplotypes, their frequencies and linkage d is equilibrium coefficients (D’), were estimated.  This information about genetic polymorphisms in  KCNK17 might be useful for host genetic studies of  other diseases, especially cerebro vascular diseases.  

# REFERENCES  

1   Sarti C, Rastenyte D, Cepaitis Z,  et  al .  International  trends in mortality from stroke, 1968 to 1994. Stroke,  2000,31(7):1588-1601  

2   Wu Z, Yao C, Zhao D,  et al . Sino-MONICA project: a  collaborative study on trends and determinants in cardiovascular diseases in China. Part i: morbidity and mortality  monitoring. Circulation, 2001,103(3):462-468  3   Bonow RO, Smaha LA, Smith SC Jr,  et al . World Heart  Day 2002: the international burden of cardiovascular disease: responding to the emerging global epidemic. Circulation, 2002,106(13):1602-1605  4   Hassan A, Markus HS. Genetics and ischaemic stroke.  Brain, 2000,123(Pt 9):1784-1812  5   Humphries SE, Morgan L. Genetic risk factors for stroke  and carotid atherosclerosis: insights into path o physiology  from candidate gene approaches. Lancet Neurol, 2004,3  (4):227-235  6   Kiely DK, Wolf PA, Cupples LA , et al . Familial aggrega- tion of stroke. The Framingham Study. Stroke, 1993,24  (9):1366-1371  7   Wellcome Trust Case Control Consortium. Genome-wide  association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature, 2007,447  (7145):661-678  8   Matarin M, Brown WM, Scholz S , et al . A genome-wide  genotyping study in patients with ischaemic stroke: initial  analysis and data release. Lancet Neurol, 2007,6  (5):414-420  9   Morohashi Y, Hatano N, Ohya S , et al . Molecular cloning  and characterization of CALP/KChIP4, a novel EF-hand  protein interacting with presenilin 2 and voltage-gated potassium channel subunit Kv4. J Biol Chem, 2002,277  (17):14965-14975.  

10   Barrett JC, Fry B, Maller J , et al . Haploview: analysis and  visualization of LD and haplotype maps. Bioinformatics,  2005,21(2):263-265  11   Schaid D J, Rowland CM, Tines DE , et al . Score tests for  association between traits and haplotypes when linkage  phase is ambiguous. Am J Hum Genet, 2002,70(2):425-434  12 Wingender E, Chen X, Hehl R, et al. TRANSFAC: an integrated system for gene expression regulation. Nucleic  Acids Res, 2000,28(1):316-319  13   Gonzalez JR, Armengol L, Sole X , et al . SNPassoc: an R  package to perform whole genome association studies.  Bioinformatics, 2007,23(5):644-645  14   Reich DE, Carill M, Bolk S , et al . Linkage d is equilibrium in  the human genome. Nature, 2001,411(6834):199-204  15   Dichgans M, Markus HS. Genetic association studies in  stroke: methodological issues and proposed standard criteria. Stroke, 2005,36(9):2027-2031  16   Hegele RA. SNP judgments and freedom of association.  Arte rios c ler Thromb Vasc Biol, 2002,22(7):1058-1061  17   Dichgans M. Genetics of ischaemic stroke. Lancet Neurol,  2007,6(2):149-161  18   Girard, C, Duprat F, Terrenoire C , et al . Genomic and  functional characteristics of novel human pancreatic 2P  domain   $\mathrm{K(+)}$   channels. Biochem Biophys Res Commun,  2001,282(1):249-256  19   Decher N, Maier M, Dittrich W , et al . Characterization of  TASK-4, a novel member of the pH-sensitive, two-pore  domain potassium channel family. FEBS Lett, 2001,492  (1-2):84-89  

(Received April 29, 2009)  